<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta http-equiv="Content-Language" content="en-us">
  <meta name="robots" content="all">
  
  
  
  
    <script type="text/javascript" async="" src="http://www.google-analytics.com/ga.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.5.1/jquery.min.js"></script>
    <script type="text/javascript">
        if (!window.location.href.match(/bioconductor\.org|\/\/localhost/i)) {
            var mirror = true;
            document.write('<base href="http://www.bioconductor.org"/>\n');
        } 
    </script>
  
  
  <script src="/js/jquery.tools.min.js"></script>

  
  
  <title>Bioconductor - BiocViews</title>
  <link rel="stylesheet" type="text/css" href="/style/bioconductor.css" media="screen">
  <link rel="SHORTCUT ICON" type="image/x-icon" href="/favicon.ico">
  <link rel="ICON" type="image/x-icon" href="/favicon.ico">
  <script type="text/javascript" src="/js/bioconductor.js"></script>
 
  <script type="text/javascript" src="/js/jquery.corner.js"></script>
  <script type="text/javascript" src="/js/jquery.timeago.js"></script>
  <script type="text/javascript" src="/js/bioc-style.js"></script>
  <script type="text/javascript" src="/js/versions.js"></script>

  <!-- Google Analytics script -->
  <script type="text/javascript">
    var _gaq = _gaq || [];
    _gaq.push(['_setAccount', 'UA-9338486-1']);
    _gaq.push(['_trackPageview']);

    (function() {
      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
    })();
  </script>
<style type="text/css" id="vakata-stylesheet">#vakata-dragged { display:block; margin:0 0 0 0; padding:4px 4px 4px 24px; position:absolute; top:-2000px; line-height:16px; z-index:10000; } #vakata-contextmenu { display:block; visibility:hidden; left:0; top:-200px; position:absolute; margin:0; padding:0; min-width:180px; background:#ebebeb; border:1px solid silver; z-index:10000; *width:180px; } #vakata-contextmenu ul { min-width:180px; *width:180px; } #vakata-contextmenu ul, #vakata-contextmenu li { margin:0; padding:0; list-style-type:none; display:block; } #vakata-contextmenu li { line-height:20px; min-height:20px; position:relative; padding:0px; } #vakata-contextmenu li a { padding:1px 6px; line-height:17px; display:block; text-decoration:none; margin:1px 1px 0 1px; } #vakata-contextmenu li ins { float:left; width:16px; height:16px; text-decoration:none; margin-right:2px; } #vakata-contextmenu li a:hover, #vakata-contextmenu li.vakata-hover > a { background:gray; color:white; } #vakata-contextmenu li ul { display:none; position:absolute; top:-2px; left:100%; background:#ebebeb; border:1px solid gray; } #vakata-contextmenu .right { right:100%; left:auto; } #vakata-contextmenu .bottom { bottom:-1px; top:auto; } #vakata-contextmenu li.vakata-separator { min-height:0; height:1px; line-height:1px; font-size:1px; overflow:hidden; margin:0 2px; background:silver; /* border-top:1px solid #fefefe; */ padding:0; } </style><style type="text/css" id="jstree-stylesheet">.jstree ul, .jstree li { display:block; margin:0 0 0 0; padding:0 0 0 0; list-style-type:none; } .jstree li { display:block; min-height:18px; line-height:18px; white-space:nowrap; margin-left:18px; min-width:18px; } .jstree-rtl li { margin-left:0; margin-right:18px; } .jstree > ul > li { margin-left:0px; } .jstree-rtl > ul > li { margin-right:0px; } .jstree ins { display:inline-block; text-decoration:none; width:18px; height:18px; margin:0 0 0 0; padding:0; } .jstree a { display:inline-block; line-height:16px; height:16px; color:black; white-space:nowrap; text-decoration:none; padding:1px 2px; margin:0; } .jstree a:focus { outline: none; } .jstree a > ins { height:16px; width:16px; } .jstree a > .jstree-icon { margin-right:3px; } .jstree-rtl a > .jstree-icon { margin-left:3px; margin-right:0; } li.jstree-open > ul { display:block; } li.jstree-closed > ul { display:none; } #vakata-dragged ins { display:block; text-decoration:none; width:16px; height:16px; margin:0 0 0 0; padding:0; position:absolute; top:4px; left:4px;  -moz-border-radius:4px; border-radius:4px; -webkit-border-radius:4px; } #vakata-dragged .jstree-ok { background:green; } #vakata-dragged .jstree-invalid { background:red; } #jstree-marker { padding:0; margin:0; font-size:12px; overflow:hidden; height:12px; width:8px; position:absolute; top:-30px; z-index:10001; background-repeat:no-repeat; display:none; background-color:transparent; text-shadow:1px 1px 1px white; color:black; line-height:10px; } #jstree-marker-line { padding:0; margin:0; line-height:0%; font-size:1px; overflow:hidden; height:1px; width:100px; position:absolute; top:-30px; z-index:10000; background-repeat:no-repeat; display:none; background-color:#456c43;  cursor:pointer; border:1px solid #eeeeee; border-left:0; -moz-box-shadow: 0px 0px 2px #666; -webkit-box-shadow: 0px 0px 2px #666; box-shadow: 0px 0px 2px #666;  -moz-border-radius:1px; border-radius:1px; -webkit-border-radius:1px; }.jstree .jstree-real-checkbox { display:none; } .jstree-themeroller .ui-icon { overflow:visible; } .jstree-themeroller a { padding:0 2px; } .jstree-themeroller .jstree-no-icon { display:none; }.jstree .jstree-wholerow-real { position:relative; z-index:1; } .jstree .jstree-wholerow-real li { cursor:pointer; } .jstree .jstree-wholerow-real a { border-left-color:transparent !important; border-right-color:transparent !important; } .jstree .jstree-wholerow { position:relative; z-index:0; height:0; } .jstree .jstree-wholerow ul, .jstree .jstree-wholerow li { width:100%; } .jstree .jstree-wholerow, .jstree .jstree-wholerow ul, .jstree .jstree-wholerow li, .jstree .jstree-wholerow a { margin:0 !important; padding:0 !important; } .jstree .jstree-wholerow, .jstree .jstree-wholerow ul, .jstree .jstree-wholerow li { background:transparent !important; }.jstree .jstree-wholerow ins, .jstree .jstree-wholerow span, .jstree .jstree-wholerow input { display:none !important; }.jstree .jstree-wholerow a, .jstree .jstree-wholerow a:hover { text-indent:-9999px; !important; width:100%; padding:0 !important; border-right-width:0px !important; border-left-width:0px !important; } .jstree .jstree-wholerow-span { position:absolute; left:0; margin:0px; padding:0; height:18px; border-width:0; padding:0; z-index:0; }</style><link rel="stylesheet" type="text/css" media="all" href="http://bioconductor.org/js/tree-widget/themes/apple/style.css"></head>

  <body>
    <a name="top"></a>

<!-- a few hooks for screen readers -->
<a href="#site-navigation" title="Jump to site nav"></a>

<a href="#site-map" title="Jump to site map"></a>


    <div id="SiteContainer" class="SiteContainer">
      
      <div id="PageContent" class="PageContent">
      

        
        
          <div id="PageBreadcrumbs" class="PageBreadcrumbs">
  <ul>
    
    
    
      
	  	
        <li><a href="/">Home</a></li>
      
    
    
    
    
      
        <li>BiocViews</li>
      
    
  </ul>
</div>

        

        
        
        
        
        

        
<script type="text/javascript" src="/packages/json/3.1/tree.js"></script>
<script type="text/javascript" src="/packages/json/3.1/bioc/packages.js"></script>
<script type="text/javascript" src="/packages/json/3.1/data/annotation/packages.js"></script>
<script type="text/javascript" src="/packages/json/3.1/data/experiment/packages.js"></script>


<script type="text/javascript" src="/js/jquery.cookie.js"></script>
<script type="text/javascript" src="/js/jquery.hotkeys.js"></script>
<script type="text/javascript" src="/js/tree-widget/jquery.jstree.js"></script>
<script type="text/javascript" src="/js/jsnetworkx.js"></script>
<script type="text/javascript" src="/js/bioc_views.js"></script>

<!-- DataTables CSS -->
<link rel="stylesheet" type="text/css" href="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.9.4/css/jquery.dataTables.css">
 
<!-- DataTables -->
<script type="text/javascript" charset="utf8" src="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.9.4/jquery.dataTables.min.js"></script>


<link rel="stylesheet" href="//code.jquery.com/ui/1.10.4/themes/smoothness/jquery-ui.css">
<script src="//code.jquery.com/ui/1.10.4/jquery-ui.js"></script>
<!--<link rel="stylesheet" href="/resources/demos/style.css">-->


<h1>All Packages</h1>



<link type="text/css" rel="stylesheet" href="/style/tree-widget/!style.css">


<div id="tree_outer" class="biocViewsTreeContainer">
    <div class="do_not_rebase">
        <div style="float:left;width:33%;">
            <h3>Bioconductor version <span id="bioc_version">3.1 (Release)</span></h3>
            <div id="tree-toggler-div" style="display: none;">
                <p>&nbsp;</p>
                <p>
                    <input id="tree-toggler" type="checkbox">
                    Developers: check this box
                    to toggle the visibility of childless biocViews.
                </p>
            </div>

            <div class="ui-widget">
                <label for="autocompleter">Autocomplete biocViews search:</label>
                <span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><input id="autocompleter" size="35" class="ui-autocomplete-input" autocomplete="off">
                <p>&nbsp;</p>
            </div>

            <div id="tree" class="jstree jstree-0 biocViewsTree jstree-focused jstree-apple"><ul class="jstree-no-dots jstree-no-icons"><li packagelist="a4,a4Base,a4Classif,a4Core,a4Preproc,a4Reporting,ABarray,ABSSeq,aCGH,ACME,ADaCGH2,adSplit,affxparser,affy,affycomp,AffyCompatible,affyContam,affycoretools,AffyExpress,affyILM,affyio,affylmGUI,affyPara,affypdnn,affyPLM,affyQCReport,AffyRNADegradation,AffyTiling,AGDEX,agilp,AgiMicroRna,AIMS,ALDEx2,AllelicImbalance,alsace,altcdfenvs,ampliQueso,AnalysisPageServer,annaffy,annmap,annotate,AnnotationDbi,AnnotationForge,AnnotationFuncs,AnnotationHub,annotationTools,anota,antiProfiles,apComplex,aroma.light,ArrayExpress,ArrayExpressHTS,arrayMvout,arrayQuality,arrayQualityMetrics,ArrayTools,ArrayTV,ARRmNormalization,ASEB,ASGSCA,ASSET,ASSIGN,AtlasRDF,attract,BAC,BADER,BAGS,ballgown,bamsignals,BaseSpaceR,Basic4Cseq,BayesPeak,baySeq,BCRANK,beadarray,beadarraySNP,BeadDataPackR,BEAT,BEclear,betr,bgafun,BGmix,bgx,BHC,BicARE,BiGGR,bigmemoryExtras,bioassayR,Biobase,BiocCaseStudies,BiocCheck,BiocGenerics,biocGraph,BiocInstaller,BiocParallel,BiocStyle,biocViews,bioDist,biomaRt,BioMVCClass,biomvRCNS,BioNet,BioSeqClass,Biostrings,biosvd,biovizBase,BiRewire,birta,birte,BiSeq,BitSeq,blima,BRAIN,BrainStars,bridge,BridgeDbR,BrowserViz,BrowserVizDemo,BSgenome,bsseq,BubbleTree,BufferedMatrix,BufferedMatrixMethods,bumphunter,BUS,CAFE,CAGEr,CALIB,CAMERA,canceR,cancerclass,CancerMutationAnalysis,CAnD,Cardinal,casper,Category,categoryCompare,ccrepe,cellGrowth,cellHTS,cellHTS2,CellNOptR,CexoR,CFAssay,CGEN,CGHbase,CGHcall,cghMCR,CGHnormaliter,CGHregions,ChAMP,charm,ChemmineOB,ChemmineR,chimera,chipenrich,ChIPpeakAnno,ChIPQC,ChIPseeker,chipseq,ChIPseqR,ChIPsim,ChIPXpress,chopsticks,chroGPS,chromDraw,ChromHeatMap,cisPath,ClassifyR,cleanUpdTSeq,cleaver,clippda,clipper,Clomial,Clonality,clonotypeR,clst,clstutils,clusterProfiler,clusterStab,CMA,cn.farms,cn.mops,CNAnorm,CNEr,CNORdt,CNORfeeder,CNORfuzzy,CNORode,CNTools,cnvGSA,CNVrd2,CNVtools,cobindR,CoCiteStats,codelink,CODEX,CoGAPS,cogena,coGPS,COHCAP,coMET,COMPASS,compcodeR,compEpiTools,CompGO,ComplexHeatmap,ConsensusClusterPlus,conumee,convert,copa,copynumber,CopyNumber450k,CopywriteR,CoRegNet,Cormotif,CorMut,coRNAi,CORREP,cosmiq,COSNet,CoverageView,cpvSNP,cqn,CRImage,CRISPRseek,crlmm,CSAR,csaw,CSSP,ctc,cummeRbund,customProDB,cycle,cytofkit,dagLogo,daMA,DART,DASiR,DAVIDQuery,DBChIP,ddCt,ddgraph,DECIPHER,DeconRNASeq,DEDS,deepSNV,DEGraph,DEGreport,DEGseq,deltaGseg,derfinder,derfinderHelper,derfinderPlot,DESeq,DESeq2,DEXSeq,dexus,DFP,DiffBind,diffGeneAnalysis,diffHic,diggit,DirichletMultinomial,dks,DMRcaller,DMRcate,DMRforPairs,DNAcopy,domainsignatures,DOQTL,DOSE,DriverNet,DrugVsDisease,DSS,DTA,dualKS,DupChecker,dyebias,DynDoc,EasyqpcR,easyRNASeq,EBarrays,EBcoexpress,EBImage,EBSeq,EBSeqHMM,ecolitk,EDASeq,EDDA,edge,edgeR,eiR,eisa,ELBOW,EMDomics,ENCODExplorer,ENmix,EnrichmentBrowser,ensembldb,ensemblVEP,ENVISIONQuery,epigenomix,epivizr,erccdashboard,ExiMiR,exomeCopy,exomePeak,explorase,ExpressionView,fabia,facopy,factDesign,farms,fastLiquidAssociation,fastseg,fdrame,FEM,ffpe,FGNet,FISHalyseR,flagme,flipflop,flowBeads,flowBin,flowcatchR,flowCHIC,flowCL,flowClean,flowClust,flowCore,flowCyBar,flowDensity,flowFit,flowFP,flowMap,flowMatch,flowMeans,flowMerge,flowPeaks,flowPlots,flowQ,flowQB,FlowRepositoryR,FlowSOM,flowStats,flowTrans,flowType,flowUtils,flowViz,flowVS,flowWorkspace,fmcsR,focalCall,FourCSeq,FRGEpistasis,frma,frmaTools,FunciSNP,gaga,gage,gaggle,gaia,gaucho,gCMAP,gCMAPWeb,gcrma,gdsfmt,geecc,genArise,GENE.E,GeneAnswers,GeneExpressionSignature,genefilter,genefu,GeneGA,GeneMeta,GeneNetworkBuilder,GeneOverlap,geneplotter,geneRecommender,GeneRegionScan,geneRxCluster,GeneSelectMMD,GeneSelector,GENESIS,geNetClassifier,GeneticsDesign,GeneticsPed,genoCN,genomation,GenomeGraphs,GenomeInfoDb,genomeIntervals,genomes,GenomicAlignments,GenomicFeatures,GenomicFiles,GenomicInteractions,GenomicRanges,GenomicTuples,Genominator,genoset,GenoView,GEOmetadb,GEOquery,GEOsubmission,gespeR,GEWIST,GGBase,ggbio,GGtools,ggtree,girafe,GLAD,GlobalAncova,globaltest,gmapR,GOexpress,GOFunction,GoogleGenomics,goProfiles,GOSemSim,goseq,GOSim,GOstats,GOsummaries,GOTHiC,goTools,gpls,gprege,gQTLBase,gQTLstats,graph,GraphAlignment,GraphAT,graphite,GraphPAC,GRENITS,GreyListChIP,groHMM,GSAR,GSCA,GSEABase,GSEAlm,GSReg,GSRI,GSVA,gtrellis,Gviz,gwascat,GWASTools,h5vc,hapFabia,Harshlight,HCsnip,HDTD,Heatplus,HELP,HEM,hiAnnotator,HIBAG,HilbertVis,HilbertVisGUI,hiReadsProcessor,HiTC,HMMcopy,hopach,hpar,HTqPCR,HTSanalyzeR,HTSeqGenie,htSeqTools,HTSFilter,HybridMTest,hyperdraw,hypergraph,iASeq,iBBiG,ibh,iBMQ,Icens,iChip,iClusterPlus,IdeoViz,idiogram,IdMappingAnalysis,IdMappingRetrieval,illuminaio,imageHTS,immunoClust,IMPCdata,impute,InPAS,INPower,inSilicoDb,inSilicoMerging,intansv,interactiveDisplay,interactiveDisplayBase,inveRsion,iontree,iPAC,IPPD,IRanges,iSeq,isobar,IsoGeneGUI,ITALICS,iterativeBMA,iterativeBMAsurv,IVAS,jmosaics,joda,KCsmart,kebabs,KEGGgraph,keggorthology,KEGGprofile,KEGGREST,lapmix,LBE,LEA,les,limma,limmaGUI,LiquidAssociation,lmdme,LMGene,logicFS,logitT,lol,LowMACA,LPE,LPEadj,lpNet,lumi,LVSmiRNA,M3D,maanova,macat,maCorrPlot,made4,maigesPack,MAIT,makecdfenv,MANOR,manta,MantelCorr,mAPKL,maPredictDSC,marray,maSigPro,maskBAD,MassArray,massiR,MassSpecWavelet,matchBox,MatrixRider,MBAmethyl,MBASED,MBCB,mBPCR,mcaGUI,MCRestimate,mdgsa,mdqc,MeasurementError.cor,MEDIPS,MEDME,MEIGOR,MergeMaid,MeSHDbi,meshr,MeSHSim,messina,metaArray,Metab,metabomxtr,metagene,metagenomeSeq,metahdep,metaMS,metaSeq,metaseqR,MethTargetedNGS,methVisual,methyAnalysis,MethylAid,MethylMix,methylMnM,methylPipe,MethylSeekR,methylumi,Mfuzz,MGFM,mgsa,MiChip,microRNA,MIMOSA,MineICA,minet,minfi,MinimumDistance,MiPP,MiRaGE,miRNApath,miRNAtap,Mirsynergy,missMethyl,mitoODE,MLInterfaces,MLP,MLSeq,MMDiff,mmnet,MmPalateMiRNA,mogsa,monocle,MoPS,mosaics,MotifDb,motifRG,motifStack,MotIV,MPFE,mQTL.NMR,msa,MSGFgui,MSGFplus,msmsEDA,msmsTests,MSnbase,MSnID,MSstats,Mulcom,MultiMed,multiscan,multtest,muscle,MVCClass,mvGST,mygene,mzID,mzR,NanoStringQCPro,NarrowPeaks,ncdfFlow,NCIgraph,neaGUI,nem,netbenchmark,netbiov,nethet,NetPathMiner,netresponse,NetSAM,networkBMA,NGScopy,nnNorm,NOISeq,nondetects,NormqPCR,npGSEA,NTW,nucleR,nudge,NuPoP,occugene,OCplus,oligo,oligoClasses,OLIN,OLINgui,omicade4,OmicCircos,OmicsMarkeR,OncoSimulR,oneChannelGUI,ontoCAT,openCyto,oposSOM,OrderedList,OrganismDbi,OSAT,OTUbase,OutlierD,PAA,PADOG,paircompviz,pandaR,PAnnBuilder,panp,PANR,PAPi,parglms,parody,pathifier,PathNet,pathRender,pathview,paxtoolsr,Pbase,pcaGoPromoter,pcaMethods,pcot2,PCpheno,pdInfoBuilder,pdmclass,PECA,pepStat,pepXMLTab,PGSEA,phenoDist,phenoTest,PhenStat,phyloseq,piano,pickgene,PICS,PING,pint,pkgDepTools,plateCore,plethy,plgem,plier,PLPE,plrs,plw,pmm,podkat,polyester,Polyfit,ppiStats,prada,prebs,PREDA,predictionet,preprocessCore,proBAMr,PROcess,procoil,ProCoNA,pRoloc,pRolocGUI,PROMISE,PROPER,prot2D,proteinProfiles,proteoQC,ProtGenerics,PSEA,PSICQUIC,puma,pvac,pvca,Pviz,PWMEnrich,pwOmics,qcmetrics,QDNAseq,qpcrNorm,qpgraph,qrqc,QUALIFIER,quantro,quantsmooth,QuartPAC,QuasR,qusage,qvalue,R3CPET,r3Cseq,R453Plus1Toolbox,rain,rama,RamiGO,randPack,RankProd,Rariant,RbcBook1,RBGL,RBioinf,rBiopaxParser,RBM,Rbowtie,rbsurv,Rcade,RCASPAR,rcellminer,Rchemcpp,RchyOptimyx,Rcpi,RCyjs,RCytoscape,RDAVIDWebService,Rdisop,RDRToolbox,ReactomePA,ReadqPCR,reb,RedeR,REDseq,RefNet,RefPlus,regioneR,regionReport,Repitools,ReportingTools,ReQON,rfPred,rGADEM,RGalaxy,Rgraphviz,rGREAT,RGSEA,rgsepd,rhdf5,Rhtslib,rHVDM,riboSeqR,Ringo,RIPSeeker,Risa,RLMM,Rmagpie,RMassBank,rMAT,RmiR,RNAinteract,RNAither,RNAprobR,rnaSeqMap,RNASeqPower,RnaSeqSampleSize,RnBeads,Rnits,roar,ROC,Roleswitch,Rolexa,rols,ROntoTools,RPA,RpsiXML,rpx,Rqc,rqubic,rRDP,RRHO,Rsamtools,rsbml,rSFFreader,Rsubread,RSVSim,rTANDEM,RTCA,RTN,RTopper,rtracklayer,Rtreemix,rTRM,rTRMui,RUVcorr,RUVnormalize,RUVSeq,RWebServices,S4Vectors,safe,sagenhaft,SAGx,SamSPECTRAL,sangerseqR,SANTA,sapFinder,saps,savR,SBMLR,SCAN.UPC,ScISI,scsR,segmentSeq,SELEX,SemDist,seq2pathway,SeqArray,seqbias,seqCNA,SeqGSEA,seqLogo,seqPattern,seqplots,seqTools,SeqVarTools,SGSeq,shinyMethyl,shinyTANDEM,ShortRead,sigaR,SigCheck,SigFuge,siggenes,sigPathway,sigsquared,SIM,SIMAT,SimBindProfiles,similaRpeak,simpleaffy,simulatorZ,sincell,sizepower,SJava,skewr,SLGI,SLqPCR,SMAP,SNAGEE,snapCGH,snm,SNPchip,SNPRelate,snpStats,soGGi,SomatiCA,SomaticSignatures,SpacePAC,spade,specL,SpeCond,SPEM,SPIA,spikeLI,spkTools,splicegear,spliceR,spliceSites,SplicingGraphs,splots,spotSegmentation,SQUADD,SRAdb,sRAP,sscore,sSeq,ssize,SSPA,ssviz,STAN,staRank,Starr,STATegRa,stepNorm,stepwiseCM,Streamer,STRINGdb,supraHex,survcomp,Sushi,sva,SVM2CRM,SwimR,switchBox,synapter,systemPipeR,TargetScore,TargetSearch,TCC,TDARACNE,TEQC,ternarynet,TFBSTools,tigre,tilingArray,timecourse,TIN,TitanCNA,tkWidgets,ToPASeq,topGO,TPP,tracktables,trackViewer,tRanslatome,TransView,triform,trigger,trio,triplex,TRONCO,TSCAN,tspair,TSSi,TurboNorm,tweeDEseq,twilight,TypeInfo,UNDO,unifiedWMWqPCR,UniProt.ws,VanillaICE,VariantAnnotation,VariantFiltering,VariantTools,vbmp,Vega,VegaMC,viper,vsn,vtpnet,wateRmelon,wavClusteR,waveTiling,weaver,webbioc,widgetTools,xcms,XDE,xmapbridge,xps,XVector,yaqcaffy,zlibbioc" id="Software" class="jstree-open"><ins class="jstree-icon">&nbsp;</ins><a href="#" class="jstree-clicked"><ins class="jstree-icon">&nbsp;</ins>Software (1024)</a><ul><li packagelist="nondetects" id="AssayDomain" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>AssayDomain (345)</a><ul><li packagelist="biomvRCNS,Clonality,copynumber,gaia,HCsnip,KCsmart,mBPCR,pint,RCASPAR,rcellminer,sigaR,Vega,VegaMC" id="aCGH" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>aCGH (13)</a></li><li packagelist="bioassayR,cellGrowth,cellHTS,cellHTS2,CellNOptR,CFAssay,ChemmineOB,ChemmineR,CNORdt,CNORfeeder,CNORode,coRNAi,eiR,flowBeads,flowBin,flowCHIC,flowCore,flowCyBar,flowFit,flowFP,flowPlots,flowQ,FlowSOM,flowStats,flowUtils,flowViz,flowVS,fmcsR,gespeR,HTSanalyzeR,imageHTS,MIMOSA,mitoODE,PANR,phenoDist,plateCore,prada,QUALIFIER,rcellminer,Rchemcpp,RNAinteract,RNAither,RTCA,staRank" id="CellBasedAssays" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CellBasedAssays (44)</a></li><li packagelist="ChIPpeakAnno,ChIPXpress,iChip,les,motifStack,MotIV,rGADEM,Starr" id="ChIPchip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ChIPchip (8)</a></li><li packagelist="aCGH,ArrayTV,beadarraySNP,biomvRCNS,BubbleTree,CGHbase,cghMCR,CGHregions,Clonality,cn.farms,cn.mops,CNAnorm,CNTools,CNVrd2,CODEX,conumee,copynumber,CopyNumber450k,CopywriteR,crlmm,DNAcopy,exomeCopy,facopy,fastseg,gaia,genoset,GLAD,HMMcopy,ITALICS,KCsmart,MANOR,mBPCR,MEDIPS,MinimumDistance,NGScopy,PREDA,QDNAseq,quantsmooth,rcellminer,reb,regioneR,seqCNA,SMAP,snapCGH,SNPchip,SomatiCA,TitanCNA,VanillaICE,Vega,VegaMC" id="CopyNumberVariation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CopyNumberVariation (50)</a></li><li packagelist="genomation,HELP,M3D,MEDIPS,MEDME,methylumi,RnBeads,TurboNorm" id="CpGIsland" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CpGIsland (8)</a></li><li packagelist="ARRmNormalization,BEAT,BEclear,BiSeq,bsseq,bumphunter,charm,COHCAP,coMET,conumee,CopyNumber450k,DMRcaller,DMRcate,DMRforPairs,DSS,ENmix,HELP,les,lumi,M3D,MassArray,MBAmethyl,MEDIPS,MEDME,methVisual,methyAnalysis,MethylAid,MethylMix,methylMnM,methylPipe,MethylSeekR,methylumi,minfi,missMethyl,MPFE,Repitools,RnBeads,shinyMethyl,skewr,TransView,TurboNorm,wateRmelon" id="DNAMethylation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DNAMethylation (42)</a></li><li packagelist="limma,oligo,puma,siggenes,snm,TIN,xps" id="ExonArray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ExonArray (7)</a></li><li packagelist="affycoretools,AffyRNADegradation,AGDEX,AIMS,ALDEx2,ampliQueso,anota,antiProfiles,ASSIGN,AtlasRDF,attract,BioNet,birta,birte,BitSeq,CAFE,canceR,casper,categoryCompare,clipper,CoGAPS,cogena,compEpiTools,CoRegNet,cummeRbund,DART,ddCt,DEGreport,DEGseq,DESeq2,dexus,diggit,DMRcate,DrugVsDisease,DTA,EasyqpcR,easyRNASeq,EBSeqHMM,edge,edgeR,eisa,ELBOW,EMDomics,EnrichmentBrowser,epigenomix,erccdashboard,ExiMiR,ExpressionView,farms,fastLiquidAssociation,ffpe,gage,geecc,GeneExpressionSignature,genefu,GeneGA,GeneMeta,GGtools,GOexpress,goseq,GOstats,GOsummaries,GRENITS,GSCA,GSEABase,GSReg,HCsnip,HDTD,HTqPCR,HTSFilter,HybridMTest,iBMQ,interactiveDisplay,interactiveDisplayBase,IVAS,limma,LiquidAssociation,manta,mAPKL,massiR,MBASED,messina,metaseqR,MethylMix,MGFM,MiRaGE,mogsa,monocle,Mulcom,netbenchmark,netresponse,networkBMA,nondetects,NormqPCR,oligo,oposSOM,pandaR,pathview,pcaGoPromoter,pcot2,PECA,piano,pint,plgem,Polyfit,prebs,PREDA,PROMISE,puma,qpcrNorm,qpgraph,R3CPET,RBioinf,Rcade,RCASPAR,rcellminer,ReadqPCR,Repitools,RmiR,rnaSeqMap,RnaSeqSampleSize,Rnits,RPA,Rsubread,RTN,RUVcorr,SAGx,saps,SeqGSEA,sigaR,SigCheck,siggenes,sincell,snm,spliceR,spliceSites,SplicingGraphs,sRAP,SSPA,supraHex,survcomp,systemPipeR,tigre,TIN,ToPASeq,tRanslatome,TransView,trigger,TSCAN,unifiedWMWqPCR,viper,waveTiling,xps" id="GeneExpression" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GeneExpression (153)</a></li><li packagelist="beadarraySNP,CAnD,chopsticks,Clomial,CNVtools,deepSNV,DOQTL,FRGEpistasis,GENESIS,GGtools,GSReg,GWASTools,hapFabia,metagenomeSeq,missMethyl,phyloseq,qpgraph,RLMM,Rsubread,RTN,SeqVarTools,SNPchip,snpStats,trigger,trio,VariantTools" id="GeneticVariability" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GeneticVariability (26)</a></li><li packagelist="beadarraySNP,BSgenome,CAnD,CGEN,chopsticks,CNVrd2,copynumber,crlmm,customProDB,deepSNV,DOQTL,ensemblVEP,gaucho,GeneRegionScan,GENESIS,GenomicAlignments,genoset,GGtools,GWASTools,hapFabia,iASeq,iBMQ,INPower,inveRsion,IVAS,logicFS,MassArray,mBPCR,MinimumDistance,mQTL.NMR,oligo,qpgraph,RLMM,Rsubread,RTN,sangerseqR,sapFinder,SeqVarTools,siggenes,SNPchip,snpStats,systemPipeR,trigger,trio,VariantAnnotation,VariantFiltering" id="SNP" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SNP (46)</a></li><li packagelist="altcdfenvs,ampliQueso,annmap,BAC,BicARE,birta,birte,BitSeq,CAGEr,casper,CexoR,CoRegNet,CSAR,customProDB,DBChIP,DTA,edgeR,erccdashboard,ExiMiR,geecc,GOexpress,goseq,GSRI,InPAS,IVAS,jmosaics,les,limma,MBASED,mosaics,motifRG,NarrowPeaks,netresponse,pandaR,pwOmics,qpgraph,Rcade,regionReport,rHVDM,rnaSeqMap,roar,RRHO,rTRM,rTRMui,SELEX,SGSeq,simpleaffy,snm,splicegear,spliceR,SplicingGraphs,tigre,TransView,xps" id="Transcription" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Transcription (54)</a></li></ul></li><li packagelist="geecc,OncoSimulR,similaRpeak" id="BiologicalQuestion" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>BiologicalQuestion (313)</a><ul><li packagelist="BitSeq,customProDB,edgeR,erccdashboard,flipflop,InPAS,IVAS,limma,puma,SGSeq,spliceR,SplicingGraphs,TIN" id="AlternativeSplicing" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>AlternativeSplicing (13)</a></li><li packagelist="bamsignals,chipseq,CNAnorm,CNVrd2,compEpiTools,CopywriteR,csaw,diffHic,DMRcaller,edgeR,ensembldb,GenomicAlignments,GenomicFiles,GenomicRanges,GreyListChIP,InPAS,M3D,MEDIPS,metagene,methylPipe,QuasR,regionReport,RNAprobR,Rsamtools,soGGi,systemPipeR" id="Coverage" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Coverage (26)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DemethylateRegionDetection</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DenovoAssembler</a></li><li packagelist="ABSSeq,AffyExpress,affylmGUI,AgiMicroRna,ALDEx2,ampliQueso,anota,ArrayTools,AtlasRDF,BADER,ballgown,baySeq,betr,BGmix,bgx,BioNet,BitSeq,bridge,casper,clippda,coGPS,compcodeR,copa,Cormotif,cqn,cummeRbund,daMA,DART,ddCt,DeconRNASeq,DEDS,DEGraph,DEGreport,DEGseq,derfinder,derfinderHelper,derfinderPlot,DESeq,DESeq2,DEXSeq,dexus,DFP,diffGeneAnalysis,DMRcate,DSS,DTA,EBarrays,EBSeq,EBSeqHMM,EDASeq,edge,edgeR,EMDomics,EnrichmentBrowser,epigenomix,erccdashboard,fabia,factDesign,fdrame,FEM,flagme,gaga,gage,genefu,GeneSelectMMD,GeneSelector,geNetClassifier,GlobalAncova,globaltest,GOexpress,gprege,GSAR,GSRI,HDTD,HEM,HTqPCR,HTSFilter,IsoGeneGUI,IVAS,lapmix,les,limma,limmaGUI,lmdme,LMGene,logitT,LPE,maanova,macat,maigesPack,manta,mAPKL,maSigPro,MEDIPS,MergeMaid,messina,metaArray,metagenomeSeq,metahdep,metaSeq,metaseqR,MethylMix,miRNApath,MmPalateMiRNA,monocle,Mulcom,multtest,mvGST,neaGUI,netresponse,NOISeq,nudge,OCplus,oligo,oneChannelGUI,oposSOM,OrderedList,PathNet,pathview,pcot2,PECA,phenoTest,piano,pickgene,pint,plgem,PLPE,plw,polyester,Polyfit,PROPER,prot2D,puma,RankProd,RBM,Rcade,RDAVIDWebService,regionReport,rgsepd,rnaSeqMap,RnaSeqSampleSize,Rnits,ROC,RUVSeq,safe,SAGx,saps,segmentSeq,SeqGSEA,sigaR,siggenes,sigPathway,similaRpeak,simpleaffy,snm,SpeCond,spliceR,spliceSites,sRAP,sscore,ssize,survcomp,TCC,timecourse,ToPASeq,tRanslatome,tweeDEseq,twilight,unifiedWMWqPCR,waveTiling,webbioc,XDE,xps" id="DifferentialExpression" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DifferentialExpression (173)</a></li><li packagelist="COHCAP,coMET,DMRcaller,DMRcate,DMRforPairs,FEM,M3D,MethylMix,missMethyl,RnBeads" id="DifferentialMethylation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DifferentialMethylation (10)</a></li><li packagelist="DiffBind,GreyListChIP" id="DifferentialPeakCalling" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DifferentialPeakCalling (2)</a></li><li packagelist="BitSeq,edgeR,erccdashboard,InPAS,IVAS,limma,puma,spliceR,TIN" id="DifferentialSplicing" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DifferentialSplicing (9)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DriverMutation</a></li><li packagelist="diggit,viper" id="FunctionalPrediction" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>FunctionalPrediction (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GeneFusionDetection</a></li><li packagelist="R3CPET" id="GenePrediction" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GenePrediction (1)</a></li><li packagelist="BCRANK,cleanUpdTSeq,CNEr,CoRegNet,CRISPRseek,diggit,GRENITS,GSReg,GSRI,InPAS,IVAS,MatrixRider,MethylMix,MoPS,pandaR,qpgraph,Rsubread,RTN,rTRM,rTRMui,SELEX,TFBSTools,tigre,tRanslatome,triplex,viper" id="GeneRegulation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GeneRegulation (26)</a></li><li packagelist="AtlasRDF,CAFE,Category,chipenrich,clusterProfiler,cogena,CompGO,cpvSNP,domainsignatures,DOSE,edgeR,EnrichmentBrowser,facopy,FGNet,gage,gCMAPWeb,geecc,GOexpress,GOFunction,GOstats,GSEABase,GSReg,GSRI,GSVA,iBBiG,limma,mdgsa,mgsa,missMethyl,mvGST,neaGUI,npGSEA,oposSOM,pathview,paxtoolsr,piano,qusage,ReactomePA,ReportingTools,rGREAT,RGSEA,rgsepd,RTN,safe,SAGx,saps,SeqGSEA,SigCheck,sincell,sRAP,systemPipeR" id="GeneSetEnrichment" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GeneSetEnrichment (51)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GeneSignaling</a></li><li packagelist="gespeR,networkBMA,pwOmics" id="GeneTarget" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GeneTarget (3)</a></li><li packagelist="AnnotationDbi,compEpiTools,DMRcate,GenomeInfoDb,GenomicFeatures,GenomicRanges,GreyListChIP,MEDIPS,microRNA,QDNAseq,rGREAT,RNAprobR,Rsubread,STAN" id="GenomeAnnotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GenomeAnnotation (14)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GenomeAssembly</a></li><li packagelist="coMET,GENESIS" id="GenomeWideAssociation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GenomeWideAssociation (2)</a></li><li packagelist="CNAnorm,cpvSNP,facopy,missMethyl,Rariant,SomaticSignatures,TitanCNA" id="GenomicVariation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GenomicVariation (7)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GermlineMutation</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>HistoneModification</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>IndelDetection</a></li><li packagelist="CNVrd2" id="LinkageDisequilibrium" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>LinkageDisequilibrium (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MetagenomeAssembly</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MicrosatelliteDetection</a></li><li packagelist="MatrixRider,MotifDb,motifStack,MotIV,SELEX,TFBSTools" id="MotifAnnotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MotifAnnotation (6)</a></li><li packagelist="BCRANK,motifRG,rGADEM,SELEX,TFBSTools" id="MotifDiscovery" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MotifDiscovery (5)</a></li><li packagelist="BioNet,CoRegNet,DEGraph,diggit,EnrichmentBrowser,FEM,FGNet,ibh,miRNApath,neaGUI,PathNet,paxtoolsr,RTN,SANTA,ToPASeq,viper" id="NetworkEnrichment" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>NetworkEnrichment (16)</a></li><li packagelist="apComplex,birte,CNORfeeder,CoRegNet,FRGEpistasis,GRENITS,joda,lpNet,minet,nem,netbenchmark,networkBMA,pandaR,PANR,ppiStats,predictionet,qpgraph,R3CPET,rHVDM,RTN,ScISI,SPEM,tigre" id="NetworkInference" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>NetworkInference (23)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>NucleosomePositioning</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>PeakDetection</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>QuantitativeTrailLocus</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Scaffolding</a></li><li packagelist="Biostrings,BSgenome,cleanUpdTSeq,cobindR,CRISPRseek,dagLogo,FunciSNP,hapFabia,LowMACA,MEDIPS,MethTargetedNGS,microRNA,MmPalateMiRNA,motifStack,muscle,PWMEnrich,REDseq,RRHO,Rsubread,seqLogo,seqPattern,triplex" id="SequenceMatching" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SequenceMatching (22)</a></li><li packagelist="gaucho,LowMACA,OncoSimulR,QuartPAC,Rariant,SomaticSignatures" id="SomaticMutation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SomaticMutation (6)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SplicedAlignment</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>StructuralVariation</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>TranscriptomeVariant</a></li><li packagelist="ensemblVEP,podkat,VariantAnnotation" id="VariantAnnotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>VariantAnnotation (3)</a></li><li packagelist="Rariant" id="VariantDetection" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>VariantDetection (1)</a></li></ul></li><li packagelist="affxparser,AffyCompatible,affyContam,affyio,AllelicImbalance,AnnotationForge,aroma.light,arrayMvout,BaseSpaceR,bigmemoryExtras,bioassayR,Biobase,BiocCaseStudies,BiocCheck,BiocGenerics,BiocInstaller,BiocParallel,biocViews,BioMVCClass,Biostrings,biovizBase,BSgenome,BufferedMatrix,BufferedMatrixMethods,bumphunter,Cardinal,CGHbase,ChemmineOB,ChemmineR,chimera,ChIPseqR,ChIPsim,convert,cummeRbund,DynDoc,eiR,ENCODExplorer,epivizr,flowBeads,flowcatchR,flowCore,flowQ,FlowRepositoryR,flowUtils,flowViz,fmcsR,FunciSNP,gdsfmt,GeneAnswers,genomeIntervals,GenomicAlignments,GenomicFeatures,GenomicFiles,GenomicInteractions,GenomicRanges,GenomicTuples,Genominator,genoset,GEOmetadb,GGBase,ggbio,Icens,illuminaio,IRanges,MeSHDbi,microRNA,monocle,MSnbase,MVCClass,mzR,oligoClasses,OrganismDbi,panp,Pbase,pdInfoBuilder,pkgDepTools,plateCore,preprocessCore,ProtGenerics,QUALIFIER,R453Plus1Toolbox,Rchemcpp,RedeR,RGalaxy,rhdf5,RpsiXML,rRDP,RTCA,RWebServices,S4Vectors,SeqArray,SJava,SNPRelate,splicegear,SRAdb,Streamer,Sushi,systemPipeR,tkWidgets,TypeInfo,UniProt.ws,weaver,webbioc,widgetTools,XVector,zlibbioc" id="Infrastructure" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Infrastructure (211)</a><ul><li packagelist="aCGH,affxparser,affyio,affylmGUI,ArrayExpress,AtlasRDF,bamsignals,BaseSpaceR,beadarraySNP,bioassayR,BioNet,Biostrings,BrainStars,cellGrowth,ChemmineOB,ChemmineR,chipseq,CNEr,codelink,cogena,ctc,cummeRbund,DECIPHER,deepSNV,edge,eiR,ENCODExplorer,ENmix,flowWorkspace,fmcsR,FunciSNP,gaggle,gdsfmt,GeneRegionScan,genomeIntervals,GenomicAlignments,GenomicFiles,GenomicInteractions,GEOquery,ggtree,GoogleGenomics,HELP,HTqPCR,ibh,illuminaio,inSilicoDb,limma,limmaGUI,LowMACA,makecdfenv,MANOR,manta,MassArray,MethTargetedNGS,minfi,monocle,motifStack,MSnbase,muscle,mzID,mzR,oligo,oneChannelGUI,openCyto,OTUbase,Pbase,plethy,podkat,PSICQUIC,puma,qrqc,R453Plus1Toolbox,Rchemcpp,Rcpi,ReadqPCR,rhdf5,Rhtslib,Ringo,Risa,RnBeads,Rolexa,rpx,Rqc,Rsamtools,rSFFreader,rtracklayer,SAGx,segmentSeq,ShortRead,simpleaffy,SRAdb,Starr,Streamer,systemPipeR,TransView,VariantAnnotation,xps" id="DataImport" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DataImport (97)</a></li><li packagelist="AnalysisPageServer,AtlasRDF,BaseSpaceR,bigmemoryExtras,Biostrings,BSgenome,cogena,cummeRbund,flowPlots,flowWorkspace,FunciSNP,GeneAnswers,GenomeInfoDb,GenomicInteractions,GenomicTuples,genoset,GOexpress,interactiveDisplay,interactiveDisplayBase,IRanges,monocle,ontoCAT,openCyto,oposSOM,OSAT,Pbase,plethy,R453Plus1Toolbox,rBiopaxParser,Rcpi,RedeR,ReportingTools,S4Vectors,SplicingGraphs,Sushi,XVector" id="DataRepresentation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DataRepresentation (36)</a></li><li packagelist="affylmGUI,AnalysisPageServer,canceR,cytofkit,epivizr,explorase,flowcatchR,gCMAPWeb,GSCA,interactiveDisplay,interactiveDisplayBase,IsoGeneGUI,limmaGUI,mcaGUI,MethylAid,MSGFgui,neaGUI,oneChannelGUI,pRolocGUI,rTRMui,spade,synapter,TSCAN" id="GUI" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GUI (23)</a></li><li packagelist="BrowserViz,BrowserVizDemo,gaggle,GENE.E,GoogleGenomics,graphite,KEGGREST,maigesPack,PSICQUIC,RamiGO,RCyjs,RCytoscape,rpx" id="ThirdPartyClient" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ThirdPartyClient (13)</a></li></ul></li><li packagelist="MethTargetedNGS" id="ResearchField" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ResearchField (225)</a><ul><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Agroinformatics</a></li><li packagelist="ChemmineOB,ChemmineR,cytofkit,eiR,fmcsR,messina,Rcpi,saps,sincell" id="BiomedicalInformatics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>BiomedicalInformatics (9)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Biophysics</a></li><li packagelist="CellNOptR,CFAssay,cn.mops,CNORdt,CNORfeeder,CNORode,cobindR,CRImage,cytofkit,dexus,fastLiquidAssociation,FISHalyseR,flowcatchR,flowDensity,flowMeans,FlowSOM,hpar,LiquidAssociation,netresponse,SamSPECTRAL,sincell,staRank,ternarynet,tRanslatome" id="CellBiology" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CellBiology (24)</a></li><li packagelist="ChemmineOB,ChemmineR,eiR,fmcsR,mQTL.NMR,rcellminer,Rcpi" id="Cheminformatics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Cheminformatics (7)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ComparativeGenomics</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ComputationalChemistry</a></li><li packagelist="coMET,FGNet,sincell" id="FunctionalGenomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>FunctionalGenomics (3)</a></li><li packagelist="aCGH,AGDEX,AllelicImbalance,BaseSpaceR,BEAT,biomvRCNS,Biostrings,BiSeq,BSgenome,CancerMutationAnalysis,CAnD,CexoR,cleanUpdTSeq,Clomial,cn.mops,coMET,copynumber,cpvSNP,CSAR,csaw,DBChIP,DECIPHER,deepSNV,DMRcate,DOQTL,easyRNASeq,edgeR,ensembldb,erccdashboard,exomeCopy,facopy,fastLiquidAssociation,FRGEpistasis,gage,gaucho,geneRxCluster,GENESIS,GeneticsDesign,GeneticsPed,genoCN,GenomeInfoDb,genomeIntervals,genomes,GenomicAlignments,GenomicFeatures,GenomicFiles,GenomicRanges,GEWIST,GGBase,GGtools,gwascat,hapFabia,HDTD,HIBAG,HybridMTest,intansv,interactiveDisplay,interactiveDisplayBase,jmosaics,limma,LiquidAssociation,logicFS,manta,MassArray,MatrixRider,MEDIPS,metagene,MethTargetedNGS,MGFM,MLP,mosaics,mQTL.NMR,muscle,NarrowPeaks,netresponse,nucleR,NuPoP,pathview,pint,podkat,PREDA,QDNAseq,qpgraph,QuasR,rain,Rcade,RCASPAR,regioneR,riboSeqR,rRDP,RRHO,Rsubread,SeqArray,seqCNA,SeqVarTools,similaRpeak,SLGI,SNPRelate,SplicingGraphs,ssviz,Sushi,systemPipeR,TEQC,TIN,TitanCNA,trigger,trio,TSSi,VariantAnnotation,VariantFiltering,VariantTools" id="Genetics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Genetics (111)</a></li><li packagelist="Cardinal" id="Lipidomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Lipidomics (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MathematicalBiology</a></li><li packagelist="BiGGR,CAMERA,cosmiq,iontree,MAIT,Metab,metabomxtr,metaMS,mQTL.NMR,mzR,OmicsMarkeR,PAPi,pathview,Rdisop,RMassBank,SIMAT,xcms" id="Metabolomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Metabolomics (17)</a></li><li packagelist="ccrepe,metagenomeSeq,phyloseq" id="Metagenomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Metagenomics (3)</a></li><li packagelist="ChemmineOB,ChemmineR,eiR,fmcsR,rcellminer" id="Pharmacogenetics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Pharmacogenetics (5)</a></li><li packagelist="ChemmineOB,ChemmineR,eiR,fmcsR,rcellminer" id="Pharmacogenomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Pharmacogenomics (5)</a></li><li packagelist="ASEB,bioassayR,BRAIN,Cardinal,CellNOptR,ChemmineOB,ChemmineR,cisPath,cleaver,clippda,CNORdt,CNORfeeder,CNORode,customProDB,deltaGseg,eiR,fmcsR,GraphPAC,hpar,iPAC,IPPD,isobar,LPEadj,MassSpecWavelet,MSGFgui,MSGFplus,msmsEDA,msmsTests,MSnbase,MSnID,MSstats,mzID,mzR,PAA,PAnnBuilder,pathview,Pbase,PCpheno,PECA,pepXMLTab,plgem,PLPE,ppiStats,proBAMr,PROcess,procoil,ProCoNA,pRoloc,pRolocGUI,prot2D,proteoQC,ProtGenerics,Pviz,qcmetrics,QuartPAC,rain,RCASPAR,Rchemcpp,Rcpi,RpsiXML,rpx,rTANDEM,sapFinder,ScISI,shinyTANDEM,SLGI,SpacePAC,specL,spliceSites,synapter,TPP" id="Proteomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Proteomics (71)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>StructuralGenomics</a></li><li packagelist="cogena,CoRegNet,diggit,FEM,gage,MEIGOR,mmnet,nem,pathview,paxtoolsr,pmm,pwOmics,rain,rcellminer,Rcpi,sincell,viper" id="SystemsBiology" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SystemsBiology (17)</a></li><li class="jstree-last jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Transcriptomics</a></li></ul></li><li packagelist="ArrayTools,attract,ballgown,CAnD,cpvSNP,EBSeq,EBSeqHMM,fabia,geecc,GeneSelector,GENESIS,GSAR,HDTD,HIBAG,joda,LEA,lol,MeasurementError.cor,MethylMix,mogsa,Mulcom,MultiMed,npGSEA,oneChannelGUI,pandaR,PING,randPack,Rariant,RGSEA,Rtreemix,RUVnormalize,safe,SNPRelate,SomaticSignatures,SSPA,STATegRa,sva,switchBox,TitanCNA,tweeDEseq" id="StatisticalMethod" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>StatisticalMethod (293)</a><ul><li packagelist="ALDEx2,ASSIGN,BAGS,birte,BitSeq,Clomial,CSSP,EBcoexpress,EBSeqHMM,edgeR,GRENITS,limma,networkBMA,pcaMethods,puma,R3CPET,tigre" id="Bayesian" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Bayesian (17)</a></li><li packagelist="AIMS,antiProfiles,bgafun,bioDist,BioSeqClass,cancerclass,Cardinal,ClassifyR,Clonality,clst,clstutils,CMA,COSNet,CRImage,ctc,dexus,DirichletMultinomial,dualKS,eisa,epigenomix,ExpressionView,fastseg,flowcatchR,gaga,genefu,geNetClassifier,gpls,interactiveDisplay,interactiveDisplayBase,iterativeBMA,kebabs,logicFS,made4,maigesPack,maPredictDSC,massiR,MCRestimate,messina,metagenomeSeq,methVisual,MiPP,miRNAtap,MLInterfaces,MLSeq,MoPS,NetPathMiner,NuPoP,omicade4,OmicsMarkeR,ontoCAT,PAA,pdmclass,phenoTest,phyloseq,procoil,pRoloc,RamiGO,RBioinf,rfPred,RGSEA,rHVDM,Rmagpie,rRDP,SemDist,SigCheck,stepwiseCM,switchBox,vbmp" id="Classification" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Classification (68)</a></li><li packagelist="adSplit,BHC,BicARE,bioDist,CAGEr,Cardinal,CGEN,ChemmineOB,ChemmineR,chroGPS,Clomial,clusterProfiler,clusterStab,cogena,ConsensusClusterPlus,CORREP,ctc,cummeRbund,cytofkit,DECIPHER,deltaGseg,diffHic,DirichletMultinomial,DrugVsDisease,EBarrays,edgeR,eiR,fabia,FGNet,flowCHIC,flowClust,flowCyBar,flowDensity,flowFP,flowMatch,flowMeans,flowMerge,flowPeaks,FlowSOM,fmcsR,genefu,geneRecommender,geneRxCluster,ggtree,GOexpress,GOSemSim,GOSim,GOsummaries,GraphPAC,hapFabia,HCsnip,hopach,iBBiG,iClusterPlus,immunoClust,iPAC,kebabs,LEA,limma,maanova,made4,maigesPack,MantelCorr,massiR,mcaGUI,MeSHSim,metagenomeSeq,methVisual,Mfuzz,Mirsynergy,MLInterfaces,MLSeq,MmPalateMiRNA,monocle,nethet,NetPathMiner,netresponse,omicade4,PANR,phenoTest,phyloseq,PICS,PING,pRoloc,puma,QuartPAC,RBioinf,Rchemcpp,RDRToolbox,rqubic,SAGx,SamSPECTRAL,SANTA,SigFuge,sincell,SomaticSignatures,SpacePAC,spade,SpeCond,STATegRa,supraHex,TransView" id="Clustering" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Clustering (102)</a></li><li packagelist="CMA,DEGraph,flowUtils,ScISI,TargetSearch" id="DecisionTree" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DecisionTree (5)</a></li><li packagelist="cytofkit,GENESIS,RDRToolbox,STATegRa" id="DimensionReduction" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DimensionReduction (4)</a></li><li packagelist="mAPKL,OmicsMarkeR,Rcpi,RDRToolbox" id="FeatureExtraction" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>FeatureExtraction (4)</a></li><li packagelist="apComplex,BiGGR,biocGraph,BioMVCClass,BioNet,birta,clipper,CoRegNet,CORREP,COSNet,DART,ddgraph,DEGraph,EnrichmentBrowser,gaggle,GeneNetworkBuilder,GOstats,graph,GraphAlignment,GraphAT,graphite,GRENITS,GSEABase,hyperdraw,hypergraph,ibh,joda,KEGGgraph,keggorthology,maigesPack,minet,MVCClass,mvGST,NCIgraph,netbenchmark,netbiov,nethet,NetPathMiner,netresponse,paircompviz,pandaR,PANR,pathRender,pathview,paxtoolsr,PCpheno,pkgDepTools,ppiStats,predictionet,ProCoNA,PSICQUIC,qpgraph,R3CPET,RamiGO,RBGL,Rcpi,RCyjs,RCytoscape,RDAVIDWebService,RedeR,RefNet,Rgraphviz,rHVDM,rsbml,RTN,rTRM,rTRMui,SBMLR,ScISI,sincell,SLGI,spade,SPIA,SQUADD,ternarynet,ToPASeq" id="GraphAndNetwork" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GraphAndNetwork (76)</a></li><li packagelist="DOQTL,EBSeqHMM,STAN,TitanCNA" id="HiddenMarkovModel" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>HiddenMarkovModel (4)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MultidimensionalScaling</a></li><li packagelist="COSNet" id="NeuralNetwork" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>NeuralNetwork (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>PatternLogic</a></li><li packagelist="GENESIS,mogsa,SNPRelate,STATegRa" id="PrincipalComponent" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>PrincipalComponent (4)</a></li><li packagelist="birte,CFAssay,edge,edgeR,flipflop,gespeR,gpls,IVAS,kebabs,LEA,limma,MoPS,plrs,pmm,SAGx" id="Regression" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Regression (15)</a></li><li packagelist="ASGSCA" id="StructuralEquationModels" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>StructuralEquationModels (1)</a></li><li packagelist="kebabs,SVM2CRM" id="SupportVectorMachine" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SupportVectorMachine (2)</a></li><li packagelist="CFAssay,ClassifyR,messina,saps,simulatorZ" id="Survival" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Survival (5)</a></li><li packagelist="betr,biosvd,cellGrowth,CellNOptR,CNORdt,CNORode,cycle,deltaGseg,DMRcate,EBSeqHMM,edge,edgeR,GOexpress,gprege,GRENITS,limma,LiquidAssociation,maSigPro,Mfuzz,mitoODE,MoPS,plethy,rain,RmiR,Rnits,RTCA,TDARACNE,tigre,timecourse,waveTiling" id="TimeCourse" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>TimeCourse (30)</a></li></ul></li><li packagelist="ACME,CoverageView,ELBOW,nondetects,spkTools" id="Technology" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Technology (645)</a><ul><li packagelist="COMPASS,flowBeads,flowBin,flowCHIC,flowCL,flowClean,flowClust,flowCore,flowCyBar,flowDensity,flowFit,flowFP,flowMap,flowMatch,flowMeans,flowMerge,flowPeaks,flowPlots,flowQ,flowQB,FlowRepositoryR,FlowSOM,flowStats,flowTrans,flowType,flowUtils,flowViz,flowVS,flowWorkspace,immunoClust,MIMOSA,ncdfFlow,openCyto,plateCore,QUALIFIER,RchyOptimyx,SamSPECTRAL,spade" id="FlowCytometry" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>FlowCytometry (38)</a></li><li packagelist="apComplex,BRAIN,CAMERA,Cardinal,cosmiq,customProDB,cytofkit,flagme,gaga,iontree,isobar,MAIT,MassArray,MassSpecWavelet,Metab,metabomxtr,metaMS,MSGFgui,MSGFplus,msmsEDA,msmsTests,MSnbase,MSnID,MSstats,mzID,mzR,PAPi,Pbase,pepXMLTab,plgem,proBAMr,PROcess,pRoloc,proteoQC,ProtGenerics,qcmetrics,Rdisop,Risa,RMassBank,rols,rpx,rTANDEM,sapFinder,shinyTANDEM,SIMAT,specL,synapter,TargetSearch,TPP,xcms" id="MassSpectrometry" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MassSpectrometry (50)</a><ul><li packagelist="Cardinal" id="ImagingMassSpectrometry" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ImagingMassSpectrometry (1)</a></li></ul></li><li packagelist="a4,a4Base,a4Classif,a4Core,a4Preproc,a4Reporting,ABarray,ACME,ADaCGH2,adSplit,affy,affycomp,AffyCompatible,affycoretools,AffyExpress,affyILM,affyio,affylmGUI,affyPara,affypdnn,affyPLM,affyQCReport,AffyRNADegradation,AffyTiling,AGDEX,AgiMicroRna,AIMS,altcdfenvs,annaffy,annmap,AnnotationDbi,annotationTools,anota,aroma.light,ArrayExpress,arrayMvout,arrayQuality,arrayQualityMetrics,ArrayTools,ARRmNormalization,AtlasRDF,BAC,beadarray,BeadDataPackR,betr,BGmix,bgx,BHC,BicARE,biomvRCNS,BioNet,birta,birte,blima,BrainStars,bridge,CAFE,CALIB,cancerclass,CGHbase,CGHcall,cghMCR,CGHnormaliter,CGHregions,ChAMP,charm,chopsticks,Clonality,cn.farms,CNTools,codelink,CoGAPS,cogena,coGPS,COHCAP,coMET,conumee,convert,CopyNumber450k,Cormotif,CORREP,crlmm,ctc,cycle,daMA,DECIPHER,DEDS,DEGraph,DFP,diffGeneAnalysis,DMRcate,DMRforPairs,DNAcopy,DrugVsDisease,DTA,dualKS,dyebias,eisa,ELBOW,EMDomics,ENmix,EnrichmentBrowser,erccdashboard,ExiMiR,explorase,ExpressionView,fabia,facopy,factDesign,farms,fdrame,ffpe,flowVS,focalCall,frma,frmaTools,gage,gCMAP,gcrma,geecc,genArise,genefilter,GeneMeta,GeneNetworkBuilder,geneRecommender,GeneRegionScan,geNetClassifier,genoCN,GenomeGraphs,genoset,GEOquery,GEOsubmission,GLAD,GlobalAncova,globaltest,GOexpress,GOFunction,goProfiles,GOstats,goTools,gpls,gprege,GSEAlm,GSRI,GSVA,Gviz,GWASTools,Harshlight,HCsnip,HDTD,Heatplus,HELP,HEM,HMMcopy,HybridMTest,iBMQ,iChip,iClusterPlus,IdeoViz,impute,inSilicoDb,inSilicoMerging,interactiveDisplay,interactiveDisplayBase,inveRsion,IsoGeneGUI,ITALICS,iterativeBMA,iterativeBMAsurv,joda,KCsmart,lapmix,les,limma,limmaGUI,lmdme,LMGene,logitT,LPE,LPEadj,lumi,LVSmiRNA,maanova,macat,maCorrPlot,maigesPack,MANOR,mAPKL,maPredictDSC,marray,maSigPro,maskBAD,massiR,matchBox,MBCB,mBPCR,mdqc,MEDIPS,MEDME,MergeMaid,metaArray,metahdep,methyAnalysis,MethylAid,Mfuzz,MGFM,MiChip,minet,minfi,MinimumDistance,MiPP,MiRaGE,miRNAtap,MmPalateMiRNA,motifStack,MotIV,Mulcom,multiscan,multtest,mvGST,NanoStringQCPro,neaGUI,nem,netbenchmark,netresponse,nnNorm,npGSEA,nucleR,nudge,OCplus,oligo,OLIN,OLINgui,oneChannelGUI,OrderedList,OutlierD,PAA,PADOG,pandaR,pathview,pcaGoPromoter,pcot2,PECA,pepStat,PGSEA,phenoTest,piano,pickgene,pint,plgem,PLPE,plw,prebs,PROMISE,puma,pvac,pvca,Pviz,qcmetrics,qpgraph,quantro,qusage,rain,rama,RBioinf,RBM,rbsurv,RDRToolbox,reb,RefPlus,ReportingTools,rGADEM,rHVDM,Ringo,RLMM,Rmagpie,rMAT,RmiR,Rnits,ROntoTools,RPA,rqubic,RRHO,RTopper,SAGx,SCAN.UPC,shinyMethyl,sigaR,siggenes,SIM,SimBindProfiles,simpleaffy,sizepower,SMAP,SNAGEE,snapCGH,snm,snpStats,SpeCond,SPIA,spikeLI,spkTools,spotSegmentation,sRAP,ssize,SSPA,STAN,Starr,stepNorm,stepwiseCM,sva,TDARACNE,TEQC,tigre,tilingArray,timecourse,TIN,ToPASeq,topGO,tRanslatome,TransView,trigger,trio,tspair,TurboNorm,twilight,vsn,wateRmelon,waveTiling,webbioc,XDE,xps,yaqcaffy" id="Microarray" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Microarray (345)</a><ul><li packagelist="puma,SVM2CRM" id="ChipOnChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ChipOnChip (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GenotypingArray</a></li><li packagelist="ChAMP,coMET,conumee,DMRcate,ENmix,MBAmethyl,MethylAid,MethylMix,missMethyl,RnBeads,TurboNorm" id="MethylationArray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MethylationArray (11)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MicroRNAArray</a></li><li packagelist="erccdashboard,limma,NanoStringQCPro,puma" id="mRNAMicroarray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mRNAMicroarray (4)</a></li><li packagelist="aroma.light,ELBOW,snm" id="MultiChannel" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MultiChannel (3)</a></li><li packagelist="ABarray,affy,affycomp,AffyCompatible,affycoretools,AffyExpress,affyILM,affylmGUI,affypdnn,affyPLM,affyQCReport,AffyRNADegradation,AgiMicroRna,altcdfenvs,annaffy,annmap,aroma.light,ArrayExpress,arrayQualityMetrics,ArrayTools,beadarray,BrainStars,bridge,CAFE,clippda,codelink,copa,DMRcate,ELBOW,ENmix,ExiMiR,gaga,gage,gcrma,GeneRegionScan,GEOquery,GlobalAncova,globaltest,Harshlight,iChip,lapmix,limma,lmdme,lumi,LVSmiRNA,makecdfenv,mvGST,oligo,oneChannelGUI,PAA,PADOG,plw,PROMISE,puma,pvac,RefPlus,RLMM,SAGx,SCAN.UPC,simpleaffy,SNAGEE,snm,Starr,tilingArray,TurboNorm,vsn,webbioc,xps,yaqcaffy" id="OneChannel" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>OneChannel (69)</a></li><li packagelist="altcdfenvs,limma" id="ProprietaryPlatforms" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ProprietaryPlatforms (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ReversePhaseProteinArray</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>TissueMicroarray</a></li><li packagelist="aroma.light,ArrayExpress,arrayQuality,arrayQualityMetrics,ARRmNormalization,beadarraySNP,bridge,CALIB,ChAMP,convert,copa,daMA,DMRcate,dyebias,ELBOW,ENmix,ExiMiR,gage,genArise,GEOquery,HELP,limma,limmaGUI,lmdme,lumi,maigesPack,MANOR,marray,MethylAid,methylumi,minfi,MmPalateMiRNA,nnNorm,nudge,oligo,OLIN,OLINgui,PADOG,plw,puma,rama,Ringo,RnBeads,SCAN.UPC,shinyMethyl,skewr,SMAP,SNAGEE,snapCGH,snm,spotSegmentation,stepNorm,TurboNorm,vsn,wateRmelon" id="TwoChannel" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>TwoChannel (55)</a></li></ul></li><li packagelist="bioassayR,cellGrowth,ChemmineOB,ChemmineR,ddCt,eiR,fmcsR,HTqPCR,NormqPCR,Rchemcpp,ReadqPCR,SLqPCR,splots,staRank,unifiedWMWqPCR" id="MicrotitrePlateAssay" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MicrotitrePlateAssay (15)</a></li><li packagelist="ddCt,DECIPHER,EasyqpcR,HTqPCR,nondetects,NormqPCR,ReadqPCR,SLqPCR,unifiedWMWqPCR" id="qPCR" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>qPCR (9)</a></li><li packagelist="BADER,baySeq,DESeq,DESeq2,edgeR,GEOquery,MiRaGE,rnaSeqMap,sagenhaft" id="SAGE" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SAGE (9)</a></li><li packagelist="ALDEx2,AllelicImbalance,ampliQueso,AnnotationDbi,anota,ArrayExpressHTS,BADER,bamsignals,BaseSpaceR,baySeq,biomvRCNS,Biostrings,birta,birte,BiSeq,BitSeq,CAGEr,casper,CexoR,ChIPQC,chipseq,cleanUpdTSeq,Clomial,clonotypeR,clstutils,cn.mops,CNAnorm,CNVrd2,cogena,coMET,compEpiTools,ComplexHeatmap,CorMut,CoverageView,csaw,CSSP,DBChIP,DECIPHER,deepSNV,derfinder,derfinderHelper,derfinderPlot,DESeq,DESeq2,DEXSeq,dexus,DiffBind,diffHic,DirichletMultinomial,DMRcaller,DSS,EBSeq,EBSeqHMM,EDASeq,EDDA,edgeR,ELBOW,ensembldb,exomeCopy,exomePeak,facopy,focalCall,FourCSeq,gage,gaucho,GeneMeta,GeneNetworkBuilder,geneRxCluster,genomation,GenomicAlignments,GenomicFeatures,GenomicFiles,GenomicRanges,GenomicTuples,girafe,GOexpress,goseq,GOTHiC,GreyListChIP,groHMM,gtrellis,hapFabia,HDTD,hiReadsProcessor,HiTC,HMMcopy,htSeqTools,HTSFilter,InPAS,intansv,interactiveDisplay,interactiveDisplayBase,iSeq,IVAS,jmosaics,LowMACA,manta,MBASED,mcaGUI,MEDIPS,metagenomeSeq,metaSeq,MethTargetedNGS,methylMnM,methylPipe,MethylSeekR,MiRaGE,miRNAtap,MLSeq,mmnet,monocle,mosaics,motifStack,msa,muscle,NarrowPeaks,NGScopy,NOISeq,nucleR,oneChannelGUI,OTUbase,pathview,phyloseq,PICS,PING,podkat,polyester,Polyfit,prebs,PROPER,QDNAseq,qrqc,quantro,QuasR,r3Cseq,R453Plus1Toolbox,Rariant,Rbowtie,Rcade,REDseq,regionReport,ReQON,rGADEM,rGREAT,Rhtslib,riboSeqR,RIPSeeker,RNAprobR,rnaSeqMap,RnaSeqSampleSize,RnBeads,roar,Rolexa,Rqc,rRDP,Rsamtools,rSFFreader,Rsubread,RSVSim,sangerseqR,savR,segmentSeq,SeqArray,seqbias,seqCNA,SeqGSEA,seqplots,seqTools,SeqVarTools,ShortRead,sincell,soGGi,SomatiCA,SomaticSignatures,spliceR,SplicingGraphs,splots,SRAdb,ssviz,STAN,Sushi,sva,systemPipeR,TCC,TEQC,TitanCNA,tracktables,TransView,triform,TSSi,tweeDEseq,VariantAnnotation,VariantFiltering,VariantTools" id="Sequencing" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Sequencing (248)</a><ul><li packagelist="ALDEx2,BayesPeak,ChIPpeakAnno,ChIPQC,ChIPseeker,chipseq,ChIPseqR,ChIPsim,ChIPXpress,cobindR,coMET,CoverageView,CSAR,csaw,CSSP,DBChIP,DESeq,DESeq2,DiffBind,DSS,EDDA,edgeR,epigenomix,GreyListChIP,iASeq,iSeq,jmosaics,MEDIPS,metagene,MMDiff,motifStack,MotIV,NarrowPeaks,nucleR,PICS,QuasR,Rcade,regioneR,rGADEM,Rsubread,seqplots,similaRpeak,soGGi,SVM2CRM,systemPipeR,TransView,triform" id="ChIPSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ChIPSeq (47)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DenovoGenome</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DenovoTranscriptome</a></li><li packagelist="ALDEx2,Clomial,CNAnorm,coMET,NGScopy,QDNAseq,regioneR,TitanCNA" id="DNASeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DNASeq (8)</a></li><li packagelist="Clomial,CODEX,coMET,CopywriteR,NGScopy,TitanCNA" id="ExomeSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ExomeSeq (6)</a></li><li packagelist="diffHic,GenomicInteractions,GOTHiC,HiTC,Sushi" id="HiC" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>HiC (5)</a></li><li packagelist="BEAT,BiSeq,COHCAP,coMET,methylPipe,MethylSeekR,MPFE,QuasR,regioneR,Repitools,RnBeads,systemPipeR,TransView" id="MethylSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MethylSeq (13)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MicrobialStrain</a></li><li packagelist="DirichletMultinomial,metagenomeSeq,mmnet,phyloseq,rRDP" id="Microbiome" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Microbiome (5)</a></li><li packagelist="miRNApath,miRNAtap,rcellminer,Roleswitch,TargetScore" id="miRNA" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>miRNA (5)</a></li><li packagelist="coMET,RIPSeeker" id="RIPSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>RIPSeq (2)</a></li><li packagelist="AIMS,ALDEx2,ampliQueso,ArrayExpressHTS,BADER,ballgown,BitSeq,casper,coMET,compcodeR,CoverageView,cqn,customProDB,DEGreport,DEGseq,derfinder,derfinderHelper,derfinderPlot,DESeq,DESeq2,DEXSeq,dexus,DSS,easyRNASeq,EBSeq,EBSeqHMM,EDASeq,EDDA,edgeR,ELBOW,erccdashboard,flipflop,gage,geecc,GenomicAlignments,GOexpress,goseq,HTSFilter,iASeq,InPAS,IVAS,limma,manta,metaSeq,metaseqR,MiRaGE,MLSeq,monocle,mvGST,NOISeq,oneChannelGUI,pathview,Polyfit,prebs,PROPER,QuasR,qusage,regionReport,ReportingTools,rgsepd,rnaSeqMap,RNASeqPower,RnaSeqSampleSize,Rsubread,RUVSeq,sapFinder,SCAN.UPC,SeqGSEA,seqplots,SGSeq,SigFuge,sincell,spliceSites,SplicingGraphs,sSeq,SSPA,ssviz,sva,systemPipeR,TCC,ToPASeq,TransView,TSSi,tweeDEseq" id="RNASeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>RNASeq (84)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SangerSeq</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SmallRNA</a></li><li packagelist="Clomial,CopywriteR,NGScopy" id="TargetedResequencing" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>TargetedResequencing (3)</a></li><li packagelist="CNAnorm,LowMACA,NGScopy,podkat,rGREAT,TitanCNA" id="WholeGenome" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>WholeGenome (6)</a></li></ul></li></ul></li><li packagelist="geecc,metaseqR,nondetects" id="WorkflowStep" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>WorkflowStep (525)</a><ul><li packagelist="bamsignals,Biostrings,DECIPHER,diffHic,GenomicAlignments,gmapR,GreyListChIP,LowMACA,metagene,MethTargetedNGS,motifStack,msa,muscle,QuasR,Rbowtie,Rsamtools,Rsubread,segmentSeq,systemPipeR" id="Alignment" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Alignment (19)</a><ul><li packagelist="LowMACA,msa,muscle" id="MultipleSequenceAlignment" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MultipleSequenceAlignment (3)</a></li></ul></li><li packagelist="AffyExpress,altcdfenvs,annaffy,annmap,annotate,AnnotationDbi,AnnotationForge,annotationTools,ArrayTools,biomaRt,BridgeDbR,BSgenome,Category,categoryCompare,ChIPpeakAnno,ChIPseeker,csaw,DASiR,DAVIDQuery,DMRforPairs,domainsignatures,DOSE,ecolitk,ensemblVEP,ENVISIONQuery,FGNet,FunciSNP,gaggle,gCMAP,genomation,GenomeInfoDb,genomes,GenomicFeatures,GenomicRanges,ggtree,GOexpress,GOSemSim,GOstats,hiAnnotator,iBBiG,IdMappingAnalysis,IdMappingRetrieval,intansv,KEGGREST,matchBox,mdgsa,MeSHDbi,meshr,miRNApath,mygene,occugene,OmicCircos,OrganismDbi,PAnnBuilder,pdInfoBuilder,podkat,RBioinf,ReactomePA,rfPred,Risa,RNAither,rnaSeqMap,rols,rtracklayer,SemDist,simpleaffy,SplicingGraphs,UniProt.ws,VariantAnnotation,VariantFiltering,xmapbridge" id="Annotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Annotation (71)</a></li><li packagelist="BEclear,edgeR,ENmix,erccdashboard,limma,MethylAid,pvca,RnBeads,sva" id="BatchEffect" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>BatchEffect (9)</a></li><li packagelist="EDDA,OSAT,RnaSeqSampleSize" id="ExperimentalDesign" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ExperimentalDesign (3)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GenomeBrowsers</a></li><li packagelist="affylmGUI,baySeq,BGmix,bumphunter,categoryCompare,ChIPseeker,clippda,Clomial,clusterProfiler,cnvGSA,cobindR,CompGO,csaw,diffHic,dks,DMRcate,DOSE,EBSeq,EBSeqHMM,edge,edgeR,erccdashboard,fabia,fdrame,flowMap,gaga,gage,GeneOverlap,GEWIST,GOexpress,GOstats,HTqPCR,HTSanalyzeR,IdMappingAnalysis,IdMappingRetrieval,LBE,limma,limmaGUI,made4,matchBox,metagene,metagenomeSeq,MineICA,MMDiff,MmPalateMiRNA,monocle,msa,Mulcom,MultiMed,multtest,OCplus,omicade4,oneChannelGUI,OrderedList,parody,PathNet,phenoTest,phyloseq,PROMISE,prot2D,quantro,rain,RBioinf,ReactomePA,SAGx,segmentSeq,siggenes,sigPathway,similaRpeak,snm,SpeCond,ssviz,staRank,sva,TransView,twilight,unifiedWMWqPCR" id="MultipleComparison" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MultipleComparison (77)</a></li><li packagelist="ACME,blima,Cardinal,ChAMP,CNAnorm,CODEX,conumee,csaw,diffHic,EDDA,edgeR,ENmix,HTSFilter,limma,metagenomeSeq,metaseqR,missMethyl,NanoStringQCPro,quantro,RNAprobR,Rnits,RUVcorr,RUVnormalize,sva,TurboNorm" id="Normalization" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Normalization (25)</a></li><li packagelist="annaffy,annotate,ASSIGN,Category,categoryCompare,clipper,clusterProfiler,cogena,cpvSNP,DART,domainsignatures,DOSE,EnrichmentBrowser,fastLiquidAssociation,FGNet,gage,gCMAP,GlobalAncova,globaltest,GOexpress,GOFunction,GOSemSim,GOSim,GOstats,graphite,GSReg,GSVA,joda,KEGGgraph,keggorthology,KEGGprofile,KEGGREST,LiquidAssociation,mdgsa,MethylMix,mgsa,miRNApath,mmnet,MmPalateMiRNA,mvGST,NCIgraph,neaGUI,nem,NetPathMiner,npGSEA,occugene,PathNet,pathRender,pathview,paxtoolsr,piano,qpgraph,ReactomePA,rGREAT,rsbml,safe,SAGx,SBMLR,sigaR,ToPASeq" id="Pathways" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Pathways (83)</a><ul><li packagelist="graphite,UniProt.ws" id="BioCarta" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>BioCarta (2)</a></li><li packagelist="annaffy,annotate,Category,categoryCompare,clusterProfiler,CompGO,ExpressionView,FGNet,gage,geecc,globaltest,GOexpress,GOFunction,goProfiles,GOSemSim,goseq,GOSim,GOstats,GOsummaries,goTools,GSEABase,interactiveDisplay,interactiveDisplayBase,mdgsa,mgsa,MmPalateMiRNA,mvGST,neaGUI,pcaGoPromoter,RamiGO,Rcpi,ReportingTools,rGREAT,RNAither,rols,SemDist,sRAP,tRanslatome,UniProt.ws" id="GO" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GO (39)</a></li><li packagelist="attract,clipper,ExpressionView,globaltest,graphite,GSEABase,KEGGgraph,keggorthology,KEGGprofile,KEGGREST,MLP,neaGUI,PathNet,pcot2,SAGx,UniProt.ws" id="KEGG" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>KEGG (16)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>NCINatureCurated</a></li><li packagelist="graphite,MLP,neaGUI,ReactomePA" id="Reactome" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Reactome (4)</a></li></ul></li><li packagelist="ABarray,affy,affycomp,AffyExpress,affyILM,affylmGUI,affyPara,affypdnn,affyPLM,AffyRNADegradation,AffyTiling,AgiMicroRna,altcdfenvs,aroma.light,ArrayTools,ARRmNormalization,ballgown,beadarray,beadarraySNP,biovizBase,blima,BUS,CAGEr,CALIB,cellHTS2,CGHcall,CGHnormaliter,clippda,codelink,conumee,CopyNumber450k,CopywriteR,cqn,crlmm,DEGseq,diffHic,DupChecker,dyebias,easyRNASeq,EDASeq,ENmix,ExiMiR,farms,flowWorkspace,focalCall,FourCSeq,frma,frmaTools,gcrma,genArise,gespeR,GOTHiC,gprege,GreyListChIP,Harshlight,HELP,hiReadsProcessor,HMMcopy,HTqPCR,HTSFilter,iBMQ,imageHTS,limma,limmaGUI,LMGene,lumi,LVSmiRNA,maCorrPlot,maigesPack,makecdfenv,MANOR,marray,MBCB,MEDIPS,metaseqR,methylumi,Mfuzz,MiChip,minfi,mitoODE,MmPalateMiRNA,multiscan,NanoStringQCPro,nnNorm,nondetects,NTW,oligo,OLIN,OLINgui,oneChannelGUI,openCyto,pepStat,piano,prebs,puma,QDNAseq,qpcrNorm,qrqc,quantro,QuasR,r3Cseq,rama,RBioinf,REDseq,RefPlus,ReQON,Ringo,rMAT,RNAinteract,RNAither,RnBeads,Rolexa,RPA,Rsubread,RUVSeq,SAGx,SCAN.UPC,scsR,shinyMethyl,simpleaffy,skewr,snapCGH,snm,spotSegmentation,sRAP,Starr,stepNorm,sva,SVM2CRM,TargetSearch,tilingArray,TSSi,TurboNorm,vsn,wateRmelon,xps" id="Preprocessing" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Preprocessing (136)</a></li><li packagelist="AffyExpress,affylmGUI,affyPLM,affyQCReport,AffyRNADegradation,altcdfenvs,arrayMvout,arrayQuality,arrayQualityMetrics,ArrayTools,Basic4Cseq,beadarray,ChIPQC,chipseq,clstutils,CODEX,conumee,CopyNumber450k,CSSP,cummeRbund,DECIPHER,dexus,dks,DMRcate,dyebias,EDASeq,edgeR,ENmix,erccdashboard,farms,ffpe,flowClean,GENESIS,GWASTools,Harshlight,HELP,HTqPCR,htSeqTools,ibh,interactiveDisplay,interactiveDisplayBase,isobar,limma,limmaGUI,lumi,MANOR,manta,massiR,mdqc,MEDIPS,metaseqR,MethylAid,methylumi,minfi,MmPalateMiRNA,monocle,MSnbase,NanoStringQCPro,OLIN,OLINgui,oneChannelGUI,OSAT,piano,pRoloc,proteoQC,pvac,qcmetrics,QDNAseq,qrqc,QuasR,R453Plus1Toolbox,rama,RBioinf,ReQON,Ringo,RNAinteract,RNAither,RnBeads,Rolexa,RPA,Rqc,Rsamtools,Rsubread,segmentSeq,seqTools,shinyMethyl,ShortRead,simpleaffy,skewr,SNAGEE,snm,spikeLI,spotSegmentation,sRAP,Starr,systemPipeR,TEQC,tracktables,tRanslatome,unifiedWMWqPCR,wateRmelon,yaqcaffy" id="QualityControl" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>QualityControl (102)</a></li><li packagelist="affycoretools,AffyExpress,ampliQueso,annaffy,annmap,arrayQualityMetrics,ArrayTools,ChIPQC,DEGreport,DMRforPairs,DynDoc,EnrichmentBrowser,Harshlight,metaseqR,MmPalateMiRNA,NanoStringQCPro,proteoQC,qcmetrics,R453Plus1Toolbox,rnaSeqMap,sapFinder,simpleaffy,SpeCond,tracktables,xmapbridge,yaqcaffy" id="ReportWriting" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ReportWriting (26)</a></li><li packagelist="AffyExpress,ampliQueso,AnalysisPageServer,annmap,aroma.light,arrayQuality,ArrayTools,Basic4Cseq,BiGGR,bioassayR,biocGraph,BioMVCClass,biomvRCNS,biosvd,biovizBase,BrowserViz,BrowserVizDemo,CAGEr,cancerclass,cellGrowth,cellHTS,cellHTS2,CGHcall,CGHregions,ChemmineOB,ChemmineR,ChIPseeker,chroGPS,ChromHeatMap,clstutils,clusterProfiler,CNEr,cogena,coMET,compEpiTools,CompGO,ComplexHeatmap,copa,copynumber,CopywriteR,CoRegNet,CoverageView,ctc,cummeRbund,dagLogo,DECIPHER,DEGreport,deltaGseg,derfinderPlot,DMRforPairs,DOSE,EBImage,ecolitk,eiR,eisa,EnrichmentBrowser,epivizr,explorase,ExpressionView,fabia,facopy,FGNet,flowcatchR,flowCHIC,flowClust,flowCyBar,flowFP,flowPlots,FlowSOM,flowViz,fmcsR,focalCall,gaggle,gCMAPWeb,GeneAnswers,genefu,GeneOverlap,geneplotter,GeneRegionScan,genomation,GenomeGraphs,GenoView,gespeR,ggbio,ggtree,GOexpress,GOsummaries,goTools,GSCA,gtrellis,Gviz,hapFabia,Heatplus,HELP,HilbertVis,HilbertVisGUI,HMMcopy,HTqPCR,hyperdraw,IdeoViz,idiogram,imageHTS,interactiveDisplay,interactiveDisplayBase,KCsmart,KEGGgraph,keggorthology,lmdme,macat,maCorrPlot,manta,MassArray,matchBox,mcaGUI,MEDIPS,MergeMaid,metagenomeSeq,methyAnalysis,MethylAid,Mfuzz,MineICA,monocle,motifStack,MSGFgui,MVCClass,NarrowPeaks,netbiov,NetSAM,NOISeq,NuPoP,OLIN,OLINgui,OmicCircos,oposSOM,OSAT,PANR,pathRender,pathview,Pbase,pcaGoPromoter,piano,PICS,PING,prada,proBAMr,pRolocGUI,proteoQC,Pviz,qrqc,quantsmooth,R453Plus1Toolbox,RamiGO,Rariant,RCASPAR,rcellminer,Rchemcpp,RCyjs,RCytoscape,RDAVIDWebService,RDRToolbox,ReactomePA,reb,RedeR,regionReport,ReportingTools,Rgraphviz,riboSeqR,RmiR,RNAinteract,RNAither,rnaSeqMap,RTCA,rtracklayer,sangerseqR,sapFinder,seqPattern,seqplots,SigFuge,SIM,simpleaffy,sincell,SNPchip,SomaticSignatures,spade,spliceR,SplicingGraphs,splots,SQUADD,sRAP,ssviz,supraHex,survcomp,Sushi,SwimR,tilingArray,ToPASeq,topGO,trackViewer,TransView,TSCAN,unifiedWMWqPCR,xmapbridge" id="Visualization" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Visualization (240)</a><ul><li packagelist="BiGGR,BioNet,BiRewire,birte,CNORfuzzy,CoRegNet,COSNet,DEGraph,DriverNet,EnrichmentBrowser,FGNet,GOSemSim,GraphAlignment,GraphAT,graphite,GRENITS,interactiveDisplay,interactiveDisplayBase,LiquidAssociation,MethylMix,minet,neaGUI,netbenchmark,netbiov,NetPathMiner,netresponse,NetSAM,networkBMA,pandaR,pathifier,PathNet,paxtoolsr,pcot2,PCpheno,ppiStats,R3CPET,RBGL,RedeR,rsbml,rTRM,rTRMui,SANTA,SBMLR,SLGI,SPEM,SQUADD,STRINGdb,vtpnet" id="Network" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Network (48)</a></li></ul></li></ul></li></ul></li><li packagelist="adme16cod.db,ag.db,agcdf,agprobe,anopheles.db0,arabidopsis.db0,ath1121501.db,ath1121501cdf,ath1121501probe,barley1cdf,barley1probe,bovine.db,bovine.db0,bovinecdf,bovineprobe,BSgenome.Alyrata.JGI.v1,BSgenome.Amellifera.BeeBase.assembly4,BSgenome.Amellifera.UCSC.apiMel2,BSgenome.Amellifera.UCSC.apiMel2.masked,BSgenome.Athaliana.TAIR.04232008,BSgenome.Athaliana.TAIR.TAIR9,BSgenome.Btaurus.UCSC.bosTau3,BSgenome.Btaurus.UCSC.bosTau3.masked,BSgenome.Btaurus.UCSC.bosTau4,BSgenome.Btaurus.UCSC.bosTau4.masked,BSgenome.Btaurus.UCSC.bosTau6,BSgenome.Btaurus.UCSC.bosTau6.masked,BSgenome.Btaurus.UCSC.bosTau8,BSgenome.Celegans.UCSC.ce10,BSgenome.Celegans.UCSC.ce2,BSgenome.Celegans.UCSC.ce6,BSgenome.Cfamiliaris.UCSC.canFam2,BSgenome.Cfamiliaris.UCSC.canFam2.masked,BSgenome.Cfamiliaris.UCSC.canFam3,BSgenome.Cfamiliaris.UCSC.canFam3.masked,BSgenome.Dmelanogaster.UCSC.dm2,BSgenome.Dmelanogaster.UCSC.dm2.masked,BSgenome.Dmelanogaster.UCSC.dm3,BSgenome.Dmelanogaster.UCSC.dm3.masked,BSgenome.Dmelanogaster.UCSC.dm6,BSgenome.Drerio.UCSC.danRer5,BSgenome.Drerio.UCSC.danRer5.masked,BSgenome.Drerio.UCSC.danRer6,BSgenome.Drerio.UCSC.danRer6.masked,BSgenome.Drerio.UCSC.danRer7,BSgenome.Drerio.UCSC.danRer7.masked,BSgenome.Ecoli.NCBI.20080805,BSgenome.Gaculeatus.UCSC.gasAcu1,BSgenome.Gaculeatus.UCSC.gasAcu1.masked,BSgenome.Ggallus.UCSC.galGal3,BSgenome.Ggallus.UCSC.galGal3.masked,BSgenome.Ggallus.UCSC.galGal4,BSgenome.Ggallus.UCSC.galGal4.masked,BSgenome.Hsapiens.NCBI.GRCh38,BSgenome.Hsapiens.UCSC.hg17,BSgenome.Hsapiens.UCSC.hg17.masked,BSgenome.Hsapiens.UCSC.hg18,BSgenome.Hsapiens.UCSC.hg18.masked,BSgenome.Hsapiens.UCSC.hg19,BSgenome.Hsapiens.UCSC.hg19.masked,BSgenome.Hsapiens.UCSC.hg38,BSgenome.Hsapiens.UCSC.hg38.masked,BSgenome.Mfascicularis.NCBI.5.0,BSgenome.Mfuro.UCSC.musFur1,BSgenome.Mmulatta.UCSC.rheMac2,BSgenome.Mmulatta.UCSC.rheMac2.masked,BSgenome.Mmulatta.UCSC.rheMac3,BSgenome.Mmulatta.UCSC.rheMac3.masked,BSgenome.Mmusculus.UCSC.mm10,BSgenome.Mmusculus.UCSC.mm10.masked,BSgenome.Mmusculus.UCSC.mm8,BSgenome.Mmusculus.UCSC.mm8.masked,BSgenome.Mmusculus.UCSC.mm9,BSgenome.Mmusculus.UCSC.mm9.masked,BSgenome.Osativa.MSU.MSU7,BSgenome.Ptroglodytes.UCSC.panTro2,BSgenome.Ptroglodytes.UCSC.panTro2.masked,BSgenome.Ptroglodytes.UCSC.panTro3,BSgenome.Ptroglodytes.UCSC.panTro3.masked,BSgenome.Rnorvegicus.UCSC.rn4,BSgenome.Rnorvegicus.UCSC.rn4.masked,BSgenome.Rnorvegicus.UCSC.rn5,BSgenome.Rnorvegicus.UCSC.rn5.masked,BSgenome.Rnorvegicus.UCSC.rn6,BSgenome.Scerevisiae.UCSC.sacCer1,BSgenome.Scerevisiae.UCSC.sacCer2,BSgenome.Scerevisiae.UCSC.sacCer3,BSgenome.Sscrofa.UCSC.susScr3,BSgenome.Sscrofa.UCSC.susScr3.masked,BSgenome.Tgondii.ToxoDB.7.0,BSgenome.Tguttata.UCSC.taeGut1,BSgenome.Tguttata.UCSC.taeGut1.masked,bsubtiliscdf,bsubtilisprobe,canine.db,canine.db0,canine2.db,canine2cdf,canine2probe,caninecdf,canineprobe,celegans.db,celeganscdf,celegansprobe,ChemmineDrugs,chicken.db,chicken.db0,chickencdf,chickenprobe,chimp.db0,citruscdf,citrusprobe,cMAP,cottoncdf,cottonprobe,cyp450cdf,DO.db,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,ecoli2.db,ecoli2cdf,ecoli2probe,ecoliasv2cdf,ecoliasv2probe,ecolicdf,ecoliK12.db0,ecoliprobe,ecoliSakai.db0,EnsDb.Hsapiens.v75,EnsDb.Hsapiens.v79,EnsDb.Mmusculus.v75,EnsDb.Mmusculus.v79,EnsDb.Rnorvegicus.v75,EnsDb.Rnorvegicus.v79,FDb.FANTOM4.promoters.hg19,FDb.InfiniumMethylation.hg18,FDb.InfiniumMethylation.hg19,FDb.UCSC.snp135common.hg19,FDb.UCSC.snp137common.hg19,FDb.UCSC.tRNAs,fly.db0,gahgu133a.db,gahgu133acdf,gahgu133aprobe,gahgu133b.db,gahgu133bcdf,gahgu133bprobe,gahgu133plus2.db,gahgu133plus2cdf,gahgu133plus2probe,gahgu95av2.db,gahgu95av2cdf,gahgu95av2probe,gahgu95b.db,gahgu95bcdf,gahgu95bprobe,gahgu95c.db,gahgu95ccdf,gahgu95cprobe,gahgu95d.db,gahgu95dcdf,gahgu95dprobe,gahgu95e.db,gahgu95ecdf,gahgu95eprobe,genomewidesnp5Crlmm,genomewidesnp6Crlmm,GGHumanMethCancerPanelv1.db,GO.db,gp53cdf,grasp2db,h10kcod.db,h20kcod.db,hapmap370k,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133atagcdf,hgu133atagprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu219.db,hgu219cdf,hgu219probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hguatlas13k.db,hgubeta7.db,hguDKFZ31.db,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hgug4845a.db,hguqiagenv3.db,hi16cod.db,hivprtplus2cdf,hom.At.inp.db,hom.Ce.inp.db,hom.Dm.inp.db,hom.Dr.inp.db,hom.Hs.inp.db,hom.Mm.inp.db,hom.Rn.inp.db,hom.Sc.inp.db,Homo.sapiens,hs25kresogen.db,Hs6UG171.db,HsAgilentDesign026652.db,Hspec,hspeccdf,hta20stprobeset.db,hta20sttranscriptcluster.db,hthgu133a.db,hthgu133acdf,hthgu133afrmavecs,hthgu133aprobe,hthgu133b.db,hthgu133bcdf,hthgu133bprobe,hthgu133pluspmcdf,hthgu133pluspmprobe,htmg430acdf,htmg430aprobe,htmg430bcdf,htmg430bprobe,htmg430pmcdf,htmg430pmprobe,htrat230pmcdf,htrat230pmprobe,htratfocuscdf,htratfocusprobe,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,huex.1.0.st.v2frmavecs,huex10stprobeset.db,huex10sttranscriptcluster.db,HuExExonProbesetLocation,HuExExonProbesetLocationHg18,HuExExonProbesetLocationHg19,hugene.1.0.st.v1frmavecs,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene10stv1cdf,hugene10stv1probe,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,human.db0,human1mduov3bCrlmm,human1mv1cCrlmm,human370quadv3cCrlmm,human370v1cCrlmm,human550v3bCrlmm,human610quadv1bCrlmm,human650v3aCrlmm,human660quadv1aCrlmm,humanCHRLOC,humancytosnp12v2p1hCrlmm,humanomni1quadv1bCrlmm,humanomni25quadv1bCrlmm,humanomni5quadv1bCrlmm,humanomniexpress12v1bCrlmm,HuO22.db,hwgcod.db,IlluminaHumanMethylation27k.db,IlluminaHumanMethylation27kmanifest,IlluminaHumanMethylation450k.db,IlluminaHumanMethylation450kanno.ilmn12.hg19,IlluminaHumanMethylation450kmanifest,IlluminaHumanMethylation450kprobe,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv2BeadID.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,illuminaRatv1.db,indac.db,JazaeriMetaData.db,KEGG.db,LAPOINTE.db,LowMACAAnnotation,lumiHumanAll.db,lumiHumanIDMapping,lumiMouseAll.db,lumiMouseIDMapping,lumiRatAll.db,lumiRatIDMapping,m10kcod.db,m20kcod.db,MafDb.ALL.wgs.phase1.release.v3.20101123,MafDb.ALL.wgs.phase3.release.v5a.20130502,MafDb.ESP6500SI.V2.SSA137,MafDb.ExAC.r0.3.sites,maizecdf,maizeprobe,malaria.db0,medicagocdf,medicagoprobe,MeSH.Aca.eg.db,MeSH.Aga.PEST.eg.db,MeSH.Ame.eg.db,MeSH.Aml.eg.db,MeSH.Ana.eg.db,MeSH.Ani.FGSC.eg.db,MeSH.AOR.db,MeSH.Ath.eg.db,MeSH.Bfl.eg.db,MeSH.Bsu.168.eg.db,MeSH.Bsu.TUB10.eg.db,MeSH.Bta.eg.db,MeSH.Cal.SC5314.eg.db,MeSH.Cbr.eg.db,MeSH.Cel.eg.db,MeSH.Cfa.eg.db,MeSH.Cin.eg.db,MeSH.Cja.eg.db,MeSH.Cpo.eg.db,MeSH.Cre.eg.db,MeSH.Dan.eg.db,MeSH.db,MeSH.Dda.3937.eg.db,MeSH.Ddi.AX4.eg.db,MeSH.Der.eg.db,MeSH.Dgr.eg.db,MeSH.Dme.eg.db,MeSH.Dmo.eg.db,MeSH.Dpe.eg.db,MeSH.Dre.eg.db,MeSH.Dse.eg.db,MeSH.Dsi.eg.db,MeSH.Dvi.eg.db,MeSH.Dya.eg.db,MeSH.Eco.55989.eg.db,MeSH.Eco.CFT073.eg.db,MeSH.Eco.ED1a.eg.db,MeSH.Eco.HS.eg.db,MeSH.Eco.IAI1.eg.db,MeSH.Eco.IAI39.eg.db,MeSH.Eco.K12.DH10B.eg.db,MeSH.Eco.K12.MG1655.eg.db,MeSH.Eco.O127.H6.E2348.69.eg.db,MeSH.Eco.O157.H7.EDL933.eg.db,MeSH.Eco.O157.H7.Sakai.eg.db,MeSH.Eco.S88.eg.db,MeSH.Eco.UMN026.eg.db,MeSH.Eqc.eg.db,MeSH.Gga.eg.db,MeSH.Gma.eg.db,MeSH.Hsa.eg.db,MeSH.Laf.eg.db,MeSH.Lma.eg.db,MeSH.Mdo.eg.db,MeSH.Mes.eg.db,MeSH.Mga.eg.db,MeSH.Miy.eg.db,MeSH.Mml.eg.db,MeSH.Mmu.eg.db,MeSH.Mtr.eg.db,MeSH.Nle.eg.db,MeSH.Oan.eg.db,MeSH.Ocu.eg.db,MeSH.Oni.eg.db,MeSH.Osa.eg.db,MeSH.Pab.eg.db,MeSH.Pae.PAO1.eg.db,MeSH.PCR.db,MeSH.Pfa.3D7.eg.db,MeSH.Pto.eg.db,MeSH.Ptr.eg.db,MeSH.Rno.eg.db,MeSH.Sau.USA300TCH1516.eg.db,MeSH.Sce.S288c.eg.db,MeSH.Sco.A32.eg.db,MeSH.Sil.eg.db,MeSH.Spo.972h.eg.db,MeSH.Spu.eg.db,MeSH.Ssc.eg.db,MeSH.Syn.eg.db,MeSH.Tbr.9274.eg.db,MeSH.Tgo.ME49.eg.db,MeSH.Tgu.eg.db,MeSH.Vvi.eg.db,MeSH.Xla.eg.db,MeSH.Xtr.eg.db,MeSH.Zma.eg.db,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mguatlas5k.db,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,mi16cod.db,mirbase.db,mirna102xgaincdf,mirna10cdf,mirna10probe,mirna20cdf,miRNAtap.db,mm24kresogen.db,MmAgilentDesign026655.db,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,moex10stprobeset.db,moex10sttranscriptcluster.db,MoExExonProbesetLocation,mogene.1.0.st.v1frmavecs,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene10stv1cdf,mogene10stv1probe,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse.db0,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mouseCHRLOC,mpedbarray.db,mta10stprobeset.db,mta10sttranscriptcluster.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,Mu15v1.db,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,Mu22v3.db,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,Mus.musculus,mwgcod.db,Norway981.db,nugohs1a520180.db,nugohs1a520180cdf,nugohs1a520180probe,nugomm1a520177.db,nugomm1a520177cdf,nugomm1a520177probe,oligoData,OperonHumanV3.db,org.Ag.eg.db,org.At.tair.db,org.Bt.eg.db,org.Ce.eg.db,org.Cf.eg.db,org.Dm.eg.db,org.Dr.eg.db,org.EcK12.eg.db,org.EcSakai.eg.db,org.Gg.eg.db,org.Hs.eg.db,org.Hs.ipi.db,org.Mm.eg.db,org.Mmu.eg.db,org.Pf.plasmo.db,org.Pt.eg.db,org.Rn.eg.db,org.Sc.sgd.db,org.Sco.eg.db,org.Ss.eg.db,org.Tgondii.eg.db,org.Xl.eg.db,paeg1acdf,paeg1aprobe,PANTHER.db,PartheenMetaData.db,pd.081229.hg18.promoter.medip.hx1,pd.2006.07.18.hg18.refseq.promoter,pd.2006.07.18.mm8.refseq.promoter,pd.2006.10.31.rn34.refseq.promoter,pd.ag,pd.aragene.1.0.st,pd.aragene.1.1.st,pd.ath1.121501,pd.barley1,pd.bovgene.1.0.st,pd.bovgene.1.1.st,pd.bovine,pd.bsubtilis,pd.cangene.1.0.st,pd.cangene.1.1.st,pd.canine,pd.canine.2,pd.celegans,pd.charm.hg18.example,pd.chicken,pd.chigene.1.0.st,pd.chigene.1.1.st,pd.chogene.2.0.st,pd.chogene.2.1.st,pd.citrus,pd.cotton,pd.cyngene.1.0.st,pd.cyngene.1.1.st,pd.cyrgene.1.0.st,pd.cyrgene.1.1.st,pd.cytogenetics.array,pd.drogene.1.0.st,pd.drogene.1.1.st,pd.drosgenome1,pd.drosophila.2,pd.e.coli.2,pd.ecoli,pd.ecoli.asv2,pd.elegene.1.0.st,pd.elegene.1.1.st,pd.equgene.1.0.st,pd.equgene.1.1.st,pd.feinberg.hg18.me.hx1,pd.feinberg.mm8.me.hx1,pd.felgene.1.0.st,pd.felgene.1.1.st,pd.fingene.1.0.st,pd.fingene.1.1.st,pd.genomewidesnp.5,pd.genomewidesnp.6,pd.guigene.1.0.st,pd.guigene.1.1.st,pd.hc.g110,pd.hg.focus,pd.hg.u133.plus.2,pd.hg.u133a,pd.hg.u133a.2,pd.hg.u133a.tag,pd.hg.u133b,pd.hg.u219,pd.hg.u95a,pd.hg.u95av2,pd.hg.u95b,pd.hg.u95c,pd.hg.u95d,pd.hg.u95e,pd.hg18.60mer.expr,pd.ht.hg.u133.plus.pm,pd.ht.hg.u133a,pd.ht.mg.430a,pd.hta.2.0,pd.hu6800,pd.huex.1.0.st.v2,pd.hugene.1.0.st.v1,pd.hugene.1.1.st.v1,pd.hugene.2.0.st,pd.hugene.2.1.st,pd.maize,pd.mapping250k.nsp,pd.mapping250k.sty,pd.mapping50k.hind240,pd.mapping50k.xba240,pd.margene.1.0.st,pd.margene.1.1.st,pd.medgene.1.0.st,pd.medgene.1.1.st,pd.medicago,pd.mg.u74a,pd.mg.u74av2,pd.mg.u74b,pd.mg.u74bv2,pd.mg.u74c,pd.mg.u74cv2,pd.mirna.1.0,pd.mirna.2.0,pd.mirna.3.0,pd.mirna.3.1,pd.mirna.4.0,pd.moe430a,pd.moe430b,pd.moex.1.0.st.v1,pd.mogene.1.0.st.v1,pd.mogene.1.1.st.v1,pd.mogene.2.0.st,pd.mogene.2.1.st,pd.mouse430.2,pd.mouse430a.2,pd.mta.1.0,pd.mu11ksuba,pd.mu11ksubb,pd.nugo.hs1a520180,pd.nugo.mm1a520177,pd.ovigene.1.0.st,pd.ovigene.1.1.st,pd.pae.g1a,pd.plasmodium.anopheles,pd.poplar,pd.porcine,pd.porgene.1.0.st,pd.porgene.1.1.st,pd.rabgene.1.0.st,pd.rabgene.1.1.st,pd.rae230a,pd.rae230b,pd.raex.1.0.st.v1,pd.ragene.1.0.st.v1,pd.ragene.1.1.st.v1,pd.ragene.2.0.st,pd.ragene.2.1.st,pd.rat230.2,pd.rcngene.1.0.st,pd.rcngene.1.1.st,pd.rg.u34a,pd.rg.u34b,pd.rg.u34c,pd.rhegene.1.0.st,pd.rhegene.1.1.st,pd.rhesus,pd.rice,pd.rjpgene.1.0.st,pd.rjpgene.1.1.st,pd.rn.u34,pd.rta.1.0,pd.rusgene.1.0.st,pd.rusgene.1.1.st,pd.s.aureus,pd.soybean,pd.soygene.1.0.st,pd.soygene.1.1.st,pd.sugar.cane,pd.tomato,pd.u133.x3p,pd.vitis.vinifera,pd.wheat,pd.x.laevis.2,pd.x.tropicalis,pd.xenopus.laevis,pd.yeast.2,pd.yg.s98,pd.zebgene.1.0.st,pd.zebgene.1.1.st,pd.zebrafish,pedbarrayv10.db,pedbarrayv9.db,PFAM.db,phastCons100way.UCSC.hg19,phastCons7way.UCSC.hg38,pig.db0,plasmodiumanophelescdf,plasmodiumanophelesprobe,POCRCannotation.db,PolyPhen.Hsapiens.dbSNP131,poplarcdf,poplarprobe,porcine.db,porcinecdf,porcineprobe,primeviewcdf,primeviewprobe,r10kcod.db,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,raex10stprobeset.db,raex10sttranscriptcluster.db,RaExExonProbesetLocation,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene10stv1cdf,ragene10stv1probe,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat.db0,rat2302.db,rat2302cdf,rat2302probe,ratCHRLOC,rattoxfxcdf,rattoxfxprobe,Rattus.norvegicus,reactome.db,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rguatlas4k.db,rgug4105a.db,rgug4130a.db,rgug4131a.db,rhesus.db0,rhesuscdf,rhesusprobe,ri16cod.db,ricecdf,riceprobe,RmiR.Hs.miRNA,RmiR.hsa,RnAgilentDesign028282.db,rnu34.db,rnu34cdf,rnu34probe,Roberts2005Annotation.db,rtu34.db,rtu34cdf,rtu34probe,rwgcod.db,saureuscdf,saureusprobe,SHDZ.db,SIFT.Hsapiens.dbSNP132,SIFT.Hsapiens.dbSNP137,SNPlocs.Hsapiens.dbSNP.20090506,SNPlocs.Hsapiens.dbSNP.20100427,SNPlocs.Hsapiens.dbSNP.20101109,SNPlocs.Hsapiens.dbSNP.20110815,SNPlocs.Hsapiens.dbSNP.20111119,SNPlocs.Hsapiens.dbSNP.20120608,SNPlocs.Hsapiens.dbSNP141.GRCh38,SNPlocs.Hsapiens.dbSNP142.GRCh37,soybeancdf,soybeanprobe,sugarcanecdf,sugarcaneprobe,targetscan.Hs.eg.db,targetscan.Mm.eg.db,test1cdf,test2cdf,test3cdf,test3probe,tomatocdf,tomatoprobe,TxDb.Athaliana.BioMart.plantsmart22,TxDb.Athaliana.BioMart.plantsmart25,TxDb.Celegans.UCSC.ce6.ensGene,TxDb.Dmelanogaster.UCSC.dm3.ensGene,TxDb.Hsapiens.BioMart.igis,TxDb.Hsapiens.UCSC.hg18.knownGene,TxDb.Hsapiens.UCSC.hg19.knownGene,TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts,TxDb.Hsapiens.UCSC.hg38.knownGene,TxDb.Mmusculus.UCSC.mm10.ensGene,TxDb.Mmusculus.UCSC.mm10.knownGene,TxDb.Mmusculus.UCSC.mm9.knownGene,TxDb.Rnorvegicus.BioMart.igis,TxDb.Rnorvegicus.UCSC.rn4.ensGene,TxDb.Rnorvegicus.UCSC.rn5.refGene,TxDb.Scerevisiae.UCSC.sacCer2.sgdGene,TxDb.Scerevisiae.UCSC.sacCer3.sgdGene,u133aaofav2cdf,u133x3p.db,u133x3pcdf,u133x3pprobe,vitisviniferacdf,vitisviniferaprobe,wheatcdf,wheatprobe,worm.db0,xenopus.db0,xenopuslaeviscdf,xenopuslaevisprobe,xlaevis.db,xlaevis2cdf,xlaevis2probe,XtraSNPlocs.Hsapiens.dbSNP141.GRCh38,xtropicaliscdf,xtropicalisprobe,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast.db0,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe,zebrafish.db,zebrafish.db0,zebrafishcdf,zebrafishprobe" id="AnnotationData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>AnnotationData (883)</a><ul><li packagelist="adme16cod.db,ag.db,agcdf,agprobe,ath1121501.db,ath1121501cdf,ath1121501probe,barley1cdf,barley1probe,bovine.db,bovinecdf,bovineprobe,bsubtiliscdf,bsubtilisprobe,canine.db,canine2.db,canine2cdf,canine2probe,caninecdf,canineprobe,celegans.db,celeganscdf,celegansprobe,chicken.db,chickencdf,chickenprobe,citruscdf,citrusprobe,cottoncdf,cottonprobe,cyp450cdf,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,ecoli2.db,ecoli2cdf,ecoli2probe,ecoliasv2cdf,ecoliasv2probe,ecolicdf,ecoliprobe,gahgu133acdf,gahgu133aprobe,gahgu133bcdf,gahgu133bprobe,gahgu133plus2cdf,gahgu133plus2probe,gahgu95av2cdf,gahgu95av2probe,gahgu95bcdf,gahgu95bprobe,gahgu95ccdf,gahgu95cprobe,gahgu95dcdf,gahgu95dprobe,gahgu95ecdf,gahgu95eprobe,GGHumanMethCancerPanelv1.db,gp53cdf,h10kcod.db,h20kcod.db,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133atagcdf,hgu133atagprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu219.db,hgu219cdf,hgu219probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hguqiagenv3.db,hi16cod.db,hivprtplus2cdf,hs25kresogen.db,HsAgilentDesign026652.db,Hspec,hspeccdf,hta20stprobeset.db,hta20sttranscriptcluster.db,hthgu133a.db,hthgu133acdf,hthgu133afrmavecs,hthgu133aprobe,hthgu133b.db,hthgu133bcdf,hthgu133bprobe,hthgu133pluspmcdf,hthgu133pluspmprobe,htmg430acdf,htmg430aprobe,htmg430bcdf,htmg430bprobe,htmg430pmcdf,htmg430pmprobe,htrat230pmcdf,htrat230pmprobe,htratfocuscdf,htratfocusprobe,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,huex.1.0.st.v2frmavecs,huex10stprobeset.db,huex10sttranscriptcluster.db,HuExExonProbesetLocation,HuExExonProbesetLocationHg18,HuExExonProbesetLocationHg19,hugene.1.0.st.v1frmavecs,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene10stv1cdf,hugene10stv1probe,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,hwgcod.db,IlluminaHumanMethylation27k.db,IlluminaHumanMethylation27kmanifest,IlluminaHumanMethylation450k.db,IlluminaHumanMethylation450kmanifest,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv2BeadID.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,illuminaRatv1.db,indac.db,lumiHumanAll.db,lumiMouseAll.db,lumiRatAll.db,m10kcod.db,m20kcod.db,maizecdf,maizeprobe,medicagocdf,medicagoprobe,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,mi16cod.db,mirna102xgaincdf,mirna10cdf,mirna10probe,mirna20cdf,mm24kresogen.db,MmAgilentDesign026655.db,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,moex10stprobeset.db,moex10sttranscriptcluster.db,MoExExonProbesetLocation,mogene.1.0.st.v1frmavecs,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene10stv1cdf,mogene10stv1probe,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mta10stprobeset.db,mta10sttranscriptcluster.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,mwgcod.db,nugohs1a520180.db,nugohs1a520180cdf,nugohs1a520180probe,nugomm1a520177.db,nugomm1a520177cdf,nugomm1a520177probe,paeg1acdf,paeg1aprobe,plasmodiumanophelescdf,plasmodiumanophelesprobe,poplarcdf,poplarprobe,porcine.db,porcinecdf,porcineprobe,primeviewcdf,primeviewprobe,r10kcod.db,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,raex10stprobeset.db,raex10sttranscriptcluster.db,RaExExonProbesetLocation,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene10stv1cdf,ragene10stv1probe,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat2302.db,rat2302cdf,rat2302probe,rattoxfxcdf,rattoxfxprobe,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rgug4105a.db,rgug4130a.db,rgug4131a.db,rhesuscdf,rhesusprobe,ri16cod.db,ricecdf,riceprobe,RnAgilentDesign028282.db,rnu34.db,rnu34cdf,rnu34probe,rtu34.db,rtu34cdf,rtu34probe,rwgcod.db,saureuscdf,saureusprobe,soybeancdf,soybeanprobe,sugarcanecdf,sugarcaneprobe,test1cdf,test2cdf,test3cdf,test3probe,tomatocdf,tomatoprobe,u133aaofav2cdf,u133x3p.db,u133x3pcdf,u133x3pprobe,vitisviniferacdf,vitisviniferaprobe,wheatcdf,wheatprobe,xenopuslaeviscdf,xenopuslaevisprobe,xlaevis.db,xlaevis2cdf,xlaevis2probe,xtropicaliscdf,xtropicalisprobe,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe,zebrafish.db,zebrafishcdf,zebrafishprobe" id="ChipManufacturer" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ChipManufacturer (388)</a><ul><li packagelist="ag.db,agcdf,agprobe,ath1121501.db,ath1121501cdf,ath1121501probe,barley1cdf,barley1probe,bovine.db,bovinecdf,bovineprobe,bsubtiliscdf,bsubtilisprobe,canine.db,canine2.db,canine2cdf,canine2probe,caninecdf,canineprobe,celegans.db,celeganscdf,celegansprobe,chicken.db,chickencdf,chickenprobe,citruscdf,citrusprobe,cottoncdf,cottonprobe,cyp450cdf,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,ecoli2.db,ecoli2cdf,ecoli2probe,ecoliasv2cdf,ecoliasv2probe,ecolicdf,ecoliprobe,gahgu133acdf,gahgu133aprobe,gahgu133bcdf,gahgu133bprobe,gahgu133plus2cdf,gahgu133plus2probe,gahgu95av2cdf,gahgu95av2probe,gahgu95bcdf,gahgu95bprobe,gahgu95ccdf,gahgu95cprobe,gahgu95dcdf,gahgu95dprobe,gahgu95ecdf,gahgu95eprobe,gp53cdf,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133atagcdf,hgu133atagprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu219.db,hgu219cdf,hgu219probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hivprtplus2cdf,Hspec,hspeccdf,hta20stprobeset.db,hta20sttranscriptcluster.db,hthgu133a.db,hthgu133acdf,hthgu133afrmavecs,hthgu133aprobe,hthgu133b.db,hthgu133bcdf,hthgu133bprobe,hthgu133pluspmcdf,hthgu133pluspmprobe,htmg430acdf,htmg430aprobe,htmg430bcdf,htmg430bprobe,htmg430pmcdf,htmg430pmprobe,htrat230pmcdf,htrat230pmprobe,htratfocuscdf,htratfocusprobe,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,huex.1.0.st.v2frmavecs,huex10stprobeset.db,huex10sttranscriptcluster.db,HuExExonProbesetLocation,HuExExonProbesetLocationHg18,HuExExonProbesetLocationHg19,hugene.1.0.st.v1frmavecs,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene10stv1cdf,hugene10stv1probe,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,maizecdf,maizeprobe,medicagocdf,medicagoprobe,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mirna102xgaincdf,mirna10cdf,mirna10probe,mirna20cdf,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,moex10stprobeset.db,moex10sttranscriptcluster.db,MoExExonProbesetLocation,mogene.1.0.st.v1frmavecs,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene10stv1cdf,mogene10stv1probe,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mta10stprobeset.db,mta10sttranscriptcluster.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,nugohs1a520180.db,nugohs1a520180cdf,nugohs1a520180probe,nugomm1a520177.db,nugomm1a520177cdf,nugomm1a520177probe,paeg1acdf,paeg1aprobe,plasmodiumanophelescdf,plasmodiumanophelesprobe,poplarcdf,poplarprobe,porcine.db,porcinecdf,porcineprobe,primeviewcdf,primeviewprobe,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,raex10stprobeset.db,raex10sttranscriptcluster.db,RaExExonProbesetLocation,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene10stv1cdf,ragene10stv1probe,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat2302.db,rat2302cdf,rat2302probe,rattoxfxcdf,rattoxfxprobe,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rhesuscdf,rhesusprobe,ricecdf,riceprobe,rnu34.db,rnu34cdf,rnu34probe,rtu34.db,rtu34cdf,rtu34probe,saureuscdf,saureusprobe,soybeancdf,soybeanprobe,sugarcanecdf,sugarcaneprobe,test1cdf,test2cdf,test3cdf,test3probe,tomatocdf,tomatoprobe,u133aaofav2cdf,u133x3p.db,u133x3pcdf,u133x3pprobe,vitisviniferacdf,vitisviniferaprobe,wheatcdf,wheatprobe,xenopuslaeviscdf,xenopuslaevisprobe,xlaevis.db,xlaevis2cdf,xlaevis2probe,xtropicaliscdf,xtropicalisprobe,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe,zebrafish.db,zebrafishcdf,zebrafishprobe" id="AffymetrixChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>AffymetrixChip (338)</a></li><li packagelist="hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,HsAgilentDesign026652.db,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,MmAgilentDesign026655.db,rgug4105a.db,rgug4130a.db,rgug4131a.db,RnAgilentDesign028282.db" id="AgilentChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>AgilentChip (15)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ClonetechChip</a></li><li packagelist="adme16cod.db,h10kcod.db,h20kcod.db,hi16cod.db,hwgcod.db,m10kcod.db,m20kcod.db,mi16cod.db,mwgcod.db,r10kcod.db,ri16cod.db,rwgcod.db" id="CodelinkChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CodelinkChip (12)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GEChip</a></li><li packagelist="GGHumanMethCancerPanelv1.db,IlluminaHumanMethylation27k.db,IlluminaHumanMethylation27kmanifest,IlluminaHumanMethylation450k.db,IlluminaHumanMethylation450kmanifest,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv2BeadID.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,illuminaRatv1.db,lumiHumanAll.db,lumiMouseAll.db,lumiRatAll.db" id="IlluminaChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>IlluminaChip (19)</a></li><li packagelist="indac.db" id="INDACChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>INDACChip (1)</a></li><li packagelist="hguqiagenv3.db" id="QiagenChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>QiagenChip (1)</a></li><li packagelist="hs25kresogen.db,mm24kresogen.db" id="RNG_MRCChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>RNG_MRCChip (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>RocheChip</a></li><li class="jstree-last jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>UniversityHealthNetwork</a></li></ul></li><li packagelist="adme16cod.db,ag.db,agcdf,agprobe,ath1121501.db,ath1121501cdf,ath1121501probe,celegans.db,celeganscdf,celegansprobe,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,h10kcod.db,h20kcod.db,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hguatlas13k.db,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hguqiagenv3.db,hi16cod.db,hs25kresogen.db,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,HuO22.db,hwgcod.db,illuminaHumanv1.db,illuminaHumanv2.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaRatv1.db,indac.db,m10kcod.db,m20kcod.db,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mguatlas5k.db,mgug4121a.db,mgug4122a.db,mi16cod.db,mm24kresogen.db,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mpedbarray.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,Mu15v1.db,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,Mu22v3.db,mwgcod.db,Norway981.db,OperonHumanV3.db,PartheenMetaData.db,pedbarrayv10.db,pedbarrayv9.db,r10kcod.db,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,rat2302.db,rat2302cdf,rat2302probe,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rgug4130a.db,ri16cod.db,rnu34.db,rnu34cdf,rnu34probe,Roberts2005Annotation.db,rtu34.db,rtu34cdf,rtu34probe,rwgcod.db,SHDZ.db,u133x3p.db,u133x3pcdf,u133x3pprobe,xenopuslaeviscdf,xenopuslaevisprobe,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe,zebrafish.db,zebrafishcdf,zebrafishprobe" id="ChipName" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ChipName (195)</a><ul><li packagelist="adme16cod.db" id="adme16cod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>adme16cod (1)</a></li><li packagelist="ag.db,agcdf,agprobe" id="ag" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ag (3)</a></li><li packagelist="ath1121501.db,ath1121501cdf,ath1121501probe" id="ath1121501" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ath1121501 (3)</a></li><li packagelist="celegans.db,celeganscdf,celegansprobe" id="celegans" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>celegans (3)</a></li><li packagelist="drosgenome1.db,drosgenome1cdf,drosgenome1probe" id="drosgenome1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>drosgenome1 (3)</a></li><li packagelist="drosophila2.db,drosophila2cdf,drosophila2probe" id="drosophila2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>drosophila2 (3)</a></li><li packagelist="h10kcod.db" id="h10kcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>h10kcod (1)</a></li><li packagelist="h20kcod.db" id="h20kcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>h20kcod (1)</a></li><li packagelist="hcg110.db,hcg110cdf,hcg110probe" id="hcg110" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hcg110 (3)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hcgi12k</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hcgi8k</a></li><li packagelist="hgfocus.db,hgfocuscdf,hgfocusprobe" id="hgfocus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgfocus (3)</a></li><li packagelist="hgu133a.db,hgu133acdf,hgu133afrmavecs,hgu133aprobe" id="hgu133a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgu133a (4)</a></li><li packagelist="hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe" id="hgu133a2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgu133a2 (4)</a></li><li packagelist="hgu133b.db,hgu133bcdf,hgu133bprobe" id="hgu133b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgu133b (3)</a></li><li packagelist="hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe" id="hgu133plus2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgu133plus2 (4)</a></li><li packagelist="hgu95a.db,hgu95acdf,hgu95aprobe" id="hgu95a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgu95a (3)</a></li><li packagelist="hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe" id="hgu95av2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgu95av2 (4)</a></li><li packagelist="hgu95b.db,hgu95bcdf,hgu95bprobe" id="hgu95b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgu95b (3)</a></li><li packagelist="hgu95c.db,hgu95ccdf,hgu95cprobe" id="hgu95c" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgu95c (3)</a></li><li packagelist="hgu95d.db,hgu95dcdf,hgu95dprobe" id="hgu95d" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgu95d (3)</a></li><li packagelist="hgu95e.db,hgu95ecdf,hgu95eprobe" id="hgu95e" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgu95e (3)</a></li><li packagelist="hguatlas13k.db" id="hguatlas13k" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hguatlas13k (1)</a></li><li packagelist="hgug4100a.db" id="hgug4100a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgug4100a (1)</a></li><li packagelist="hgug4101a.db" id="hgug4101a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgug4101a (1)</a></li><li packagelist="hgug4110b.db" id="hgug4110b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgug4110b (1)</a></li><li packagelist="hgug4111a.db" id="hgug4111a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgug4111a (1)</a></li><li packagelist="hgug4112a.db" id="hgug4112a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hgug4112a (1)</a></li><li packagelist="hguqiagenv3.db" id="hguqiagenv3" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hguqiagenv3 (1)</a></li><li packagelist="hi16cod.db" id="hi16cod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hi16cod (1)</a></li><li packagelist="hs25kresogen.db" id="hs25kresogen" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hs25kresogen (1)</a></li><li packagelist="hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe" id="hu35ksuba" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hu35ksuba (3)</a></li><li packagelist="hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe" id="hu35ksubb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hu35ksubb (3)</a></li><li packagelist="hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe" id="hu35ksubc" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hu35ksubc (3)</a></li><li packagelist="hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe" id="hu35ksubd" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hu35ksubd (3)</a></li><li packagelist="hu6800.db,hu6800cdf,hu6800probe" id="hu6800" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hu6800 (3)</a></li><li packagelist="HuO22.db" id="HuO22" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>HuO22 (1)</a></li><li packagelist="hwgcod.db" id="hwgcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>hwgcod (1)</a></li><li packagelist="illuminaHumanv1.db" id="illuminaHumanv1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>illuminaHumanv1 (1)</a></li><li packagelist="illuminaHumanv2.db" id="illuminaHumanv2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>illuminaHumanv2 (1)</a></li><li packagelist="illuminaMousev1.db" id="illuminaMousev1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>illuminaMousev1 (1)</a></li><li packagelist="illuminaMousev1p1.db" id="illuminaMousev1p1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>illuminaMousev1p1 (1)</a></li><li packagelist="illuminaRatv1.db" id="illuminaRatv1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>illuminaRatv1 (1)</a></li><li packagelist="indac.db" id="indac" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>indac (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>JazaerimetaData</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>lumiHumanV1</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>lumiHumanV2</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>lumiMouseV1</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>lumiRatV1</a></li><li packagelist="m10kcod.db" id="m10kcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>m10kcod (1)</a></li><li packagelist="m20kcod.db" id="m20kcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>m20kcod (1)</a></li><li packagelist="mgu74a.db,mgu74acdf,mgu74aprobe" id="mgu74a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mgu74a (3)</a></li><li packagelist="mgu74av2.db,mgu74av2cdf,mgu74av2probe" id="mgu74av2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mgu74av2 (3)</a></li><li packagelist="mgu74b.db,mgu74bcdf,mgu74bprobe" id="mgu74b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mgu74b (3)</a></li><li packagelist="mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe" id="mgu74bv2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mgu74bv2 (3)</a></li><li packagelist="mgu74c.db,mgu74ccdf,mgu74cprobe" id="mgu74c" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mgu74c (3)</a></li><li packagelist="mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe" id="mgu74cv2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mgu74cv2 (3)</a></li><li packagelist="mguatlas5k.db" id="mguatlas5k" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mguatlas5k (1)</a></li><li packagelist="mgug4121a.db" id="mgug4121a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mgug4121a (1)</a></li><li packagelist="mgug4122a.db" id="mgug4122a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mgug4122a (1)</a></li><li packagelist="mi16cod.db" id="mi16cod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mi16cod (1)</a></li><li packagelist="mm24kresogen.db" id="mm24kresogen" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mm24kresogen (1)</a></li><li packagelist="moe430a.db,moe430acdf,moe430aprobe" id="moe430a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>moe430a (3)</a></li><li packagelist="moe430b.db,moe430bcdf,moe430bprobe" id="moe430b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>moe430b (3)</a></li><li packagelist="mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe" id="mouse4302" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mouse4302 (4)</a></li><li packagelist="mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe" id="mouse430a2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mouse430a2 (4)</a></li><li packagelist="mpedbarray.db" id="mpedbarray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mpedbarray (1)</a></li><li packagelist="mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe" id="mu11ksuba" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mu11ksuba (3)</a></li><li packagelist="mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe" id="mu11ksubb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mu11ksubb (3)</a></li><li packagelist="Mu15v1.db" id="Mu15v1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Mu15v1 (1)</a></li><li packagelist="mu19ksuba.db,mu19ksubacdf" id="mu19ksuba" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mu19ksuba (2)</a></li><li packagelist="mu19ksubb.db,mu19ksubbcdf" id="mu19ksubb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mu19ksubb (2)</a></li><li packagelist="mu19ksubc.db,mu19ksubccdf" id="mu19ksubc" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mu19ksubc (2)</a></li><li packagelist="Mu22v3.db" id="Mu22v3" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Mu22v3 (1)</a></li><li packagelist="mwgcod.db" id="mwgcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mwgcod (1)</a></li><li packagelist="Norway981.db" id="Norway981" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Norway981 (1)</a></li><li packagelist="OperonHumanV3.db" id="OperonHumanV3" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>OperonHumanV3 (1)</a></li><li packagelist="PartheenMetaData.db" id="PartheenMetaData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>PartheenMetaData (1)</a></li><li packagelist="pedbarrayv10.db" id="pedbarrayv10" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>pedbarrayv10 (1)</a></li><li packagelist="pedbarrayv9.db" id="pedbarrayv9" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>pedbarrayv9 (1)</a></li><li packagelist="r10kcod.db" id="r10kcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>r10kcod (1)</a></li><li packagelist="rae230a.db,rae230acdf,rae230aprobe" id="rae230a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>rae230a (3)</a></li><li packagelist="rae230b.db,rae230bcdf,rae230bprobe" id="rae230b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>rae230b (3)</a></li><li packagelist="rat2302.db,rat2302cdf,rat2302probe" id="rat2302" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>rat2302 (3)</a></li><li packagelist="rgu34a.db,rgu34acdf,rgu34aprobe" id="rgu34a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>rgu34a (3)</a></li><li packagelist="rgu34b.db,rgu34bcdf,rgu34bprobe" id="rgu34b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>rgu34b (3)</a></li><li packagelist="rgu34c.db,rgu34ccdf,rgu34cprobe" id="rgu34c" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>rgu34c (3)</a></li><li packagelist="rgug4130a.db" id="rgug4130a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>rgug4130a (1)</a></li><li packagelist="ri16cod.db" id="ri16cod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ri16cod (1)</a></li><li packagelist="rnu34.db,rnu34cdf,rnu34probe" id="rnu34" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>rnu34 (3)</a></li><li packagelist="Roberts2005Annotation.db" id="Roberts2005Annotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Roberts2005Annotation (1)</a></li><li packagelist="rtu34.db,rtu34cdf,rtu34probe" id="rtu34" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>rtu34 (3)</a></li><li packagelist="rwgcod.db" id="rwgcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>rwgcod (1)</a></li><li packagelist="SHDZ.db" id="SHDZ" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SHDZ (1)</a></li><li packagelist="u133x3p.db,u133x3pcdf,u133x3pprobe" id="u133x3p" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>u133x3p (3)</a></li><li packagelist="xenopuslaeviscdf,xenopuslaevisprobe" id="xenopuslaevis" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>xenopuslaevis (2)</a></li><li packagelist="yeast2.db,yeast2cdf,yeast2probe" id="yeast2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>yeast2 (3)</a></li><li packagelist="ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe" id="ygs98" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ygs98 (4)</a></li><li packagelist="zebrafish.db,zebrafishcdf,zebrafishprobe" id="zebrafish" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>zebrafish (3)</a></li></ul></li><li packagelist="hgubeta7.db,hguDKFZ31.db" id="CustomArray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CustomArray (2)</a></li><li packagelist="gahgu133acdf,gahgu133aprobe,gahgu133bcdf,gahgu133bprobe,gahgu133plus2cdf,gahgu133plus2probe,gahgu95av2cdf,gahgu95av2probe,gahgu95bcdf,gahgu95bprobe,gahgu95ccdf,gahgu95cprobe,gahgu95dcdf,gahgu95dprobe,gahgu95ecdf,gahgu95eprobe" id="CustomCDF" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CustomCDF (16)</a><ul><li packagelist="gahgu133acdf,gahgu133aprobe,gahgu133bcdf,gahgu133bprobe,gahgu133plus2cdf,gahgu133plus2probe,gahgu95av2cdf,gahgu95av2probe,gahgu95bcdf,gahgu95bprobe,gahgu95ccdf,gahgu95cprobe,gahgu95dcdf,gahgu95dprobe,gahgu95ecdf,gahgu95eprobe" id="GACustomCDF" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GACustomCDF (16)</a></li><li class="jstree-last jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MBNICustomCDF</a></li></ul></li><li packagelist="ChemmineDrugs,RmiR.Hs.miRNA,RmiR.hsa" id="CustomDBSchema" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CustomDBSchema (11)</a><ul><li packagelist="gahgu133a.db,gahgu133b.db,gahgu133plus2.db,gahgu95av2.db,gahgu95b.db,gahgu95c.db,gahgu95d.db,gahgu95e.db" id="GeneCardsCustomSchema" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GeneCardsCustomSchema (8)</a></li></ul></li><li packagelist="DO.db,GO.db,humanCHRLOC,KEGG.db,LowMACAAnnotation,MeSH.AOR.db,MeSH.db,MeSH.PCR.db,mirbase.db,mouseCHRLOC,ratCHRLOC,reactome.db,targetscan.Hs.eg.db,targetscan.Mm.eg.db" id="FunctionalAnnotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>FunctionalAnnotation (14)</a></li><li packagelist="adme16cod.db,ag.db,agcdf,agprobe,anopheles.db0,arabidopsis.db0,ath1121501.db,ath1121501cdf,ath1121501probe,barley1cdf,barley1probe,bovine.db,bovine.db0,bovinecdf,bovineprobe,BSgenome.Amellifera.BeeBase.assembly4,BSgenome.Amellifera.UCSC.apiMel2,BSgenome.Amellifera.UCSC.apiMel2.masked,BSgenome.Athaliana.TAIR.04232008,BSgenome.Athaliana.TAIR.TAIR9,BSgenome.Btaurus.UCSC.bosTau3,BSgenome.Btaurus.UCSC.bosTau3.masked,BSgenome.Btaurus.UCSC.bosTau4,BSgenome.Btaurus.UCSC.bosTau4.masked,BSgenome.Btaurus.UCSC.bosTau6,BSgenome.Btaurus.UCSC.bosTau6.masked,BSgenome.Btaurus.UCSC.bosTau8,BSgenome.Celegans.UCSC.ce10,BSgenome.Celegans.UCSC.ce2,BSgenome.Celegans.UCSC.ce6,BSgenome.Cfamiliaris.UCSC.canFam2,BSgenome.Cfamiliaris.UCSC.canFam2.masked,BSgenome.Cfamiliaris.UCSC.canFam3,BSgenome.Cfamiliaris.UCSC.canFam3.masked,BSgenome.Dmelanogaster.UCSC.dm2,BSgenome.Dmelanogaster.UCSC.dm2.masked,BSgenome.Dmelanogaster.UCSC.dm3,BSgenome.Dmelanogaster.UCSC.dm3.masked,BSgenome.Dmelanogaster.UCSC.dm6,BSgenome.Drerio.UCSC.danRer5,BSgenome.Drerio.UCSC.danRer5.masked,BSgenome.Drerio.UCSC.danRer6,BSgenome.Drerio.UCSC.danRer6.masked,BSgenome.Drerio.UCSC.danRer7,BSgenome.Drerio.UCSC.danRer7.masked,BSgenome.Ecoli.NCBI.20080805,BSgenome.Gaculeatus.UCSC.gasAcu1,BSgenome.Gaculeatus.UCSC.gasAcu1.masked,BSgenome.Ggallus.UCSC.galGal3,BSgenome.Ggallus.UCSC.galGal3.masked,BSgenome.Ggallus.UCSC.galGal4,BSgenome.Ggallus.UCSC.galGal4.masked,BSgenome.Hsapiens.NCBI.GRCh38,BSgenome.Hsapiens.UCSC.hg17,BSgenome.Hsapiens.UCSC.hg17.masked,BSgenome.Hsapiens.UCSC.hg18,BSgenome.Hsapiens.UCSC.hg18.masked,BSgenome.Hsapiens.UCSC.hg19,BSgenome.Hsapiens.UCSC.hg19.masked,BSgenome.Hsapiens.UCSC.hg38,BSgenome.Hsapiens.UCSC.hg38.masked,BSgenome.Mfascicularis.NCBI.5.0,BSgenome.Mmulatta.UCSC.rheMac2,BSgenome.Mmulatta.UCSC.rheMac2.masked,BSgenome.Mmulatta.UCSC.rheMac3,BSgenome.Mmulatta.UCSC.rheMac3.masked,BSgenome.Mmusculus.UCSC.mm10,BSgenome.Mmusculus.UCSC.mm10.masked,BSgenome.Mmusculus.UCSC.mm8,BSgenome.Mmusculus.UCSC.mm8.masked,BSgenome.Mmusculus.UCSC.mm9,BSgenome.Mmusculus.UCSC.mm9.masked,BSgenome.Osativa.MSU.MSU7,BSgenome.Ptroglodytes.UCSC.panTro2,BSgenome.Ptroglodytes.UCSC.panTro2.masked,BSgenome.Ptroglodytes.UCSC.panTro3,BSgenome.Ptroglodytes.UCSC.panTro3.masked,BSgenome.Rnorvegicus.UCSC.rn4,BSgenome.Rnorvegicus.UCSC.rn4.masked,BSgenome.Rnorvegicus.UCSC.rn5,BSgenome.Rnorvegicus.UCSC.rn5.masked,BSgenome.Rnorvegicus.UCSC.rn6,BSgenome.Scerevisiae.UCSC.sacCer1,BSgenome.Scerevisiae.UCSC.sacCer2,BSgenome.Scerevisiae.UCSC.sacCer3,BSgenome.Sscrofa.UCSC.susScr3,BSgenome.Sscrofa.UCSC.susScr3.masked,BSgenome.Tguttata.UCSC.taeGut1,BSgenome.Tguttata.UCSC.taeGut1.masked,bsubtiliscdf,bsubtilisprobe,canine.db,canine.db0,canine2.db,canine2cdf,canine2probe,caninecdf,canineprobe,celegans.db,celeganscdf,celegansprobe,chicken.db,chicken.db0,chickencdf,chickenprobe,chimp.db0,cyp450cdf,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,ecoli2.db,ecoli2cdf,ecoli2probe,ecoliasv2cdf,ecoliasv2probe,ecolicdf,ecoliK12.db0,ecoliprobe,ecoliSakai.db0,EnsDb.Hsapiens.v75,EnsDb.Hsapiens.v79,EnsDb.Mmusculus.v75,EnsDb.Mmusculus.v79,EnsDb.Rnorvegicus.v75,EnsDb.Rnorvegicus.v79,FDb.FANTOM4.promoters.hg19,FDb.InfiniumMethylation.hg18,FDb.InfiniumMethylation.hg19,FDb.UCSC.snp135common.hg19,FDb.UCSC.snp137common.hg19,FDb.UCSC.tRNAs,fly.db0,gahgu133aprobe,gahgu133bprobe,gahgu133plus2probe,gahgu95av2probe,gahgu95bprobe,gahgu95cprobe,gahgu95dprobe,gahgu95eprobe,GGHumanMethCancerPanelv1.db,gp53cdf,h10kcod.db,h20kcod.db,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133atagcdf,hgu133atagprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu219.db,hgu219cdf,hgu219probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hguatlas13k.db,hgubeta7.db,hguDKFZ31.db,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hgug4845a.db,hguqiagenv3.db,hi16cod.db,hivprtplus2cdf,hom.At.inp.db,hom.Ce.inp.db,hom.Dm.inp.db,hom.Dr.inp.db,hom.Hs.inp.db,hom.Mm.inp.db,hom.Rn.inp.db,hom.Sc.inp.db,Homo.sapiens,hs25kresogen.db,Hs6UG171.db,HsAgilentDesign026652.db,Hspec,hspeccdf,hta20stprobeset.db,hta20sttranscriptcluster.db,hthgu133a.db,hthgu133acdf,hthgu133afrmavecs,hthgu133aprobe,hthgu133b.db,hthgu133bcdf,hthgu133bprobe,hthgu133pluspmcdf,hthgu133pluspmprobe,htmg430acdf,htmg430aprobe,htmg430bcdf,htmg430bprobe,htmg430pmcdf,htmg430pmprobe,htrat230pmcdf,htrat230pmprobe,htratfocuscdf,htratfocusprobe,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,huex.1.0.st.v2frmavecs,huex10stprobeset.db,huex10sttranscriptcluster.db,HuExExonProbesetLocation,HuExExonProbesetLocationHg18,HuExExonProbesetLocationHg19,hugene.1.0.st.v1frmavecs,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene10stv1cdf,hugene10stv1probe,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,human.db0,humanCHRLOC,HuO22.db,hwgcod.db,IlluminaHumanMethylation27k.db,IlluminaHumanMethylation27kmanifest,IlluminaHumanMethylation450k.db,IlluminaHumanMethylation450kmanifest,IlluminaHumanMethylation450kprobe,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv2BeadID.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,illuminaRatv1.db,indac.db,JazaeriMetaData.db,LAPOINTE.db,lumiHumanAll.db,lumiMouseAll.db,lumiRatAll.db,m10kcod.db,m20kcod.db,MafDb.ALL.wgs.phase1.release.v3.20101123,MafDb.ALL.wgs.phase3.release.v5a.20130502,MafDb.ESP6500SI.V2.SSA137,MafDb.ExAC.r0.3.sites,maizecdf,maizeprobe,malaria.db0,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mguatlas5k.db,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,mi16cod.db,mirna102xgaincdf,mirna10cdf,mirna10probe,mirna20cdf,mm24kresogen.db,MmAgilentDesign026655.db,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,moex10stprobeset.db,moex10sttranscriptcluster.db,mogene.1.0.st.v1frmavecs,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene10stv1cdf,mogene10stv1probe,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse.db0,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mouseCHRLOC,mpedbarray.db,mta10stprobeset.db,mta10sttranscriptcluster.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,Mu15v1.db,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,Mu22v3.db,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,Mus.musculus,mwgcod.db,Norway981.db,nugohs1a520180.db,nugohs1a520180cdf,nugohs1a520180probe,nugomm1a520177.db,nugomm1a520177cdf,nugomm1a520177probe,OperonHumanV3.db,org.Ag.eg.db,org.At.tair.db,org.Bt.eg.db,org.Ce.eg.db,org.Cf.eg.db,org.Dm.eg.db,org.Dr.eg.db,org.EcK12.eg.db,org.EcSakai.eg.db,org.Gg.eg.db,org.Hs.eg.db,org.Mm.eg.db,org.Mmu.eg.db,org.Pf.plasmo.db,org.Pt.eg.db,org.Rn.eg.db,org.Sc.sgd.db,org.Ss.eg.db,org.Xl.eg.db,paeg1acdf,paeg1aprobe,PartheenMetaData.db,pedbarrayv10.db,pedbarrayv9.db,phastCons100way.UCSC.hg19,phastCons7way.UCSC.hg38,pig.db0,plasmodiumanophelescdf,plasmodiumanophelescdf,plasmodiumanophelesprobe,plasmodiumanophelesprobe,POCRCannotation.db,PolyPhen.Hsapiens.dbSNP131,porcine.db,porcinecdf,porcineprobe,primeviewcdf,primeviewprobe,r10kcod.db,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,raex10stprobeset.db,raex10sttranscriptcluster.db,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene10stv1cdf,ragene10stv1probe,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat.db0,rat2302.db,rat2302cdf,rat2302probe,ratCHRLOC,rattoxfxcdf,rattoxfxprobe,Rattus.norvegicus,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rguatlas4k.db,rgug4105a.db,rgug4130a.db,rgug4131a.db,rhesuscdf,rhesusprobe,ri16cod.db,RmiR.Hs.miRNA,RmiR.hsa,RnAgilentDesign028282.db,rnu34.db,rnu34cdf,rnu34probe,Roberts2005Annotation.db,rtu34.db,rtu34cdf,rtu34probe,rwgcod.db,saureuscdf,saureusprobe,SHDZ.db,SIFT.Hsapiens.dbSNP132,SIFT.Hsapiens.dbSNP137,SNPlocs.Hsapiens.dbSNP.20090506,SNPlocs.Hsapiens.dbSNP.20100427,SNPlocs.Hsapiens.dbSNP.20101109,SNPlocs.Hsapiens.dbSNP.20110815,SNPlocs.Hsapiens.dbSNP.20111119,SNPlocs.Hsapiens.dbSNP.20120608,SNPlocs.Hsapiens.dbSNP141.GRCh38,SNPlocs.Hsapiens.dbSNP142.GRCh37,sugarcanecdf,sugarcaneprobe,test1cdf,test2cdf,test3cdf,test3probe,TxDb.Athaliana.BioMart.plantsmart22,TxDb.Athaliana.BioMart.plantsmart25,TxDb.Celegans.UCSC.ce6.ensGene,TxDb.Dmelanogaster.UCSC.dm3.ensGene,TxDb.Hsapiens.BioMart.igis,TxDb.Hsapiens.UCSC.hg18.knownGene,TxDb.Hsapiens.UCSC.hg19.knownGene,TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts,TxDb.Hsapiens.UCSC.hg38.knownGene,TxDb.Mmusculus.UCSC.mm10.ensGene,TxDb.Mmusculus.UCSC.mm10.knownGene,TxDb.Mmusculus.UCSC.mm9.knownGene,TxDb.Rnorvegicus.BioMart.igis,TxDb.Rnorvegicus.UCSC.rn4.ensGene,TxDb.Rnorvegicus.UCSC.rn5.refGene,TxDb.Scerevisiae.UCSC.sacCer2.sgdGene,TxDb.Scerevisiae.UCSC.sacCer3.sgdGene,u133aaofav2cdf,u133x3p.db,u133x3pcdf,u133x3pprobe,vitisviniferacdf,vitisviniferaprobe,worm.db0,xenopus.db0,xlaevis.db,XtraSNPlocs.Hsapiens.dbSNP141.GRCh38,xtropicalisprobe,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast.db0,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98probe,zebrafish.db,zebrafish.db0,zebrafishcdf,zebrafishprobe" id="Organism" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Organism (552)</a><ul><li packagelist="anopheles.db0,org.Ag.eg.db,plasmodiumanophelescdf,plasmodiumanophelesprobe" id="Anopheles_gambiae" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Anopheles_gambiae (4)</a></li><li packagelist="BSgenome.Amellifera.BeeBase.assembly4,BSgenome.Amellifera.UCSC.apiMel2,BSgenome.Amellifera.UCSC.apiMel2.masked" id="Apis_mellifera" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Apis_mellifera (3)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Arabidopsis_lyrata</a></li><li packagelist="ag.db,agcdf,agprobe,arabidopsis.db0,ath1121501.db,ath1121501cdf,ath1121501probe,BSgenome.Athaliana.TAIR.04232008,BSgenome.Athaliana.TAIR.TAIR9,hom.At.inp.db,org.At.tair.db,TxDb.Athaliana.BioMart.plantsmart22,TxDb.Athaliana.BioMart.plantsmart25" id="Arabidopsis_thaliana" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Arabidopsis_thaliana (13)</a></li><li packagelist="bsubtiliscdf,bsubtilisprobe" id="Bacillus_subtilis" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Bacillus_subtilis (2)</a></li><li packagelist="bovine.db,bovine.db0,bovinecdf,bovineprobe,BSgenome.Btaurus.UCSC.bosTau3,BSgenome.Btaurus.UCSC.bosTau3.masked,BSgenome.Btaurus.UCSC.bosTau4,BSgenome.Btaurus.UCSC.bosTau4.masked,BSgenome.Btaurus.UCSC.bosTau6,BSgenome.Btaurus.UCSC.bosTau6.masked,BSgenome.Btaurus.UCSC.bosTau8,org.Bt.eg.db" id="Bos_taurus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Bos_taurus (12)</a></li><li packagelist="BSgenome.Celegans.UCSC.ce10,BSgenome.Celegans.UCSC.ce2,BSgenome.Celegans.UCSC.ce6,celegans.db,celeganscdf,celegansprobe,hom.Ce.inp.db,org.Ce.eg.db,TxDb.Celegans.UCSC.ce6.ensGene,worm.db0" id="Caenorhabditis_elegans" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Caenorhabditis_elegans (10)</a></li><li packagelist="BSgenome.Cfamiliaris.UCSC.canFam2,BSgenome.Cfamiliaris.UCSC.canFam2.masked,BSgenome.Cfamiliaris.UCSC.canFam3,BSgenome.Cfamiliaris.UCSC.canFam3.masked,canine.db,canine.db0,canine2.db,canine2cdf,canine2probe,caninecdf,canineprobe,org.Cf.eg.db" id="Canis_familiaris" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Canis_familiaris (12)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Ciona_intestinalis</a></li><li packagelist="BSgenome.Drerio.UCSC.danRer5,BSgenome.Drerio.UCSC.danRer5.masked,BSgenome.Drerio.UCSC.danRer6,BSgenome.Drerio.UCSC.danRer6.masked,BSgenome.Drerio.UCSC.danRer7,BSgenome.Drerio.UCSC.danRer7.masked,hom.Dr.inp.db,org.Dr.eg.db,zebrafish.db,zebrafish.db0,zebrafishcdf,zebrafishprobe" id="Danio_rerio" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Danio_rerio (12)</a></li><li packagelist="BSgenome.Dmelanogaster.UCSC.dm2,BSgenome.Dmelanogaster.UCSC.dm2.masked,BSgenome.Dmelanogaster.UCSC.dm3,BSgenome.Dmelanogaster.UCSC.dm3.masked,BSgenome.Dmelanogaster.UCSC.dm6,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,fly.db0,hom.Dm.inp.db,indac.db,org.Dm.eg.db,TxDb.Dmelanogaster.UCSC.dm3.ensGene" id="Drosophila_melanogaster" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Drosophila_melanogaster (16)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Eremothecium_gossypii</a></li><li packagelist="BSgenome.Ecoli.NCBI.20080805,ecoli2.db,ecoli2cdf,ecoli2probe,ecoliasv2cdf,ecoliasv2probe,ecolicdf,ecoliK12.db0,ecoliprobe,ecoliSakai.db0,org.EcK12.eg.db,org.EcSakai.eg.db" id="Escherichia_coli" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Escherichia_coli (12)</a></li><li packagelist="BSgenome.Ggallus.UCSC.galGal3,BSgenome.Ggallus.UCSC.galGal3.masked,BSgenome.Ggallus.UCSC.galGal4,BSgenome.Ggallus.UCSC.galGal4.masked,chicken.db,chicken.db0,chickencdf,chickenprobe,org.Gg.eg.db" id="Gallus_gallus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Gallus_gallus (9)</a></li><li packagelist="BSgenome.Gaculeatus.UCSC.gasAcu1,BSgenome.Gaculeatus.UCSC.gasAcu1.masked" id="Gasterosteus_aculeatus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Gasterosteus_aculeatus (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Glycine_max</a></li><li packagelist="BSgenome.Hsapiens.NCBI.GRCh38,BSgenome.Hsapiens.UCSC.hg17,BSgenome.Hsapiens.UCSC.hg17.masked,BSgenome.Hsapiens.UCSC.hg18,BSgenome.Hsapiens.UCSC.hg18.masked,BSgenome.Hsapiens.UCSC.hg19,BSgenome.Hsapiens.UCSC.hg19.masked,BSgenome.Hsapiens.UCSC.hg38,BSgenome.Hsapiens.UCSC.hg38.masked,cyp450cdf,EnsDb.Hsapiens.v75,EnsDb.Hsapiens.v79,FDb.FANTOM4.promoters.hg19,FDb.InfiniumMethylation.hg18,FDb.InfiniumMethylation.hg19,FDb.UCSC.snp135common.hg19,FDb.UCSC.snp137common.hg19,FDb.UCSC.tRNAs,gahgu133aprobe,gahgu133bprobe,gahgu133plus2probe,gahgu95av2probe,gahgu95bprobe,gahgu95cprobe,gahgu95dprobe,gahgu95eprobe,GGHumanMethCancerPanelv1.db,gp53cdf,h10kcod.db,h20kcod.db,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133atagcdf,hgu133atagprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu219.db,hgu219cdf,hgu219probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hguatlas13k.db,hgubeta7.db,hguDKFZ31.db,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hgug4845a.db,hguqiagenv3.db,hi16cod.db,hivprtplus2cdf,hom.Hs.inp.db,Homo.sapiens,hs25kresogen.db,Hs6UG171.db,HsAgilentDesign026652.db,Hspec,hspeccdf,hta20stprobeset.db,hta20sttranscriptcluster.db,hthgu133a.db,hthgu133acdf,hthgu133afrmavecs,hthgu133aprobe,hthgu133b.db,hthgu133bcdf,hthgu133bprobe,hthgu133pluspmcdf,hthgu133pluspmprobe,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,huex.1.0.st.v2frmavecs,huex10stprobeset.db,huex10sttranscriptcluster.db,HuExExonProbesetLocation,HuExExonProbesetLocationHg18,HuExExonProbesetLocationHg19,hugene.1.0.st.v1frmavecs,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene10stv1cdf,hugene10stv1probe,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,human.db0,humanCHRLOC,HuO22.db,hwgcod.db,IlluminaHumanMethylation27k.db,IlluminaHumanMethylation27kmanifest,IlluminaHumanMethylation450k.db,IlluminaHumanMethylation450kmanifest,IlluminaHumanMethylation450kprobe,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv2BeadID.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,JazaeriMetaData.db,LAPOINTE.db,lumiHumanAll.db,MafDb.ALL.wgs.phase1.release.v3.20101123,MafDb.ALL.wgs.phase3.release.v5a.20130502,MafDb.ESP6500SI.V2.SSA137,MafDb.ExAC.r0.3.sites,mirna102xgaincdf,mirna10cdf,mirna10probe,mirna20cdf,mogene.1.0.st.v1frmavecs,Norway981.db,nugohs1a520180.db,nugohs1a520180cdf,nugohs1a520180probe,OperonHumanV3.db,org.Hs.eg.db,PartheenMetaData.db,pedbarrayv10.db,pedbarrayv9.db,phastCons100way.UCSC.hg19,phastCons7way.UCSC.hg38,POCRCannotation.db,PolyPhen.Hsapiens.dbSNP131,primeviewcdf,primeviewprobe,RmiR.Hs.miRNA,RmiR.hsa,Roberts2005Annotation.db,SHDZ.db,SIFT.Hsapiens.dbSNP132,SIFT.Hsapiens.dbSNP137,SNPlocs.Hsapiens.dbSNP.20090506,SNPlocs.Hsapiens.dbSNP.20100427,SNPlocs.Hsapiens.dbSNP.20101109,SNPlocs.Hsapiens.dbSNP.20110815,SNPlocs.Hsapiens.dbSNP.20111119,SNPlocs.Hsapiens.dbSNP.20120608,SNPlocs.Hsapiens.dbSNP141.GRCh38,SNPlocs.Hsapiens.dbSNP142.GRCh37,test1cdf,test2cdf,test3cdf,test3probe,TxDb.Hsapiens.BioMart.igis,TxDb.Hsapiens.UCSC.hg18.knownGene,TxDb.Hsapiens.UCSC.hg19.knownGene,TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts,TxDb.Hsapiens.UCSC.hg38.knownGene,u133aaofav2cdf,u133x3p.db,u133x3pcdf,u133x3pprobe,XtraSNPlocs.Hsapiens.dbSNP141.GRCh38" id="Homo_sapiens" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Homo_sapiens (212)</a></li><li packagelist="barley1cdf,barley1probe" id="Hordeum_vulgare" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Hordeum_vulgare (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Kluyveromyces_lactis</a></li><li packagelist="BSgenome.Mfascicularis.NCBI.5.0" id="Macaca_fascicularis" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Macaca_fascicularis (1)</a></li><li packagelist="BSgenome.Mmulatta.UCSC.rheMac2,BSgenome.Mmulatta.UCSC.rheMac2.masked,BSgenome.Mmulatta.UCSC.rheMac3,BSgenome.Mmulatta.UCSC.rheMac3.masked,org.Mmu.eg.db,rhesuscdf,rhesusprobe" id="Macaca_mulatta" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Macaca_mulatta (7)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Magnaporthe_grisea</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Medicago_truncatula</a></li><li packagelist="BSgenome.Mmusculus.UCSC.mm10,BSgenome.Mmusculus.UCSC.mm10.masked,BSgenome.Mmusculus.UCSC.mm8,BSgenome.Mmusculus.UCSC.mm8.masked,BSgenome.Mmusculus.UCSC.mm9,BSgenome.Mmusculus.UCSC.mm9.masked,EnsDb.Mmusculus.v75,EnsDb.Mmusculus.v79,hom.Mm.inp.db,htmg430acdf,htmg430aprobe,htmg430bcdf,htmg430bprobe,htmg430pmcdf,htmg430pmprobe,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,lumiMouseAll.db,m10kcod.db,m20kcod.db,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mguatlas5k.db,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,mi16cod.db,mm24kresogen.db,MmAgilentDesign026655.db,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,moex10stprobeset.db,moex10sttranscriptcluster.db,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene10stv1cdf,mogene10stv1probe,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse.db0,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mouseCHRLOC,mpedbarray.db,mta10stprobeset.db,mta10sttranscriptcluster.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,Mu15v1.db,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,Mu22v3.db,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,Mus.musculus,mwgcod.db,nugomm1a520177.db,nugomm1a520177cdf,nugomm1a520177probe,org.Mm.eg.db,TxDb.Mmusculus.UCSC.mm10.ensGene,TxDb.Mmusculus.UCSC.mm10.knownGene,TxDb.Mmusculus.UCSC.mm9.knownGene" id="Mus_musculus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Mus_musculus (105)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Neurospora_crassa</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Oncorhynchus_mykiss</a></li><li packagelist="BSgenome.Osativa.MSU.MSU7" id="Oryza_sativa" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Oryza_sativa (1)</a></li><li packagelist="BSgenome.Ptroglodytes.UCSC.panTro2,BSgenome.Ptroglodytes.UCSC.panTro2.masked,BSgenome.Ptroglodytes.UCSC.panTro3,BSgenome.Ptroglodytes.UCSC.panTro3.masked,chimp.db0,org.Pt.eg.db" id="Pan_troglodytes" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Pan_troglodytes (6)</a></li><li packagelist="malaria.db0,org.Pf.plasmo.db,plasmodiumanophelescdf,plasmodiumanophelesprobe" id="Plasmodium_falciparum" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Plasmodium_falciparum (4)</a></li><li packagelist="paeg1acdf,paeg1aprobe" id="Pseudomonas_aeruginosa" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Pseudomonas_aeruginosa (2)</a></li><li packagelist="adme16cod.db,BSgenome.Rnorvegicus.UCSC.rn4,BSgenome.Rnorvegicus.UCSC.rn4.masked,BSgenome.Rnorvegicus.UCSC.rn5,BSgenome.Rnorvegicus.UCSC.rn5.masked,BSgenome.Rnorvegicus.UCSC.rn6,EnsDb.Rnorvegicus.v75,EnsDb.Rnorvegicus.v79,hom.Rn.inp.db,htrat230pmcdf,htrat230pmprobe,htratfocuscdf,htratfocusprobe,illuminaRatv1.db,lumiRatAll.db,org.Rn.eg.db,r10kcod.db,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,raex10stprobeset.db,raex10sttranscriptcluster.db,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene10stv1cdf,ragene10stv1probe,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat.db0,rat2302.db,rat2302cdf,rat2302probe,ratCHRLOC,rattoxfxcdf,rattoxfxprobe,Rattus.norvegicus,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rguatlas4k.db,rgug4105a.db,rgug4130a.db,rgug4131a.db,ri16cod.db,RnAgilentDesign028282.db,rnu34.db,rnu34cdf,rnu34probe,rtu34.db,rtu34cdf,rtu34probe,rwgcod.db,TxDb.Rnorvegicus.BioMart.igis,TxDb.Rnorvegicus.UCSC.rn4.ensGene,TxDb.Rnorvegicus.UCSC.rn5.refGene" id="Rattus_norvegicus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Rattus_norvegicus (68)</a></li><li packagelist="BSgenome.Scerevisiae.UCSC.sacCer1,BSgenome.Scerevisiae.UCSC.sacCer2,BSgenome.Scerevisiae.UCSC.sacCer3,hom.Sc.inp.db,org.Sc.sgd.db,TxDb.Scerevisiae.UCSC.sacCer2.sgdGene,TxDb.Scerevisiae.UCSC.sacCer3.sgdGene,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast.db0,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98probe" id="Saccharomyces_cerevisiae" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Saccharomyces_cerevisiae (18)</a></li><li packagelist="sugarcanecdf,sugarcaneprobe" id="Saccharum_officinarum" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Saccharum_officinarum (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Schizosaccharomyces_pombe</a></li><li packagelist="saureuscdf,saureusprobe" id="Staphylococcus_aureus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Staphylococcus_aureus (2)</a></li><li packagelist="BSgenome.Sscrofa.UCSC.susScr3,BSgenome.Sscrofa.UCSC.susScr3.masked,org.Ss.eg.db,pig.db0,porcine.db,porcinecdf,porcineprobe" id="Sus_scrofa" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Sus_scrofa (7)</a></li><li packagelist="BSgenome.Tguttata.UCSC.taeGut1,BSgenome.Tguttata.UCSC.taeGut1.masked" id="Taeniopygia_guttata" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Taeniopygia_guttata (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Triticum_aestivum</a></li><li packagelist="vitisviniferacdf,vitisviniferaprobe" id="Vitis_vinifera" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Vitis_vinifera (2)</a></li><li packagelist="org.Xl.eg.db,xenopus.db0,xlaevis.db" id="Xenopus_laevis" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Xenopus_laevis (3)</a></li><li packagelist="xtropicalisprobe" id="Xenopus_tropicalis" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Xenopus_tropicalis (1)</a></li><li packagelist="maizecdf,maizeprobe" id="Zea_mays" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Zea_mays (2)</a></li></ul></li><li packagelist="adme16cod.db,ag.db,agcdf,agprobe,anopheles.db0,arabidopsis.db0,ath1121501.db,ath1121501cdf,ath1121501probe,barley1cdf,barley1probe,bovine.db,bovine.db0,bovinecdf,bovineprobe,BSgenome.Amellifera.BeeBase.assembly4,BSgenome.Amellifera.UCSC.apiMel2,BSgenome.Amellifera.UCSC.apiMel2.masked,BSgenome.Athaliana.TAIR.04232008,BSgenome.Athaliana.TAIR.TAIR9,BSgenome.Btaurus.UCSC.bosTau3,BSgenome.Btaurus.UCSC.bosTau3.masked,BSgenome.Btaurus.UCSC.bosTau4,BSgenome.Btaurus.UCSC.bosTau4.masked,BSgenome.Btaurus.UCSC.bosTau6,BSgenome.Btaurus.UCSC.bosTau6.masked,BSgenome.Btaurus.UCSC.bosTau8,BSgenome.Celegans.UCSC.ce10,BSgenome.Celegans.UCSC.ce2,BSgenome.Celegans.UCSC.ce6,BSgenome.Cfamiliaris.UCSC.canFam2,BSgenome.Cfamiliaris.UCSC.canFam2.masked,BSgenome.Cfamiliaris.UCSC.canFam3,BSgenome.Cfamiliaris.UCSC.canFam3.masked,BSgenome.Dmelanogaster.UCSC.dm2,BSgenome.Dmelanogaster.UCSC.dm2.masked,BSgenome.Dmelanogaster.UCSC.dm3,BSgenome.Dmelanogaster.UCSC.dm3.masked,BSgenome.Dmelanogaster.UCSC.dm6,BSgenome.Drerio.UCSC.danRer5,BSgenome.Drerio.UCSC.danRer5.masked,BSgenome.Drerio.UCSC.danRer6,BSgenome.Drerio.UCSC.danRer6.masked,BSgenome.Drerio.UCSC.danRer7,BSgenome.Drerio.UCSC.danRer7.masked,BSgenome.Ecoli.NCBI.20080805,BSgenome.Gaculeatus.UCSC.gasAcu1,BSgenome.Gaculeatus.UCSC.gasAcu1.masked,BSgenome.Ggallus.UCSC.galGal3,BSgenome.Ggallus.UCSC.galGal3.masked,BSgenome.Ggallus.UCSC.galGal4,BSgenome.Ggallus.UCSC.galGal4.masked,BSgenome.Hsapiens.NCBI.GRCh38,BSgenome.Hsapiens.UCSC.hg17,BSgenome.Hsapiens.UCSC.hg17.masked,BSgenome.Hsapiens.UCSC.hg18,BSgenome.Hsapiens.UCSC.hg18.masked,BSgenome.Hsapiens.UCSC.hg19,BSgenome.Hsapiens.UCSC.hg19.masked,BSgenome.Hsapiens.UCSC.hg38,BSgenome.Hsapiens.UCSC.hg38.masked,BSgenome.Mfascicularis.NCBI.5.0,BSgenome.Mfuro.UCSC.musFur1,BSgenome.Mmulatta.UCSC.rheMac2,BSgenome.Mmulatta.UCSC.rheMac2.masked,BSgenome.Mmulatta.UCSC.rheMac3,BSgenome.Mmulatta.UCSC.rheMac3.masked,BSgenome.Mmusculus.UCSC.mm10,BSgenome.Mmusculus.UCSC.mm10.masked,BSgenome.Mmusculus.UCSC.mm8,BSgenome.Mmusculus.UCSC.mm8.masked,BSgenome.Mmusculus.UCSC.mm9,BSgenome.Mmusculus.UCSC.mm9.masked,BSgenome.Osativa.MSU.MSU7,BSgenome.Ptroglodytes.UCSC.panTro2,BSgenome.Ptroglodytes.UCSC.panTro2.masked,BSgenome.Ptroglodytes.UCSC.panTro3,BSgenome.Ptroglodytes.UCSC.panTro3.masked,BSgenome.Rnorvegicus.UCSC.rn4,BSgenome.Rnorvegicus.UCSC.rn4.masked,BSgenome.Rnorvegicus.UCSC.rn5,BSgenome.Rnorvegicus.UCSC.rn5.masked,BSgenome.Rnorvegicus.UCSC.rn6,BSgenome.Scerevisiae.UCSC.sacCer1,BSgenome.Scerevisiae.UCSC.sacCer2,BSgenome.Scerevisiae.UCSC.sacCer3,BSgenome.Sscrofa.UCSC.susScr3,BSgenome.Sscrofa.UCSC.susScr3.masked,BSgenome.Tguttata.UCSC.taeGut1,BSgenome.Tguttata.UCSC.taeGut1.masked,bsubtiliscdf,bsubtilisprobe,canine.db,canine.db0,canine2.db,canine2cdf,canine2probe,caninecdf,canineprobe,celegans.db,celeganscdf,celegansprobe,chicken.db,chicken.db0,chickencdf,chickenprobe,chimp.db0,citruscdf,citrusprobe,cottoncdf,cottonprobe,cyp450cdf,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,ecoli2.db,ecoli2cdf,ecoli2probe,ecoliasv2cdf,ecoliasv2probe,ecolicdf,ecoliK12.db0,ecoliprobe,ecoliSakai.db0,FDb.UCSC.tRNAs,fly.db0,gahgu133acdf,gahgu133aprobe,gahgu133bcdf,gahgu133bprobe,gahgu133plus2cdf,gahgu133plus2probe,gahgu95av2cdf,gahgu95av2probe,gahgu95bcdf,gahgu95bprobe,gahgu95ccdf,gahgu95cprobe,gahgu95dcdf,gahgu95dprobe,gahgu95ecdf,gahgu95eprobe,gp53cdf,h10kcod.db,h20kcod.db,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133atagcdf,hgu133atagprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu219.db,hgu219cdf,hgu219probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hguatlas13k.db,hgubeta7.db,hguDKFZ31.db,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hguqiagenv3.db,hi16cod.db,hivprtplus2cdf,hom.At.inp.db,hom.Ce.inp.db,hom.Dm.inp.db,hom.Dr.inp.db,hom.Hs.inp.db,hom.Mm.inp.db,hom.Rn.inp.db,hom.Sc.inp.db,Homo.sapiens,Hs6UG171.db,HsAgilentDesign026652.db,Hspec,hspeccdf,hta20stprobeset.db,hta20sttranscriptcluster.db,hthgu133a.db,hthgu133acdf,hthgu133afrmavecs,hthgu133aprobe,hthgu133b.db,hthgu133bcdf,hthgu133bprobe,hthgu133pluspmcdf,hthgu133pluspmprobe,htmg430acdf,htmg430aprobe,htmg430bcdf,htmg430bprobe,htmg430pmcdf,htmg430pmprobe,htrat230pmcdf,htrat230pmprobe,htratfocuscdf,htratfocusprobe,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,huex.1.0.st.v2frmavecs,huex10stprobeset.db,huex10sttranscriptcluster.db,HuExExonProbesetLocation,hugene.1.0.st.v1frmavecs,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene10stv1cdf,hugene10stv1probe,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,human.db0,HuO22.db,hwgcod.db,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,illuminaRatv1.db,indac.db,JazaeriMetaData.db,LAPOINTE.db,lumiHumanAll.db,lumiMouseAll.db,lumiRatAll.db,m10kcod.db,m20kcod.db,maizecdf,maizeprobe,malaria.db0,medicagocdf,medicagoprobe,MeSH.AOR.db,MeSH.db,MeSH.PCR.db,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mguatlas5k.db,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,mi16cod.db,mirna102xgaincdf,mirna10cdf,mirna10probe,mirna20cdf,MmAgilentDesign026655.db,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,moex10stprobeset.db,moex10sttranscriptcluster.db,MoExExonProbesetLocation,mogene.1.0.st.v1frmavecs,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene10stv1cdf,mogene10stv1probe,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse.db0,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mpedbarray.db,mta10stprobeset.db,mta10sttranscriptcluster.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,Mu15v1.db,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,Mu22v3.db,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,Mus.musculus,mwgcod.db,Norway981.db,nugohs1a520180.db,nugohs1a520180cdf,nugohs1a520180probe,nugomm1a520177.db,nugomm1a520177cdf,nugomm1a520177probe,OperonHumanV3.db,org.Ag.eg.db,org.At.tair.db,org.Bt.eg.db,org.Ce.eg.db,org.Cf.eg.db,org.Dm.eg.db,org.Dr.eg.db,org.EcK12.eg.db,org.EcSakai.eg.db,org.Gg.eg.db,org.Hs.eg.db,org.Mm.eg.db,org.Mmu.eg.db,org.Pf.plasmo.db,org.Pt.eg.db,org.Rn.eg.db,org.Sc.sgd.db,org.Ss.eg.db,org.Xl.eg.db,paeg1acdf,paeg1aprobe,PartheenMetaData.db,pedbarrayv10.db,pedbarrayv9.db,pig.db0,plasmodiumanophelescdf,plasmodiumanophelesprobe,POCRCannotation.db,poplarcdf,poplarprobe,porcine.db,porcinecdf,porcineprobe,primeviewcdf,primeviewprobe,r10kcod.db,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,raex10stprobeset.db,raex10sttranscriptcluster.db,RaExExonProbesetLocation,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene10stv1cdf,ragene10stv1probe,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat.db0,rat2302.db,rat2302cdf,rat2302probe,rattoxfxcdf,rattoxfxprobe,Rattus.norvegicus,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rguatlas4k.db,rgug4105a.db,rgug4130a.db,rgug4131a.db,rhesus.db0,rhesuscdf,rhesusprobe,ri16cod.db,ricecdf,riceprobe,RnAgilentDesign028282.db,rnu34.db,rnu34cdf,rnu34probe,Roberts2005Annotation.db,rtu34.db,rtu34cdf,rtu34probe,rwgcod.db,saureuscdf,saureusprobe,SHDZ.db,SNPlocs.Hsapiens.dbSNP141.GRCh38,soybeancdf,soybeanprobe,sugarcanecdf,sugarcaneprobe,test1cdf,test2cdf,test3cdf,test3probe,tomatocdf,tomatoprobe,TxDb.Athaliana.BioMart.plantsmart22,TxDb.Athaliana.BioMart.plantsmart25,TxDb.Celegans.UCSC.ce6.ensGene,TxDb.Dmelanogaster.UCSC.dm3.ensGene,TxDb.Hsapiens.BioMart.igis,TxDb.Hsapiens.UCSC.hg18.knownGene,TxDb.Hsapiens.UCSC.hg19.knownGene,TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts,TxDb.Hsapiens.UCSC.hg38.knownGene,TxDb.Mmusculus.UCSC.mm10.ensGene,TxDb.Mmusculus.UCSC.mm10.knownGene,TxDb.Mmusculus.UCSC.mm9.knownGene,TxDb.Rnorvegicus.BioMart.igis,TxDb.Rnorvegicus.UCSC.rn4.ensGene,TxDb.Rnorvegicus.UCSC.rn5.refGene,TxDb.Scerevisiae.UCSC.sacCer2.sgdGene,TxDb.Scerevisiae.UCSC.sacCer3.sgdGene,u133aaofav2cdf,u133x3p.db,u133x3pcdf,u133x3pprobe,vitisviniferacdf,vitisviniferaprobe,wheatcdf,wheatprobe,worm.db0,xenopus.db0,xenopuslaeviscdf,xenopuslaevisprobe,xlaevis.db,xlaevis2cdf,xlaevis2probe,XtraSNPlocs.Hsapiens.dbSNP141.GRCh38,xtropicaliscdf,xtropicalisprobe,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast.db0,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe,zebrafish.db,zebrafish.db0,zebrafishcdf,zebrafishprobe" id="PackageType" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>PackageType (544)</a><ul><li packagelist="BSgenome.Amellifera.BeeBase.assembly4,BSgenome.Amellifera.UCSC.apiMel2,BSgenome.Amellifera.UCSC.apiMel2.masked,BSgenome.Athaliana.TAIR.04232008,BSgenome.Athaliana.TAIR.TAIR9,BSgenome.Btaurus.UCSC.bosTau3,BSgenome.Btaurus.UCSC.bosTau3.masked,BSgenome.Btaurus.UCSC.bosTau4,BSgenome.Btaurus.UCSC.bosTau4.masked,BSgenome.Btaurus.UCSC.bosTau6,BSgenome.Btaurus.UCSC.bosTau6.masked,BSgenome.Btaurus.UCSC.bosTau8,BSgenome.Celegans.UCSC.ce10,BSgenome.Celegans.UCSC.ce2,BSgenome.Celegans.UCSC.ce6,BSgenome.Cfamiliaris.UCSC.canFam2,BSgenome.Cfamiliaris.UCSC.canFam2.masked,BSgenome.Cfamiliaris.UCSC.canFam3,BSgenome.Cfamiliaris.UCSC.canFam3.masked,BSgenome.Dmelanogaster.UCSC.dm2,BSgenome.Dmelanogaster.UCSC.dm2.masked,BSgenome.Dmelanogaster.UCSC.dm3,BSgenome.Dmelanogaster.UCSC.dm3.masked,BSgenome.Dmelanogaster.UCSC.dm6,BSgenome.Drerio.UCSC.danRer5,BSgenome.Drerio.UCSC.danRer5.masked,BSgenome.Drerio.UCSC.danRer6,BSgenome.Drerio.UCSC.danRer6.masked,BSgenome.Drerio.UCSC.danRer7,BSgenome.Drerio.UCSC.danRer7.masked,BSgenome.Ecoli.NCBI.20080805,BSgenome.Gaculeatus.UCSC.gasAcu1,BSgenome.Gaculeatus.UCSC.gasAcu1.masked,BSgenome.Ggallus.UCSC.galGal3,BSgenome.Ggallus.UCSC.galGal3.masked,BSgenome.Ggallus.UCSC.galGal4,BSgenome.Ggallus.UCSC.galGal4.masked,BSgenome.Hsapiens.NCBI.GRCh38,BSgenome.Hsapiens.UCSC.hg17,BSgenome.Hsapiens.UCSC.hg17.masked,BSgenome.Hsapiens.UCSC.hg18,BSgenome.Hsapiens.UCSC.hg18.masked,BSgenome.Hsapiens.UCSC.hg19,BSgenome.Hsapiens.UCSC.hg19.masked,BSgenome.Hsapiens.UCSC.hg38,BSgenome.Hsapiens.UCSC.hg38.masked,BSgenome.Mfascicularis.NCBI.5.0,BSgenome.Mfuro.UCSC.musFur1,BSgenome.Mmulatta.UCSC.rheMac2,BSgenome.Mmulatta.UCSC.rheMac2.masked,BSgenome.Mmulatta.UCSC.rheMac3,BSgenome.Mmulatta.UCSC.rheMac3.masked,BSgenome.Mmusculus.UCSC.mm10,BSgenome.Mmusculus.UCSC.mm10.masked,BSgenome.Mmusculus.UCSC.mm8,BSgenome.Mmusculus.UCSC.mm8.masked,BSgenome.Mmusculus.UCSC.mm9,BSgenome.Mmusculus.UCSC.mm9.masked,BSgenome.Osativa.MSU.MSU7,BSgenome.Ptroglodytes.UCSC.panTro2,BSgenome.Ptroglodytes.UCSC.panTro2.masked,BSgenome.Ptroglodytes.UCSC.panTro3,BSgenome.Ptroglodytes.UCSC.panTro3.masked,BSgenome.Rnorvegicus.UCSC.rn4,BSgenome.Rnorvegicus.UCSC.rn4.masked,BSgenome.Rnorvegicus.UCSC.rn5,BSgenome.Rnorvegicus.UCSC.rn5.masked,BSgenome.Rnorvegicus.UCSC.rn6,BSgenome.Scerevisiae.UCSC.sacCer1,BSgenome.Scerevisiae.UCSC.sacCer2,BSgenome.Scerevisiae.UCSC.sacCer3,BSgenome.Sscrofa.UCSC.susScr3,BSgenome.Sscrofa.UCSC.susScr3.masked,BSgenome.Tguttata.UCSC.taeGut1,BSgenome.Tguttata.UCSC.taeGut1.masked" id="BSgenome" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>BSgenome (75)</a></li><li packagelist="agcdf,ath1121501cdf,barley1cdf,bovinecdf,bsubtiliscdf,canine2cdf,caninecdf,celeganscdf,chickencdf,citruscdf,cottoncdf,cyp450cdf,drosgenome1cdf,drosophila2cdf,ecoli2cdf,ecoliasv2cdf,ecolicdf,gahgu133acdf,gahgu133bcdf,gahgu133plus2cdf,gahgu95av2cdf,gahgu95bcdf,gahgu95ccdf,gahgu95dcdf,gahgu95ecdf,gp53cdf,hcg110cdf,hgfocuscdf,hgu133a2cdf,hgu133acdf,hgu133atagcdf,hgu133bcdf,hgu133plus2cdf,hgu219cdf,hgu95acdf,hgu95av2cdf,hgu95bcdf,hgu95ccdf,hgu95dcdf,hgu95ecdf,hivprtplus2cdf,Hspec,hspeccdf,hthgu133acdf,hthgu133bcdf,hthgu133pluspmcdf,htmg430acdf,htmg430bcdf,htmg430pmcdf,htrat230pmcdf,htratfocuscdf,hu35ksubacdf,hu35ksubbcdf,hu35ksubccdf,hu35ksubdcdf,hu6800cdf,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,hugene10stv1cdf,maizecdf,medicagocdf,mgu74acdf,mgu74av2cdf,mgu74bcdf,mgu74bv2cdf,mgu74ccdf,mgu74cv2cdf,mirna102xgaincdf,mirna10cdf,mirna20cdf,moe430acdf,moe430bcdf,mogene10stv1cdf,mouse4302cdf,mouse430a2cdf,mu11ksubacdf,mu11ksubbcdf,mu19ksubacdf,mu19ksubbcdf,mu19ksubccdf,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,nugohs1a520180cdf,nugomm1a520177cdf,paeg1acdf,plasmodiumanophelescdf,poplarcdf,porcinecdf,primeviewcdf,rae230acdf,rae230bcdf,ragene10stv1cdf,rat2302cdf,rattoxfxcdf,rgu34acdf,rgu34bcdf,rgu34ccdf,rhesuscdf,ricecdf,rnu34cdf,rtu34cdf,saureuscdf,soybeancdf,sugarcanecdf,test1cdf,test2cdf,test3cdf,tomatocdf,u133aaofav2cdf,u133x3pcdf,vitisviniferacdf,wheatcdf,xenopuslaeviscdf,xlaevis2cdf,xtropicaliscdf,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast2cdf,ygs98cdf,zebrafishcdf" id="cdf" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>cdf (126)</a></li><li packagelist="adme16cod.db,ag.db,ath1121501.db,bovine.db,canine.db,canine2.db,celegans.db,chicken.db,drosgenome1.db,drosophila2.db,ecoli2.db,h10kcod.db,h20kcod.db,hcg110.db,hgfocus.db,hgu133a.db,hgu133a2.db,hgu133b.db,hgu133plus2.db,hgu219.db,hgu95a.db,hgu95av2.db,hgu95b.db,hgu95c.db,hgu95d.db,hgu95e.db,hguatlas13k.db,hgubeta7.db,hguDKFZ31.db,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hguqiagenv3.db,hi16cod.db,Hs6UG171.db,HsAgilentDesign026652.db,hta20stprobeset.db,hta20sttranscriptcluster.db,hthgu133a.db,hthgu133b.db,hu35ksuba.db,hu35ksubb.db,hu35ksubc.db,hu35ksubd.db,hu6800.db,huex10stprobeset.db,huex10sttranscriptcluster.db,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,HuO22.db,hwgcod.db,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,illuminaRatv1.db,indac.db,JazaeriMetaData.db,LAPOINTE.db,lumiHumanAll.db,lumiMouseAll.db,lumiRatAll.db,m10kcod.db,m20kcod.db,mgu74a.db,mgu74av2.db,mgu74b.db,mgu74bv2.db,mgu74c.db,mgu74cv2.db,mguatlas5k.db,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,mi16cod.db,MmAgilentDesign026655.db,moe430a.db,moe430b.db,moex10stprobeset.db,moex10sttranscriptcluster.db,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse4302.db,mouse430a2.db,mpedbarray.db,mta10stprobeset.db,mta10sttranscriptcluster.db,mu11ksuba.db,mu11ksubb.db,Mu15v1.db,mu19ksuba.db,mu19ksubb.db,mu19ksubc.db,Mu22v3.db,mwgcod.db,Norway981.db,nugohs1a520180.db,nugomm1a520177.db,OperonHumanV3.db,PartheenMetaData.db,pedbarrayv10.db,pedbarrayv9.db,POCRCannotation.db,porcine.db,r10kcod.db,rae230a.db,rae230b.db,raex10stprobeset.db,raex10sttranscriptcluster.db,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat2302.db,rgu34a.db,rgu34b.db,rgu34c.db,rguatlas4k.db,rgug4105a.db,rgug4130a.db,rgug4131a.db,ri16cod.db,RnAgilentDesign028282.db,rnu34.db,Roberts2005Annotation.db,rtu34.db,rwgcod.db,SHDZ.db,u133x3p.db,xlaevis.db,yeast2.db,ygs98.db,zebrafish.db" id="ChipDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ChipDb (157)</a></li><li packagelist="anopheles.db0,arabidopsis.db0,bovine.db0,canine.db0,chicken.db0,chimp.db0,ecoliK12.db0,ecoliSakai.db0,fly.db0,human.db0,malaria.db0,mouse.db0,pig.db0,rat.db0,rhesus.db0,worm.db0,xenopus.db0,yeast.db0,zebrafish.db0" id="db0" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>db0 (19)</a></li><li packagelist="hgu133a2frmavecs,hgu133afrmavecs,hgu133plus2frmavecs,hthgu133afrmavecs,huex.1.0.st.v2frmavecs,hugene.1.0.st.v1frmavecs,mogene.1.0.st.v1frmavecs,mouse4302frmavecs,mouse430a2frmavecs,ygs98frmavecs" id="FRMA" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>FRMA (10)</a></li><li packagelist="hom.At.inp.db,hom.Ce.inp.db,hom.Dm.inp.db,hom.Dr.inp.db,hom.Hs.inp.db,hom.Mm.inp.db,hom.Rn.inp.db,hom.Sc.inp.db" id="InparanoidDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>InparanoidDb (8)</a></li><li packagelist="MeSH.AOR.db,MeSH.db,MeSH.PCR.db" id="MeSHDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MeSHDb (3)</a></li><li packagelist="Homo.sapiens,Mus.musculus,Rattus.norvegicus" id="OrganismDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>OrganismDb (3)</a></li><li packagelist="org.Ag.eg.db,org.At.tair.db,org.Bt.eg.db,org.Ce.eg.db,org.Cf.eg.db,org.Dm.eg.db,org.Dr.eg.db,org.EcK12.eg.db,org.EcSakai.eg.db,org.Gg.eg.db,org.Hs.eg.db,org.Mm.eg.db,org.Mmu.eg.db,org.Pf.plasmo.db,org.Pt.eg.db,org.Rn.eg.db,org.Sc.sgd.db,org.Ss.eg.db,org.Xl.eg.db" id="OrgDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>OrgDb (19)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>PolyPhen</a></li><li packagelist="agprobe,ath1121501probe,barley1probe,bovineprobe,bsubtilisprobe,canine2probe,canineprobe,celegansprobe,chickenprobe,citrusprobe,cottonprobe,drosgenome1probe,drosophila2probe,ecoli2probe,ecoliasv2probe,ecoliprobe,gahgu133aprobe,gahgu133bprobe,gahgu133plus2probe,gahgu95av2probe,gahgu95bprobe,gahgu95cprobe,gahgu95dprobe,gahgu95eprobe,hcg110probe,hgfocusprobe,hgu133a2probe,hgu133aprobe,hgu133atagprobe,hgu133bprobe,hgu133plus2probe,hgu219probe,hgu95aprobe,hgu95av2probe,hgu95bprobe,hgu95cprobe,hgu95dprobe,hgu95eprobe,hthgu133aprobe,hthgu133bprobe,hthgu133pluspmprobe,htmg430aprobe,htmg430bprobe,htmg430pmprobe,htrat230pmprobe,htratfocusprobe,hu35ksubaprobe,hu35ksubbprobe,hu35ksubcprobe,hu35ksubdprobe,hu6800probe,HuExExonProbesetLocation,hugene10stv1probe,maizeprobe,medicagoprobe,mgu74aprobe,mgu74av2probe,mgu74bprobe,mgu74bv2probe,mgu74cprobe,mgu74cv2probe,mirna10probe,moe430aprobe,moe430bprobe,MoExExonProbesetLocation,mogene10stv1probe,mouse4302probe,mouse430a2probe,mu11ksubaprobe,mu11ksubbprobe,nugohs1a520180probe,nugomm1a520177probe,paeg1aprobe,plasmodiumanophelesprobe,poplarprobe,porcineprobe,primeviewprobe,rae230aprobe,rae230bprobe,RaExExonProbesetLocation,ragene10stv1probe,rat2302probe,rattoxfxprobe,rgu34aprobe,rgu34bprobe,rgu34cprobe,rhesusprobe,riceprobe,rnu34probe,rtu34probe,saureusprobe,soybeanprobe,sugarcaneprobe,test3probe,tomatoprobe,u133x3pprobe,vitisviniferaprobe,wheatprobe,xenopuslaevisprobe,xlaevis2probe,xtropicalisprobe,yeast2probe,ygs98probe,zebrafishprobe" id="probe" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>probe (104)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SIFT</a></li><li packagelist="SNPlocs.Hsapiens.dbSNP141.GRCh38" id="SNPlocs" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SNPlocs (1)</a></li><li packagelist="FDb.UCSC.tRNAs,TxDb.Athaliana.BioMart.plantsmart22,TxDb.Athaliana.BioMart.plantsmart25,TxDb.Celegans.UCSC.ce6.ensGene,TxDb.Dmelanogaster.UCSC.dm3.ensGene,TxDb.Hsapiens.BioMart.igis,TxDb.Hsapiens.UCSC.hg18.knownGene,TxDb.Hsapiens.UCSC.hg19.knownGene,TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts,TxDb.Hsapiens.UCSC.hg38.knownGene,TxDb.Mmusculus.UCSC.mm10.ensGene,TxDb.Mmusculus.UCSC.mm10.knownGene,TxDb.Mmusculus.UCSC.mm9.knownGene,TxDb.Rnorvegicus.BioMart.igis,TxDb.Rnorvegicus.UCSC.rn4.ensGene,TxDb.Rnorvegicus.UCSC.rn5.refGene,TxDb.Scerevisiae.UCSC.sacCer2.sgdGene,TxDb.Scerevisiae.UCSC.sacCer3.sgdGene" id="TxDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>TxDb (18)</a></li><li packagelist="XtraSNPlocs.Hsapiens.dbSNP141.GRCh38" id="XtraSNPlocs" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>XtraSNPlocs (1)</a></li></ul></li><li packagelist="miRNAtap.db,RmiR.Hs.miRNA,RmiR.hsa" id="SequenceAnnotation" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SequenceAnnotation (3)</a><ul><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GenomicSequence</a></li><li packagelist="miRNAtap.db,RmiR.Hs.miRNA,RmiR.hsa" id="miRNA" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>miRNA (3)</a></li><li class="jstree-last jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MotifAnnotation</a></li></ul></li></ul></li><li packagelist="affycompData,affydata,Affyhgu133A2Expr,Affyhgu133aExpr,Affyhgu133Plus2Expr,AffymetrixDataTestFiles,Affymoe4302Expr,airway,ALL,ALLMLL,AmpAffyExample,antiProfilesData,ARRmData,bcellViper,beadarrayExampleData,BeadArrayUseCases,beta7,bladderbatch,blimaTestingData,breastCancerMAINZ,breastCancerNKI,breastCancerTRANSBIG,breastCancerUNT,breastCancerUPP,breastCancerVDX,bronchialIL13,bsseqData,cancerdata,CardinalWorkflows,CCl4,ccTutorial,ceu1kg,ceu1kgv,ceuhm3,cgdv17,ChAMPdata,charmData,cheung2010,ChimpHumanBrainData,chipenrich.data,ChIPXpressData,CLL,cMap2data,cnvGSAdata,COHCAPanno,colonCA,ConnectivityMap,COPDSexualDimorphism.data,CopyhelpeR,CopyNumber450kData,COSMIC.67,CRCL18,curatedBladderData,curatedBreastData,curatedCRCData,curatedOvarianData,davidTiling,derfinderData,DeSousa2013,diggitdata,DLBCL,DmelSGI,DMRcatedata,DonaPLLP2013,DREAM4,dressCheck,DrugVsDiseasedata,dsQTL,DvDdata,dyebiasexamples,EatonEtAlChIPseq,ecoliLeucine,encoDnaseI,estrogen,faahKO,fabiaData,facopy.annot,facsDorit,FANTOM3and4CAGE,ffpeExampleData,fibroEset,fission,Fletcher2013a,Fletcher2013b,flowFitExampleData,FlowSorted.Blood.450k,FlowSorted.DLPFC.450k,flowWorkspaceData,frmaExampleData,FunciSNP.data,gageData,gaschYHS,gatingMLData,gcspikelite,geneLenDataBase,genomationData,geuvPack,geuvStore,GGdata,ggtut,golubEsets,grndata,GSBenchMark,gskb,GSVAdata,GWASdata,h5vcData,hapmap100khind,hapmap100kxba,hapmap500knsp,hapmap500ksty,hapmapsnp5,hapmapsnp6,harbChIP,HD2013SGI,healthyFlowData,HEEBOdata,hgu133abarcodevecs,hgu133plus2barcodevecs,hgu2beta7,HiCDataHumanIMR90,HiCDataLymphoblast,Hiiragi2013,HIVcDNAvantWout03,hmyriB36,HSMMSingleCell,humanStemCell,Illumina450ProbeVariants.db,IlluminaDataTestFiles,ind1KG,iontreeData,ITALICSData,Iyer517,JASPAR2014,KEGGandMetacoreDzPathwaysGEO,KEGGdzPathwaysGEO,kidpack,leeBamViews,leukemiasEset,LiebermanAidenHiC2009,ListerEtAlBSseq,lumiBarnes,LungCancerACvsSCCGEO,LungCancerLines,lungExpression,mammaPrintData,mAPKLData,maqcExpression4plex,MAQCsubset,MAQCsubsetAFX,MAQCsubsetILM,MEDIPSData,MEEBOdata,metaMSdata,MethylAidData,minfiData,miRNATarget,mitoODEdata,MMDiffBamSubset,mosaicsExample,mouse4302barcodevecs,MSBdata,msd16s,msdata,MUGAExampleData,Mulder2012,mvoutData,NCIgraphData,Neve2006,NGScopyData,parathyroidSE,pasilla,pasillaBamSubset,PathNetData,pcaGoPromoter.Hs.hg19,pcaGoPromoter.Mm.mm9,pcaGoPromoter.Rn.rn4,pd.atdschip.tiling,pepDat,ppiData,prebsdata,PREDAsampledata,ProData,pRolocdata,pumadata,PWMEnrich.Dmelanogaster.background,PWMEnrich.Hsapiens.background,PWMEnrich.Mmusculus.background,rcellminerData,RforProteomics,rheumaticConditionWOLLBOLD,RIPSeekerData,RMassBankData,RNAinteractMAPK,RNAseqData.HNRNPC.bam.chr14,RnaSeqSampleSizeData,RnaSeqTutorial,RnBeads.hg19,RnBeads.mm10,RnBeads.mm9,RnBeads.rn5,RRBSdata,rRDPData,RUVnormalizeData,SCLCBam,seq2pathway.data,seqc,seqCNA.annot,serumStimulation,seventyGeneData,shinyMethylData,simpIntLists,SNAData,SNAGEEdata,SomatiCAData,SomaticCancerAlterations,SpikeIn,SpikeInSubset,stemHypoxia,stjudem,SVM2CRMdata,synapterdata,TargetScoreData,TargetSearchData,TBX20BamSubset,TCGAcrcmiRNA,TCGAcrcmRNA,TCGAMethylation450k,tinesath1cdf,tinesath1probe,tweeDEseqCountData,waveTilingData,WES.1KG.WUGSC,XhybCasneuf,yeastCC,yeastExpData,yeastGSData,yeastNagalakshmi,yeastRNASeq,yri1kgv,zebrafishRNASeq" id="ExperimentData" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ExperimentData (241)</a><ul><li packagelist="ceu1kg,ceuhm3,cgdv17,cheung2010,cnvGSAdata,COHCAPanno,curatedBladderData,DMRcatedata,encoDnaseI,Fletcher2013a,Fletcher2013b,FunciSNP.data,GGdata,ggtut,GWASdata,hapmapsnp5,hapmapsnp6,hmyriB36,Illumina450ProbeVariants.db,ind1KG,leeBamViews,mammaPrintData,mAPKLData,pd.atdschip.tiling,pumadata,rcellminerData,rcellminerData,rcellminerData,RRBSdata,seqCNA.annot" id="AssayDomainData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>AssayDomainData (28)</a><ul><li packagelist="cnvGSAdata,rcellminerData,seqCNA.annot" id="CopyNumberVariationData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CopyNumberVariationData (3)</a></li><li packagelist="COHCAPanno,RRBSdata" id="CpGIslandData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CpGIslandData (2)</a></li><li packagelist="curatedBladderData,Fletcher2013a,mammaPrintData,mAPKLData,rcellminerData" id="ExpressionData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ExpressionData (5)</a></li><li packagelist="ceu1kg,ceuhm3,cgdv17,cheung2010,DMRcatedata,encoDnaseI,Fletcher2013b,FunciSNP.data,GGdata,ggtut,GWASdata,hapmapsnp5,hapmapsnp6,hmyriB36,Illumina450ProbeVariants.db,ind1KG,leeBamViews,pd.atdschip.tiling,pumadata,rcellminerData" id="SNPData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SNPData (20)</a></li></ul></li><li packagelist="ALL,ALL,ALLMLL,ALLMLL,antiProfilesData,antiProfilesData,bcellViper,bladderbatch,breastCancerMAINZ,breastCancerMAINZ,breastCancerNKI,breastCancerNKI,breastCancerTRANSBIG,breastCancerTRANSBIG,breastCancerUNT,breastCancerUNT,breastCancerUPP,breastCancerUPP,breastCancerVDX,breastCancerVDX,breastCancerVDX,bsseqData,bsseqData,cancerdata,CLL,CLL,colonCA,colonCA,ConnectivityMap,COPDSexualDimorphism.data,COSMIC.67,CRCL18,curatedBladderData,curatedBladderData,curatedCRCData,curatedOvarianData,curatedOvarianData,DeSousa2013,DeSousa2013,DLBCL,fabiaData,fabiaData,Fletcher2013a,Fletcher2013a,Fletcher2013b,Fletcher2013b,gageData,gageData,golubEsets,golubEsets,GSVAdata,GSVAdata,h5vcData,hapmap100khind,hapmap100kxba,hapmap500knsp,hapmap500ksty,hapmapsnp5,HD2013SGI,HD2013SGI,HIVcDNAvantWout03,Illumina450ProbeVariants.db,kidpack,kidpack,leukemiasEset,leukemiasEset,LungCancerACvsSCCGEO,LungCancerACvsSCCGEO,LungCancerLines,LungCancerLines,lungExpression,lungExpression,mammaPrintData,mammaPrintData,maqcExpression4plex,Mulder2012,Neve2006,Neve2006,NGScopyData,NGScopyData,PREDAsampledata,PREDAsampledata,ProData,ProData,rcellminerData,RnaSeqSampleSizeData,RRBSdata,SCLCBam,seventyGeneData,seventyGeneData,shinyMethylData,SomatiCAData,SomaticCancerAlterations,stemHypoxia,stjudem,stjudem,TCGAcrcmiRNA,TCGAcrcmRNA,TCGAMethylation450k,yeastCC" id="DiseaseModel" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DiseaseModel (65)</a><ul><li packagelist="ALL,ALLMLL,antiProfilesData,bcellViper,bladderbatch,breastCancerMAINZ,breastCancerNKI,breastCancerTRANSBIG,breastCancerUNT,breastCancerUPP,breastCancerVDX,bsseqData,cancerdata,CLL,colonCA,ConnectivityMap,COSMIC.67,CRCL18,curatedBladderData,curatedCRCData,curatedOvarianData,DeSousa2013,DLBCL,fabiaData,Fletcher2013a,Fletcher2013b,gageData,golubEsets,GSVAdata,h5vcData,hapmap100khind,hapmap100kxba,hapmap500knsp,hapmap500ksty,hapmapsnp5,HD2013SGI,Illumina450ProbeVariants.db,kidpack,leukemiasEset,LungCancerACvsSCCGEO,LungCancerLines,lungExpression,mammaPrintData,maqcExpression4plex,Mulder2012,Neve2006,NGScopyData,PREDAsampledata,ProData,rcellminerData,RnaSeqSampleSizeData,RRBSdata,seventyGeneData,shinyMethylData,SomatiCAData,SomaticCancerAlterations,stemHypoxia,stjudem,TCGAcrcmiRNA,TCGAcrcmRNA,TCGAMethylation450k,yeastCC" id="CancerData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CancerData (63)</a><ul><li packagelist="breastCancerMAINZ,breastCancerNKI,breastCancerTRANSBIG,breastCancerUNT,breastCancerUPP,breastCancerVDX,fabiaData,Fletcher2013a,Fletcher2013b,gageData,mammaPrintData,Neve2006,ProData,seventyGeneData" id="BreastCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>BreastCancerData (14)</a></li><li packagelist="antiProfilesData,bsseqData,colonCA,DeSousa2013,HD2013SGI" id="ColonCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ColonCancerData (5)</a></li><li packagelist="kidpack,PREDAsampledata" id="KidneyCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>KidneyCancerData (2)</a></li><li packagelist="ALL,ALLMLL,CLL,golubEsets,GSVAdata,leukemiasEset,stjudem" id="LeukemiaCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>LeukemiaCancerData (7)</a></li><li packagelist="breastCancerVDX,LungCancerACvsSCCGEO,LungCancerLines,lungExpression,NGScopyData,SCLCBam" id="LungCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>LungCancerData (6)</a></li><li packagelist="curatedBladderData,curatedOvarianData" id="OvarianCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>OvarianCancerData (2)</a></li><li class="jstree-last jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ProstateCancerData</a></li></ul></li><li packagelist="COPDSexualDimorphism.data" id="COPDData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>COPDData (1)</a></li><li packagelist="HIVcDNAvantWout03" id="HIVData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>HIVData (1)</a></li></ul></li><li packagelist="Affyhgu133A2Expr,Affyhgu133aExpr,Affyhgu133Plus2Expr,Affymoe4302Expr,bcellViper,beadarrayExampleData,beta7,breastCancerVDX,CCl4,ccTutorial,cheung2010,ChimpHumanBrainData,ChimpHumanBrainData,ChIPXpressData,ChIPXpressData,cMap2data,COHCAPanno,CopyNumber450kData,davidTiling,derfinderData,DmelSGI,DREAM4,DrugVsDiseasedata,EatonEtAlChIPseq,encoDnaseI,fibroEset,fission,FlowSorted.Blood.450k,FlowSorted.DLPFC.450k,frmaExampleData,gaschYHS,GSVAdata,harbChIP,HD2013SGI,hgu133abarcodevecs,hgu133plus2barcodevecs,HiCDataHumanIMR90,HiCDataLymphoblast,humanStemCell,Illumina450ProbeVariants.db,leeBamViews,leukemiasEset,ListerEtAlBSseq,MAQCsubsetAFX,MAQCsubsetILM,minfiData,miRNATarget,MMDiffBamSubset,mosaicsExample,mouse4302barcodevecs,MUGAExampleData,pasilla,pcaGoPromoter.Hs.hg19,pcaGoPromoter.Mm.mm9,pcaGoPromoter.Rn.rn4,pd.atdschip.tiling,pRolocdata,pRolocdata,pRolocdata,pRolocdata,PWMEnrich.Dmelanogaster.background,PWMEnrich.Hsapiens.background,PWMEnrich.Mmusculus.background,RNAinteractMAPK,RNAseqData.HNRNPC.bam.chr14,RnaSeqTutorial,simpIntLists,stemHypoxia,TargetSearchData,tinesath1cdf,tinesath1probe,tweeDEseqCountData,waveTilingData,WES.1KG.WUGSC,yeastCC,yeastExpData,yeastGSData,yeastNagalakshmi,yeastRNASeq,zebrafishRNASeq" id="OrganismData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>OrganismData (75)</a><ul><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Anopheles_gambiae_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Apis_mellifera_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Arabidopsis_lyrata_Data</a></li><li packagelist="pd.atdschip.tiling,pRolocdata,simpIntLists,tinesath1cdf,tinesath1probe,waveTilingData" id="Arabidopsis_thaliana_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Arabidopsis_thaliana_Data (6)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Bacillus_subtilis_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Bos_taurus_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Caenorhabditis_elegans_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Canis_familiaris_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Ciona_intestinalis_Data</a></li><li packagelist="zebrafishRNASeq" id="Danio_rerio_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Danio_rerio_Data (1)</a></li><li packagelist="DmelSGI,pasilla,pRolocdata,PWMEnrich.Dmelanogaster.background,RNAinteractMAPK,RnaSeqTutorial" id="Drosophila_melanogaster_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Drosophila_melanogaster_Data (6)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Eremothecium_gossypii_Data</a></li><li packagelist="TargetSearchData" id="Escherichia_coli_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Escherichia_coli_Data (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Gallus_gallus_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Gasterosteus_aculeatus_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Glycine_max_Data</a></li><li packagelist="Affyhgu133A2Expr,Affyhgu133aExpr,Affyhgu133Plus2Expr,bcellViper,beadarrayExampleData,beta7,breastCancerVDX,cheung2010,ChimpHumanBrainData,ChIPXpressData,cMap2data,COHCAPanno,CopyNumber450kData,derfinderData,DrugVsDiseasedata,encoDnaseI,fibroEset,FlowSorted.Blood.450k,FlowSorted.DLPFC.450k,frmaExampleData,GSVAdata,HD2013SGI,hgu133abarcodevecs,hgu133plus2barcodevecs,HiCDataHumanIMR90,HiCDataLymphoblast,humanStemCell,Illumina450ProbeVariants.db,leukemiasEset,ListerEtAlBSseq,MAQCsubsetAFX,MAQCsubsetILM,minfiData,miRNATarget,mosaicsExample,pcaGoPromoter.Hs.hg19,pRolocdata,PWMEnrich.Hsapiens.background,RNAseqData.HNRNPC.bam.chr14,stemHypoxia,tweeDEseqCountData,WES.1KG.WUGSC" id="Homo_sapiens_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Homo_sapiens_Data (42)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Hordeum_vulgare_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Kluyveromyces_lactis_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Macaca_mulatta_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Magnaporthe_grisea_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Medicago_truncatul_Dataa</a></li><li packagelist="Affymoe4302Expr,ccTutorial,ChIPXpressData,MMDiffBamSubset,mouse4302barcodevecs,MUGAExampleData,pcaGoPromoter.Mm.mm9,pRolocdata,PWMEnrich.Mmusculus.background" id="Mus_musculus_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Mus_musculus_Data (9)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Neurospora_crassa_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Oncorhynchus_mykiss_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Oryza_sativa_Data</a></li><li packagelist="ChimpHumanBrainData" id="Pan_troglodytes_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Pan_troglodytes_Data (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Plasmodium_falciparum_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Pseudomonas_aeruginosa_Data</a></li><li packagelist="CCl4,pcaGoPromoter.Rn.rn4" id="Rattus_norvegicus_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Rattus_norvegicus_Data (2)</a></li><li packagelist="davidTiling,DREAM4,EatonEtAlChIPseq,fission,gaschYHS,harbChIP,leeBamViews,yeastCC,yeastExpData,yeastGSData,yeastNagalakshmi,yeastRNASeq" id="Saccharomyces_cerevisiae_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Saccharomyces_cerevisiae_Data (12)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Saccharum_officinarum_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Schizosaccharomyces_pombe_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Staphylococcus_aureus_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Sus_scrofa_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Taeniopygia_guttata_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Triticum_aestivum_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Vitis_vinifera_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Xenopus_laevis_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Xenopus_tropicalis_Data</a></li><li class="jstree-last jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Zea_mays_Data</a></li></ul></li><li packagelist="Affyhgu133A2Expr,Affyhgu133aExpr,Affyhgu133Plus2Expr,Affymoe4302Expr,airway,blimaTestingData,breastCancerMAINZ,breastCancerTRANSBIG,breastCancerUNT,breastCancerUPP,breastCancerVDX,ceu1kg,ceu1kg,ceu1kgv,ceuhm3,cheung2010,ChimpHumanBrainData,ChIPXpressData,cMap2data,COHCAPanno,COHCAPanno,dsQTL,dsQTL,DvDdata,dyebiasexamples,EatonEtAlChIPseq,encoDnaseI,ffpeExampleData,fission,FunciSNP.data,FunciSNP.data,gageData,genomationData,geuvPack,GGdata,GWASdata,hapmap100khind,hapmap100kxba,hapmap500knsp,hapmap500ksty,hapmapsnp5,hapmapsnp6,HiCDataHumanIMR90,hmyriB36,ITALICSData,KEGGandMetacoreDzPathwaysGEO,KEGGdzPathwaysGEO,kidpack,leukemiasEset,LiebermanAidenHiC2009,LiebermanAidenHiC2009,LiebermanAidenHiC2009,LungCancerACvsSCCGEO,MAQCsubset,MMDiffBamSubset,NCIgraphData,PathNetData,PREDAsampledata,ProData,rcellminerData,rheumaticConditionWOLLBOLD,rheumaticConditionWOLLBOLD,RIPSeekerData,RNAseqData.HNRNPC.bam.chr14,RNAseqData.HNRNPC.bam.chr14,RNAseqData.HNRNPC.bam.chr14,SomaticCancerAlterations,SomaticCancerAlterations,stemHypoxia,WES.1KG.WUGSC,yeastCC,yri1kgv" id="RepositoryData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>RepositoryData (62)</a><ul><li packagelist="ceu1kgv,COHCAPanno,dyebiasexamples,geuvPack,kidpack,MMDiffBamSubset,PREDAsampledata,rheumaticConditionWOLLBOLD,RNAseqData.HNRNPC.bam.chr14,yri1kgv" id="ArrayExpress" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ArrayExpress (10)</a></li><li packagelist="encoDnaseI,FunciSNP.data,genomationData,LiebermanAidenHiC2009" id="ENCODE" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ENCODE (4)</a></li><li packagelist="Affyhgu133A2Expr,Affyhgu133aExpr,Affyhgu133Plus2Expr,Affymoe4302Expr,airway,blimaTestingData,breastCancerMAINZ,breastCancerTRANSBIG,breastCancerUNT,breastCancerUPP,breastCancerVDX,ChimpHumanBrainData,ChIPXpressData,cMap2data,COHCAPanno,DvDdata,EatonEtAlChIPseq,ffpeExampleData,fission,gageData,HiCDataHumanIMR90,KEGGandMetacoreDzPathwaysGEO,KEGGdzPathwaysGEO,leukemiasEset,LiebermanAidenHiC2009,LungCancerACvsSCCGEO,MAQCsubset,rheumaticConditionWOLLBOLD,RIPSeekerData,RNAseqData.HNRNPC.bam.chr14,stemHypoxia,yeastCC" id="GEO" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GEO (32)</a></li><li packagelist="ceu1kg,ceuhm3,cheung2010,GGdata,GWASdata,hapmap100khind,hapmap100kxba,hapmap500knsp,hapmap500ksty,hapmapsnp5,hapmapsnp6,hmyriB36,ITALICSData" id="HapMap" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>HapMap (13)</a></li><li packagelist="dsQTL,LiebermanAidenHiC2009,NCIgraphData,ProData,rcellminerData,RNAseqData.HNRNPC.bam.chr14,SomaticCancerAlterations" id="NCI" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>NCI (7)</a></li><li packagelist="PathNetData" id="PathwayInteractionDatabase" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>PathwayInteractionDatabase (1)</a></li><li packagelist="ceu1kg,dsQTL,FunciSNP.data,SomaticCancerAlterations,WES.1KG.WUGSC" id="Project1000genomes" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Project1000genomes (5)</a></li></ul></li><li packagelist="affydata,Affyhgu133A2Expr,Affyhgu133Plus2Expr,Affymoe4302Expr,antiProfilesData,bcellViper,breastCancerVDX,bsseqData,CCl4,ChimpHumanBrainData,chipenrich.data,cnvGSAdata,colonCA,COPDSexualDimorphism.data,CopyNumber450kData,COSMIC.67,davidTiling,DmelSGI,DonaPLLP2013,DREAM4,dressCheck,dsQTL,encoDnaseI,facopy.annot,FANTOM3and4CAGE,ffpeExampleData,ffpeExampleData,fibroEset,fission,gageData,gaschYHS,geneLenDataBase,genomationData,genomationData,geuvPack,GGdata,golubEsets,HD2013SGI,hgu2beta7,HiCDataHumanIMR90,hmyriB36,ind1KG,leukemiasEset,leukemiasEset,LiebermanAidenHiC2009,lumiBarnes,LungCancerLines,MAQCsubsetAFX,MEDIPSData,mitoODEdata,MMDiffBamSubset,MMDiffBamSubset,mosaicsExample,Mulder2012,pasilla,pasillaBamSubset,PREDAsampledata,pRolocdata,rheumaticConditionWOLLBOLD,RIPSeekerData,RIPSeekerData,RNAinteractMAPK,RNAseqData.HNRNPC.bam.chr14,seqCNA.annot,shinyMethylData,SomatiCAData,SomaticCancerAlterations,stemHypoxia,stemHypoxia,TCGAMethylation450k,tweeDEseqCountData,XhybCasneuf,yeastCC,yeastExpData,yeastNagalakshmi" id="SpecimenSource" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SpecimenSource (69)</a><ul><li packagelist="DmelSGI,HD2013SGI,RNAinteractMAPK,yeastCC" id="CellCulture" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CellCulture (4)</a></li><li packagelist="Affyhgu133A2Expr,Affyhgu133Plus2Expr,Affymoe4302Expr,bcellViper,breastCancerVDX,bsseqData,CCl4,chipenrich.data,cnvGSAdata,COSMIC.67,davidTiling,DREAM4,dressCheck,dsQTL,encoDnaseI,facopy.annot,ffpeExampleData,fibroEset,fission,gaschYHS,geneLenDataBase,genomationData,geuvPack,GGdata,golubEsets,hgu2beta7,HiCDataHumanIMR90,hmyriB36,ind1KG,leukemiasEset,LiebermanAidenHiC2009,LungCancerLines,MAQCsubsetAFX,MEDIPSData,mitoODEdata,MMDiffBamSubset,mosaicsExample,pasilla,pasillaBamSubset,RIPSeekerData,RNAseqData.HNRNPC.bam.chr14,seqCNA.annot,shinyMethylData,SomatiCAData,SomaticCancerAlterations,TCGAMethylation450k,tweeDEseqCountData,yeastExpData,yeastNagalakshmi" id="Genome" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Genome (49)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Germline</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Proteome</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Somatic</a></li><li packagelist="gageData,genomationData,MMDiffBamSubset,Mulder2012,pRolocdata,RIPSeekerData,stemHypoxia" id="StemCell" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>StemCell (7)</a></li><li packagelist="affydata,antiProfilesData,ChimpHumanBrainData,colonCA,COPDSexualDimorphism.data,CopyNumber450kData,DonaPLLP2013,FANTOM3and4CAGE,ffpeExampleData,leukemiasEset,lumiBarnes,PREDAsampledata,rheumaticConditionWOLLBOLD,stemHypoxia,XhybCasneuf" id="Tissue" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>Tissue (15)</a></li></ul></li><li packagelist="affycompData,affydata,affydata,AffymetrixDataTestFiles,airway,airway,ALLMLL,AmpAffyExample,antiProfilesData,ARRmData,beadarrayExampleData,BeadArrayUseCases,beta7,beta7,bladderbatch,blimaTestingData,breastCancerMAINZ,breastCancerNKI,breastCancerNKI,breastCancerNKI,breastCancerNKI,breastCancerTRANSBIG,breastCancerUNT,breastCancerUNT,breastCancerUPP,breastCancerVDX,breastCancerVDX,bronchialIL13,bsseqData,cancerdata,CardinalWorkflows,CardinalWorkflows,CCl4,ccTutorial,ccTutorial,cgdv17,charmData,cheung2010,cheung2010,ChimpHumanBrainData,ChimpHumanBrainData,chipenrich.data,CLL,cMap2data,COHCAPanno,COHCAPanno,colonCA,colonCA,ConnectivityMap,CopyNumber450kData,CopyNumber450kData,CopyNumber450kData,curatedBladderData,curatedOvarianData,curatedOvarianData,davidTiling,derfinderData,DeSousa2013,DLBCL,DLBCL,DmelSGI,DmelSGI,DREAM4,DrugVsDiseasedata,dsQTL,dsQTL,dyebiasexamples,dyebiasexamples,dyebiasexamples,dyebiasexamples,EatonEtAlChIPseq,EatonEtAlChIPseq,ecoliLeucine,estrogen,faahKO,fabiaData,facsDorit,ffpeExampleData,ffpeExampleData,fibroEset,fibroEset,fission,fission,Fletcher2013a,Fletcher2013b,flowFitExampleData,FlowSorted.Blood.450k,FlowSorted.Blood.450k,FlowSorted.DLPFC.450k,flowWorkspaceData,frmaExampleData,gageData,gatingMLData,gcspikelite,genomationData,genomationData,genomationData,geuvPack,geuvPack,geuvStore,geuvStore,GGdata,GGdata,ggtut,ggtut,GSBenchMark,GSVAdata,GWASdata,harbChIP,HD2013SGI,HD2013SGI,healthyFlowData,hgu133abarcodevecs,hgu133plus2barcodevecs,Hiiragi2013,Hiiragi2013,HIVcDNAvantWout03,HIVcDNAvantWout03,HSMMSingleCell,Illumina450ProbeVariants.db,IlluminaDataTestFiles,ind1KG,iontreeData,JASPAR2014,KEGGdzPathwaysGEO,kidpack,leeBamViews,leeBamViews,leukemiasEset,leukemiasEset,leukemiasEset,LiebermanAidenHiC2009,LiebermanAidenHiC2009,LiebermanAidenHiC2009,LiebermanAidenHiC2009,LiebermanAidenHiC2009,LiebermanAidenHiC2009,ListerEtAlBSseq,lumiBarnes,lumiBarnes,LungCancerACvsSCCGEO,LungCancerLines,mammaPrintData,mammaPrintData,MAQCsubset,MAQCsubsetAFX,MAQCsubsetILM,MEDIPSData,metaMSdata,minfiData,MMDiffBamSubset,MMDiffBamSubset,mosaicsExample,mouse4302barcodevecs,msd16s,msd16s,msdata,mvoutData,NGScopyData,parathyroidSE,parathyroidSE,pasilla,pasillaBamSubset,pasillaBamSubset,pd.atdschip.tiling,pepDat,prebsdata,prebsdata,PREDAsampledata,PREDAsampledata,ProData,pRolocdata,pumadata,PWMEnrich.Hsapiens.background,rcellminerData,rcellminerData,RforProteomics,rheumaticConditionWOLLBOLD,rheumaticConditionWOLLBOLD,rheumaticConditionWOLLBOLD,rheumaticConditionWOLLBOLD,RIPSeekerData,RIPSeekerData,RMassBankData,RNAinteractMAPK,RNAseqData.HNRNPC.bam.chr14,RNAseqData.HNRNPC.bam.chr14,RnaSeqSampleSizeData,RnaSeqTutorial,RRBSdata,rRDPData,rRDPData,RUVnormalizeData,SCLCBam,seqc,seqc,seqc,serumStimulation,seventyGeneData,SNAGEEdata,SomatiCAData,SpikeIn,SpikeInSubset,stemHypoxia,stemHypoxia,stjudem,stjudem,synapterdata,TargetScoreData,TargetScoreData,TBX20BamSubset,TBX20BamSubset,TCGAMethylation450k,tinesath1cdf,tinesath1probe,tinesath1probe,tweeDEseqCountData,waveTilingData,WES.1KG.WUGSC,WES.1KG.WUGSC,XhybCasneuf,XhybCasneuf,yeastCC,yeastCC,yeastNagalakshmi,yeastNagalakshmi,yeastRNASeq,yeastRNASeq,zebrafishRNASeq" id="TechnologyData" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>TechnologyData (157)</a><ul><li packagelist="beta7,breastCancerNKI,dyebiasexamples,PWMEnrich.Hsapiens.background" id="CGHData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>CGHData (4)</a></li><li packagelist="flowFitExampleData,flowWorkspaceData,gatingMLData,healthyFlowData" id="FlowCytometryData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>FlowCytometryData (4)</a></li><li packagelist="DmelSGI,HD2013SGI" id="HighThroughputImagingData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>HighThroughputImagingData (2)</a></li><li packagelist="CardinalWorkflows,faahKO,gcspikelite,iontreeData,metaMSdata,msdata,ProData,pRolocdata,RforProteomics,RMassBankData,synapterdata" id="MassSpectrometryData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MassSpectrometryData (11)</a><ul><li packagelist="CardinalWorkflows" id="ImagingMassSpectrometryData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ImagingMassSpectrometryData (1)</a></li></ul></li><li packagelist="affycompData,affydata,AffymetrixDataTestFiles,ALLMLL,AmpAffyExample,antiProfilesData,beadarrayExampleData,BeadArrayUseCases,beta7,bladderbatch,blimaTestingData,breastCancerMAINZ,breastCancerNKI,breastCancerTRANSBIG,breastCancerUNT,breastCancerUPP,breastCancerVDX,bronchialIL13,cancerdata,CCl4,ccTutorial,charmData,ChimpHumanBrainData,CLL,cMap2data,COHCAPanno,colonCA,ConnectivityMap,CopyNumber450kData,curatedBladderData,curatedOvarianData,davidTiling,DeSousa2013,DLBCL,DrugVsDiseasedata,dyebiasexamples,ecoliLeucine,estrogen,fabiaData,ffpeExampleData,fibroEset,Fletcher2013a,FlowSorted.Blood.450k,FlowSorted.DLPFC.450k,frmaExampleData,gageData,geuvPack,geuvStore,GGdata,ggtut,GSBenchMark,GWASdata,hgu133abarcodevecs,hgu133plus2barcodevecs,Hiiragi2013,HIVcDNAvantWout03,IlluminaDataTestFiles,ind1KG,KEGGdzPathwaysGEO,kidpack,leukemiasEset,lumiBarnes,LungCancerACvsSCCGEO,mammaPrintData,MAQCsubset,MAQCsubsetAFX,MAQCsubsetILM,mouse4302barcodevecs,mvoutData,pd.atdschip.tiling,pepDat,PREDAsampledata,pumadata,rcellminerData,rheumaticConditionWOLLBOLD,RUVnormalizeData,serumStimulation,seventyGeneData,SNAGEEdata,SpikeIn,SpikeInSubset,stemHypoxia,stjudem,tinesath1probe,waveTilingData,XhybCasneuf,yeastCC" id="MicroarrayData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MicroarrayData (94)</a><ul><li packagelist="breastCancerNKI,ccTutorial,DLBCL,fibroEset,genomationData,Illumina450ProbeVariants.db,leukemiasEset,LiebermanAidenHiC2009,lumiBarnes,rheumaticConditionWOLLBOLD,stjudem,tinesath1cdf" id="ChipOnChipData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ChipOnChipData (12)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>GenotypingArrayData</a></li><li packagelist="ARRmData,COHCAPanno,CopyNumber450kData,FlowSorted.Blood.450k,minfiData,TCGAMethylation450k" id="MethylationArrayData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MethylationArrayData (6)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MicroRNAArrayData</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>mRNAArrayData</a></li><li packagelist="rheumaticConditionWOLLBOLD" id="MultiChannelData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MultiChannelData (1)</a></li><li packagelist="breastCancerNKI,breastCancerVDX,yeastCC" id="OneChannelData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>OneChannelData (3)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ProprietaryPlatformsData</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ReversePhaseProteinArrayData</a></li><li packagelist="affydata,ChimpHumanBrainData,colonCA,CopyNumber450kData,ffpeExampleData,leukemiasEset,PREDAsampledata,rheumaticConditionWOLLBOLD,stemHypoxia,XhybCasneuf" id="TissueMicroarrayData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>TissueMicroarrayData (10)</a></li><li packagelist="breastCancerUNT,dyebiasexamples,HIVcDNAvantWout03,mammaPrintData" id="TwoChannelData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>TwoChannelData (4)</a></li></ul></li><li packagelist="DmelSGI,facsDorit,HD2013SGI,RNAinteractMAPK" id="MicrotitrePlateAssayData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MicrotitrePlateAssayData (4)</a></li><li packagelist="Hiiragi2013,seqc" id="qPCRData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>qPCRData (2)</a></li><li packagelist="dyebiasexamples" id="SAGEData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SAGEData (1)</a></li><li packagelist="airway,bsseqData,cgdv17,cheung2010,DREAM4,dsQTL,EatonEtAlChIPseq,fission,genomationData,geuvPack,geuvStore,GGdata,ggtut,harbChIP,JASPAR2014,leeBamViews,LiebermanAidenHiC2009,ListerEtAlBSseq,MEDIPSData,msd16s,NGScopyData,parathyroidSE,prebsdata,RIPSeekerData,RNAseqData.HNRNPC.bam.chr14,RRBSdata,rRDPData,SCLCBam,seqc,SomatiCAData,TBX20BamSubset,tinesath1probe,WES.1KG.WUGSC,yeastNagalakshmi,yeastRNASeq" id="SequencingData" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SequencingData (52)</a><ul><li packagelist="chipenrich.data,EatonEtAlChIPseq,Fletcher2013b,genomationData,LiebermanAidenHiC2009,MMDiffBamSubset,mosaicsExample,yeastNagalakshmi" id="ChIPSeqData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ChIPSeqData (8)</a></li><li packagelist="dsQTL,LiebermanAidenHiC2009,MMDiffBamSubset,pasillaBamSubset,WES.1KG.WUGSC" id="DNASeqData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>DNASeqData (5)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>ExomeSeqData</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MethylSeqData</a></li><li packagelist="msd16s,rRDPData" id="MicrobiomeData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>MicrobiomeData (2)</a></li><li packagelist="LiebermanAidenHiC2009,rcellminerData,TargetScoreData" id="miRNAData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>miRNAData (3)</a></li><li packagelist="RIPSeekerData" id="RIPSeqData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>RIPSeqData (1)</a></li><li packagelist="airway,cheung2010,curatedOvarianData,derfinderData,fission,GSVAdata,HSMMSingleCell,leeBamViews,LungCancerLines,parathyroidSE,pasilla,pasillaBamSubset,prebsdata,RNAseqData.HNRNPC.bam.chr14,RnaSeqSampleSizeData,RnaSeqTutorial,seqc,TargetScoreData,TBX20BamSubset,tweeDEseqCountData,yeastRNASeq,zebrafishRNASeq" id="RNASeqData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>RNASeqData (22)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SangerSeqData</a></li><li packagelist="LiebermanAidenHiC2009" id="SmallRNAData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="#"><ins class="jstree-icon">&nbsp;</ins>SmallRNAData (1)</a></li></ul></li></ul></li></ul></li></ul></div>
        </div>
        <div id="packages" class="biocViewsPackages"><h3>Packages found under Software:</h3>
<div id="biocViews_package_table_wrapper" class="dataTables_wrapper" role="grid"><div id="biocViews_package_table_length" class="dataTables_length"><label>Show <select size="1" name="biocViews_package_table_length" aria-controls="biocViews_package_table"><option value="-1">All</option><option value="10">10</option><option value="25">25</option><option value="50">50</option><option value="100">100</option></select> entries</label></div><div class="dataTables_filter" id="biocViews_package_table_filter"><label>Search table: <input type="text" aria-controls="biocViews_package_table"></label></div><div class="dataTables_scroll"><div class="dataTables_scrollHead" style="overflow: hidden; position: relative; border: 0px; width: 100%;"><div class="dataTables_scrollHeadInner" style="width: 599px; padding-right: 0px;"><table class="dataTable" style="margin-left: 0px; width: 599px;"><thead><tr role="row"><th class="sorting_asc" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-sort="ascending" aria-label="Package: activate to sort column descending" style="width: 140px;">Package</th><th class="sorting" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-label="Maintainer: activate to sort column ascending" style="width: 93px;">Maintainer</th><th class="sorting" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-label="Title: activate to sort column ascending" style="width: 274px;">Title</th></tr></thead></table></div></div><div class="dataTables_scrollBody" style="overflow: auto; width: 100%;"><table id="biocViews_package_table" class="dataTable" aria-describedby="biocViews_package_table_info" style="margin-left: 0px; width: 599px;"><thead><tr role="row" style="height: 0px;"><th class="sorting_asc" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-sort="ascending" aria-label="Package: activate to sort column descending" style="width: 140px; padding-top: 0px; padding-bottom: 0px; border-top-width: 0px; border-bottom-width: 0px; height: 0px;"></th><th class="sorting" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-label="Maintainer: activate to sort column ascending" style="width: 93px; padding-top: 0px; padding-bottom: 0px; border-top-width: 0px; border-bottom-width: 0px; height: 0px;"></th><th class="sorting" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-label="Title: activate to sort column ascending" style="width: 274px; padding-top: 0px; padding-bottom: 0px; border-top-width: 0px; border-bottom-width: 0px; height: 0px;"></th></tr></thead><tbody role="alert" aria-live="polite" aria-relevant="all"><tr id="pkg_a4" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/a4.html">a4</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Umbrella Package</td>
</tr><tr id="pkg_a4Base" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/a4Base.html">a4Base</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Base Package</td>
</tr><tr id="pkg_a4Classif" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/a4Classif.html">a4Classif</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Classification Package</td>
</tr><tr id="pkg_a4Core" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/a4Core.html">a4Core</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Core Package</td>
</tr><tr id="pkg_a4Preproc" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/a4Preproc.html">a4Preproc</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Preprocessing Package</td>
</tr><tr id="pkg_a4Reporting" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/a4Reporting.html">a4Reporting</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Reporting Package</td>
</tr><tr id="pkg_ABarray" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ABarray.html">ABarray</a></td>
	<td class=" ">Yongming Andrew Sun</td>
	<td class=" ">Microarray QA and statistical data analysis for Applied Biosystems Genome Survey Microrarray (AB1700) gene expression data.</td>
</tr><tr id="pkg_ABSSeq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ABSSeq.html">ABSSeq</a></td>
	<td class=" ">Wentao Yang</td>
	<td class=" ">ABSSeq: a new RNA-Seq analysis method based on absolute expression differences and generalized Poisson model</td>
</tr><tr id="pkg_aCGH" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/aCGH.html">aCGH</a></td>
	<td class=" ">Peter Dimitrov</td>
	<td class=" ">Classes and functions for Array Comparative Genomic Hybridization data.</td>
</tr><tr id="pkg_ACME" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ACME.html">ACME</a></td>
	<td class=" ">Sean Davis</td>
	<td class=" ">Algorithms for Calculating Microarray Enrichment (ACME)</td>
</tr><tr id="pkg_ADaCGH2" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ADaCGH2.html">ADaCGH2</a></td>
	<td class=" ">Ramon Diaz-Uriarte</td>
	<td class=" ">Analysis of big data from aCGH experiments using parallel computing and ff objects</td>
</tr><tr id="pkg_adSplit" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/adSplit.html">adSplit</a></td>
	<td class=" ">Claudio Lottaz</td>
	<td class=" ">Annotation-Driven Clustering</td>
</tr><tr id="pkg_affxparser" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affxparser.html">affxparser</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Affymetrix File Parsing SDK</td>
</tr><tr id="pkg_affy" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affy.html">affy</a></td>
	<td class=" ">Rafael A. Irizarry</td>
	<td class=" ">Methods for Affymetrix Oligonucleotide Arrays</td>
</tr><tr id="pkg_affycomp" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affycomp.html">affycomp</a></td>
	<td class=" ">Rafael A. Irizarry</td>
	<td class=" ">Graphics Toolbox for Assessment of Affymetrix Expression Measures</td>
</tr><tr id="pkg_AffyCompatible" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AffyCompatible.html">AffyCompatible</a></td>
	<td class=" ">Martin Morgan</td>
	<td class=" ">Affymetrix GeneChip software compatibility</td>
</tr><tr id="pkg_affyContam" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affyContam.html">affyContam</a></td>
	<td class=" ">V. Carey</td>
	<td class=" ">structured corruption of affymetrix cel file data</td>
</tr><tr id="pkg_affycoretools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affycoretools.html">affycoretools</a></td>
	<td class=" ">James W. MacDonald</td>
	<td class=" ">Functions useful for those doing repetitive analyses with Affymetrix GeneChips</td>
</tr><tr id="pkg_AffyExpress" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AffyExpress.html">AffyExpress</a></td>
	<td class=" ">Xuejun Arthur Li</td>
	<td class=" ">Affymetrix Quality Assessment and Analysis Tool</td>
</tr><tr id="pkg_affyILM" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affyILM.html">affyILM</a></td>
	<td class=" ">Myriam Kroll and Fabrice Berger</td>
	<td class=" ">Linear Model of background subtraction and the Langmuir isotherm</td>
</tr><tr id="pkg_affyio" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affyio.html">affyio</a></td>
	<td class=" ">Benjamin Milo Bolstad</td>
	<td class=" ">Tools for parsing Affymetrix data files</td>
</tr><tr id="pkg_affylmGUI" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affylmGUI.html">affylmGUI</a></td>
	<td class=" ">Keith Satterley</td>
	<td class=" ">GUI for affy analysis using limma package</td>
</tr><tr id="pkg_affyPara" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affyPara.html">affyPara</a></td>
	<td class=" ">Markus Schmidberger</td>
	<td class=" ">Parallelized preprocessing methods for Affymetrix Oligonucleotide Arrays</td>
</tr><tr id="pkg_affypdnn" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affypdnn.html">affypdnn</a></td>
	<td class=" ">Laurent Gautier</td>
	<td class=" ">Probe Dependent Nearest Neighbours (PDNN) for the affy package</td>
</tr><tr id="pkg_affyPLM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affyPLM.html">affyPLM</a></td>
	<td class=" ">Ben Bolstad</td>
	<td class=" ">Methods for fitting probe-level models</td>
</tr><tr id="pkg_affyQCReport" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/affyQCReport.html">affyQCReport</a></td>
	<td class=" ">Craig Parman</td>
	<td class=" ">QC Report Generation for affyBatch objects</td>
</tr><tr id="pkg_AffyRNADegradation" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AffyRNADegradation.html">AffyRNADegradation</a></td>
	<td class=" ">Mario Fasold</td>
	<td class=" ">Analyze and correct probe positional bias in microarray data due to RNA degradation</td>
</tr><tr id="pkg_AffyTiling" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AffyTiling.html">AffyTiling</a></td>
	<td class=" ">Charles G. Danko</td>
	<td class=" ">Easy extraction of individual probes in Affymetrix tiling arrays</td>
</tr><tr id="pkg_AGDEX" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AGDEX.html">AGDEX</a></td>
	<td class=" ">Cuilan lani Gao</td>
	<td class=" ">Agreement of Differential Expression Analysis</td>
</tr><tr id="pkg_agilp" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/agilp.html">agilp</a></td>
	<td class=" ">Benny Chain</td>
	<td class=" ">Agilent expression array processing package</td>
</tr><tr id="pkg_AgiMicroRna" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AgiMicroRna.html">AgiMicroRna</a></td>
	<td class=" ">Pedro Lopez-Romero</td>
	<td class=" ">Processing and Differential Expression Analysis of Agilent microRNA chips</td>
</tr><tr id="pkg_AIMS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AIMS.html">AIMS</a></td>
	<td class=" ">Eric R Paquet</td>
	<td class=" ">AIMS : Absolute Assignment of Breast Cancer Intrinsic Molecular Subtype</td>
</tr><tr id="pkg_ALDEx2" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ALDEx2.html">ALDEx2</a></td>
	<td class=" ">Greg Gloor</td>
	<td class=" ">Analysis of differential abundance taking sample variation into account</td>
</tr><tr id="pkg_AllelicImbalance" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AllelicImbalance.html">AllelicImbalance</a></td>
	<td class=" ">Jesper R Gadin</td>
	<td class=" ">Investigates allele specific expression</td>
</tr><tr id="pkg_alsace" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/alsace.html">alsace</a></td>
	<td class=" ">Ron Wehrens</td>
	<td class=" ">ALS for the Automatic Chemical Exploration of mixtures</td>
</tr><tr id="pkg_altcdfenvs" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/altcdfenvs.html">altcdfenvs</a></td>
	<td class=" ">Laurent Gautier</td>
	<td class=" ">alternative CDF environments (aka probeset mappings)</td>
</tr><tr id="pkg_ampliQueso" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ampliQueso.html">ampliQueso</a></td>
	<td class=" ">Michal Okoniewski</td>
	<td class=" ">Analysis of amplicon enrichment panels</td>
</tr><tr id="pkg_AnalysisPageServer" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AnalysisPageServer.html">AnalysisPageServer</a></td>
	<td class=" ">Brad Friedman</td>
	<td class=" ">A framework for sharing interactive data and plots from R through the web.</td>
</tr><tr id="pkg_annaffy" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/annaffy.html">annaffy</a></td>
	<td class=" ">Colin A. Smith</td>
	<td class=" ">Annotation tools for Affymetrix biological metadata</td>
</tr><tr id="pkg_annmap" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/annmap.html">annmap</a></td>
	<td class=" ">Chris Wirth</td>
	<td class=" ">Genome annotation and visualisation package pertaining to Affymetrix arrays and NGS analysis.</td>
</tr><tr id="pkg_annotate" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/annotate.html">annotate</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Annotation for microarrays</td>
</tr><tr id="pkg_AnnotationDbi" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AnnotationDbi.html">AnnotationDbi</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Annotation Database Interface</td>
</tr><tr id="pkg_AnnotationForge" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AnnotationForge.html">AnnotationForge</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Code for Building Annotation Database Packages</td>
</tr><tr id="pkg_AnnotationFuncs" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AnnotationFuncs.html">AnnotationFuncs</a></td>
	<td class=" ">Stefan McKinnon Edwards</td>
	<td class=" ">Annotation translation functions</td>
</tr><tr id="pkg_AnnotationHub" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AnnotationHub.html">AnnotationHub</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Client to access AnnotationHub resources</td>
</tr><tr id="pkg_annotationTools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/annotationTools.html">annotationTools</a></td>
	<td class=" ">Alexandre Kuhn</td>
	<td class=" ">Annotate microarrays and perform cross-species gene expression analyses using flat file databases.</td>
</tr><tr id="pkg_anota" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/anota.html">anota</a></td>
	<td class=" ">Ola Larsson</td>
	<td class=" ">ANalysis Of Translational Activity (ANOTA).</td>
</tr><tr id="pkg_antiProfiles" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/antiProfiles.html">antiProfiles</a></td>
	<td class=" ">Hector Corrada Bravo</td>
	<td class=" ">Implementation of gene expression anti-profiles</td>
</tr><tr id="pkg_apComplex" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/apComplex.html">apComplex</a></td>
	<td class=" ">Denise Scholtens</td>
	<td class=" ">Estimate protein complex membership using AP-MS protein data</td>
</tr><tr id="pkg_aroma.light" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/aroma.light.html">aroma.light</a></td>
	<td class=" ">Henrik Bengtsson</td>
	<td class=" ">Light-weight Methods for Normalization and Visualization of Microarray Data using Only Basic R Data Types</td>
</tr><tr id="pkg_ArrayExpress" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ArrayExpress.html">ArrayExpress</a></td>
	<td class=" ">Ugis Sarkans</td>
	<td class=" ">Access the ArrayExpress Microarray Database at EBI and build Bioconductor data structures: ExpressionSet, AffyBatch, NChannelSet</td>
</tr><tr id="pkg_ArrayExpressHTS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ArrayExpressHTS.html">ArrayExpressHTS</a></td>
	<td class=" ">Angela Goncalves, Andrew Tikhonov</td>
	<td class=" ">ArrayExpress High Throughput Sequencing Processing Pipeline</td>
</tr><tr id="pkg_arrayMvout" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/arrayMvout.html">arrayMvout</a></td>
	<td class=" ">V. Carey</td>
	<td class=" ">multivariate outlier detection for expression array QA</td>
</tr><tr id="pkg_arrayQuality" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/arrayQuality.html">arrayQuality</a></td>
	<td class=" ">Agnes Paquet</td>
	<td class=" ">Assessing array quality on spotted arrays</td>
</tr><tr id="pkg_arrayQualityMetrics" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/arrayQualityMetrics.html">arrayQualityMetrics</a></td>
	<td class=" ">Audrey Kauffmann</td>
	<td class=" ">Quality metrics report for microarray data sets</td>
</tr><tr id="pkg_ArrayTools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ArrayTools.html">ArrayTools</a></td>
	<td class=" ">Arthur Li</td>
	<td class=" ">geneChip Analysis Package</td>
</tr><tr id="pkg_ArrayTV" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ArrayTV.html">ArrayTV</a></td>
	<td class=" ">Eitan Halper-Stromberg</td>
	<td class=" ">Implementation of wave correction for arrays</td>
</tr><tr id="pkg_ARRmNormalization" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ARRmNormalization.html">ARRmNormalization</a></td>
	<td class=" ">Jean-Philippe Fortin</td>
	<td class=" ">Adaptive Robust Regression normalization for Illumina methylation data</td>
</tr><tr id="pkg_ASEB" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ASEB.html">ASEB</a></td>
	<td class=" ">Likun Wang</td>
	<td class=" ">Predict Acetylated Lysine Sites</td>
</tr><tr id="pkg_ASGSCA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ASGSCA.html">ASGSCA</a></td>
	<td class=" ">Hela Romdhani</td>
	<td class=" ">Association Studies for multiple SNPs and multiple traits using Generalized Structured Equation Models</td>
</tr><tr id="pkg_ASSET" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ASSET.html">ASSET</a></td>
	<td class=" ">William Wheeler</td>
	<td class=" ">An R package for subset-based association analysis of heterogeneous traits and subtypes</td>
</tr><tr id="pkg_ASSIGN" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ASSIGN.html">ASSIGN</a></td>
	<td class=" ">Ying Shen</td>
	<td class=" ">Adaptive Signature Selection and InteGratioN (ASSIGN)</td>
</tr><tr id="pkg_AtlasRDF" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/AtlasRDF.html">AtlasRDF</a></td>
	<td class=" ">James Malone</td>
	<td class=" ">Gene Expression Atlas query and gene set enrichment package.</td>
</tr><tr id="pkg_attract" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/attract.html">attract</a></td>
	<td class=" ">Jessica Mar</td>
	<td class=" ">Methods to Find the Gene Expression Modules that Represent the Drivers of Kauffman's Attractor Landscape</td>
</tr><tr id="pkg_BAC" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BAC.html">BAC</a></td>
	<td class=" ">Raphael Gottardo</td>
	<td class=" ">Bayesian Analysis of Chip-chip experiment</td>
</tr><tr id="pkg_BADER" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BADER.html">BADER</a></td>
	<td class=" ">Andreas Neudecker</td>
	<td class=" ">Bayesian Analysis of Differential Expression in RNA Sequencing Data</td>
</tr><tr id="pkg_BAGS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BAGS.html">BAGS</a></td>
	<td class=" ">Alejandro Quiroz-Zarate</td>
	<td class=" ">A Bayesian Approach for Geneset Selection</td>
</tr><tr id="pkg_ballgown" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ballgown.html">ballgown</a></td>
	<td class=" ">Alyssa Frazee</td>
	<td class=" ">Flexible, isoform-level differential expression analysis</td>
</tr><tr id="pkg_bamsignals" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/bamsignals.html">bamsignals</a></td>
	<td class=" ">Alessandro Mammana</td>
	<td class=" ">Extract read count signals from bam files</td>
</tr><tr id="pkg_BaseSpaceR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BaseSpaceR.html">BaseSpaceR</a></td>
	<td class=" ">Jared O'Connell</td>
	<td class=" ">R SDK for BaseSpace RESTful API</td>
</tr><tr id="pkg_Basic4Cseq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Basic4Cseq.html">Basic4Cseq</a></td>
	<td class=" ">Carolin Walter</td>
	<td class=" ">Basic4Cseq: an R/Bioconductor package for analyzing 4C-seq data</td>
</tr><tr id="pkg_BayesPeak" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BayesPeak.html">BayesPeak</a></td>
	<td class=" ">Jonathan Cairns</td>
	<td class=" ">Bayesian Analysis of ChIP-seq Data</td>
</tr><tr id="pkg_baySeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/baySeq.html">baySeq</a></td>
	<td class=" ">Thomas J. Hardcastle</td>
	<td class=" ">Empirical Bayesian analysis of patterns of differential expression in count data</td>
</tr><tr id="pkg_BCRANK" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BCRANK.html">BCRANK</a></td>
	<td class=" ">Adam Ameur</td>
	<td class=" ">Predicting binding site consensus from ranked DNA sequences</td>
</tr><tr id="pkg_beadarray" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/beadarray.html">beadarray</a></td>
	<td class=" ">Mark Dunning</td>
	<td class=" ">Quality assessment and low-level analysis for Illumina BeadArray data</td>
</tr><tr id="pkg_beadarraySNP" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/beadarraySNP.html">beadarraySNP</a></td>
	<td class=" ">Jan Oosting</td>
	<td class=" ">Normalization and reporting of Illumina SNP bead arrays</td>
</tr><tr id="pkg_BeadDataPackR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BeadDataPackR.html">BeadDataPackR</a></td>
	<td class=" ">Mike Smith</td>
	<td class=" ">Compression of Illumina BeadArray data</td>
</tr><tr id="pkg_BEAT" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BEAT.html">BEAT</a></td>
	<td class=" ">Kemal Akman</td>
	<td class=" ">BEAT - BS-Seq Epimutation Analysis Toolkit</td>
</tr><tr id="pkg_BEclear" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BEclear.html">BEclear</a></td>
	<td class=" ">Markus Merl</td>
	<td class=" ">Correct for batch effects in DNA methylation data</td>
</tr><tr id="pkg_betr" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/betr.html">betr</a></td>
	<td class=" ">Martin Aryee</td>
	<td class=" ">Identify differentially expressed genes in microarray time-course data</td>
</tr><tr id="pkg_bgafun" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/bgafun.html">bgafun</a></td>
	<td class=" ">Iain Wallace</td>
	<td class=" ">BGAfun A method to identify specifity determining residues in protein families</td>
</tr><tr id="pkg_BGmix" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BGmix.html">BGmix</a></td>
	<td class=" ">Alex Lewin</td>
	<td class=" ">Bayesian models for differential gene expression</td>
</tr><tr id="pkg_bgx" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/bgx.html">bgx</a></td>
	<td class=" ">Ernest Turro</td>
	<td class=" ">Bayesian Gene eXpression</td>
</tr><tr id="pkg_BHC" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BHC.html">BHC</a></td>
	<td class=" ">Rich Savage</td>
	<td class=" ">Bayesian Hierarchical Clustering</td>
</tr><tr id="pkg_BicARE" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BicARE.html">BicARE</a></td>
	<td class=" ">Pierre Gestraud</td>
	<td class=" ">Biclustering Analysis and Results Exploration</td>
</tr><tr id="pkg_BiGGR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BiGGR.html">BiGGR</a></td>
	<td class=" ">Anand K. Gavai, Hannes Hettling</td>
	<td class=" ">Constraint based modeling in R using metabolic reconstruction databases.</td>
</tr><tr id="pkg_bigmemoryExtras" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/bigmemoryExtras.html">bigmemoryExtras</a></td>
	<td class=" ">Peter M. Haverty</td>
	<td class=" ">An extension of the bigmemory package with added safety, convenience, and a factor class.</td>
</tr><tr id="pkg_bioassayR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/bioassayR.html">bioassayR</a></td>
	<td class=" ">Tyler Backman</td>
	<td class=" ">R library for Bioactivity analysis</td>
</tr><tr id="pkg_Biobase" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Biobase.html">Biobase</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Biobase: Base functions for Bioconductor</td>
</tr><tr id="pkg_BiocCaseStudies" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BiocCaseStudies.html">BiocCaseStudies</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">BiocCaseStudies: Support for the Case Studies Monograph</td>
</tr><tr id="pkg_BiocCheck" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BiocCheck.html">BiocCheck</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Bioconductor-specific package checks</td>
</tr><tr id="pkg_BiocGenerics" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BiocGenerics.html">BiocGenerics</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">S4 generic functions for Bioconductor</td>
</tr><tr id="pkg_biocGraph" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/biocGraph.html">biocGraph</a></td>
	<td class=" ">Florian Hahne</td>
	<td class=" ">Graph examples and use cases in Bioinformatics</td>
</tr><tr id="pkg_BiocInstaller" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BiocInstaller.html">BiocInstaller</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Install/Update Bioconductor and CRAN Packages</td>
</tr><tr id="pkg_BiocParallel" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BiocParallel.html">BiocParallel</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Bioconductor facilities for parallel evaluation</td>
</tr><tr id="pkg_BiocStyle" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BiocStyle.html">BiocStyle</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Standard styles for vignettes and other Bioconductor documents</td>
</tr><tr id="pkg_biocViews" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/biocViews.html">biocViews</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Categorized views of R package repositories</td>
</tr><tr id="pkg_bioDist" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/bioDist.html">bioDist</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Different distance measures</td>
</tr><tr id="pkg_biomaRt" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/biomaRt.html">biomaRt</a></td>
	<td class=" ">Steffen Durinck</td>
	<td class=" ">Interface to BioMart databases (e.g. Ensembl, COSMIC ,Wormbase and Gramene)</td>
</tr><tr id="pkg_BioMVCClass" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BioMVCClass.html">BioMVCClass</a></td>
	<td class=" ">Elizabeth Whalen</td>
	<td class=" ">Model-View-Controller (MVC) Classes That Use Biobase</td>
</tr><tr id="pkg_biomvRCNS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/biomvRCNS.html">biomvRCNS</a></td>
	<td class=" ">Yang Du</td>
	<td class=" ">Copy Number study and Segmentation for multivariate biological data</td>
</tr><tr id="pkg_BioNet" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BioNet.html">BioNet</a></td>
	<td class=" ">Marcus Dittrich</td>
	<td class=" ">Routines for the functional analysis of biological networks</td>
</tr><tr id="pkg_BioSeqClass" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BioSeqClass.html">BioSeqClass</a></td>
	<td class=" ">Li Hong</td>
	<td class=" ">Classification for Biological Sequences</td>
</tr><tr id="pkg_Biostrings" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Biostrings.html">Biostrings</a></td>
	<td class=" ">H. Pages</td>
	<td class=" ">String objects representing biological sequences, and matching algorithms</td>
</tr><tr id="pkg_biosvd" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/biosvd.html">biosvd</a></td>
	<td class=" ">Anneleen Daemen, Matthew Brauer</td>
	<td class=" ">Package for high-throughput data processing, outlier detection, noise removal and dynamic modeling</td>
</tr><tr id="pkg_biovizBase" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/biovizBase.html">biovizBase</a></td>
	<td class=" ">Tengfei Yin</td>
	<td class=" ">Basic graphic utilities for visualization of genomic data.</td>
</tr><tr id="pkg_BiRewire" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BiRewire.html">BiRewire</a></td>
	<td class=" ">Andrea Gobbi</td>
	<td class=" ">High-performing routines for the randomization of a bipartite graph (or a binary event matrix) and directed signed graph preserving degree distribution (or marginal totals).</td>
</tr><tr id="pkg_birta" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/birta.html">birta</a></td>
	<td class=" ">Benedikt Zacher, Holger Froehlich</td>
	<td class=" ">Bayesian Inference of Regulation of Transcriptional Activity</td>
</tr><tr id="pkg_birte" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/birte.html">birte</a></td>
	<td class=" ">Holger Froehlich</td>
	<td class=" ">Bayesian Inference of Regulatory Influence on Expression (biRte)</td>
</tr><tr id="pkg_BiSeq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BiSeq.html">BiSeq</a></td>
	<td class=" ">Katja Hebestreit</td>
	<td class=" ">Processing and analyzing bisulfite sequencing data</td>
</tr><tr id="pkg_BitSeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BitSeq.html">BitSeq</a></td>
	<td class=" ">Antti Honkela, Panagiotis Papastamoulis</td>
	<td class=" ">Transcript expression inference and differential expression analysis for RNA-seq data</td>
</tr><tr id="pkg_blima" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/blima.html">blima</a></td>
	<td class=" ">Vojtech Kulvait</td>
	<td class=" ">Package for the preprocessing and analysis of the Illumina microarrays on the detector (bead) level.</td>
</tr><tr id="pkg_BRAIN" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BRAIN.html">BRAIN</a></td>
	<td class=" ">Piotr Dittwald</td>
	<td class=" ">Baffling Recursive Algorithm for Isotope distributioN calculations</td>
</tr><tr id="pkg_BrainStars" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BrainStars.html">BrainStars</a></td>
	<td class=" ">Itoshi NIKAIDO</td>
	<td class=" ">query gene expression data and plots from BrainStars (B*)</td>
</tr><tr id="pkg_bridge" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/bridge.html">bridge</a></td>
	<td class=" ">Raphael Gottardo</td>
	<td class=" ">Bayesian Robust Inference for Differential Gene Expression</td>
</tr><tr id="pkg_BridgeDbR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BridgeDbR.html">BridgeDbR</a></td>
	<td class=" ">Anwesha Bohler</td>
	<td class=" ">Code for using BridgeDb identifier mapping framework from within R</td>
</tr><tr id="pkg_BrowserViz" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BrowserViz.html">BrowserViz</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">BrowserViz: interactive R/browser graphics using websockets and JSON</td>
</tr><tr id="pkg_BrowserVizDemo" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BrowserVizDemo.html">BrowserVizDemo</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">BrowserVizDemo: How to subclass BrowserViz</td>
</tr><tr id="pkg_BSgenome" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BSgenome.html">BSgenome</a></td>
	<td class=" ">H. Pages</td>
	<td class=" ">Infrastructure for Biostrings-based genome data packages</td>
</tr><tr id="pkg_bsseq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/bsseq.html">bsseq</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Analyze, manage and store bisulfite sequencing data.</td>
</tr><tr id="pkg_BubbleTree" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BubbleTree.html">BubbleTree</a></td>
	<td class=" ">Wei Zhu</td>
	<td class=" ">A method to elucidate purity and clonality in tumors using copy number ratio and allele frequency</td>
</tr><tr id="pkg_BufferedMatrix" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BufferedMatrix.html">BufferedMatrix</a></td>
	<td class=" ">Benjamin Milo Bolstad</td>
	<td class=" ">A matrix data storage object held in temporary files</td>
</tr><tr id="pkg_BufferedMatrixMethods" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BufferedMatrixMethods.html">BufferedMatrixMethods</a></td>
	<td class=" ">B. M. Bolstad</td>
	<td class=" ">Microarray Data related methods that utlize BufferedMatrix objects</td>
</tr><tr id="pkg_bumphunter" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/bumphunter.html">bumphunter</a></td>
	<td class=" ">Rafael A. Irizarry</td>
	<td class=" ">Bump Hunter</td>
</tr><tr id="pkg_BUS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/BUS.html">BUS</a></td>
	<td class=" ">Yuanhua Liu</td>
	<td class=" ">Gene network reconstruction</td>
</tr><tr id="pkg_CAFE" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CAFE.html">CAFE</a></td>
	<td class=" ">Sander Bollen</td>
	<td class=" ">Chromosmal Aberrations Finder in Expression data</td>
</tr><tr id="pkg_CAGEr" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CAGEr.html">CAGEr</a></td>
	<td class=" ">Vanja Haberle</td>
	<td class=" ">Analysis of CAGE (Cap Analysis of Gene Expression) sequencing data for precise mapping of transcription start sites and promoterome mining</td>
</tr><tr id="pkg_CALIB" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CALIB.html">CALIB</a></td>
	<td class=" ">Hui Zhao</td>
	<td class=" ">Calibration model for estimating absolute expression levels from microarray data</td>
</tr><tr id="pkg_CAMERA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CAMERA.html">CAMERA</a></td>
	<td class=" ">Carsten Kuhl</td>
	<td class=" ">Collection of annotation related methods for mass spectrometry data</td>
</tr><tr id="pkg_canceR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/canceR.html">canceR</a></td>
	<td class=" ">Karim Mezhoud</td>
	<td class=" ">A Graphical User Interface for accessing and modeling the Cancer Genomics Data of MSKCC.</td>
</tr><tr id="pkg_cancerclass" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cancerclass.html">cancerclass</a></td>
	<td class=" ">Daniel Kosztyla</td>
	<td class=" ">Development and validation of diagnostic tests from high-dimensional molecular data</td>
</tr><tr id="pkg_CancerMutationAnalysis" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CancerMutationAnalysis.html">CancerMutationAnalysis</a></td>
	<td class=" ">Simina M. Boca</td>
	<td class=" ">Cancer mutation analysis</td>
</tr><tr id="pkg_CAnD" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CAnD.html">CAnD</a></td>
	<td class=" ">Caitlin McHugh</td>
	<td class=" ">Perform Chromosomal Ancestry Differences (CAnD) Analyses</td>
</tr><tr id="pkg_Cardinal" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Cardinal.html">Cardinal</a></td>
	<td class=" ">Kyle D. Bemis</td>
	<td class=" ">A mass spectrometry imaging toolbox for statistical analysis</td>
</tr><tr id="pkg_casper" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/casper.html">casper</a></td>
	<td class=" ">David Rossell</td>
	<td class=" ">Characterization of Alternative Splicing based on Paired-End Reads</td>
</tr><tr id="pkg_Category" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Category.html">Category</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Category Analysis</td>
</tr><tr id="pkg_categoryCompare" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/categoryCompare.html">categoryCompare</a></td>
	<td class=" ">Robert M. Flight</td>
	<td class=" ">Meta-analysis of high-throughput experiments using feature annotations</td>
</tr><tr id="pkg_ccrepe" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ccrepe.html">ccrepe</a></td>
	<td class=" ">Emma Schwager,Craig Bielski, George Weingart</td>
	<td class=" ">ccrepe_and_nc.score</td>
</tr><tr id="pkg_cellGrowth" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cellGrowth.html">cellGrowth</a></td>
	<td class=" ">Julien Gagneur</td>
	<td class=" ">Fitting cell population growth models</td>
</tr><tr id="pkg_cellHTS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cellHTS.html">cellHTS</a></td>
	<td class=" ">Ligia Bras</td>
	<td class=" ">Analysis of cell-based screens</td>
</tr><tr id="pkg_cellHTS2" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cellHTS2.html">cellHTS2</a></td>
	<td class=" ">Joseph Barry</td>
	<td class=" ">Analysis of cell-based screens - revised version of cellHTS</td>
</tr><tr id="pkg_CellNOptR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CellNOptR.html">CellNOptR</a></td>
	<td class=" ">T.Cokelaer</td>
	<td class=" ">Training of boolean logic models of signalling networks using prior knowledge networks and perturbation data.</td>
</tr><tr id="pkg_CexoR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CexoR.html">CexoR</a></td>
	<td class=" ">Pedro Madrigal</td>
	<td class=" ">An R package to uncover high-resolution protein-DNA interactions in ChIP-exo replicates</td>
</tr><tr id="pkg_CFAssay" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CFAssay.html">CFAssay</a></td>
	<td class=" ">Herbert Braselmann</td>
	<td class=" ">Statistical analysis for the Colony Formation Assay</td>
</tr><tr id="pkg_CGEN" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CGEN.html">CGEN</a></td>
	<td class=" ">William Wheeler</td>
	<td class=" ">An R package for analysis of case-control studies in genetic epidemiology</td>
</tr><tr id="pkg_CGHbase" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CGHbase.html">CGHbase</a></td>
	<td class=" ">Mark van de Wiel</td>
	<td class=" ">CGHbase: Base functions and classes for arrayCGH data analysis.</td>
</tr><tr id="pkg_CGHcall" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CGHcall.html">CGHcall</a></td>
	<td class=" ">Mark van de Wiel</td>
	<td class=" ">Calling aberrations for array CGH tumor profiles.</td>
</tr><tr id="pkg_cghMCR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cghMCR.html">cghMCR</a></td>
	<td class=" ">J. Zhang</td>
	<td class=" ">Find chromosome regions showing common gains/losses</td>
</tr><tr id="pkg_CGHnormaliter" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CGHnormaliter.html">CGHnormaliter</a></td>
	<td class=" ">Bart P.P. van Houte</td>
	<td class=" ">Normalization of array CGH data with imbalanced aberrations.</td>
</tr><tr id="pkg_CGHregions" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CGHregions.html">CGHregions</a></td>
	<td class=" ">Sjoerd Vosse</td>
	<td class=" ">Dimension Reduction for Array CGH Data with Minimal Information Loss.</td>
</tr><tr id="pkg_ChAMP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ChAMP.html">ChAMP</a></td>
	<td class=" ">Tiffany Morris</td>
	<td class=" ">Chip Analysis Methylation Pipeline for Illumina HumanMethylation450</td>
</tr><tr id="pkg_charm" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/charm.html">charm</a></td>
	<td class=" ">Peter Murakami</td>
	<td class=" ">Analysis of DNA methylation data from CHARM microarrays</td>
</tr><tr id="pkg_ChemmineOB" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ChemmineOB.html">ChemmineOB</a></td>
	<td class=" ">Thomas Girke</td>
	<td class=" ">R interface to a subset of OpenBabel functionalities</td>
</tr><tr id="pkg_ChemmineR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ChemmineR.html">ChemmineR</a></td>
	<td class=" ">Thomas Girke</td>
	<td class=" ">Cheminformatics Toolkit for R</td>
</tr><tr id="pkg_chimera" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/chimera.html">chimera</a></td>
	<td class=" ">Raffaele A Calogero</td>
	<td class=" ">A package for secondary analysis of fusion products</td>
</tr><tr id="pkg_chipenrich" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/chipenrich.html">chipenrich</a></td>
	<td class=" ">Ryan P. Welch, Chee Lee, Raymond G. Cavalcante</td>
	<td class=" ">Gene set enrichment for ChIP-seq peak data</td>
</tr><tr id="pkg_ChIPpeakAnno" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ChIPpeakAnno.html">ChIPpeakAnno</a></td>
	<td class=" ">Lihua Julie Zhu, Jianhong Ou</td>
	<td class=" ">Batch annotation of the peaks identified from either ChIP-seq, ChIP-chip experiments or any experiments resulted in large number of chromosome ranges.</td>
</tr><tr id="pkg_ChIPQC" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ChIPQC.html">ChIPQC</a></td>
	<td class=" ">Tom Carroll, Rory Stark</td>
	<td class=" ">Quality metrics for ChIPseq data</td>
</tr><tr id="pkg_ChIPseeker" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ChIPseeker.html">ChIPseeker</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">ChIPseeker for ChIP peak Annotation, Comparison, and Visualization</td>
</tr><tr id="pkg_chipseq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/chipseq.html">chipseq</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">chipseq: A package for analyzing chipseq data</td>
</tr><tr id="pkg_ChIPseqR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ChIPseqR.html">ChIPseqR</a></td>
	<td class=" ">Peter Humburg</td>
	<td class=" ">Identifying Protein Binding Sites in High-Throughput Sequencing Data</td>
</tr><tr id="pkg_ChIPsim" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ChIPsim.html">ChIPsim</a></td>
	<td class=" ">Peter Humburg</td>
	<td class=" ">Simulation of ChIP-seq experiments</td>
</tr><tr id="pkg_ChIPXpress" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ChIPXpress.html">ChIPXpress</a></td>
	<td class=" ">George Wu</td>
	<td class=" ">ChIPXpress: enhanced transcription factor target gene identification from ChIP-seq and ChIP-chip data using publicly available gene expression profiles</td>
</tr><tr id="pkg_chopsticks" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/chopsticks.html">chopsticks</a></td>
	<td class=" ">Hin-Tak Leung</td>
	<td class=" ">The snp.matrix and X.snp.matrix classes</td>
</tr><tr id="pkg_chroGPS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/chroGPS.html">chroGPS</a></td>
	<td class=" ">Oscar Reina</td>
	<td class=" ">chroGPS: visualizing the epigenome</td>
</tr><tr id="pkg_chromDraw" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/chromDraw.html">chromDraw</a></td>
	<td class=" ">Jan Janecka</td>
	<td class=" ">chromDraw an R package for visualization of linear and circular karyotypes.</td>
</tr><tr id="pkg_ChromHeatMap" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ChromHeatMap.html">ChromHeatMap</a></td>
	<td class=" ">Tim F. Rayner</td>
	<td class=" ">Heat map plotting by genome coordinate</td>
</tr><tr id="pkg_cisPath" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cisPath.html">cisPath</a></td>
	<td class=" ">Likun Wang</td>
	<td class=" ">Visualization and management of the protein-protein interaction networks.</td>
</tr><tr id="pkg_ClassifyR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ClassifyR.html">ClassifyR</a></td>
	<td class=" ">Dario Strbenac</td>
	<td class=" ">A framework for two-class classification problems, with applications to differential variability and differential distribution testing.</td>
</tr><tr id="pkg_cleanUpdTSeq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cleanUpdTSeq.html">cleanUpdTSeq</a></td>
	<td class=" ">Sarah Sheppard; Jianhong Ou; Lihua Julie Zhu</td>
	<td class=" ">This package classifies putative polyadenylation sites as true or false/internally oligodT primed.</td>
</tr><tr id="pkg_cleaver" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cleaver.html">cleaver</a></td>
	<td class=" ">Sebastian Gibb</td>
	<td class=" ">Cleavage of Polypeptide Sequences</td>
</tr><tr id="pkg_clippda" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/clippda.html">clippda</a></td>
	<td class=" ">Stephen Nyangoma</td>
	<td class=" ">A package for the clinical proteomic profiling data analysis</td>
</tr><tr id="pkg_clipper" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/clipper.html">clipper</a></td>
	<td class=" ">Paolo Martini</td>
	<td class=" ">Gene set analysis exploiting pathway topology</td>
</tr><tr id="pkg_Clomial" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Clomial.html">Clomial</a></td>
	<td class=" ">Habil Zare</td>
	<td class=" ">Infers clonal composition of a tumor</td>
</tr><tr id="pkg_Clonality" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Clonality.html">Clonality</a></td>
	<td class=" ">Irina Ostrovnaya</td>
	<td class=" ">Clonality testing</td>
</tr><tr id="pkg_clonotypeR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/clonotypeR.html">clonotypeR</a></td>
	<td class=" ">Charles Plessy</td>
	<td class=" ">High throughput analysis of T cell antigen receptor sequences</td>
</tr><tr id="pkg_clst" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/clst.html">clst</a></td>
	<td class=" ">Noah Hoffman</td>
	<td class=" ">Classification by local similarity threshold</td>
</tr><tr id="pkg_clstutils" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/clstutils.html">clstutils</a></td>
	<td class=" ">Noah Hoffman</td>
	<td class=" ">Tools for performing taxonomic assignment.</td>
</tr><tr id="pkg_clusterProfiler" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/clusterProfiler.html">clusterProfiler</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">statistical analysis and visulization of functional profiles for genes and gene clusters</td>
</tr><tr id="pkg_clusterStab" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/clusterStab.html">clusterStab</a></td>
	<td class=" ">James W. MacDonald</td>
	<td class=" ">Compute cluster stability scores for microarray data</td>
</tr><tr id="pkg_CMA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CMA.html">CMA</a></td>
	<td class=" ">Christoph Bernau</td>
	<td class=" ">Synthesis of microarray-based classification</td>
</tr><tr id="pkg_cn.farms" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cn.farms.html">cn.farms</a></td>
	<td class=" ">Andreas Mitterecker</td>
	<td class=" ">cn.FARMS - factor analysis for copy number estimation</td>
</tr><tr id="pkg_cn.mops" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cn.mops.html">cn.mops</a></td>
	<td class=" ">Guenter Klambauer</td>
	<td class=" ">cn.mops - Mixture of Poissons for CNV detection in NGS data</td>
</tr><tr id="pkg_CNAnorm" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CNAnorm.html">CNAnorm</a></td>
	<td class=" ">Stefano Berri</td>
	<td class=" ">A normalization method for Copy Number Aberration in cancer samples</td>
</tr><tr id="pkg_CNEr" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CNEr.html">CNEr</a></td>
	<td class=" ">Ge Tan</td>
	<td class=" ">CNE detection and visualization.</td>
</tr><tr id="pkg_CNORdt" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CNORdt.html">CNORdt</a></td>
	<td class=" ">A. MacNamara</td>
	<td class=" ">Add-on to CellNOptR: Discretized time treatments</td>
</tr><tr id="pkg_CNORfeeder" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CNORfeeder.html">CNORfeeder</a></td>
	<td class=" ">F.Eduati</td>
	<td class=" ">Integration of CellNOptR to add missing links</td>
</tr><tr id="pkg_CNORfuzzy" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CNORfuzzy.html">CNORfuzzy</a></td>
	<td class=" ">T. Cokelaer</td>
	<td class=" ">Addon to CellNOptR: Fuzzy Logic</td>
</tr><tr id="pkg_CNORode" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CNORode.html">CNORode</a></td>
	<td class=" ">David Henriques</td>
	<td class=" ">ODE add-on to CellNOptR</td>
</tr><tr id="pkg_CNTools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CNTools.html">CNTools</a></td>
	<td class=" ">J. Zhang</td>
	<td class=" ">Convert segment data into a region by sample matrix to allow for other high level computational analyses.</td>
</tr><tr id="pkg_cnvGSA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cnvGSA.html">cnvGSA</a></td>
	<td class=" ">Joseph Lugo</td>
	<td class=" ">Gene Set Analysis of (Rare) Copy Number Variants</td>
</tr><tr id="pkg_CNVrd2" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CNVrd2.html">CNVrd2</a></td>
	<td class=" ">Hoang Tan Nguyen</td>
	<td class=" ">CNVrd2: a read depth-based method to detect and genotype complex common copy number variants from next generation sequencing data.</td>
</tr><tr id="pkg_CNVtools" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CNVtools.html">CNVtools</a></td>
	<td class=" ">Chris Barnes</td>
	<td class=" ">A package to test genetic association with CNV data</td>
</tr><tr id="pkg_cobindR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cobindR.html">cobindR</a></td>
	<td class=" ">Manuela Benary</td>
	<td class=" ">Finding Co-occuring motifs of transcription factor binding sites</td>
</tr><tr id="pkg_CoCiteStats" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CoCiteStats.html">CoCiteStats</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Different test statistics based on co-citation.</td>
</tr><tr id="pkg_codelink" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/codelink.html">codelink</a></td>
	<td class=" ">Diego Diez</td>
	<td class=" ">Manipulation of Codelink microarray data.</td>
</tr><tr id="pkg_CODEX" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CODEX.html">CODEX</a></td>
	<td class=" ">Yuchao Jiang</td>
	<td class=" ">A Normalization and Copy Number Variation Detection Method for Whole Exome Sequencing</td>
</tr><tr id="pkg_CoGAPS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CoGAPS.html">CoGAPS</a></td>
	<td class=" ">Elana J. Fertig, Michael F. Ochs</td>
	<td class=" ">Coordinated Gene Activity in Pattern Sets</td>
</tr><tr id="pkg_cogena" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cogena.html">cogena</a></td>
	<td class=" ">Zhilong Jia</td>
	<td class=" ">co-expressed gene-set enrichment analysis</td>
</tr><tr id="pkg_coGPS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/coGPS.html">coGPS</a></td>
	<td class=" ">Yingying Wei</td>
	<td class=" ">cancer outlier Gene Profile Sets</td>
</tr><tr id="pkg_COHCAP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/COHCAP.html">COHCAP</a></td>
	<td class=" ">Charles Warden</td>
	<td class=" ">CpG Island Analysis Pipeline for Illumina Methylation Array and Targeted BS-Seq Data</td>
</tr><tr id="pkg_coMET" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/coMET.html">coMET</a></td>
	<td class=" ">Tiphaine Martin</td>
	<td class=" ">coMET: visualisation of regional epigenome-wide association scan (EWAS) results and DNA co-methylation patterns.</td>
</tr><tr id="pkg_COMPASS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/COMPASS.html">COMPASS</a></td>
	<td class=" ">Greg Finak</td>
	<td class=" ">Combinatorial Polyfunctionality Analysis of Single Cells</td>
</tr><tr id="pkg_compcodeR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/compcodeR.html">compcodeR</a></td>
	<td class=" ">Charlotte Soneson</td>
	<td class=" ">RNAseq data simulation, differential expression analysis and performance comparison of differential expression methods</td>
</tr><tr id="pkg_compEpiTools" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/compEpiTools.html">compEpiTools</a></td>
	<td class=" ">Kamal Kishore</td>
	<td class=" ">Tools for computational epigenomics</td>
</tr><tr id="pkg_CompGO" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CompGO.html">CompGO</a></td>
	<td class=" ">Ashley J. Waardenberg</td>
	<td class=" ">An R pipeline for .bed file annotation, comparing GO term enrichment between gene sets and data visualisation</td>
</tr><tr id="pkg_ComplexHeatmap" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html">ComplexHeatmap</a></td>
	<td class=" ">Zuguang Gu</td>
	<td class=" ">Making Complex Heatmaps</td>
</tr><tr id="pkg_ConsensusClusterPlus" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html">ConsensusClusterPlus</a></td>
	<td class=" ">Matt Wilkerson</td>
	<td class=" ">ConsensusClusterPlus</td>
</tr><tr id="pkg_conumee" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/conumee.html">conumee</a></td>
	<td class=" ">Volker Hovestadt</td>
	<td class=" ">Enhanced copy-number variation analysis using Illumina 450k methylation arrays</td>
</tr><tr id="pkg_convert" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/convert.html">convert</a></td>
	<td class=" ">Yee Hwa (Jean) Yang</td>
	<td class=" ">Convert Microarray Data Objects</td>
</tr><tr id="pkg_copa" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/copa.html">copa</a></td>
	<td class=" ">James W. MacDonald</td>
	<td class=" ">Functions to perform cancer outlier profile analysis.</td>
</tr><tr id="pkg_copynumber" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/copynumber.html">copynumber</a></td>
	<td class=" ">Gro Nilsen</td>
	<td class=" ">Segmentation of single- and multi-track copy number data by penalized least squares regression.</td>
</tr><tr id="pkg_CopyNumber450k" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CopyNumber450k.html">CopyNumber450k</a></td>
	<td class=" ">Simon Papillon-Cavanagh</td>
	<td class=" ">R package for calling CNV from Illumina 450k methylation microarrays</td>
</tr><tr id="pkg_CopywriteR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CopywriteR.html">CopywriteR</a></td>
	<td class=" ">Thomas Kuilman</td>
	<td class=" ">Copy number information from targeted sequencing using off-target reads</td>
</tr><tr id="pkg_CoRegNet" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CoRegNet.html">CoRegNet</a></td>
	<td class=" ">Remy Nicolle</td>
	<td class=" ">CoRegNet : reconstruction and integrated analysis of co-regulatory networks</td>
</tr><tr id="pkg_Cormotif" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Cormotif.html">Cormotif</a></td>
	<td class=" ">Yingying Wei</td>
	<td class=" ">Correlation Motif Fit</td>
</tr><tr id="pkg_CorMut" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CorMut.html">CorMut</a></td>
	<td class=" ">Zhenpeng Li</td>
	<td class=" ">Detect the correlated mutations based on selection pressure</td>
</tr><tr id="pkg_coRNAi" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/coRNAi.html">coRNAi</a></td>
	<td class=" ">Elin Axelsson</td>
	<td class=" ">Analysis of co-knock-down RNAi data</td>
</tr><tr id="pkg_CORREP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CORREP.html">CORREP</a></td>
	<td class=" ">Dongxiao Zhu</td>
	<td class=" ">Multivariate Correlation Estimator and Statistical Inference Procedures.</td>
</tr><tr id="pkg_cosmiq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cosmiq.html">cosmiq</a></td>
	<td class=" ">David Fischer, Christian Panse</td>
	<td class=" ">cosmiq - COmbining Single Masses Into Quantities</td>
</tr><tr id="pkg_COSNet" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/COSNet.html">COSNet</a></td>
	<td class=" ">Marco Frasca</td>
	<td class=" ">Cost Sensitive Network for node label prediction on graphs with highly unbalanced labelings</td>
</tr><tr id="pkg_CoverageView" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CoverageView.html">CoverageView</a></td>
	<td class=" ">Ernesto Lowy</td>
	<td class=" ">Coverage visualization package for R</td>
</tr><tr id="pkg_cpvSNP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cpvSNP.html">cpvSNP</a></td>
	<td class=" ">Caitlin McHugh</td>
	<td class=" ">Gene set analysis methods for SNP association p-values that lie in genes in given gene sets</td>
</tr><tr id="pkg_cqn" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cqn.html">cqn</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Conditional quantile normalization</td>
</tr><tr id="pkg_CRImage" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CRImage.html">CRImage</a></td>
	<td class=" ">Henrik Failmezger, Yinyin Yuan</td>
	<td class=" ">CRImage a package to classify cells and calculate tumour cellularity</td>
</tr><tr id="pkg_CRISPRseek" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CRISPRseek.html">CRISPRseek</a></td>
	<td class=" ">Lihua Julie Zhu</td>
	<td class=" ">Design of target-specific guide RNAs in CRISPR-Cas9, genome-editing systems</td>
</tr><tr id="pkg_crlmm" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/crlmm.html">crlmm</a></td>
	<td class=" ">Benilton S Carvalho, Robert Scharpf, Matt Ritchie</td>
	<td class=" ">Genotype Calling (CRLMM) and Copy Number Analysis tool for Affymetrix SNP 5.0 and 6.0 and Illumina arrays.</td>
</tr><tr id="pkg_CSAR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CSAR.html">CSAR</a></td>
	<td class=" ">Jose M Muino</td>
	<td class=" ">Statistical tools for the analysis of ChIP-seq data</td>
</tr><tr id="pkg_csaw" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/csaw.html">csaw</a></td>
	<td class=" ">Aaron Lun</td>
	<td class=" ">ChIP-seq analysis with windows</td>
</tr><tr id="pkg_CSSP" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/CSSP.html">CSSP</a></td>
	<td class=" ">Chandler Zuo</td>
	<td class=" ">ChIP-Seq Statistical Power</td>
</tr><tr id="pkg_ctc" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ctc.html">ctc</a></td>
	<td class=" ">Antoine Lucas</td>
	<td class=" ">Cluster and Tree Conversion.</td>
</tr><tr id="pkg_cummeRbund" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cummeRbund.html">cummeRbund</a></td>
	<td class=" ">Loyal A. Goff</td>
	<td class=" ">Analysis, exploration, manipulation, and visualization of Cufflinks high-throughput sequencing data.</td>
</tr><tr id="pkg_customProDB" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/customProDB.html">customProDB</a></td>
	<td class=" ">xiaojing wang</td>
	<td class=" ">Generate customized protein database from NGS data, with a focus on RNA-Seq data, for proteomics search.</td>
</tr><tr id="pkg_cycle" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cycle.html">cycle</a></td>
	<td class=" ">Matthias Futschik</td>
	<td class=" ">Significance of periodic expression pattern in time-series data</td>
</tr><tr id="pkg_cytofkit" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/cytofkit.html">cytofkit</a></td>
	<td class=" ">Jinmiao Chen</td>
	<td class=" ">cytofkit: an integrated analysis pipeline for mass cytometry data</td>
</tr><tr id="pkg_dagLogo" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/dagLogo.html">dagLogo</a></td>
	<td class=" ">Jianhong Ou</td>
	<td class=" ">dagLogo</td>
</tr><tr id="pkg_daMA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/daMA.html">daMA</a></td>
	<td class=" ">Jobst Landgrebe</td>
	<td class=" ">Efficient design and analysis of factorial two-colour microarray data</td>
</tr><tr id="pkg_DART" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DART.html">DART</a></td>
	<td class=" ">Charles Shijie Zheng</td>
	<td class=" ">Denoising Algorithm based on Relevance network Topology</td>
</tr><tr id="pkg_DASiR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DASiR.html">DASiR</a></td>
	<td class=" ">Oscar Flores, Anna Mantsoki</td>
	<td class=" ">Distributed Annotation System in R</td>
</tr><tr id="pkg_DAVIDQuery" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DAVIDQuery.html">DAVIDQuery</a></td>
	<td class=" ">Roger Day</td>
	<td class=" ">Retrieval from the DAVID bioinformatics data resource into R</td>
</tr><tr id="pkg_DBChIP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DBChIP.html">DBChIP</a></td>
	<td class=" ">Kun Liang</td>
	<td class=" ">Differential Binding of Transcription Factor with ChIP-seq</td>
</tr><tr id="pkg_ddCt" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ddCt.html">ddCt</a></td>
	<td class=" ">Jitao David Zhang</td>
	<td class=" ">The ddCt Algorithm for the Analysis of Quantitative Real-Time PCR (qRT-PCR)</td>
</tr><tr id="pkg_ddgraph" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ddgraph.html">ddgraph</a></td>
	<td class=" ">Robert Stojnic</td>
	<td class=" ">Distinguish direct and indirect interactions with Graphical Modelling</td>
</tr><tr id="pkg_DECIPHER" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DECIPHER.html">DECIPHER</a></td>
	<td class=" ">Erik Wright</td>
	<td class=" ">Database Enabled Code for Ideal Probe Hybridization Employing R</td>
</tr><tr id="pkg_DeconRNASeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DeconRNASeq.html">DeconRNASeq</a></td>
	<td class=" ">Ting Gong</td>
	<td class=" ">Deconvolution of Heterogeneous Tissue Samples for mRNA-Seq data</td>
</tr><tr id="pkg_DEDS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DEDS.html">DEDS</a></td>
	<td class=" ">Yuanyuan Xiao</td>
	<td class=" ">Differential Expression via Distance Summary for Microarray Data</td>
</tr><tr id="pkg_deepSNV" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/deepSNV.html">deepSNV</a></td>
	<td class=" ">Moritz Gerstung</td>
	<td class=" ">Detection of subclonal SNVs in deep sequencing data.</td>
</tr><tr id="pkg_DEGraph" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DEGraph.html">DEGraph</a></td>
	<td class=" ">Laurent Jacob</td>
	<td class=" ">Two-sample tests on a graph</td>
</tr><tr id="pkg_DEGreport" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DEGreport.html">DEGreport</a></td>
	<td class=" ">Lorena Pantano</td>
	<td class=" ">Report of DEG analysis</td>
</tr><tr id="pkg_DEGseq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DEGseq.html">DEGseq</a></td>
	<td class=" ">Likun Wang</td>
	<td class=" ">Identify Differentially Expressed Genes from RNA-seq data</td>
</tr><tr id="pkg_deltaGseg" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/deltaGseg.html">deltaGseg</a></td>
	<td class=" ">Diana Low</td>
	<td class=" ">deltaGseg</td>
</tr><tr id="pkg_derfinder" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/derfinder.html">derfinder</a></td>
	<td class=" ">Leonardo Collado-Torres</td>
	<td class=" ">Annotation-agnostic differential expression analysis of RNA-seq data at base-pair resolution</td>
</tr><tr id="pkg_derfinderHelper" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/derfinderHelper.html">derfinderHelper</a></td>
	<td class=" ">Leonardo Collado-Torres</td>
	<td class=" ">derfinder helper package</td>
</tr><tr id="pkg_derfinderPlot" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/derfinderPlot.html">derfinderPlot</a></td>
	<td class=" ">Leonardo Collado-Torres</td>
	<td class=" ">Plotting functions for derfinder</td>
</tr><tr id="pkg_DESeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DESeq.html">DESeq</a></td>
	<td class=" ">Simon Anders</td>
	<td class=" ">Differential gene expression analysis based on the negative binomial distribution</td>
</tr><tr id="pkg_DESeq2" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DESeq2.html">DESeq2</a></td>
	<td class=" ">Michael Love</td>
	<td class=" ">Differential gene expression analysis based on the negative binomial distribution</td>
</tr><tr id="pkg_DEXSeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DEXSeq.html">DEXSeq</a></td>
	<td class=" ">Alejandro Reyes</td>
	<td class=" ">Inference of differential exon usage in RNA-Seq</td>
</tr><tr id="pkg_dexus" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/dexus.html">dexus</a></td>
	<td class=" ">Guenter Klambauer</td>
	<td class=" ">DEXUS - Identifying Differential Expression in RNA-Seq Studies with Unknown Conditions or without Replicates</td>
</tr><tr id="pkg_DFP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DFP.html">DFP</a></td>
	<td class=" ">Rodrigo Alvarez-Glez</td>
	<td class=" ">Gene Selection</td>
</tr><tr id="pkg_DiffBind" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DiffBind.html">DiffBind</a></td>
	<td class=" ">Rory Stark</td>
	<td class=" ">Differential Binding Analysis of ChIP-Seq peak data</td>
</tr><tr id="pkg_diffGeneAnalysis" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/diffGeneAnalysis.html">diffGeneAnalysis</a></td>
	<td class=" ">Choudary Jagarlamudi</td>
	<td class=" ">Performs differential gene expression Analysis</td>
</tr><tr id="pkg_diffHic" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/diffHic.html">diffHic</a></td>
	<td class=" ">Aaron Lun</td>
	<td class=" ">Differential analyis of Hi-C data</td>
</tr><tr id="pkg_diggit" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/diggit.html">diggit</a></td>
	<td class=" ">Mariano J Alvarez</td>
	<td class=" ">Inference of Genetic Variants Driving Cellular Phenotypes</td>
</tr><tr id="pkg_DirichletMultinomial" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DirichletMultinomial.html">DirichletMultinomial</a></td>
	<td class=" ">Martin Morgan</td>
	<td class=" ">Dirichlet-Multinomial Mixture Model Machine Learning for Microbiome Data</td>
</tr><tr id="pkg_dks" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/dks.html">dks</a></td>
	<td class=" ">Jeffrey T. Leek</td>
	<td class=" ">The double Kolmogorov-Smirnov package for evaluating multiple testing procedures.</td>
</tr><tr id="pkg_DMRcaller" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DMRcaller.html">DMRcaller</a></td>
	<td class=" ">Nicolae Radu Zabet</td>
	<td class=" ">Differentially Methylated Regions caller</td>
</tr><tr id="pkg_DMRcate" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DMRcate.html">DMRcate</a></td>
	<td class=" ">Tim Peters</td>
	<td class=" ">Illumina 450K methylation array spatial analysis methods</td>
</tr><tr id="pkg_DMRforPairs" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DMRforPairs.html">DMRforPairs</a></td>
	<td class=" ">Martin Rijlaarsdam</td>
	<td class=" ">DMRforPairs: identifying Differentially Methylated Regions between unique samples using array based methylation profiles</td>
</tr><tr id="pkg_DNAcopy" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DNAcopy.html">DNAcopy</a></td>
	<td class=" ">Venkatraman E. Seshan</td>
	<td class=" ">DNA copy number data analysis</td>
</tr><tr id="pkg_domainsignatures" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/domainsignatures.html">domainsignatures</a></td>
	<td class=" ">Florian Hahne</td>
	<td class=" ">Geneset enrichment based on InterPro domain signatures</td>
</tr><tr id="pkg_DOQTL" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DOQTL.html">DOQTL</a></td>
	<td class=" ">Daniel Gatti</td>
	<td class=" ">Genotyping and QTL Mapping in DO Mice</td>
</tr><tr id="pkg_DOSE" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DOSE.html">DOSE</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">Disease Ontology Semantic and Enrichment analysis</td>
</tr><tr id="pkg_DriverNet" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DriverNet.html">DriverNet</a></td>
	<td class=" ">Jiarui Ding</td>
	<td class=" ">Drivernet: uncovering somatic driver mutations modulating transcriptional networks in cancer</td>
</tr><tr id="pkg_DrugVsDisease" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DrugVsDisease.html">DrugVsDisease</a></td>
	<td class=" ">j. Saez-Rodriguez</td>
	<td class=" ">Comparison of disease and drug profiles using Gene set Enrichment Analysis</td>
</tr><tr id="pkg_DSS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DSS.html">DSS</a></td>
	<td class=" ">Hao Wu</td>
	<td class=" ">Dispersion shrinakge for sequencing data.</td>
</tr><tr id="pkg_DTA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DTA.html">DTA</a></td>
	<td class=" ">Bjoern Schwalb</td>
	<td class=" ">Dynamic Transcriptome Analysis</td>
</tr><tr id="pkg_dualKS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/dualKS.html">dualKS</a></td>
	<td class=" ">Eric J. Kort, Yarong Yang</td>
	<td class=" ">Dual KS Discriminant Analysis and Classification</td>
</tr><tr id="pkg_DupChecker" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DupChecker.html">DupChecker</a></td>
	<td class=" ">"Quanhu SHENG"</td>
	<td class=" ">a package for checking high-throughput genomic data redundancy in meta-analysis</td>
</tr><tr id="pkg_dyebias" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/dyebias.html">dyebias</a></td>
	<td class=" ">Philip Lijnzaad</td>
	<td class=" ">The GASSCO method for correcting for slide-dependent gene-specific dye bias</td>
</tr><tr id="pkg_DynDoc" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/DynDoc.html">DynDoc</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Dynamic document tools</td>
</tr><tr id="pkg_EasyqpcR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/EasyqpcR.html">EasyqpcR</a></td>
	<td class=" ">Le Pape Sylvain</td>
	<td class=" ">EasyqpcR for low-throughput real-time quantitative PCR data analysis</td>
</tr><tr id="pkg_easyRNASeq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/easyRNASeq.html">easyRNASeq</a></td>
	<td class=" ">Nicolas Delhomme</td>
	<td class=" ">Count summarization and normalization for RNA-Seq data</td>
</tr><tr id="pkg_EBarrays" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/EBarrays.html">EBarrays</a></td>
	<td class=" ">Ming Yuan</td>
	<td class=" ">Unified Approach for Simultaneous Gene Clustering and Differential Expression Identification</td>
</tr><tr id="pkg_EBcoexpress" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/EBcoexpress.html">EBcoexpress</a></td>
	<td class=" ">John A. Dawson</td>
	<td class=" ">EBcoexpress for Differential Co-Expression Analysis</td>
</tr><tr id="pkg_EBImage" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/EBImage.html">EBImage</a></td>
	<td class=" ">Andrzej OleÅ</td>
	<td class=" ">Image processing and analysis toolbox for R</td>
</tr><tr id="pkg_EBSeq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/EBSeq.html">EBSeq</a></td>
	<td class=" ">Ning Leng</td>
	<td class=" ">An R package for gene and isoform differential expression analysis of RNA-seq data</td>
</tr><tr id="pkg_EBSeqHMM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/EBSeqHMM.html">EBSeqHMM</a></td>
	<td class=" ">Ning Leng</td>
	<td class=" ">Bayesian analysis for identifying gene or isoform expression changes in ordered RNA-seq experiments</td>
</tr><tr id="pkg_ecolitk" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ecolitk.html">ecolitk</a></td>
	<td class=" ">Laurent Gautier</td>
	<td class=" ">Meta-data and tools for E. coli</td>
</tr><tr id="pkg_EDASeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/EDASeq.html">EDASeq</a></td>
	<td class=" ">Davide Risso</td>
	<td class=" ">Exploratory Data Analysis and Normalization for RNA-Seq</td>
</tr><tr id="pkg_EDDA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/EDDA.html">EDDA</a></td>
	<td class=" ">Chia Kuan Hui Burton, Niranjan Nagarajan</td>
	<td class=" ">Experimental Design in Differential Abundance analysis</td>
</tr><tr id="pkg_edge" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/edge.html">edge</a></td>
	<td class=" ">John D. Storey</td>
	<td class=" ">Extraction of Differential Gene Expression</td>
</tr><tr id="pkg_edgeR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/edgeR.html">edgeR</a></td>
	<td class=" ">Yunshun Chen, Aaron Lun, Mark Robinson, Davis McCarthy, Gordon Smyth</td>
	<td class=" ">Empirical analysis of digital gene expression data in R</td>
</tr><tr id="pkg_eiR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/eiR.html">eiR</a></td>
	<td class=" ">Thomas Girke</td>
	<td class=" ">Accelerated similarity searching of small molecules</td>
</tr><tr id="pkg_eisa" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/eisa.html">eisa</a></td>
	<td class=" ">Gabor Csardi</td>
	<td class=" ">Expression data analysis via the Iterative Signature Algorithm</td>
</tr><tr id="pkg_ELBOW" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ELBOW.html">ELBOW</a></td>
	<td class=" ">Graham Alvare, Xiangli Zhang</td>
	<td class=" ">ELBOW - Evaluating foLd change By the lOgit Way</td>
</tr><tr id="pkg_EMDomics" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/EMDomics.html">EMDomics</a></td>
	<td class=" ">Daniel Schmolze</td>
	<td class=" ">Earth Mover's Distance for Differential Analysis of Genomics Data</td>
</tr><tr id="pkg_ENCODExplorer" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ENCODExplorer.html">ENCODExplorer</a></td>
	<td class=" ">Charles Joly Beauparlant</td>
	<td class=" ">A compilation of ENCODE metadata</td>
</tr><tr id="pkg_ENmix" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ENmix.html">ENmix</a></td>
	<td class=" ">Zongli Xu</td>
	<td class=" ">Data preprocessing and quality control for Illumina HumanMethylation450 BeadChip</td>
</tr><tr id="pkg_EnrichmentBrowser" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/EnrichmentBrowser.html">EnrichmentBrowser</a></td>
	<td class=" ">Ludwig Geistlinger</td>
	<td class=" ">Seamless navigation through combined results of set-based and network-based enrichment analysis</td>
</tr><tr id="pkg_ensembldb" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ensembldb.html">ensembldb</a></td>
	<td class=" ">Johannes Rainer</td>
	<td class=" ">Utilities to create and use an Ensembl based annotation database</td>
</tr><tr id="pkg_ensemblVEP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ensemblVEP.html">ensemblVEP</a></td>
	<td class=" ">Valerie Obenchain</td>
	<td class=" ">R Interface to Ensembl Variant Effect Predictor</td>
</tr><tr id="pkg_ENVISIONQuery" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ENVISIONQuery.html">ENVISIONQuery</a></td>
	<td class=" ">Alex Lisovich, Roger Day</td>
	<td class=" ">Retrieval from the ENVISION bioinformatics data portal into R</td>
</tr><tr id="pkg_epigenomix" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/epigenomix.html">epigenomix</a></td>
	<td class=" ">Hans-Ulrich Klein</td>
	<td class=" ">Epigenetic and gene transcription data normalization and integration with mixture models</td>
</tr><tr id="pkg_epivizr" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/epivizr.html">epivizr</a></td>
	<td class=" ">Hector Corrada Bravo</td>
	<td class=" ">R Interface to epiviz web app</td>
</tr><tr id="pkg_erccdashboard" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/erccdashboard.html">erccdashboard</a></td>
	<td class=" ">Sarah Munro</td>
	<td class=" ">Assess Differential Gene Expression Experiments with ERCC Controls</td>
</tr><tr id="pkg_ExiMiR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ExiMiR.html">ExiMiR</a></td>
	<td class=" ">Sylvain Gubian</td>
	<td class=" ">R functions for the normalization of Exiqon miRNA array data</td>
</tr><tr id="pkg_exomeCopy" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/exomeCopy.html">exomeCopy</a></td>
	<td class=" ">Michael Love</td>
	<td class=" ">Copy number variant detection from exome sequencing read depth</td>
</tr><tr id="pkg_exomePeak" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/exomePeak.html">exomePeak</a></td>
	<td class=" ">Jia Meng</td>
	<td class=" ">exome-based anlaysis of MeRIP-Seq data: peak calling and differential analysis</td>
</tr><tr id="pkg_explorase" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/explorase.html">explorase</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">GUI for exploratory data analysis of systems biology data</td>
</tr><tr id="pkg_ExpressionView" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ExpressionView.html">ExpressionView</a></td>
	<td class=" ">Gabor Csardi</td>
	<td class=" ">Visualize biclusters identified in gene expression data</td>
</tr><tr id="pkg_fabia" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/fabia.html">fabia</a></td>
	<td class=" ">Sepp Hochreiter</td>
	<td class=" ">FABIA: Factor Analysis for Bicluster Acquisition</td>
</tr><tr id="pkg_facopy" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/facopy.html">facopy</a></td>
	<td class=" ">David Mosen-Ansorena</td>
	<td class=" ">Feature-based association and gene-set enrichment for copy number alteration analysis in cancer</td>
</tr><tr id="pkg_factDesign" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/factDesign.html">factDesign</a></td>
	<td class=" ">Denise Scholtens</td>
	<td class=" ">Factorial designed microarray experiment analysis</td>
</tr><tr id="pkg_farms" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/farms.html">farms</a></td>
	<td class=" ">Djork-Arne Clevert</td>
	<td class=" ">FARMS - Factor Analysis for Robust Microarray Summarization</td>
</tr><tr id="pkg_fastLiquidAssociation" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/fastLiquidAssociation.html">fastLiquidAssociation</a></td>
	<td class=" ">Tina Gunderson</td>
	<td class=" ">functions for genome-wide application of Liquid Association</td>
</tr><tr id="pkg_fastseg" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/fastseg.html">fastseg</a></td>
	<td class=" ">Guenter Klambauer</td>
	<td class=" ">fastseg - a fast segmentation algorithm</td>
</tr><tr id="pkg_fdrame" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/fdrame.html">fdrame</a></td>
	<td class=" ">Effi Kenigsberg</td>
	<td class=" ">FDR adjustments of Microarray Experiments (FDR-AME)</td>
</tr><tr id="pkg_FEM" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/FEM.html">FEM</a></td>
	<td class=" ">Andrew E. Teschendorff</td>
	<td class=" ">Identification of Functional Epigenetic Modules</td>
</tr><tr id="pkg_ffpe" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ffpe.html">ffpe</a></td>
	<td class=" ">Levi Waldron</td>
	<td class=" ">Quality assessment and control for FFPE microarray expression data</td>
</tr><tr id="pkg_FGNet" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/FGNet.html">FGNet</a></td>
	<td class=" ">Sara Aibar</td>
	<td class=" ">Functional Gene Networks derived from biological enrichment analyses</td>
</tr><tr id="pkg_FISHalyseR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/FISHalyseR.html">FISHalyseR</a></td>
	<td class=" ">Karesh Arunakirinathan, Andreas Heindl</td>
	<td class=" ">FISHalyseR a package for automated FISH quantification</td>
</tr><tr id="pkg_flagme" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flagme.html">flagme</a></td>
	<td class=" ">Mark Robinson, Riccardo Romoli</td>
	<td class=" ">Analysis of Metabolomics GC/MS Data</td>
</tr><tr id="pkg_flipflop" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flipflop.html">flipflop</a></td>
	<td class=" ">Elsa Bernard</td>
	<td class=" ">Fast lasso-based isoform prediction as a flow problem</td>
</tr><tr id="pkg_flowBeads" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowBeads.html">flowBeads</a></td>
	<td class=" ">Nikolas Pontikos</td>
	<td class=" ">flowBeads: Analysis of flow bead data</td>
</tr><tr id="pkg_flowBin" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowBin.html">flowBin</a></td>
	<td class=" ">Kieran O'Neill</td>
	<td class=" ">Combining multitube flow cytometry data by binning</td>
</tr><tr id="pkg_flowcatchR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowcatchR.html">flowcatchR</a></td>
	<td class=" ">Federico Marini</td>
	<td class=" ">Tools to analyze in vivo microscopy imaging data focused on tracking flowing blood cells</td>
</tr><tr id="pkg_flowCHIC" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowCHIC.html">flowCHIC</a></td>
	<td class=" ">Author: Joachim Schumann</td>
	<td class=" ">Analyze flow cytometric data using histogram information</td>
</tr><tr id="pkg_flowCL" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowCL.html">flowCL</a></td>
	<td class=" ">Justin Meskas</td>
	<td class=" ">Semantic labelling of flow cytometric cell populations</td>
</tr><tr id="pkg_flowClean" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowClean.html">flowClean</a></td>
	<td class=" ">Kipper Fletez-Brant</td>
	<td class=" ">flowClean</td>
</tr><tr id="pkg_flowClust" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowClust.html">flowClust</a></td>
	<td class=" ">Greg Finak, Mike Jiang</td>
	<td class=" ">Clustering for Flow Cytometry</td>
</tr><tr id="pkg_flowCore" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowCore.html">flowCore</a></td>
	<td class=" ">M.Jiang</td>
	<td class=" ">flowCore: Basic structures for flow cytometry data</td>
</tr><tr id="pkg_flowCyBar" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowCyBar.html">flowCyBar</a></td>
	<td class=" ">Joachim Schumann</td>
	<td class=" ">Analyze flow cytometric data using gate information</td>
</tr><tr id="pkg_flowDensity" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowDensity.html">flowDensity</a></td>
	<td class=" ">Mehrnoush Malek</td>
	<td class=" ">Sequential Flow Cytometry Data Gating</td>
</tr><tr id="pkg_flowFit" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowFit.html">flowFit</a></td>
	<td class=" ">Davide Rambaldi</td>
	<td class=" ">Estimate proliferation in cell-tracking dye studies</td>
</tr><tr id="pkg_flowFP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowFP.html">flowFP</a></td>
	<td class=" ">Herb Holyst</td>
	<td class=" ">Fingerprinting for Flow Cytometry</td>
</tr><tr id="pkg_flowMap" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowMap.html">flowMap</a></td>
	<td class=" ">Chiaowen Joyce Hsiao</td>
	<td class=" ">Mapping cell populations in flow cytometry data for cross-sample comparisons using the Friedman-Rafsky Test</td>
</tr><tr id="pkg_flowMatch" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowMatch.html">flowMatch</a></td>
	<td class=" ">Ariful Azad</td>
	<td class=" ">Matching and meta-clustering in flow cytometry</td>
</tr><tr id="pkg_flowMeans" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowMeans.html">flowMeans</a></td>
	<td class=" ">Nima Aghaeepour</td>
	<td class=" ">Non-parametric Flow Cytometry Data Gating</td>
</tr><tr id="pkg_flowMerge" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowMerge.html">flowMerge</a></td>
	<td class=" ">Greg Finak</td>
	<td class=" ">Cluster Merging for Flow Cytometry Data</td>
</tr><tr id="pkg_flowPeaks" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowPeaks.html">flowPeaks</a></td>
	<td class=" ">Yongchao Ge</td>
	<td class=" ">An R package for flow data clustering</td>
</tr><tr id="pkg_flowPlots" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowPlots.html">flowPlots</a></td>
	<td class=" ">N. Hawkins</td>
	<td class=" ">flowPlots: analysis plots and data class for gated flow cytometry data</td>
</tr><tr id="pkg_flowQ" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowQ.html">flowQ</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">Quality control for flow cytometry</td>
</tr><tr id="pkg_flowQB" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowQB.html">flowQB</a></td>
	<td class=" ">Faysal El Khettabi</td>
	<td class=" ">Automated Quadratic Characterization of Flow Cytometer Instrument Sensitivity: Q, B and CVinstrinsic calculations.</td>
</tr><tr id="pkg_FlowRepositoryR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/FlowRepositoryR.html">FlowRepositoryR</a></td>
	<td class=" ">Josef Spidlen</td>
	<td class=" ">FlowRepository R Interface</td>
</tr><tr id="pkg_FlowSOM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/FlowSOM.html">FlowSOM</a></td>
	<td class=" ">Sofie Van Gassen</td>
	<td class=" ">Using self-organizing maps for visualization and interpretation of cytometry data</td>
</tr><tr id="pkg_flowStats" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowStats.html">flowStats</a></td>
	<td class=" ">Greg Finak and Mike Jiang</td>
	<td class=" ">Statistical methods for the analysis of flow cytometry data</td>
</tr><tr id="pkg_flowTrans" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowTrans.html">flowTrans</a></td>
	<td class=" ">Greg Finak</td>
	<td class=" ">Parameter Optimization for Flow Cytometry Data Transformation</td>
</tr><tr id="pkg_flowType" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowType.html">flowType</a></td>
	<td class=" ">Nima Aghaeepour</td>
	<td class=" ">Phenotyping Flow Cytometry Assays</td>
</tr><tr id="pkg_flowUtils" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowUtils.html">flowUtils</a></td>
	<td class=" ">Josef Spidlen</td>
	<td class=" ">Utilities for flow cytometry</td>
</tr><tr id="pkg_flowViz" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowViz.html">flowViz</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">Visualization for flow cytometry</td>
</tr><tr id="pkg_flowVS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowVS.html">flowVS</a></td>
	<td class=" ">Ariful Azad</td>
	<td class=" ">Variance stabilization in flow cytometry (and microarrays)</td>
</tr><tr id="pkg_flowWorkspace" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/flowWorkspace.html">flowWorkspace</a></td>
	<td class=" ">Greg Finak,Mike Jiang</td>
	<td class=" ">Import flowJo Workspaces into BioConductor and replicate flowJo gating with flowCore</td>
</tr><tr id="pkg_fmcsR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/fmcsR.html">fmcsR</a></td>
	<td class=" ">Thomas Girke</td>
	<td class=" ">Mismatch Tolerant Maximum Common Substructure Searching</td>
</tr><tr id="pkg_focalCall" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/focalCall.html">focalCall</a></td>
	<td class=" ">Oscar Krijgsman</td>
	<td class=" ">Detection of focal aberrations in DNA copy number data</td>
</tr><tr id="pkg_FourCSeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/FourCSeq.html">FourCSeq</a></td>
	<td class=" ">Felix A. Klein</td>
	<td class=" ">Package analyse 4C sequencing data</td>
</tr><tr id="pkg_FRGEpistasis" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/FRGEpistasis.html">FRGEpistasis</a></td>
	<td class=" ">Futao Zhang</td>
	<td class=" ">Epistasis Analysis for Quantitative Traits by Functional Regression Model</td>
</tr><tr id="pkg_frma" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/frma.html">frma</a></td>
	<td class=" ">Matthew N. McCall</td>
	<td class=" ">Frozen RMA and Barcode</td>
</tr><tr id="pkg_frmaTools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/frmaTools.html">frmaTools</a></td>
	<td class=" ">Matthew N. McCall</td>
	<td class=" ">Frozen RMA Tools</td>
</tr><tr id="pkg_FunciSNP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/FunciSNP.html">FunciSNP</a></td>
	<td class=" ">Simon G. Coetzee</td>
	<td class=" ">Integrating Functional Non-coding Datasets with Genetic Association Studies to Identify Candidate Regulatory SNPs</td>
</tr><tr id="pkg_gaga" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gaga.html">gaga</a></td>
	<td class=" ">David Rossell</td>
	<td class=" ">GaGa hierarchical model for high-throughput data analysis</td>
</tr><tr id="pkg_gage" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gage.html">gage</a></td>
	<td class=" ">Weijun Luo</td>
	<td class=" ">Generally Applicable Gene-set Enrichment for Pathway Analysis</td>
</tr><tr id="pkg_gaggle" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gaggle.html">gaggle</a></td>
	<td class=" ">Christopher Bare</td>
	<td class=" ">Broadcast data between R and Gaggle</td>
</tr><tr id="pkg_gaia" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gaia.html">gaia</a></td>
	<td class=" ">S. Morganella</td>
	<td class=" ">GAIA: An R package for genomic analysis of significant chromosomal aberrations.</td>
</tr><tr id="pkg_gaucho" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gaucho.html">gaucho</a></td>
	<td class=" ">Alex Murison, Christopher Wardell</td>
	<td class=" ">Genetic Algorithms for Understanding Clonal Heterogeneity and Ordering</td>
</tr><tr id="pkg_gCMAP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gCMAP.html">gCMAP</a></td>
	<td class=" ">Thomas Sandmann</td>
	<td class=" ">Tools for Connectivity Map-like analyses</td>
</tr><tr id="pkg_gCMAPWeb" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gCMAPWeb.html">gCMAPWeb</a></td>
	<td class=" ">Thomas Sandmann</td>
	<td class=" ">A web interface for gene-set enrichment analyses</td>
</tr><tr id="pkg_gcrma" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gcrma.html">gcrma</a></td>
	<td class=" ">Z. Wu</td>
	<td class=" ">Background Adjustment Using Sequence Information</td>
</tr><tr id="pkg_gdsfmt" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gdsfmt.html">gdsfmt</a></td>
	<td class=" ">Xiuwen Zheng</td>
	<td class=" ">R Interface to CoreArray Genomic Data Structure (GDS) Files</td>
</tr><tr id="pkg_geecc" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/geecc.html">geecc</a></td>
	<td class=" ">Markus Boenn</td>
	<td class=" ">Gene set Enrichment analysis Extended to Contingency Cubes</td>
</tr><tr id="pkg_genArise" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/genArise.html">genArise</a></td>
	<td class=" ">IFC Development Team</td>
	<td class=" ">Microarray Analysis tool</td>
</tr><tr id="pkg_GENE.E" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GENE.E.html">GENE.E</a></td>
	<td class=" ">Joshua Gould</td>
	<td class=" ">Interact with GENE-E from R</td>
</tr><tr id="pkg_GeneAnswers" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GeneAnswers.html">GeneAnswers</a></td>
	<td class=" ">Lei Huang and Gang Feng</td>
	<td class=" ">Integrated Interpretation of Genes</td>
</tr><tr id="pkg_GeneExpressionSignature" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GeneExpressionSignature.html">GeneExpressionSignature</a></td>
	<td class=" ">Yang Cao, Fei Li,Lu Han</td>
	<td class=" ">Gene Expression Signature based Similarity Metric</td>
</tr><tr id="pkg_genefilter" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/genefilter.html">genefilter</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">genefilter: methods for filtering genes from high-throughput experiments</td>
</tr><tr id="pkg_genefu" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/genefu.html">genefu</a></td>
	<td class=" ">Benjamin Haibe-Kains, Markus Schroeder</td>
	<td class=" ">Relevant Functions for Gene Expression Analysis, Especially in Breast Cancer.</td>
</tr><tr id="pkg_GeneGA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GeneGA.html">GeneGA</a></td>
	<td class=" ">Zhenpeng Li</td>
	<td class=" ">Design gene based on both mRNA secondary structure and codon usage bias using Genetic algorithm</td>
</tr><tr id="pkg_GeneMeta" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GeneMeta.html">GeneMeta</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">MetaAnalysis for High Throughput Experiments</td>
</tr><tr id="pkg_GeneNetworkBuilder" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GeneNetworkBuilder.html">GeneNetworkBuilder</a></td>
	<td class=" ">Jianhong Ou</td>
	<td class=" ">Build Regulatory Network from ChIP-chip/ChIP-seq and Expression Data</td>
</tr><tr id="pkg_GeneOverlap" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GeneOverlap.html">GeneOverlap</a></td>
	<td class=" ">Li Shen, Mount Sinai</td>
	<td class=" ">Test and visualize gene overlaps</td>
</tr><tr id="pkg_geneplotter" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/geneplotter.html">geneplotter</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Graphics related functions for Bioconductor</td>
</tr><tr id="pkg_geneRecommender" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/geneRecommender.html">geneRecommender</a></td>
	<td class=" ">Greg Hather</td>
	<td class=" ">A gene recommender algorithm to identify genes coexpressed with a query set of genes</td>
</tr><tr id="pkg_GeneRegionScan" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GeneRegionScan.html">GeneRegionScan</a></td>
	<td class=" ">Lasse Folkersen</td>
	<td class=" ">GeneRegionScan</td>
</tr><tr id="pkg_geneRxCluster" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/geneRxCluster.html">geneRxCluster</a></td>
	<td class=" ">Charles Berry</td>
	<td class=" ">gRx Differential Clustering</td>
</tr><tr id="pkg_GeneSelectMMD" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GeneSelectMMD.html">GeneSelectMMD</a></td>
	<td class=" ">Weiliang Qiu</td>
	<td class=" ">Gene selection based on the marginal distributions of gene profiles that characterized by a mixture of three-component multivariate distributions</td>
</tr><tr id="pkg_GeneSelector" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GeneSelector.html">GeneSelector</a></td>
	<td class=" ">Martin Slawski</td>
	<td class=" ">Stability and Aggregation of ranked gene lists</td>
</tr><tr id="pkg_GENESIS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GENESIS.html">GENESIS</a></td>
	<td class=" ">Matthew P. Conomos</td>
	<td class=" ">GENetic EStimation and Inference in Structured samples (GENESIS): Statistical methods for analyzing genetic data from samples with population structure and/or relatedness</td>
</tr><tr id="pkg_geNetClassifier" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/geNetClassifier.html">geNetClassifier</a></td>
	<td class=" ">Sara Aibar</td>
	<td class=" ">classify diseases and build associated gene networks using gene expression profiles</td>
</tr><tr id="pkg_GeneticsDesign" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GeneticsDesign.html">GeneticsDesign</a></td>
	<td class=" ">The R Genetics Project</td>
	<td class=" ">Functions for designing genetics studies</td>
</tr><tr id="pkg_GeneticsPed" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GeneticsPed.html">GeneticsPed</a></td>
	<td class=" ">David Henderson</td>
	<td class=" ">Pedigree and genetic relationship functions</td>
</tr><tr id="pkg_genoCN" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/genoCN.html">genoCN</a></td>
	<td class=" ">Wei Sun</td>
	<td class=" ">genotyping and copy number study tools</td>
</tr><tr id="pkg_genomation" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/genomation.html">genomation</a></td>
	<td class=" ">Altuna Akalin, Vedran Franke</td>
	<td class=" ">Summary, annotation and visualization of genomic data</td>
</tr><tr id="pkg_GenomeGraphs" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GenomeGraphs.html">GenomeGraphs</a></td>
	<td class=" ">Steffen Durinck</td>
	<td class=" ">Plotting genomic information from Ensembl</td>
</tr><tr id="pkg_GenomeInfoDb" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html">GenomeInfoDb</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Utilities for manipulating chromosome and other 'seqname' identifiers</td>
</tr><tr id="pkg_genomeIntervals" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/genomeIntervals.html">genomeIntervals</a></td>
	<td class=" ">Julien Gagneur</td>
	<td class=" ">Operations on genomic intervals</td>
</tr><tr id="pkg_genomes" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/genomes.html">genomes</a></td>
	<td class=" ">Chris Stubben</td>
	<td class=" ">Genome sequencing project metadata</td>
</tr><tr id="pkg_GenomicAlignments" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GenomicAlignments.html">GenomicAlignments</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Representation and manipulation of short genomic alignments</td>
</tr><tr id="pkg_GenomicFeatures" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GenomicFeatures.html">GenomicFeatures</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Tools for making and manipulating transcript centric annotations</td>
</tr><tr id="pkg_GenomicFiles" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GenomicFiles.html">GenomicFiles</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Distributed computing by file or by range</td>
</tr><tr id="pkg_GenomicInteractions" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GenomicInteractions.html">GenomicInteractions</a></td>
	<td class=" ">Malcolm Perry, Liz Ing-Simmons</td>
	<td class=" ">R package for handling genomic interaction data</td>
</tr><tr id="pkg_GenomicRanges" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GenomicRanges.html">GenomicRanges</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Representation and manipulation of genomic intervals</td>
</tr><tr id="pkg_GenomicTuples" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GenomicTuples.html">GenomicTuples</a></td>
	<td class=" ">Peter Hickey</td>
	<td class=" ">Representation and Manipulation of Genomic Tuples</td>
</tr><tr id="pkg_Genominator" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Genominator.html">Genominator</a></td>
	<td class=" ">James Bullard</td>
	<td class=" ">Analyze, manage and store genomic data</td>
</tr><tr id="pkg_genoset" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/genoset.html">genoset</a></td>
	<td class=" ">Peter M. Haverty</td>
	<td class=" ">Provides classes similar to ExpressionSet for copy number analysis</td>
</tr><tr id="pkg_GenoView" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GenoView.html">GenoView</a></td>
	<td class=" ">Sharon Lee</td>
	<td class=" ">Condensed, overlapped plotting of genomic data tracks</td>
</tr><tr id="pkg_GEOmetadb" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GEOmetadb.html">GEOmetadb</a></td>
	<td class=" ">Jack Zhu</td>
	<td class=" ">A compilation of metadata from NCBI GEO</td>
</tr><tr id="pkg_GEOquery" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GEOquery.html">GEOquery</a></td>
	<td class=" ">Sean Davis</td>
	<td class=" ">Get data from NCBI Gene Expression Omnibus (GEO)</td>
</tr><tr id="pkg_GEOsubmission" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GEOsubmission.html">GEOsubmission</a></td>
	<td class=" ">Alexandre Kuhn</td>
	<td class=" ">Prepares microarray data for submission to GEO</td>
</tr><tr id="pkg_gespeR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gespeR.html">gespeR</a></td>
	<td class=" ">Fabian Schmich</td>
	<td class=" ">Gene-Specific Phenotype EstimatoR</td>
</tr><tr id="pkg_GEWIST" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GEWIST.html">GEWIST</a></td>
	<td class=" ">Wei Q. Deng</td>
	<td class=" ">Gene Environment Wide Interaction Search Threshold</td>
</tr><tr id="pkg_GGBase" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GGBase.html">GGBase</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">GGBase infrastructure for genetics of gene expression package GGtools</td>
</tr><tr id="pkg_ggbio" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ggbio.html">ggbio</a></td>
	<td class=" ">Tengfei Yin</td>
	<td class=" ">Visualization tools for genomic data.</td>
</tr><tr id="pkg_GGtools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GGtools.html">GGtools</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">software and data for analyses in genetics of gene expression</td>
</tr><tr id="pkg_ggtree" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ggtree.html">ggtree</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">a phylogenetic tree viewer for different types of tree annotations</td>
</tr><tr id="pkg_girafe" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/girafe.html">girafe</a></td>
	<td class=" ">J. Toedling</td>
	<td class=" ">Genome Intervals and Read Alignments for Functional Exploration</td>
</tr><tr id="pkg_GLAD" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GLAD.html">GLAD</a></td>
	<td class=" ">Philippe Hupe</td>
	<td class=" ">Gain and Loss Analysis of DNA</td>
</tr><tr id="pkg_GlobalAncova" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GlobalAncova.html">GlobalAncova</a></td>
	<td class=" ">Manuela Hummel</td>
	<td class=" ">Calculates a global test for differential gene expression between groups</td>
</tr><tr id="pkg_globaltest" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/globaltest.html">globaltest</a></td>
	<td class=" ">Jelle Goeman</td>
	<td class=" ">Testing groups of covariates/features for association with a response variable, with applications to gene set testing</td>
</tr><tr id="pkg_gmapR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gmapR.html">gmapR</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">An R interface to the GMAP/GSNAP/GSTRUCT suite</td>
</tr><tr id="pkg_GOexpress" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GOexpress.html">GOexpress</a></td>
	<td class=" ">Kevin Rue-Albrecht</td>
	<td class=" ">Visualise microarray and RNAseq data using gene ontology annotations</td>
</tr><tr id="pkg_GOFunction" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GOFunction.html">GOFunction</a></td>
	<td class=" ">Jing Wang</td>
	<td class=" ">GO-function: deriving biologcially relevant functions from statistically significant functions</td>
</tr><tr id="pkg_GoogleGenomics" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GoogleGenomics.html">GoogleGenomics</a></td>
	<td class=" ">Siddhartha Bagaria</td>
	<td class=" ">R Client for Google Genomics API</td>
</tr><tr id="pkg_goProfiles" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/goProfiles.html">goProfiles</a></td>
	<td class=" ">Alex Sanchez</td>
	<td class=" ">goProfiles: an R package for the statistical analysis of functional profiles</td>
</tr><tr id="pkg_GOSemSim" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GOSemSim.html">GOSemSim</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">GO-terms Semantic Similarity Measures</td>
</tr><tr id="pkg_goseq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/goseq.html">goseq</a></td>
	<td class=" ">Nadia Davidson</td>
	<td class=" ">Gene Ontology analyser for RNA-seq and other length biased data</td>
</tr><tr id="pkg_GOSim" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GOSim.html">GOSim</a></td>
	<td class=" ">Holger Froehlich</td>
	<td class=" ">Computation of functional similarities between GO terms and gene products; GO enrichment analysis</td>
</tr><tr id="pkg_GOstats" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GOstats.html">GOstats</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Tools for manipulating GO and microarrays.</td>
</tr><tr id="pkg_GOsummaries" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GOsummaries.html">GOsummaries</a></td>
	<td class=" ">Raivo Kolde</td>
	<td class=" ">Word cloud summaries of GO enrichment analysis</td>
</tr><tr id="pkg_GOTHiC" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GOTHiC.html">GOTHiC</a></td>
	<td class=" ">Borbala Mifsud</td>
	<td class=" ">Binomial test for Hi-C data analysis</td>
</tr><tr id="pkg_goTools" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/goTools.html">goTools</a></td>
	<td class=" ">Agnes Paquet</td>
	<td class=" ">Functions for Gene Ontology database</td>
</tr><tr id="pkg_gpls" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gpls.html">gpls</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Classification using generalized partial least squares</td>
</tr><tr id="pkg_gprege" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gprege.html">gprege</a></td>
	<td class=" ">Alfredo Kalaitzis</td>
	<td class=" ">Gaussian Process Ranking and Estimation of Gene Expression time-series</td>
</tr><tr id="pkg_gQTLBase" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gQTLBase.html">gQTLBase</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">gQTLBase: infrastructure for eQTL, mQTL and similar studies</td>
</tr><tr id="pkg_gQTLstats" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gQTLstats.html">gQTLstats</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">gQTLstats: computationally efficient analysis for eQTL and allied studies</td>
</tr><tr id="pkg_graph" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/graph.html">graph</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">graph: A package to handle graph data structures</td>
</tr><tr id="pkg_GraphAlignment" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GraphAlignment.html">GraphAlignment</a></td>
	<td class=" ">Joern P. Meier</td>
	<td class=" ">GraphAlignment</td>
</tr><tr id="pkg_GraphAT" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GraphAT.html">GraphAT</a></td>
	<td class=" ">Thomas LaFramboise</td>
	<td class=" ">Graph Theoretic Association Tests</td>
</tr><tr id="pkg_graphite" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/graphite.html">graphite</a></td>
	<td class=" ">Gabriele Sales</td>
	<td class=" ">GRAPH Interaction from pathway Topological Environment</td>
</tr><tr id="pkg_GraphPAC" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GraphPAC.html">GraphPAC</a></td>
	<td class=" ">Gregory Ryslik</td>
	<td class=" ">Identification of Mutational Clusters in Proteins via a Graph Theoretical Approach.</td>
</tr><tr id="pkg_GRENITS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GRENITS.html">GRENITS</a></td>
	<td class=" ">Edward Morrissey</td>
	<td class=" ">Gene Regulatory Network Inference Using Time Series</td>
</tr><tr id="pkg_GreyListChIP" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GreyListChIP.html">GreyListChIP</a></td>
	<td class=" ">Gordon Brown</td>
	<td class=" ">Grey Lists -- Mask Artefact Regions Based on ChIP Inputs</td>
</tr><tr id="pkg_groHMM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/groHMM.html">groHMM</a></td>
	<td class=" ">Minho Chae, W. Lee Kraus</td>
	<td class=" ">GRO-seq Analysis Pipeline.</td>
</tr><tr id="pkg_GSAR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GSAR.html">GSAR</a></td>
	<td class=" ">Yasir Rahmatallah, Galina Glazko</td>
	<td class=" ">Gene Set Analysis in R</td>
</tr><tr id="pkg_GSCA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GSCA.html">GSCA</a></td>
	<td class=" ">Zhicheng Ji</td>
	<td class=" ">GSCA: Gene Set Context Analysis</td>
</tr><tr id="pkg_GSEABase" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GSEABase.html">GSEABase</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Gene set enrichment data structures and methods</td>
</tr><tr id="pkg_GSEAlm" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GSEAlm.html">GSEAlm</a></td>
	<td class=" ">Assaf Oron</td>
	<td class=" ">Linear Model Toolset for Gene Set Enrichment Analysis</td>
</tr><tr id="pkg_GSReg" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GSReg.html">GSReg</a></td>
	<td class=" ">Bahman Afsari</td>
	<td class=" ">Gene Set Regulation (GS-Reg)</td>
</tr><tr id="pkg_GSRI" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GSRI.html">GSRI</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Gene Set Regulation Index</td>
</tr><tr id="pkg_GSVA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GSVA.html">GSVA</a></td>
	<td class=" ">Justin Guinney</td>
	<td class=" ">Gene Set Variation Analysis for microarray and RNA-seq data</td>
</tr><tr id="pkg_gtrellis" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gtrellis.html">gtrellis</a></td>
	<td class=" ">Zuguang Gu</td>
	<td class=" ">Genome Level Trellis Layout</td>
</tr><tr id="pkg_Gviz" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Gviz.html">Gviz</a></td>
	<td class=" ">Florian Hahne</td>
	<td class=" ">Plotting data and annotation information along genomic coordinates</td>
</tr><tr id="pkg_gwascat" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/gwascat.html">gwascat</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">representing and modeling data in the NHGRI GWAS catalog</td>
</tr><tr id="pkg_GWASTools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/GWASTools.html">GWASTools</a></td>
	<td class=" ">Stephanie M. Gogarten, Adrienne Stilp</td>
	<td class=" ">Tools for Genome Wide Association Studies</td>
</tr><tr id="pkg_h5vc" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/h5vc.html">h5vc</a></td>
	<td class=" ">Paul Theodor Pyl</td>
	<td class=" ">Managing alignment tallies using a hdf5 backend</td>
</tr><tr id="pkg_hapFabia" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/hapFabia.html">hapFabia</a></td>
	<td class=" ">Sepp Hochreiter</td>
	<td class=" ">hapFabia: Identification of very short segments of identity by descent (IBD) characterized by rare variants in large sequencing data</td>
</tr><tr id="pkg_Harshlight" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Harshlight.html">Harshlight</a></td>
	<td class=" ">Maurizio Pellegrino</td>
	<td class=" ">A "corrective make-up" program for microarray chips</td>
</tr><tr id="pkg_HCsnip" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HCsnip.html">HCsnip</a></td>
	<td class=" ">Askar Obulkasim</td>
	<td class=" ">Semi-supervised adaptive-height snipping of the Hierarchical Clustering tree</td>
</tr><tr id="pkg_HDTD" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HDTD.html">HDTD</a></td>
	<td class=" ">Anestis Touloumis</td>
	<td class=" ">Statistical Inference about the Mean Matrix and the Covariance Matrices in High-Dimensional Transposable Data (HDTD)</td>
</tr><tr id="pkg_Heatplus" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Heatplus.html">Heatplus</a></td>
	<td class=" ">Alexander Ploner</td>
	<td class=" ">Heatmaps with row and/or column covariates and colored clusters</td>
</tr><tr id="pkg_HELP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HELP.html">HELP</a></td>
	<td class=" ">Reid F. Thompson</td>
	<td class=" ">Tools for HELP data analysis</td>
</tr><tr id="pkg_HEM" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HEM.html">HEM</a></td>
	<td class=" ">HyungJun Cho</td>
	<td class=" ">Heterogeneous error model for identification of differentially expressed genes under multiple conditions</td>
</tr><tr id="pkg_hiAnnotator" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/hiAnnotator.html">hiAnnotator</a></td>
	<td class=" ">Nirav V Malani</td>
	<td class=" ">Functions for annotating GRanges objects.</td>
</tr><tr id="pkg_HIBAG" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HIBAG.html">HIBAG</a></td>
	<td class=" ">Xiuwen Zheng</td>
	<td class=" ">HLA Genotype Imputation with Attribute Bagging</td>
</tr><tr id="pkg_HilbertVis" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HilbertVis.html">HilbertVis</a></td>
	<td class=" ">Simon Anders</td>
	<td class=" ">Hilbert curve visualization</td>
</tr><tr id="pkg_HilbertVisGUI" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HilbertVisGUI.html">HilbertVisGUI</a></td>
	<td class=" ">Simon Anders</td>
	<td class=" ">HilbertVisGUI</td>
</tr><tr id="pkg_hiReadsProcessor" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/hiReadsProcessor.html">hiReadsProcessor</a></td>
	<td class=" ">Nirav V Malani</td>
	<td class=" ">Functions to process LM-PCR reads from 454/Illumina data.</td>
</tr><tr id="pkg_HiTC" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HiTC.html">HiTC</a></td>
	<td class=" ">Nicolas Servant</td>
	<td class=" ">High Throughput Chromosome Conformation Capture analysis</td>
</tr><tr id="pkg_HMMcopy" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HMMcopy.html">HMMcopy</a></td>
	<td class=" ">Daniel Lai, Gavin Ha, Sohrab Shah</td>
	<td class=" ">Copy number prediction with correction for GC and mappability bias for HTS data</td>
</tr><tr id="pkg_hopach" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/hopach.html">hopach</a></td>
	<td class=" ">Katherine S. Pollard</td>
	<td class=" ">Hierarchical Ordered Partitioning and Collapsing Hybrid (HOPACH)</td>
</tr><tr id="pkg_hpar" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/hpar.html">hpar</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">Human Protein Atlas in R</td>
</tr><tr id="pkg_HTqPCR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HTqPCR.html">HTqPCR</a></td>
	<td class=" ">Heidi Dvinge</td>
	<td class=" ">Automated analysis of high-throughput qPCR data</td>
</tr><tr id="pkg_HTSanalyzeR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HTSanalyzeR.html">HTSanalyzeR</a></td>
	<td class=" ">Xin Wang</td>
	<td class=" ">Gene set over-representation, enrichment and network analyses for high-throughput screens</td>
</tr><tr id="pkg_HTSeqGenie" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HTSeqGenie.html">HTSeqGenie</a></td>
	<td class=" ">Jens Reeder</td>
	<td class=" ">A NGS analysis pipeline.</td>
</tr><tr id="pkg_htSeqTools" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/htSeqTools.html">htSeqTools</a></td>
	<td class=" ">Oscar Reina</td>
	<td class=" ">Quality Control, Visualization and Processing for High-Throughput Sequencing data</td>
</tr><tr id="pkg_HTSFilter" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HTSFilter.html">HTSFilter</a></td>
	<td class=" ">Andrea Rau</td>
	<td class=" ">Filter replicated high-throughput transcriptome sequencing data</td>
</tr><tr id="pkg_HybridMTest" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/HybridMTest.html">HybridMTest</a></td>
	<td class=" ">Demba Fofana</td>
	<td class=" ">Hybrid Multiple Testing</td>
</tr><tr id="pkg_hyperdraw" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/hyperdraw.html">hyperdraw</a></td>
	<td class=" ">Paul Murrell</td>
	<td class=" ">Visualizing Hypergaphs</td>
</tr><tr id="pkg_hypergraph" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/hypergraph.html">hypergraph</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">A package providing hypergraph data structures</td>
</tr><tr id="pkg_iASeq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/iASeq.html">iASeq</a></td>
	<td class=" ">Yingying Wei</td>
	<td class=" ">iASeq: integrating multiple sequencing datasets for detecting allele-specific events</td>
</tr><tr id="pkg_iBBiG" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/iBBiG.html">iBBiG</a></td>
	<td class=" ">Aedin Culhane</td>
	<td class=" ">Iterative Binary Biclustering of Genesets</td>
</tr><tr id="pkg_ibh" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ibh.html">ibh</a></td>
	<td class=" ">Kircicegi Korkmaz</td>
	<td class=" ">Interaction Based Homogeneity for Evaluating Gene Lists</td>
</tr><tr id="pkg_iBMQ" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/iBMQ.html">iBMQ</a></td>
	<td class=" ">Greg Imholte</td>
	<td class=" ">integrated Bayesian Modeling of eQTL data</td>
</tr><tr id="pkg_Icens" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Icens.html">Icens</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">NPMLE for Censored and Truncated Data</td>
</tr><tr id="pkg_iChip" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/iChip.html">iChip</a></td>
	<td class=" ">Qianxing Mo</td>
	<td class=" ">Bayesian Modeling of ChIP-chip Data Through Hidden Ising Models</td>
</tr><tr id="pkg_iClusterPlus" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/iClusterPlus.html">iClusterPlus</a></td>
	<td class=" ">Qianxing Mo, Ronglai Shen</td>
	<td class=" ">Integrative clustering of multi-type genomic data</td>
</tr><tr id="pkg_IdeoViz" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/IdeoViz.html">IdeoViz</a></td>
	<td class=" ">Shraddha Pai</td>
	<td class=" ">Plots data (continuous/discrete) along chromosomal ideogram</td>
</tr><tr id="pkg_idiogram" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/idiogram.html">idiogram</a></td>
	<td class=" ">Karl J. Dykema</td>
	<td class=" ">idiogram</td>
</tr><tr id="pkg_IdMappingAnalysis" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/IdMappingAnalysis.html">IdMappingAnalysis</a></td>
	<td class=" ">Alex Lisovich, Roger Day</td>
	<td class=" ">ID Mapping Analysis</td>
</tr><tr id="pkg_IdMappingRetrieval" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/IdMappingRetrieval.html">IdMappingRetrieval</a></td>
	<td class=" ">Alex Lisovich, Roger Day</td>
	<td class=" ">ID Mapping Data Retrieval</td>
</tr><tr id="pkg_illuminaio" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/illuminaio.html">illuminaio</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Parsing Illumina microarray output files</td>
</tr><tr id="pkg_imageHTS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/imageHTS.html">imageHTS</a></td>
	<td class=" ">Joseph Barry</td>
	<td class=" ">Analysis of high-throughput microscopy-based screens</td>
</tr><tr id="pkg_immunoClust" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/immunoClust.html">immunoClust</a></td>
	<td class=" ">Till Soerensen</td>
	<td class=" ">immunoClust - Automated Pipeline for Population Detection in Flow Cytometry</td>
</tr><tr id="pkg_IMPCdata" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/IMPCdata.html">IMPCdata</a></td>
	<td class=" ">Jeremy Mason</td>
	<td class=" ">Retrieves data from IMPC database</td>
</tr><tr id="pkg_impute" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/impute.html">impute</a></td>
	<td class=" ">Balasubramanian Narasimhan</td>
	<td class=" ">impute: Imputation for microarray data</td>
</tr><tr id="pkg_InPAS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/InPAS.html">InPAS</a></td>
	<td class=" ">Jianhong Ou</td>
	<td class=" ">Identification of Novel alternative PolyAdenylation Sites (PAS)</td>
</tr><tr id="pkg_INPower" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/INPower.html">INPower</a></td>
	<td class=" ">Bill Wheeler</td>
	<td class=" ">An R package for computing the number of susceptibility SNPs</td>
</tr><tr id="pkg_inSilicoDb" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/inSilicoDb.html">inSilicoDb</a></td>
	<td class=" ">InSilico DB</td>
	<td class=" ">Access to the InSilico Database</td>
</tr><tr id="pkg_inSilicoMerging" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/inSilicoMerging.html">inSilicoMerging</a></td>
	<td class=" ">InSilico DB</td>
	<td class=" ">Collection of Merging Techniques for Gene Expression Data</td>
</tr><tr id="pkg_intansv" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/intansv.html">intansv</a></td>
	<td class=" ">Wen Yao</td>
	<td class=" ">Integrative analysis of structural variations</td>
</tr><tr id="pkg_interactiveDisplay" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/interactiveDisplay.html">interactiveDisplay</a></td>
	<td class=" ">Shawn Balcome</td>
	<td class=" ">Package for enabling powerful shiny web displays of Bioconductor objects</td>
</tr><tr id="pkg_interactiveDisplayBase" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/interactiveDisplayBase.html">interactiveDisplayBase</a></td>
	<td class=" ">Shawn Balcome</td>
	<td class=" ">Base package for enabling powerful shiny web displays of Bioconductor objects</td>
</tr><tr id="pkg_inveRsion" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/inveRsion.html">inveRsion</a></td>
	<td class=" ">Alejandro Caceres</td>
	<td class=" ">Inversions in genotype data</td>
</tr><tr id="pkg_iontree" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/iontree.html">iontree</a></td>
	<td class=" ">Mingshu Cao</td>
	<td class=" ">Data management and analysis of ion trees from ion-trap mass spectrometry</td>
</tr><tr id="pkg_iPAC" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/iPAC.html">iPAC</a></td>
	<td class=" ">Gregory Ryslik</td>
	<td class=" ">Identification of Protein Amino acid Clustering</td>
</tr><tr id="pkg_IPPD" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/IPPD.html">IPPD</a></td>
	<td class=" ">Martin Slawski</td>
	<td class=" ">Isotopic peak pattern deconvolution for Protein Mass Spectrometry by template matching</td>
</tr><tr id="pkg_IRanges" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/IRanges.html">IRanges</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Infrastructure for manipulating intervals on sequences</td>
</tr><tr id="pkg_iSeq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/iSeq.html">iSeq</a></td>
	<td class=" ">Qianxing Mo</td>
	<td class=" ">Bayesian Hierarchical Modeling of ChIP-seq Data Through Hidden Ising Models</td>
</tr><tr id="pkg_isobar" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/isobar.html">isobar</a></td>
	<td class=" ">Florian P Breitwieser</td>
	<td class=" ">Analysis and quantitation of isobarically tagged MSMS proteomics data</td>
</tr><tr id="pkg_IsoGeneGUI" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/IsoGeneGUI.html">IsoGeneGUI</a></td>
	<td class=" ">Setia Pramana</td>
	<td class=" ">A graphical user interface to conduct a dose-response analysis of microarray data</td>
</tr><tr id="pkg_ITALICS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ITALICS.html">ITALICS</a></td>
	<td class=" ">Guillem Rigaill</td>
	<td class=" ">ITALICS</td>
</tr><tr id="pkg_iterativeBMA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/iterativeBMA.html">iterativeBMA</a></td>
	<td class=" ">Ka Yee Yeung</td>
	<td class=" ">The Iterative Bayesian Model Averaging (BMA) algorithm</td>
</tr><tr id="pkg_iterativeBMAsurv" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/iterativeBMAsurv.html">iterativeBMAsurv</a></td>
	<td class=" ">Ka Yee Yeung</td>
	<td class=" ">The Iterative Bayesian Model Averaging (BMA) Algorithm For Survival Analysis</td>
</tr><tr id="pkg_IVAS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/IVAS.html">IVAS</a></td>
	<td class=" ">Seonggyun Han</td>
	<td class=" ">Identification of genetic Variants affecting Alternative Splicing</td>
</tr><tr id="pkg_jmosaics" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/jmosaics.html">jmosaics</a></td>
	<td class=" ">Xin Zeng</td>
	<td class=" ">Joint analysis of multiple ChIP-Seq data sets</td>
</tr><tr id="pkg_joda" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/joda.html">joda</a></td>
	<td class=" ">Ewa Szczurek</td>
	<td class=" ">JODA algorithm for quantifying gene deregulation using knowledge</td>
</tr><tr id="pkg_KCsmart" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/KCsmart.html">KCsmart</a></td>
	<td class=" ">Jorma de Ronde</td>
	<td class=" ">Multi sample aCGH analysis package using kernel convolution</td>
</tr><tr id="pkg_kebabs" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/kebabs.html">kebabs</a></td>
	<td class=" ">Ulrich Bodenhofer</td>
	<td class=" ">Kernel-Based Analysis Of Biological Sequences</td>
</tr><tr id="pkg_KEGGgraph" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/KEGGgraph.html">KEGGgraph</a></td>
	<td class=" ">Jitao David Zhang</td>
	<td class=" ">KEGGgraph: A graph approach to KEGG PATHWAY in R and Bioconductor</td>
</tr><tr id="pkg_keggorthology" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/keggorthology.html">keggorthology</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">graph support for KO, KEGG Orthology</td>
</tr><tr id="pkg_KEGGprofile" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/KEGGprofile.html">KEGGprofile</a></td>
	<td class=" ">Shilin Zhao</td>
	<td class=" ">An annotation and visualization package for multi-types and multi-groups expression data in KEGG pathway</td>
</tr><tr id="pkg_KEGGREST" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/KEGGREST.html">KEGGREST</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Client-side REST access to KEGG</td>
</tr><tr id="pkg_lapmix" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/lapmix.html">lapmix</a></td>
	<td class=" ">Yann Ruffieux</td>
	<td class=" ">Laplace Mixture Model in Microarray Experiments</td>
</tr><tr id="pkg_LBE" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/LBE.html">LBE</a></td>
	<td class=" ">Cyril Dalmasso</td>
	<td class=" ">Estimation of the false discovery rate.</td>
</tr><tr id="pkg_LEA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/LEA.html">LEA</a></td>
	<td class=" ">Eric Frichot</td>
	<td class=" ">LEA: an R package for Landscape and Ecological Association Studies</td>
</tr><tr id="pkg_les" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/les.html">les</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Identifying Differential Effects in Tiling Microarray Data</td>
</tr><tr id="pkg_limma" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/limma.html">limma</a></td>
	<td class=" ">Gordon Smyth</td>
	<td class=" ">Linear Models for Microarray Data</td>
</tr><tr id="pkg_limmaGUI" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/limmaGUI.html">limmaGUI</a></td>
	<td class=" ">Keith Satterley</td>
	<td class=" ">GUI for limma package</td>
</tr><tr id="pkg_LiquidAssociation" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/LiquidAssociation.html">LiquidAssociation</a></td>
	<td class=" ">Yen-Yi Ho</td>
	<td class=" ">LiquidAssociation</td>
</tr><tr id="pkg_lmdme" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/lmdme.html">lmdme</a></td>
	<td class=" ">Cristobal Fresno</td>
	<td class=" ">Linear Model decomposition for Designed Multivariate Experiments</td>
</tr><tr id="pkg_LMGene" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/LMGene.html">LMGene</a></td>
	<td class=" ">Blythe Durbin-Johnson</td>
	<td class=" ">LMGene Software for Data Transformation and Identification of Differentially Expressed Genes in Gene Expression Arrays</td>
</tr><tr id="pkg_logicFS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/logicFS.html">logicFS</a></td>
	<td class=" ">Holger Schwender</td>
	<td class=" ">Identification of SNP Interactions</td>
</tr><tr id="pkg_logitT" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/logitT.html">logitT</a></td>
	<td class=" ">Tobias Guennel</td>
	<td class=" ">logit-t Package</td>
</tr><tr id="pkg_lol" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/lol.html">lol</a></td>
	<td class=" ">Yinyin Yuan</td>
	<td class=" ">Lots Of Lasso</td>
</tr><tr id="pkg_LowMACA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/LowMACA.html">LowMACA</a></td>
	<td class=" ">Stefano de Pretis, Giorgio Melloni</td>
	<td class=" ">LowMACA - Low frequency Mutation Analysis via Consensus Alignment</td>
</tr><tr id="pkg_LPE" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/LPE.html">LPE</a></td>
	<td class=" ">Nitin Jain</td>
	<td class=" ">Methods for analyzing microarray data using Local Pooled Error (LPE) method</td>
</tr><tr id="pkg_LPEadj" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/LPEadj.html">LPEadj</a></td>
	<td class=" ">Carl Murie</td>
	<td class=" ">A correction of the local pooled error (LPE) method to replace the asymptotic variance adjustment with an unbiased adjustment based on sample size.</td>
</tr><tr id="pkg_lpNet" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/lpNet.html">lpNet</a></td>
	<td class=" ">Bettina Knapp</td>
	<td class=" ">Linear Programming Model for Network Inference</td>
</tr><tr id="pkg_lumi" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/lumi.html">lumi</a></td>
	<td class=" ">Pan Du</td>
	<td class=" ">BeadArray Specific Methods for Illumina Methylation and Expression Microarrays</td>
</tr><tr id="pkg_LVSmiRNA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/LVSmiRNA.html">LVSmiRNA</a></td>
	<td class=" ">Stefano Calza</td>
	<td class=" ">LVS normalization for Agilent miRNA data</td>
</tr><tr id="pkg_M3D" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/M3D.html">M3D</a></td>
	<td class=" ">Tom Mayo</td>
	<td class=" ">Identifies differentially methylated regions across testing groups.</td>
</tr><tr id="pkg_maanova" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/maanova.html">maanova</a></td>
	<td class=" ">Keith Sheppard</td>
	<td class=" ">Tools for analyzing Micro Array experiments</td>
</tr><tr id="pkg_macat" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/macat.html">macat</a></td>
	<td class=" ">Joern Toedling</td>
	<td class=" ">MicroArray Chromosome Analysis Tool</td>
</tr><tr id="pkg_maCorrPlot" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/maCorrPlot.html">maCorrPlot</a></td>
	<td class=" ">Alexander Ploner</td>
	<td class=" ">Visualize artificial correlation in microarray data</td>
</tr><tr id="pkg_made4" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/made4.html">made4</a></td>
	<td class=" ">Aedin Culhane</td>
	<td class=" ">Multivariate analysis of microarray data using ADE4</td>
</tr><tr id="pkg_maigesPack" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/maigesPack.html">maigesPack</a></td>
	<td class=" ">Gustavo H. Esteves</td>
	<td class=" ">Functions to handle cDNA microarray data, including several methods of data analysis</td>
</tr><tr id="pkg_MAIT" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MAIT.html">MAIT</a></td>
	<td class=" ">Francesc Fernandez-Albert</td>
	<td class=" ">Statistical Analysis of Metabolomic Data</td>
</tr><tr id="pkg_makecdfenv" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/makecdfenv.html">makecdfenv</a></td>
	<td class=" ">James W. MacDonald</td>
	<td class=" ">CDF Environment Maker</td>
</tr><tr id="pkg_MANOR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MANOR.html">MANOR</a></td>
	<td class=" ">Pierre Neuvial</td>
	<td class=" ">CGH Micro-Array NORmalization</td>
</tr><tr id="pkg_manta" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/manta.html">manta</a></td>
	<td class=" ">Chris Berthiaume, Adrian Marchetti</td>
	<td class=" ">Microbial Assemblage Normalized Transcript Analysis</td>
</tr><tr id="pkg_MantelCorr" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MantelCorr.html">MantelCorr</a></td>
	<td class=" ">Brian Steinmeyer</td>
	<td class=" ">Compute Mantel Cluster Correlations</td>
</tr><tr id="pkg_mAPKL" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mAPKL.html">mAPKL</a></td>
	<td class=" ">Argiris Sakellariou</td>
	<td class=" ">A Hybrid Feature Selection method for gene expression data</td>
</tr><tr id="pkg_maPredictDSC" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/maPredictDSC.html">maPredictDSC</a></td>
	<td class=" ">Adi Laurentiu Tarca</td>
	<td class=" ">Phenotype prediction using microarray data: approach of the best overall team in the IMPROVER Diagnostic Signature Challenge</td>
</tr><tr id="pkg_marray" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/marray.html">marray</a></td>
	<td class=" ">Yee Hwa (Jean) Yang</td>
	<td class=" ">Exploratory analysis for two-color spotted microarray data</td>
</tr><tr id="pkg_maSigPro" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/maSigPro.html">maSigPro</a></td>
	<td class=" ">Maria Jose Nueda</td>
	<td class=" ">Significant Gene Expression Profile Differences in Time Course Microarray Data</td>
</tr><tr id="pkg_maskBAD" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/maskBAD.html">maskBAD</a></td>
	<td class=" ">Michael Dannemann</td>
	<td class=" ">Masking probes with binding affinity differences</td>
</tr><tr id="pkg_MassArray" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MassArray.html">MassArray</a></td>
	<td class=" ">Reid F. Thompson</td>
	<td class=" ">Analytical Tools for MassArray Data</td>
</tr><tr id="pkg_massiR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/massiR.html">massiR</a></td>
	<td class=" ">Sam Buckberry</td>
	<td class=" ">massiR: MicroArray Sample Sex Identifier</td>
</tr><tr id="pkg_MassSpecWavelet" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MassSpecWavelet.html">MassSpecWavelet</a></td>
	<td class=" ">Pan Du</td>
	<td class=" ">Mass spectrum processing by wavelet-based algorithms</td>
</tr><tr id="pkg_matchBox" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/matchBox.html">matchBox</a></td>
	<td class=" ">Luigi Marchionni, Anuj Gupta</td>
	<td class=" ">Utilities to compute, compare, and plot the agreement between ordered vectors of features (ie. distinct genomic experiments). The package includes Correspondence-At-the-TOP (CAT) analysis.</td>
</tr><tr id="pkg_MatrixRider" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MatrixRider.html">MatrixRider</a></td>
	<td class=" ">Elena Grassi</td>
	<td class=" ">Obtain total affinity and occupancies for binding site matrices on a given sequence</td>
</tr><tr id="pkg_MBAmethyl" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MBAmethyl.html">MBAmethyl</a></td>
	<td class=" ">Tao Wang</td>
	<td class=" ">Model-based analysis of DNA methylation data</td>
</tr><tr id="pkg_MBASED" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MBASED.html">MBASED</a></td>
	<td class=" ">Oleg Mayba</td>
	<td class=" ">Package containing functions for ASE analysis using Meta-analysis Based Allele-Specific Expression Detection</td>
</tr><tr id="pkg_MBCB" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MBCB.html">MBCB</a></td>
	<td class=" ">Jeff Allen</td>
	<td class=" ">MBCB (Model-based Background Correction for Beadarray)</td>
</tr><tr id="pkg_mBPCR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mBPCR.html">mBPCR</a></td>
	<td class=" ">P.M.V. Rancoita</td>
	<td class=" ">Bayesian Piecewise Constant Regression for DNA copy number estimation</td>
</tr><tr id="pkg_mcaGUI" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mcaGUI.html">mcaGUI</a></td>
	<td class=" ">Wade K. Copeland</td>
	<td class=" ">Microbial Community Analysis GUI</td>
</tr><tr id="pkg_MCRestimate" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MCRestimate.html">MCRestimate</a></td>
	<td class=" ">Marc Johannes</td>
	<td class=" ">Misclassification error estimation with cross-validation</td>
</tr><tr id="pkg_mdgsa" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mdgsa.html">mdgsa</a></td>
	<td class=" ">David Montaner</td>
	<td class=" ">Multi Dimensional Gene Set Analysis.</td>
</tr><tr id="pkg_mdqc" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mdqc.html">mdqc</a></td>
	<td class=" ">Gabriela Cohen-Freue</td>
	<td class=" ">Mahalanobis Distance Quality Control for microarrays</td>
</tr><tr id="pkg_MeasurementError.cor" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MeasurementError.cor.html">MeasurementError.cor</a></td>
	<td class=" ">Beiying Ding</td>
	<td class=" ">Measurement Error model estimate for correlation coefficient</td>
</tr><tr id="pkg_MEDIPS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MEDIPS.html">MEDIPS</a></td>
	<td class=" ">Lukas Chavez</td>
	<td class=" ">DNA IP-seq data analysis</td>
</tr><tr id="pkg_MEDME" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MEDME.html">MEDME</a></td>
	<td class=" ">Mattia Pelizzola</td>
	<td class=" ">Modelling Experimental Data from MeDIP Enrichment</td>
</tr><tr id="pkg_MEIGOR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MEIGOR.html">MEIGOR</a></td>
	<td class=" ">Jose Egea</td>
	<td class=" ">MEIGO - MEtaheuristics for bIoinformatics Global Optimization</td>
</tr><tr id="pkg_MergeMaid" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MergeMaid.html">MergeMaid</a></td>
	<td class=" ">Xiaogang Zhong</td>
	<td class=" ">Merge Maid</td>
</tr><tr id="pkg_MeSHDbi" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MeSHDbi.html">MeSHDbi</a></td>
	<td class=" ">Koki Tsuyuzaki</td>
	<td class=" ">DBI to construct MeSH-related package from sqlite file</td>
</tr><tr id="pkg_meshr" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/meshr.html">meshr</a></td>
	<td class=" ">Koki Tsuyuzaki</td>
	<td class=" ">Tools for conducting enrichment analysis of MeSH</td>
</tr><tr id="pkg_MeSHSim" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MeSHSim.html">MeSHSim</a></td>
	<td class=" ">Jing ZHou</td>
	<td class=" ">MeSH(Medical Subject Headings) Semantic Similarity Measures</td>
</tr><tr id="pkg_messina" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/messina.html">messina</a></td>
	<td class=" ">Mark Pinese</td>
	<td class=" ">Single-gene classifiers and outlier-resistant detection of differential expression for two-group and survival problems.</td>
</tr><tr id="pkg_metaArray" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/metaArray.html">metaArray</a></td>
	<td class=" ">Hyungwon Choi</td>
	<td class=" ">Integration of Microarray Data for Meta-analysis</td>
</tr><tr id="pkg_Metab" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Metab.html">Metab</a></td>
	<td class=" ">Raphael Aggio</td>
	<td class=" ">Metab: An R Package for a High-Throughput Analysis of Metabolomics Data Generated by GC-MS.</td>
</tr><tr id="pkg_metabomxtr" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/metabomxtr.html">metabomxtr</a></td>
	<td class=" ">Michael Nodzenski</td>
	<td class=" ">A package to run mixture models for truncated metabolomics data with normal or lognormal distributions.</td>
</tr><tr id="pkg_metagene" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/metagene.html">metagene</a></td>
	<td class=" ">Charles Joly Beauparlant</td>
	<td class=" ">A package to produce metagene plots</td>
</tr><tr id="pkg_metagenomeSeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/metagenomeSeq.html">metagenomeSeq</a></td>
	<td class=" ">Joseph N. Paulson</td>
	<td class=" ">Statistical analysis for sparse high-throughput sequencing</td>
</tr><tr id="pkg_metahdep" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/metahdep.html">metahdep</a></td>
	<td class=" ">John R. Stevens</td>
	<td class=" ">Hierarchical Dependence in Meta-Analysis</td>
</tr><tr id="pkg_metaMS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/metaMS.html">metaMS</a></td>
	<td class=" ">Ron Wehrens</td>
	<td class=" ">MS-based metabolomics annotation pipeline</td>
</tr><tr id="pkg_metaSeq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/metaSeq.html">metaSeq</a></td>
	<td class=" ">Koki Tsuyuzaki</td>
	<td class=" ">Meta-analysis of RNA-Seq count data in multiple studies</td>
</tr><tr id="pkg_metaseqR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/metaseqR.html">metaseqR</a></td>
	<td class=" ">Panagiotis Moulos</td>
	<td class=" ">An R package for the analysis and result reporting of RNA-Seq data by combining multiple statistical algorithms.</td>
</tr><tr id="pkg_MethTargetedNGS" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MethTargetedNGS.html">MethTargetedNGS</a></td>
	<td class=" ">Muhammad Ahmer Jamil</td>
	<td class=" ">Perform Methylation Analysis on Next Generation Sequencing Data</td>
</tr><tr id="pkg_methVisual" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/methVisual.html">methVisual</a></td>
	<td class=" ">Arie Zackay</td>
	<td class=" ">Methods for visualization and statistics on DNA methylation data</td>
</tr><tr id="pkg_methyAnalysis" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/methyAnalysis.html">methyAnalysis</a></td>
	<td class=" ">Pan Du</td>
	<td class=" ">DNA methylation data analysis and visualization</td>
</tr><tr id="pkg_MethylAid" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MethylAid.html">MethylAid</a></td>
	<td class=" ">M. van Iterson</td>
	<td class=" ">Visual and interactive quality control of large Illumina 450k data sets</td>
</tr><tr id="pkg_MethylMix" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MethylMix.html">MethylMix</a></td>
	<td class=" ">Olivier Gevaert</td>
	<td class=" ">MethylMix: Identifying methylation driven cancer genes.</td>
</tr><tr id="pkg_methylMnM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/methylMnM.html">methylMnM</a></td>
	<td class=" ">Yan Zhou</td>
	<td class=" ">detect different methylation level (DMR)</td>
</tr><tr id="pkg_methylPipe" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/methylPipe.html">methylPipe</a></td>
	<td class=" ">Kamal Kishore</td>
	<td class=" ">Base resolution DNA methylation data analysis</td>
</tr><tr id="pkg_MethylSeekR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MethylSeekR.html">MethylSeekR</a></td>
	<td class=" ">Lukas Burger</td>
	<td class=" ">Segmentation of Bis-seq data</td>
</tr><tr id="pkg_methylumi" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/methylumi.html">methylumi</a></td>
	<td class=" ">Sean Davis</td>
	<td class=" ">Handle Illumina methylation data</td>
</tr><tr id="pkg_Mfuzz" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Mfuzz.html">Mfuzz</a></td>
	<td class=" ">Matthias Futschik</td>
	<td class=" ">Soft clustering of time series gene expression data</td>
</tr><tr id="pkg_MGFM" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MGFM.html">MGFM</a></td>
	<td class=" ">Khadija El Amrani</td>
	<td class=" ">Marker Gene Finder in Microarray gene expression data</td>
</tr><tr id="pkg_mgsa" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mgsa.html">mgsa</a></td>
	<td class=" ">Sebastian Bauer</td>
	<td class=" ">Model-based gene set analysis</td>
</tr><tr id="pkg_MiChip" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MiChip.html">MiChip</a></td>
	<td class=" ">Jonathon Blake</td>
	<td class=" ">MiChip Parsing and Summarizing Functions</td>
</tr><tr id="pkg_microRNA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/microRNA.html">microRNA</a></td>
	<td class=" ">"James F. Reid"</td>
	<td class=" ">Data and functions for dealing with microRNAs</td>
</tr><tr id="pkg_MIMOSA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MIMOSA.html">MIMOSA</a></td>
	<td class=" ">Greg Finak</td>
	<td class=" ">Mixture Models for Single-Cell Assays</td>
</tr><tr id="pkg_MineICA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MineICA.html">MineICA</a></td>
	<td class=" ">Anne Biton</td>
	<td class=" ">Analysis of an ICA decomposition obtained on genomics data</td>
</tr><tr id="pkg_minet" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/minet.html">minet</a></td>
	<td class=" ">Patrick E. Meyer</td>
	<td class=" ">Mutual Information NETworks</td>
</tr><tr id="pkg_minfi" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/minfi.html">minfi</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Analyze Illumina's 450k methylation arrays</td>
</tr><tr id="pkg_MinimumDistance" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MinimumDistance.html">MinimumDistance</a></td>
	<td class=" ">Robert B Scharpf</td>
	<td class=" ">A package for de novo CNV detection in case-parent trios</td>
</tr><tr id="pkg_MiPP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MiPP.html">MiPP</a></td>
	<td class=" ">Sukwoo Kim</td>
	<td class=" ">Misclassification Penalized Posterior Classification</td>
</tr><tr id="pkg_MiRaGE" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MiRaGE.html">MiRaGE</a></td>
	<td class=" ">Y-h. Taguchi</td>
	<td class=" ">MiRNA Ranking by Gene Expression</td>
</tr><tr id="pkg_miRNApath" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/miRNApath.html">miRNApath</a></td>
	<td class=" ">James M. Ward</td>
	<td class=" ">miRNApath: Pathway Enrichment for miRNA Expression Data</td>
</tr><tr id="pkg_miRNAtap" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/miRNAtap.html">miRNAtap</a></td>
	<td class=" ">Maciej Pajak</td>
	<td class=" ">miRNAtap: microRNA Targets - Aggregated Predictions</td>
</tr><tr id="pkg_miRNAtap.db" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/miRNAtap.db.html">miRNAtap.db</a></td>
	<td class=" ">Maciej Pajak</td>
	<td class=" ">Data for miRNAtap</td>
</tr><tr id="pkg_Mirsynergy" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Mirsynergy.html">Mirsynergy</a></td>
	<td class=" ">Yue Li</td>
	<td class=" ">Mirsynergy</td>
</tr><tr id="pkg_missMethyl" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/missMethyl.html">missMethyl</a></td>
	<td class=" ">Belinda Phipson, Jovana Maksimovic</td>
	<td class=" ">Analysis of methylation array data</td>
</tr><tr id="pkg_mitoODE" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mitoODE.html">mitoODE</a></td>
	<td class=" ">Gregoire Pau</td>
	<td class=" ">Implementation of the differential equation model described in "Dynamical modelling of phenotypes in a genome-wide RNAi live-cell imaging assay"</td>
</tr><tr id="pkg_MLInterfaces" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MLInterfaces.html">MLInterfaces</a></td>
	<td class=" ">V. Carey</td>
	<td class=" ">Uniform interfaces to R machine learning procedures for data in Bioconductor containers</td>
</tr><tr id="pkg_MLP" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MLP.html">MLP</a></td>
	<td class=" ">Tobias Verbeke</td>
	<td class=" ">MLP</td>
</tr><tr id="pkg_MLSeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MLSeq.html">MLSeq</a></td>
	<td class=" ">Gokmen Zararsiz</td>
	<td class=" ">Machine learning interface for RNA-Seq data</td>
</tr><tr id="pkg_MMDiff" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MMDiff.html">MMDiff</a></td>
	<td class=" ">Gabriele Schweikert</td>
	<td class=" ">Statistical Testing for ChIP-Seq data sets</td>
</tr><tr id="pkg_mmnet" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mmnet.html">mmnet</a></td>
	<td class=" ">Yang Cao, Fei Li</td>
	<td class=" ">A metagenomic pipeline for systems biology</td>
</tr><tr id="pkg_MmPalateMiRNA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MmPalateMiRNA.html">MmPalateMiRNA</a></td>
	<td class=" ">Guy Brock</td>
	<td class=" ">Murine Palate miRNA Expression Analysis</td>
</tr><tr id="pkg_mogsa" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mogsa.html">mogsa</a></td>
	<td class=" ">Chen Meng</td>
	<td class=" ">Multiple omics data integration and gene set analysis</td>
</tr><tr id="pkg_monocle" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/monocle.html">monocle</a></td>
	<td class=" ">Cole Trapnell</td>
	<td class=" ">Analysis tools for single-cell expression experiments.</td>
</tr><tr id="pkg_MoPS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MoPS.html">MoPS</a></td>
	<td class=" ">Philipp Eser</td>
	<td class=" ">MoPS - Model-based Periodicity Screening</td>
</tr><tr id="pkg_mosaics" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mosaics.html">mosaics</a></td>
	<td class=" ">Dongjun Chung</td>
	<td class=" ">MOSAiCS (MOdel-based one and two Sample Analysis and Inference for ChIP-Seq)</td>
</tr><tr id="pkg_MotifDb" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MotifDb.html">MotifDb</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">An Annotated Collection of Protein-DNA Binding Sequence Motifs</td>
</tr><tr id="pkg_motifRG" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/motifRG.html">motifRG</a></td>
	<td class=" ">Zizhen Yao</td>
	<td class=" ">A package for discriminative motif discovery, designed for high throughput sequencing dataset</td>
</tr><tr id="pkg_motifStack" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/motifStack.html">motifStack</a></td>
	<td class=" ">Jianhong Ou</td>
	<td class=" ">Plot stacked logos for single or multiple DNA, RNA and amino acid sequence</td>
</tr><tr id="pkg_MotIV" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MotIV.html">MotIV</a></td>
	<td class=" ">Eloi Mercier, Raphael Gottardo</td>
	<td class=" ">Motif Identification and Validation</td>
</tr><tr id="pkg_MPFE" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MPFE.html">MPFE</a></td>
	<td class=" ">Conrad Burden</td>
	<td class=" ">Estimation of the amplicon methylation pattern distribution from bisulphite sequencing data.</td>
</tr><tr id="pkg_mQTL.NMR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mQTL.NMR.html">mQTL.NMR</a></td>
	<td class=" ">Lyamine Hedjazi</td>
	<td class=" ">Metabolomic Quantitative Trait Locus Mapping for 1H NMR data</td>
</tr><tr id="pkg_msa" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/msa.html">msa</a></td>
	<td class=" ">Ulrich Bodenhofer</td>
	<td class=" ">Multiple Sequence Alignment</td>
</tr><tr id="pkg_MSGFgui" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MSGFgui.html">MSGFgui</a></td>
	<td class=" ">Thomas Lin Pedersen</td>
	<td class=" ">A shiny GUI for MSGFplus</td>
</tr><tr id="pkg_MSGFplus" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MSGFplus.html">MSGFplus</a></td>
	<td class=" ">Thomas Lin Pedersen</td>
	<td class=" ">An interface between R and MS-GF+</td>
</tr><tr id="pkg_msmsEDA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/msmsEDA.html">msmsEDA</a></td>
	<td class=" ">Josep Gregori</td>
	<td class=" ">Exploratory Data Analysis of LC-MS/MS data by spectral counts</td>
</tr><tr id="pkg_msmsTests" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/msmsTests.html">msmsTests</a></td>
	<td class=" ">Josep Gregori i Font</td>
	<td class=" ">LC-MS/MS Differential Expression Tests</td>
</tr><tr id="pkg_MSnbase" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MSnbase.html">MSnbase</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">Base Functions and Classes for MS-based Proteomics</td>
</tr><tr id="pkg_MSnID" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MSnID.html">MSnID</a></td>
	<td class=" ">Vlad Petyuk</td>
	<td class=" ">Utilities for Exploration and Assessment of Confidence of LC-MSn Proteomics Identifications.</td>
</tr><tr id="pkg_MSstats" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MSstats.html">MSstats</a></td>
	<td class=" ">Meena Choi</td>
	<td class=" ">Protein Significance Analysis in DDA, SRM and DIA for Label-free or Label-based Proteomics Experiments</td>
</tr><tr id="pkg_Mulcom" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Mulcom.html">Mulcom</a></td>
	<td class=" ">Claudio Isella</td>
	<td class=" ">Calculates Mulcom test</td>
</tr><tr id="pkg_MultiMed" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MultiMed.html">MultiMed</a></td>
	<td class=" ">Simina M. Boca</td>
	<td class=" ">Testing multiple biological mediators simultaneously</td>
</tr><tr id="pkg_multiscan" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/multiscan.html">multiscan</a></td>
	<td class=" ">Mizanur Khondoker</td>
	<td class=" ">R package for combining multiple scans</td>
</tr><tr id="pkg_multtest" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/multtest.html">multtest</a></td>
	<td class=" ">Katherine S. Pollard</td>
	<td class=" ">Resampling-based multiple hypothesis testing</td>
</tr><tr id="pkg_muscle" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/muscle.html">muscle</a></td>
	<td class=" ">Alex T. Kalinka</td>
	<td class=" ">Multiple Sequence Alignment with MUSCLE</td>
</tr><tr id="pkg_MVCClass" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/MVCClass.html">MVCClass</a></td>
	<td class=" ">Elizabeth Whalen</td>
	<td class=" ">Model-View-Controller (MVC) Classes</td>
</tr><tr id="pkg_mvGST" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mvGST.html">mvGST</a></td>
	<td class=" ">John R. Stevens</td>
	<td class=" ">Multivariate and directional gene set testing</td>
</tr><tr id="pkg_mygene" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mygene.html">mygene</a></td>
	<td class=" ">Adam Mark, Chunlei Wu</td>
	<td class=" ">Access MyGene.Info_ services</td>
</tr><tr id="pkg_mzID" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mzID.html">mzID</a></td>
	<td class=" ">Thomas Lin Pedersen</td>
	<td class=" ">An mzIdentML parser for R</td>
</tr><tr id="pkg_mzR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/mzR.html">mzR</a></td>
	<td class=" ">Bernd Fischer, Steffen Neumann, Laurent Gatto, Qiang Kou</td>
	<td class=" ">parser for netCDF, mzXML, mzData and mzML and mzIdentML files (mass spectrometry data)</td>
</tr><tr id="pkg_NanoStringQCPro" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/NanoStringQCPro.html">NanoStringQCPro</a></td>
	<td class=" ">Robert Ziman</td>
	<td class=" ">Quality metrics and data processing methods for NanoString mRNA gene expression data</td>
</tr><tr id="pkg_NarrowPeaks" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/NarrowPeaks.html">NarrowPeaks</a></td>
	<td class=" ">Pedro Madrigal</td>
	<td class=" ">Shape-based Analysis of Variation in ChIP-Seq using Functional PCA</td>
</tr><tr id="pkg_ncdfFlow" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ncdfFlow.html">ncdfFlow</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">ncdfFlow: A package that provides HDF5 based storage for flow cytometry data.</td>
</tr><tr id="pkg_NCIgraph" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/NCIgraph.html">NCIgraph</a></td>
	<td class=" ">Laurent Jacob</td>
	<td class=" ">Pathways from the NCI Pathways Database</td>
</tr><tr id="pkg_neaGUI" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/neaGUI.html">neaGUI</a></td>
	<td class=" ">Setia Pramana</td>
	<td class=" ">An R package to perform the network enrichment analysis (NEA).</td>
</tr><tr id="pkg_nem" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/nem.html">nem</a></td>
	<td class=" ">Holger Froehlich</td>
	<td class=" ">(Dynamic) Nested Effects Models and Deterministic Effects Propagation Networks to reconstruct phenotypic hierarchies</td>
</tr><tr id="pkg_netbenchmark" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/netbenchmark.html">netbenchmark</a></td>
	<td class=" ">Pau Bellot</td>
	<td class=" ">Benchmarking of several gene network inference methods</td>
</tr><tr id="pkg_netbiov" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/netbiov.html">netbiov</a></td>
	<td class=" ">Shailesh tripathi</td>
	<td class=" ">A package for visualizing complex biological network</td>
</tr><tr id="pkg_nethet" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/nethet.html">nethet</a></td>
	<td class=" ">Nicolas Staedler, Frank Dondelinger</td>
	<td class=" ">A bioconductor package for high-dimensional exploration of biological network heterogeneity</td>
</tr><tr id="pkg_NetPathMiner" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/NetPathMiner.html">NetPathMiner</a></td>
	<td class=" ">Ahmed Mohamed</td>
	<td class=" ">NetPathMiner for Biological Network Construction, Path Mining and Visualization</td>
</tr><tr id="pkg_netresponse" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/netresponse.html">netresponse</a></td>
	<td class=" ">Leo Lahti</td>
	<td class=" ">Functional Network Analysis</td>
</tr><tr id="pkg_NetSAM" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/NetSAM.html">NetSAM</a></td>
	<td class=" ">Bing Zhang</td>
	<td class=" ">Network Seriation And Modularization</td>
</tr><tr id="pkg_networkBMA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/networkBMA.html">networkBMA</a></td>
	<td class=" ">Ka Yee Yeung</td>
	<td class=" ">Regression-based network inference using Bayesian Model Averaging</td>
</tr><tr id="pkg_NGScopy" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/NGScopy.html">NGScopy</a></td>
	<td class=" ">Xiaobei Zhao</td>
	<td class=" ">NGScopy: Detection of copy number variations in next generation sequencing</td>
</tr><tr id="pkg_nnNorm" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/nnNorm.html">nnNorm</a></td>
	<td class=" ">Adi Laurentiu Tarca</td>
	<td class=" ">Spatial and intensity based normalization of cDNA microarray data based on robust neural nets</td>
</tr><tr id="pkg_NOISeq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/NOISeq.html">NOISeq</a></td>
	<td class=" ">Sonia Tarazona</td>
	<td class=" ">Exploratory analysis and differential expression for RNA-seq data</td>
</tr><tr id="pkg_nondetects" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/nondetects.html">nondetects</a></td>
	<td class=" ">Matthew N. McCall</td>
	<td class=" ">Non-detects in qPCR data</td>
</tr><tr id="pkg_NormqPCR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/NormqPCR.html">NormqPCR</a></td>
	<td class=" ">James Perkins</td>
	<td class=" ">Functions for normalisation of RT-qPCR data</td>
</tr><tr id="pkg_npGSEA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/npGSEA.html">npGSEA</a></td>
	<td class=" ">Jessica Larson</td>
	<td class=" ">Permutation approximation methods for gene set enrichment analysis (non-permutation GSEA)</td>
</tr><tr id="pkg_NTW" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/NTW.html">NTW</a></td>
	<td class=" ">Yuanhua Liu</td>
	<td class=" ">Predict gene network using an Ordinary Differential Equation (ODE) based method</td>
</tr><tr id="pkg_nucleR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/nucleR.html">nucleR</a></td>
	<td class=" ">Ricard Illa</td>
	<td class=" ">Nucleosome positioning package for R</td>
</tr><tr id="pkg_nudge" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/nudge.html">nudge</a></td>
	<td class=" ">N. Dean</td>
	<td class=" ">Normal Uniform Differential Gene Expression detection</td>
</tr><tr id="pkg_NuPoP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/NuPoP.html">NuPoP</a></td>
	<td class=" ">Ji-Ping Wang</td>
	<td class=" ">An R package for nucleosome positioning prediction</td>
</tr><tr id="pkg_occugene" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/occugene.html">occugene</a></td>
	<td class=" ">Oliver Will</td>
	<td class=" ">Functions for Multinomial Occupancy Distribution</td>
</tr><tr id="pkg_OCplus" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/OCplus.html">OCplus</a></td>
	<td class=" ">Alexander Ploner</td>
	<td class=" ">Operating characteristics plus sample size and local fdr for microarray experiments</td>
</tr><tr id="pkg_oligo" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/oligo.html">oligo</a></td>
	<td class=" ">Benilton Carvalho</td>
	<td class=" ">Preprocessing tools for oligonucleotide arrays</td>
</tr><tr id="pkg_oligoClasses" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/oligoClasses.html">oligoClasses</a></td>
	<td class=" ">Benilton Carvalho and Robert Scharpf</td>
	<td class=" ">Classes for high-throughput arrays supported by oligo and crlmm</td>
</tr><tr id="pkg_OLIN" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/OLIN.html">OLIN</a></td>
	<td class=" ">Matthias Futschik</td>
	<td class=" ">Optimized local intensity-dependent normalisation of two-color microarrays</td>
</tr><tr id="pkg_OLINgui" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/OLINgui.html">OLINgui</a></td>
	<td class=" ">Matthias Futschik</td>
	<td class=" ">Graphical user interface for OLIN</td>
</tr><tr id="pkg_omicade4" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/omicade4.html">omicade4</a></td>
	<td class=" ">Chen Meng</td>
	<td class=" ">Multiple co-inertia analysis of omics datasets</td>
</tr><tr id="pkg_OmicCircos" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/OmicCircos.html">OmicCircos</a></td>
	<td class=" ">Ying Hu</td>
	<td class=" ">High-quality circular visualization of omics data</td>
</tr><tr id="pkg_OmicsMarkeR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/OmicsMarkeR.html">OmicsMarkeR</a></td>
	<td class=" ">Charles E. Determan Jr.</td>
	<td class=" ">Classification and Feature Selection for 'Omics' Datasets</td>
</tr><tr id="pkg_OncoSimulR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/OncoSimulR.html">OncoSimulR</a></td>
	<td class=" ">Ramon Diaz-Uriarte</td>
	<td class=" ">Simulation of cancer progresion with order restrictions</td>
</tr><tr id="pkg_oneChannelGUI" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/oneChannelGUI.html">oneChannelGUI</a></td>
	<td class=" ">Raffaele A Calogero</td>
	<td class=" ">A graphical interface designed to facilitate analysis of microarrays and miRNA/RNA-seq data on laptops</td>
</tr><tr id="pkg_ontoCAT" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ontoCAT.html">ontoCAT</a></td>
	<td class=" ">Natalja Kurbatova</td>
	<td class=" ">Ontology traversal and search</td>
</tr><tr id="pkg_openCyto" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/openCyto.html">openCyto</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">Hierarchical Gating Pipeline for flow cytometry data</td>
</tr><tr id="pkg_oposSOM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/oposSOM.html">oposSOM</a></td>
	<td class=" ">Henry Wirth</td>
	<td class=" ">Comprehensive analysis of transciptome data</td>
</tr><tr id="pkg_OrderedList" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/OrderedList.html">OrderedList</a></td>
	<td class=" ">Claudio Lottaz</td>
	<td class=" ">Similarities of Ordered Gene Lists</td>
</tr><tr id="pkg_OrganismDbi" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/OrganismDbi.html">OrganismDbi</a></td>
	<td class=" ">Biocore Data Team</td>
	<td class=" ">Software to enable the smooth interfacing of different database packages</td>
</tr><tr id="pkg_OSAT" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/OSAT.html">OSAT</a></td>
	<td class=" ">Li Yan</td>
	<td class=" ">OSAT: Optimal Sample Assignment Tool</td>
</tr><tr id="pkg_OTUbase" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/OTUbase.html">OTUbase</a></td>
	<td class=" ">Daniel Beck</td>
	<td class=" ">Provides structure and functions for the analysis of OTU data</td>
</tr><tr id="pkg_OutlierD" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/OutlierD.html">OutlierD</a></td>
	<td class=" ">Sukwoo Kim</td>
	<td class=" ">Outlier detection using quantile regression on the M-A scatterplots of high-throughput data</td>
</tr><tr id="pkg_PAA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PAA.html">PAA</a></td>
	<td class=" ">Michael Turewicz, Martin Eisenacher</td>
	<td class=" ">PAA (Protein Array Analyzer)</td>
</tr><tr id="pkg_PADOG" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PADOG.html">PADOG</a></td>
	<td class=" ">Adi Laurentiu Tarca</td>
	<td class=" ">Pathway Analysis with Down-weighting of Overlapping Genes (PADOG)</td>
</tr><tr id="pkg_paircompviz" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/paircompviz.html">paircompviz</a></td>
	<td class=" ">Michal Burda</td>
	<td class=" ">Multiple comparison test visualization</td>
</tr><tr id="pkg_pandaR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pandaR.html">pandaR</a></td>
	<td class=" ">Dan Schlauch</td>
	<td class=" ">PANDA algorithm</td>
</tr><tr id="pkg_PAnnBuilder" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PAnnBuilder.html">PAnnBuilder</a></td>
	<td class=" ">Li Hong</td>
	<td class=" ">Protein annotation data package builder</td>
</tr><tr id="pkg_panp" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/panp.html">panp</a></td>
	<td class=" ">Peter Warren</td>
	<td class=" ">Presence-Absence Calls from Negative Strand Matching Probesets</td>
</tr><tr id="pkg_PANR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PANR.html">PANR</a></td>
	<td class=" ">Xin Wang</td>
	<td class=" ">Posterior association networks and functional modules inferred from rich phenotypes of gene perturbations</td>
</tr><tr id="pkg_PAPi" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PAPi.html">PAPi</a></td>
	<td class=" ">Raphael Aggio</td>
	<td class=" ">Predict metabolic pathway activity based on metabolomics data</td>
</tr><tr id="pkg_parglms" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/parglms.html">parglms</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">support for parallelized estimation of GLMs/GEEs</td>
</tr><tr id="pkg_parody" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/parody.html">parody</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">Parametric And Resistant Outlier DYtection</td>
</tr><tr id="pkg_pathifier" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pathifier.html">pathifier</a></td>
	<td class=" ">Assif Yitzhaky</td>
	<td class=" ">Quantify deregulation of pathways in cancer</td>
</tr><tr id="pkg_PathNet" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PathNet.html">PathNet</a></td>
	<td class=" ">Jason B. Smith</td>
	<td class=" ">An R package for pathway analysis using topological information</td>
</tr><tr id="pkg_pathRender" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pathRender.html">pathRender</a></td>
	<td class=" ">Vince Carey</td>
	<td class=" ">Render molecular pathways</td>
</tr><tr id="pkg_pathview" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pathview.html">pathview</a></td>
	<td class=" ">Weijun Luo</td>
	<td class=" ">a tool set for pathway based data integration and visualization</td>
</tr><tr id="pkg_paxtoolsr" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/paxtoolsr.html">paxtoolsr</a></td>
	<td class=" ">Augustin Luna</td>
	<td class=" ">PaxtoolsR: Access Pathways from Multiple Databases through BioPAX and Pathway Commons</td>
</tr><tr id="pkg_Pbase" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Pbase.html">Pbase</a></td>
	<td class=" ">Sebastian Gibb, Laurent Gatto</td>
	<td class=" ">Manipulating and exploring protein and proteomics data</td>
</tr><tr id="pkg_pcaGoPromoter" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pcaGoPromoter.html">pcaGoPromoter</a></td>
	<td class=" ">Morten Hansen</td>
	<td class=" ">pcaGoPromoter is used to analyze DNA micro array data</td>
</tr><tr id="pkg_pcaMethods" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pcaMethods.html">pcaMethods</a></td>
	<td class=" ">Henning Redestig</td>
	<td class=" ">A collection of PCA methods</td>
</tr><tr id="pkg_pcot2" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pcot2.html">pcot2</a></td>
	<td class=" ">Sarah Song</td>
	<td class=" ">Principal Coordinates and Hotelling's T-Square method</td>
</tr><tr id="pkg_PCpheno" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PCpheno.html">PCpheno</a></td>
	<td class=" ">Nolwenn Le Meur</td>
	<td class=" ">Phenotypes and cellular organizational units</td>
</tr><tr id="pkg_pdInfoBuilder" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pdInfoBuilder.html">pdInfoBuilder</a></td>
	<td class=" ">Benilton Carvalho</td>
	<td class=" ">Platform Design Information Package Builder</td>
</tr><tr id="pkg_pdmclass" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pdmclass.html">pdmclass</a></td>
	<td class=" ">James W. MacDonald</td>
	<td class=" ">Classification of Microarray Samples using Penalized Discriminant Methods</td>
</tr><tr id="pkg_PECA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PECA.html">PECA</a></td>
	<td class=" ">Tomi Suomi</td>
	<td class=" ">Probe-level Expression Change Averaging</td>
</tr><tr id="pkg_pepStat" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pepStat.html">pepStat</a></td>
	<td class=" ">Gregory C Imholte</td>
	<td class=" ">Statistical analysis of peptide microarrays</td>
</tr><tr id="pkg_pepXMLTab" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pepXMLTab.html">pepXMLTab</a></td>
	<td class=" ">Xiaojing Wang</td>
	<td class=" ">Parsing pepXML files and filter based on peptide FDR.</td>
</tr><tr id="pkg_PGSEA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PGSEA.html">PGSEA</a></td>
	<td class=" ">Karl Dykema</td>
	<td class=" ">Parametric Gene Set Enrichment Analysis</td>
</tr><tr id="pkg_phenoDist" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/phenoDist.html">phenoDist</a></td>
	<td class=" ">Xian Zhang</td>
	<td class=" ">Phenotypic distance measures</td>
</tr><tr id="pkg_phenoTest" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/phenoTest.html">phenoTest</a></td>
	<td class=" ">Evarist Planet</td>
	<td class=" ">Tools to test association between gene expression and phenotype in a way that is efficient, structured, fast and scalable. We also provide tools to do GSEA (Gene set enrichment analysis) and copy number variation.</td>
</tr><tr id="pkg_PhenStat" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PhenStat.html">PhenStat</a></td>
	<td class=" ">Natasha Karp</td>
	<td class=" ">Statistical analysis of phenotypic data</td>
</tr><tr id="pkg_phyloseq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/phyloseq.html">phyloseq</a></td>
	<td class=" ">Paul J. McMurdie</td>
	<td class=" ">Handling and analysis of high-throughput microbiome census data.</td>
</tr><tr id="pkg_piano" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/piano.html">piano</a></td>
	<td class=" ">Leif Varemo</td>
	<td class=" ">Platform for integrative analysis of omics data</td>
</tr><tr id="pkg_pickgene" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pickgene.html">pickgene</a></td>
	<td class=" ">Brian S. Yandell</td>
	<td class=" ">Adaptive Gene Picking for Microarray Expression Data Analysis</td>
</tr><tr id="pkg_PICS" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PICS.html">PICS</a></td>
	<td class=" ">Renan Sauteraud</td>
	<td class=" ">Probabilistic inference of ChIP-seq</td>
</tr><tr id="pkg_PING" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PING.html">PING</a></td>
	<td class=" ">Renan Sauteraud</td>
	<td class=" ">Probabilistic inference for Nucleosome Positioning with MNase-based or Sonicated Short-read Data</td>
</tr><tr id="pkg_pint" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pint.html">pint</a></td>
	<td class=" ">Olli-Pekka Huovilainen</td>
	<td class=" ">Pairwise INTegration of functional genomics data</td>
</tr><tr id="pkg_pkgDepTools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pkgDepTools.html">pkgDepTools</a></td>
	<td class=" ">Seth Falcon</td>
	<td class=" ">Package Dependency Tools</td>
</tr><tr id="pkg_plateCore" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/plateCore.html">plateCore</a></td>
	<td class=" ">Errol Strain</td>
	<td class=" ">Statistical tools and data structures for plate-based flow cytometry</td>
</tr><tr id="pkg_plethy" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/plethy.html">plethy</a></td>
	<td class=" ">Daniel Bottomly</td>
	<td class=" ">R framework for exploration and analysis of respirometry data</td>
</tr><tr id="pkg_plgem" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/plgem.html">plgem</a></td>
	<td class=" ">Norman Pavelka</td>
	<td class=" ">Detect differential expression in microarray and proteomics datasets with the Power Law Global Error Model (PLGEM)</td>
</tr><tr id="pkg_plier" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/plier.html">plier</a></td>
	<td class=" ">Crispin Miller</td>
	<td class=" ">Implements the Affymetrix PLIER algorithm</td>
</tr><tr id="pkg_PLPE" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PLPE.html">PLPE</a></td>
	<td class=" ">Soo-heang Eo</td>
	<td class=" ">Local Pooled Error Test for Differential Expression with Paired High-throughput Data</td>
</tr><tr id="pkg_plrs" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/plrs.html">plrs</a></td>
	<td class=" ">Gwenael G.R. Leday to</td>
	<td class=" ">Piecewise Linear Regression Splines (PLRS) for the association between DNA copy number and gene expression</td>
</tr><tr id="pkg_plw" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/plw.html">plw</a></td>
	<td class=" ">Magnus Astrand</td>
	<td class=" ">Probe level Locally moderated Weighted t-tests.</td>
</tr><tr id="pkg_pmm" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pmm.html">pmm</a></td>
	<td class=" ">Anna Drewek</td>
	<td class=" ">Parallel Mixed Model</td>
</tr><tr id="pkg_podkat" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/podkat.html">podkat</a></td>
	<td class=" ">Ulrich Bodenhofer</td>
	<td class=" ">Position-Dependent Kernel Association Test</td>
</tr><tr id="pkg_polyester" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/polyester.html">polyester</a></td>
	<td class=" ">Alyssa Frazee, Jeff Leek</td>
	<td class=" ">Simulate RNA-seq reads</td>
</tr><tr id="pkg_Polyfit" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Polyfit.html">Polyfit</a></td>
	<td class=" ">Conrad Burden</td>
	<td class=" ">Add-on to DESeq to improve p-values and q-values</td>
</tr><tr id="pkg_ppiStats" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ppiStats.html">ppiStats</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Protein-Protein Interaction Statistical Package</td>
</tr><tr id="pkg_prada" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/prada.html">prada</a></td>
	<td class=" ">Florian Hahne</td>
	<td class=" ">Data analysis for cell-based functional assays</td>
</tr><tr id="pkg_prebs" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/prebs.html">prebs</a></td>
	<td class=" ">Karolis Uziela</td>
	<td class=" ">Probe region expression estimation for RNA-seq data for improved microarray comparability</td>
</tr><tr id="pkg_PREDA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PREDA.html">PREDA</a></td>
	<td class=" ">Francesco Ferrari</td>
	<td class=" ">Position RElated Data Anlysis</td>
</tr><tr id="pkg_predictionet" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/predictionet.html">predictionet</a></td>
	<td class=" ">Benjamin Haibe-Kains, Catharina Olsen</td>
	<td class=" ">Inference for predictive networks designed for (but not limited to) genomic data</td>
</tr><tr id="pkg_preprocessCore" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/preprocessCore.html">preprocessCore</a></td>
	<td class=" ">Benjamin Milo Bolstad</td>
	<td class=" ">A collection of pre-processing functions</td>
</tr><tr id="pkg_proBAMr" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/proBAMr.html">proBAMr</a></td>
	<td class=" ">Xiaojing Wang</td>
	<td class=" ">Generating SAM file for PSMs in shotgun proteomics data</td>
</tr><tr id="pkg_PROcess" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PROcess.html">PROcess</a></td>
	<td class=" ">Xiaochun Li</td>
	<td class=" ">Ciphergen SELDI-TOF Processing</td>
</tr><tr id="pkg_procoil" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/procoil.html">procoil</a></td>
	<td class=" ">Ulrich Bodenhofer</td>
	<td class=" ">Prediction of Oligomerization of Coiled Coil Proteins</td>
</tr><tr id="pkg_ProCoNA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ProCoNA.html">ProCoNA</a></td>
	<td class=" ">David L Gibbs</td>
	<td class=" ">Protein co-expression network analysis (ProCoNA).</td>
</tr><tr id="pkg_pRoloc" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pRoloc.html">pRoloc</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">A unifying bioinformatics framework for spatial proteomics</td>
</tr><tr id="pkg_pRolocGUI" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pRolocGUI.html">pRolocGUI</a></td>
	<td class=" ">Laurent Gatto, Thomas Naake</td>
	<td class=" ">Interactive visualisation of spatial proteomics data</td>
</tr><tr id="pkg_PROMISE" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PROMISE.html">PROMISE</a></td>
	<td class=" ">Stan Pounds, Xueyuan Cao</td>
	<td class=" ">PRojection Onto the Most Interesting Statistical Evidence</td>
</tr><tr id="pkg_PROPER" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PROPER.html">PROPER</a></td>
	<td class=" ">Hao Wu</td>
	<td class=" ">PROspective Power Evaluation for RNAseq</td>
</tr><tr id="pkg_prot2D" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/prot2D.html">prot2D</a></td>
	<td class=" ">Sebastien Artigaud</td>
	<td class=" ">Statistical Tools for volume data from 2D Gel Electrophoresis</td>
</tr><tr id="pkg_proteinProfiles" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/proteinProfiles.html">proteinProfiles</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Protein Profiling</td>
</tr><tr id="pkg_proteoQC" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/proteoQC.html">proteoQC</a></td>
	<td class=" ">Bo Wen</td>
	<td class=" ">An R package for proteomics data quality control</td>
</tr><tr id="pkg_ProtGenerics" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ProtGenerics.html">ProtGenerics</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">S4 generic functions for Bioconductor proteomics infrastructure</td>
</tr><tr id="pkg_PSEA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PSEA.html">PSEA</a></td>
	<td class=" ">Alexandre Kuhn</td>
	<td class=" ">Population-Specific Expression Analysis.</td>
</tr><tr id="pkg_PSICQUIC" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PSICQUIC.html">PSICQUIC</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">Protemics Standard Initiative Common QUery InterfaCe</td>
</tr><tr id="pkg_puma" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/puma.html">puma</a></td>
	<td class=" ">Xuejun Liu</td>
	<td class=" ">Propagating Uncertainty in Microarray Analysis(including Affymetrix tranditional 3' arrays and exon arrays and Human Transcriptome Array 2.0)</td>
</tr><tr id="pkg_pvac" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pvac.html">pvac</a></td>
	<td class=" ">Jun Lu, Pierre R. Bushel</td>
	<td class=" ">PCA-based gene filtering for Affymetrix arrays</td>
</tr><tr id="pkg_pvca" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pvca.html">pvca</a></td>
	<td class=" ">Jianying LI</td>
	<td class=" ">Principal Variance Component Analysis (PVCA)</td>
</tr><tr id="pkg_Pviz" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Pviz.html">Pviz</a></td>
	<td class=" ">Renan Sauteraud</td>
	<td class=" ">Peptide Annotation and Data Visualization using Gviz</td>
</tr><tr id="pkg_PWMEnrich" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/PWMEnrich.html">PWMEnrich</a></td>
	<td class=" ">Robert Stojnic</td>
	<td class=" ">PWM enrichment analysis</td>
</tr><tr id="pkg_pwOmics" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/pwOmics.html">pwOmics</a></td>
	<td class=" ">Astrid Wachter</td>
	<td class=" ">Pathway-based data integration of omics data</td>
</tr><tr id="pkg_qcmetrics" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/qcmetrics.html">qcmetrics</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">A Framework for Quality Control</td>
</tr><tr id="pkg_QDNAseq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/QDNAseq.html">QDNAseq</a></td>
	<td class=" ">Daoud Sie</td>
	<td class=" ">Quantitative DNA sequencing for chromosomal aberrations</td>
</tr><tr id="pkg_qpcrNorm" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/qpcrNorm.html">qpcrNorm</a></td>
	<td class=" ">Jessica Mar</td>
	<td class=" ">Data-driven normalization strategies for high-throughput qPCR data.</td>
</tr><tr id="pkg_qpgraph" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/qpgraph.html">qpgraph</a></td>
	<td class=" ">Robert Castelo</td>
	<td class=" ">Estimation of genetic and molecular regulatory networks from high-throughput genomics data</td>
</tr><tr id="pkg_qrqc" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/qrqc.html">qrqc</a></td>
	<td class=" ">Vince Buffalo</td>
	<td class=" ">Quick Read Quality Control</td>
</tr><tr id="pkg_QUALIFIER" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/QUALIFIER.html">QUALIFIER</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">Quality Control of Gated Flow Cytometry Experiments</td>
</tr><tr id="pkg_quantro" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/quantro.html">quantro</a></td>
	<td class=" ">Stephanie Hicks</td>
	<td class=" ">A test for when to use quantile normalization</td>
</tr><tr id="pkg_quantsmooth" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/quantsmooth.html">quantsmooth</a></td>
	<td class=" ">Jan Oosting</td>
	<td class=" ">Quantile smoothing and genomic visualization of array data</td>
</tr><tr id="pkg_QuartPAC" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/QuartPAC.html">QuartPAC</a></td>
	<td class=" ">Gregory Ryslik</td>
	<td class=" ">Identification of mutational clusters in protein quaternary structures.</td>
</tr><tr id="pkg_QuasR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/QuasR.html">QuasR</a></td>
	<td class=" ">Michael Stadler</td>
	<td class=" ">Quantify and Annotate Short Reads in R</td>
</tr><tr id="pkg_qusage" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/qusage.html">qusage</a></td>
	<td class=" ">Christopher Bolen</td>
	<td class=" ">qusage: Quantitative Set Analysis for Gene Expression</td>
</tr><tr id="pkg_qvalue" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/qvalue.html">qvalue</a></td>
	<td class=" ">John D. Storey</td>
	<td class=" ">Q-value estimation for false discovery rate control</td>
</tr><tr id="pkg_R3CPET" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/R3CPET.html">R3CPET</a></td>
	<td class=" ">Mohamed Nadhir Djekidel</td>
	<td class=" ">3CPET: Finding Co-factor Complexes in Chia-PET experiment using a Hierarchical Dirichlet Process</td>
</tr><tr id="pkg_r3Cseq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/r3Cseq.html">r3Cseq</a></td>
	<td class=" ">Supat Thongjuea</td>
	<td class=" ">Analysis of Chromosome Conformation Capture and Next-generation Sequencing (3C-seq)</td>
</tr><tr id="pkg_R453Plus1Toolbox" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/R453Plus1Toolbox.html">R453Plus1Toolbox</a></td>
	<td class=" ">Hans-Ulrich Klein</td>
	<td class=" ">A package for importing and analyzing data from Roche's Genome Sequencer System</td>
</tr><tr id="pkg_rain" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rain.html">rain</a></td>
	<td class=" ">Paul F. Thaben</td>
	<td class=" ">Rhythmicity Analysis Incorporating Non-parametric Methods</td>
</tr><tr id="pkg_rama" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rama.html">rama</a></td>
	<td class=" ">Raphael Gottardo</td>
	<td class=" ">Robust Analysis of MicroArrays</td>
</tr><tr id="pkg_RamiGO" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RamiGO.html">RamiGO</a></td>
	<td class=" ">Markus Schroeder</td>
	<td class=" ">AmiGO visualize R interface</td>
</tr><tr id="pkg_randPack" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/randPack.html">randPack</a></td>
	<td class=" ">Robert Gentleman</td>
	<td class=" ">Randomization routines for Clinical Trials</td>
</tr><tr id="pkg_RankProd" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RankProd.html">RankProd</a></td>
	<td class=" ">Fangxin Hong</td>
	<td class=" ">Rank Product method for identifying differentially expressed genes with application in meta-analysis</td>
</tr><tr id="pkg_Rariant" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rariant.html">Rariant</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Identification and Assessment of Single Nucleotide Variants through Shifts in Non-Consensus Base Call Frequencies</td>
</tr><tr id="pkg_RbcBook1" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RbcBook1.html">RbcBook1</a></td>
	<td class=" ">Vince Carey</td>
	<td class=" ">Support for Springer monograph on Bioconductor</td>
</tr><tr id="pkg_RBGL" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RBGL.html">RBGL</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">An interface to the BOOST graph library</td>
</tr><tr id="pkg_RBioinf" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RBioinf.html">RBioinf</a></td>
	<td class=" ">Robert Gentleman</td>
	<td class=" ">RBioinf</td>
</tr><tr id="pkg_rBiopaxParser" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rBiopaxParser.html">rBiopaxParser</a></td>
	<td class=" ">Frank Kramer</td>
	<td class=" ">Parses BioPax files and represents them in R</td>
</tr><tr id="pkg_RBM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RBM.html">RBM</a></td>
	<td class=" ">Dongmei Li</td>
	<td class=" ">RBM: a R package for microarray and RNA-Seq data analysis</td>
</tr><tr id="pkg_Rbowtie" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rbowtie.html">Rbowtie</a></td>
	<td class=" ">Michael Stadler</td>
	<td class=" ">R bowtie wrapper</td>
</tr><tr id="pkg_rbsurv" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rbsurv.html">rbsurv</a></td>
	<td class=" ">Soo-heang Eo</td>
	<td class=" ">Robust likelihood-based survival modeling with microarray data</td>
</tr><tr id="pkg_Rcade" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rcade.html">Rcade</a></td>
	<td class=" ">Jonathan Cairns</td>
	<td class=" ">R-based analysis of ChIP-seq And Differential Expression - a tool for integrating a count-based ChIP-seq analysis with differential expression summary data.</td>
</tr><tr id="pkg_RCASPAR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RCASPAR.html">RCASPAR</a></td>
	<td class=" ">Douaa Mugahid, Lars Kaderali</td>
	<td class=" ">A package for survival time prediction based on a piecewise baseline hazard Cox regression model.</td>
</tr><tr id="pkg_rcellminer" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rcellminer.html">rcellminer</a></td>
	<td class=" ">Augustin Luna, Vinodh Rajapakse</td>
	<td class=" ">rcellminer: Molecular Profiles and Drug Response for the NCI-60 Cell Lines</td>
</tr><tr id="pkg_Rchemcpp" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rchemcpp.html">Rchemcpp</a></td>
	<td class=" ">Guenter Klambauer</td>
	<td class=" ">Similarity measures for chemical compounds</td>
</tr><tr id="pkg_RchyOptimyx" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RchyOptimyx.html">RchyOptimyx</a></td>
	<td class=" ">Adrin Jalali, Nima Aghaeepour</td>
	<td class=" ">Optimyzed Cellular Hierarchies for Flow Cytometry</td>
</tr><tr id="pkg_Rcpi" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rcpi.html">Rcpi</a></td>
	<td class=" ">Nan Xiao</td>
	<td class=" ">Toolkit for Compound-Protein Interaction in Drug Discovery</td>
</tr><tr id="pkg_RCyjs" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RCyjs.html">RCyjs</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">Display and manipulate graphs in Cytoscape.js</td>
</tr><tr id="pkg_RCytoscape" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RCytoscape.html">RCytoscape</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">Display and manipulate graphs in Cytoscape</td>
</tr><tr id="pkg_RDAVIDWebService" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RDAVIDWebService.html">RDAVIDWebService</a></td>
	<td class=" ">Cristobal Fresno</td>
	<td class=" ">An R Package for retrieving data from DAVID into R objects using Web Services API.</td>
</tr><tr id="pkg_Rdisop" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rdisop.html">Rdisop</a></td>
	<td class=" ">Steffen Neumann</td>
	<td class=" ">Decomposition of Isotopic Patterns</td>
</tr><tr id="pkg_RDRToolbox" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RDRToolbox.html">RDRToolbox</a></td>
	<td class=" ">Christoph Bartenhagen</td>
	<td class=" ">A package for nonlinear dimension reduction with Isomap and LLE.</td>
</tr><tr id="pkg_ReactomePA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ReactomePA.html">ReactomePA</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">Reactome Pathway Analysis</td>
</tr><tr id="pkg_ReadqPCR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ReadqPCR.html">ReadqPCR</a></td>
	<td class=" ">James Perkins</td>
	<td class=" ">Read qPCR data</td>
</tr><tr id="pkg_reb" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/reb.html">reb</a></td>
	<td class=" ">Karl J. Dykema</td>
	<td class=" ">Regional Expression Biases</td>
</tr><tr id="pkg_RedeR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RedeR.html">RedeR</a></td>
	<td class=" ">Mauro Castro</td>
	<td class=" ">Interactive visualization and manipulation of nested networks</td>
</tr><tr id="pkg_REDseq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/REDseq.html">REDseq</a></td>
	<td class=" ">Lihua Julie Zhu</td>
	<td class=" ">Analysis of high-throughput sequencing data processed by restriction enzyme digestion</td>
</tr><tr id="pkg_RefNet" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RefNet.html">RefNet</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">A queryable collection of molecular interactions, from many sources</td>
</tr><tr id="pkg_RefPlus" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RefPlus.html">RefPlus</a></td>
	<td class=" ">Kai-Ming Chang</td>
	<td class=" ">A function set for the Extrapolation Strategy (RMA+) and Extrapolation Averaging (RMA++) methods.</td>
</tr><tr id="pkg_regioneR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/regioneR.html">regioneR</a></td>
	<td class=" ">Bernat Gel</td>
	<td class=" ">Association analysis of genomic regions based on permutation tests</td>
</tr><tr id="pkg_regionReport" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/regionReport.html">regionReport</a></td>
	<td class=" ">Leonardo Collado-Torres</td>
	<td class=" ">Generate HTML reports for exploring a set of regions</td>
</tr><tr id="pkg_Repitools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Repitools.html">Repitools</a></td>
	<td class=" ">Mark Robinson</td>
	<td class=" ">Epigenomic tools</td>
</tr><tr id="pkg_ReportingTools" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ReportingTools.html">ReportingTools</a></td>
	<td class=" ">Jason A. Hackney, Gabriel Becker, Jessica L. Larson</td>
	<td class=" ">Tools for making reports in various formats</td>
</tr><tr id="pkg_ReQON" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ReQON.html">ReQON</a></td>
	<td class=" ">Christopher Cabanski</td>
	<td class=" ">Recalibrating Quality Of Nucleotides</td>
</tr><tr id="pkg_rfPred" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rfPred.html">rfPred</a></td>
	<td class=" ">Hugo Varet</td>
	<td class=" ">Assign rfPred functional prediction scores to a missense variants list</td>
</tr><tr id="pkg_rGADEM" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rGADEM.html">rGADEM</a></td>
	<td class=" ">Arnaud Droit</td>
	<td class=" ">de novo motif discovery</td>
</tr><tr id="pkg_RGalaxy" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RGalaxy.html">RGalaxy</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Make an R function available in the Galaxy web platform</td>
</tr><tr id="pkg_Rgraphviz" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rgraphviz.html">Rgraphviz</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Provides plotting capabilities for R graph objects</td>
</tr><tr id="pkg_rGREAT" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rGREAT.html">rGREAT</a></td>
	<td class=" ">Zuguang Gu</td>
	<td class=" ">Client for GREAT Analysis</td>
</tr><tr id="pkg_RGSEA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RGSEA.html">RGSEA</a></td>
	<td class=" ">Chengcheng Ma</td>
	<td class=" ">Random Gene Set Enrichment Analysis</td>
</tr><tr id="pkg_rgsepd" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rgsepd.html">rgsepd</a></td>
	<td class=" ">Karl Stamm</td>
	<td class=" ">Gene Set Enrichment / Projection Displays</td>
</tr><tr id="pkg_rhdf5" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rhdf5.html">rhdf5</a></td>
	<td class=" ">Bernd Fischer</td>
	<td class=" ">HDF5 interface to R</td>
</tr><tr id="pkg_Rhtslib" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rhtslib.html">Rhtslib</a></td>
	<td class=" ">Nathaniel Hayden</td>
	<td class=" ">HTSlib high-throughput sequencing library as an R package</td>
</tr><tr id="pkg_rHVDM" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rHVDM.html">rHVDM</a></td>
	<td class=" ">Martino Barenco</td>
	<td class=" ">Hidden Variable Dynamic Modeling</td>
</tr><tr id="pkg_riboSeqR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/riboSeqR.html">riboSeqR</a></td>
	<td class=" ">Thomas J. Hardcastle</td>
	<td class=" ">Analysis of sequencing data from ribosome profiling experiments.</td>
</tr><tr id="pkg_Ringo" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Ringo.html">Ringo</a></td>
	<td class=" ">J. Toedling</td>
	<td class=" ">R Investigation of ChIP-chip Oligoarrays</td>
</tr><tr id="pkg_RIPSeeker" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RIPSeeker.html">RIPSeeker</a></td>
	<td class=" ">Yue Li</td>
	<td class=" ">RIPSeeker: a statistical package for identifying protein-associated transcripts from RIP-seq experiments</td>
</tr><tr id="pkg_Risa" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Risa.html">Risa</a></td>
	<td class=" ">Alejandra Gonzalez-Beltran, ISA Team</td>
	<td class=" ">Converting experimental metadata from ISA-tab into Bioconductor data structures</td>
</tr><tr id="pkg_RLMM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RLMM.html">RLMM</a></td>
	<td class=" ">Nusrat Rabbee</td>
	<td class=" ">A Genotype Calling Algorithm for Affymetrix SNP Arrays</td>
</tr><tr id="pkg_Rmagpie" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rmagpie.html">Rmagpie</a></td>
	<td class=" ">Camille Maumet</td>
	<td class=" ">MicroArray Gene-expression-based Program In Error rate estimation</td>
</tr><tr id="pkg_RMassBank" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RMassBank.html">RMassBank</a></td>
	<td class=" ">RMassBank at Eawag</td>
	<td class=" ">Workflow to process tandem MS files and build MassBank records</td>
</tr><tr id="pkg_rMAT" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rMAT.html">rMAT</a></td>
	<td class=" ">Arnaud Droit and Raphael Gottardo</td>
	<td class=" ">R implementation from MAT program to normalize and analyze tiling arrays and ChIP-chip data.</td>
</tr><tr id="pkg_RmiR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RmiR.html">RmiR</a></td>
	<td class=" ">Francesco Favero</td>
	<td class=" ">Package to work with miRNAs and miRNA targets with R</td>
</tr><tr id="pkg_RmiR.Hs.miRNA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RmiR.Hs.miRNA.html">RmiR.Hs.miRNA</a></td>
	<td class=" ">Francesco Favero</td>
	<td class=" ">Various databases of microRNA Targets</td>
</tr><tr id="pkg_RmiR.hsa" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RmiR.hsa.html">RmiR.hsa</a></td>
	<td class=" ">Francesco Favero</td>
	<td class=" ">Various databases of microRNA Targets</td>
</tr><tr id="pkg_RNAinteract" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RNAinteract.html">RNAinteract</a></td>
	<td class=" ">Bernd Fischer</td>
	<td class=" ">Estimate Pairwise Interactions from multidimensional features</td>
</tr><tr id="pkg_RNAither" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RNAither.html">RNAither</a></td>
	<td class=" ">Lars Kaderali</td>
	<td class=" ">Statistical analysis of high-throughput RNAi screens</td>
</tr><tr id="pkg_RNAprobR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RNAprobR.html">RNAprobR</a></td>
	<td class=" ">Nikos Sidiropoulos</td>
	<td class=" ">An R package for analysis of massive parallel sequencing based RNA structure probing data</td>
</tr><tr id="pkg_rnaSeqMap" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rnaSeqMap.html">rnaSeqMap</a></td>
	<td class=" ">Michal Okoniewski</td>
	<td class=" ">rnaSeq secondary analyses</td>
</tr><tr id="pkg_RNASeqPower" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RNASeqPower.html">RNASeqPower</a></td>
	<td class=" ">Terry M Therneau</td>
	<td class=" ">Sample size for RNAseq studies</td>
</tr><tr id="pkg_RnaSeqSampleSize" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RnaSeqSampleSize.html">RnaSeqSampleSize</a></td>
	<td class=" ">Shilin Zhao</td>
	<td class=" ">RnaSeqSampleSize</td>
</tr><tr id="pkg_RnBeads" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RnBeads.html">RnBeads</a></td>
	<td class=" ">Fabian Mueller</td>
	<td class=" ">RnBeads</td>
</tr><tr id="pkg_Rnits" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rnits.html">Rnits</a></td>
	<td class=" ">Dipen P. Sangurdekar</td>
	<td class=" ">R Normalization and Inference of Time Series data</td>
</tr><tr id="pkg_roar" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/roar.html">roar</a></td>
	<td class=" ">Elena Grassi</td>
	<td class=" ">Identify differential APA usage from RNA-seq alignments</td>
</tr><tr id="pkg_ROC" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ROC.html">ROC</a></td>
	<td class=" ">Vince Carey</td>
	<td class=" ">utilities for ROC, with uarray focus</td>
</tr><tr id="pkg_Roleswitch" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Roleswitch.html">Roleswitch</a></td>
	<td class=" ">Yue Li</td>
	<td class=" ">Infer miRNA-mRNA interactions using paired expression data from a single sample</td>
</tr><tr id="pkg_Rolexa" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rolexa.html">Rolexa</a></td>
	<td class=" ">Jacques Rougemont</td>
	<td class=" ">Statistical analysis of Solexa sequencing data</td>
</tr><tr id="pkg_rols" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rols.html">rols</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">An R interface to the Ontology Lookup Service</td>
</tr><tr id="pkg_ROntoTools" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ROntoTools.html">ROntoTools</a></td>
	<td class=" ">Calin Voichita</td>
	<td class=" ">R Onto-Tools suite</td>
</tr><tr id="pkg_RPA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RPA.html">RPA</a></td>
	<td class=" ">Leo Lahti</td>
	<td class=" ">RPA: Robust Probabilistic Averaging for probe-level analysis</td>
</tr><tr id="pkg_RpsiXML" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RpsiXML.html">RpsiXML</a></td>
	<td class=" ">Jitao David Zhang</td>
	<td class=" ">R interface to PSI-MI 2.5 files</td>
</tr><tr id="pkg_rpx" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rpx.html">rpx</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">R Interface to the ProteomeXchange Repository</td>
</tr><tr id="pkg_Rqc" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rqc.html">Rqc</a></td>
	<td class=" ">Welliton Souza</td>
	<td class=" ">Quality Control Tool for High-Throughput Sequencing Data</td>
</tr><tr id="pkg_rqubic" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rqubic.html">rqubic</a></td>
	<td class=" ">Jitao David Zhang</td>
	<td class=" ">Qualitative biclustering algorithm for expression data analysis in R</td>
</tr><tr id="pkg_rRDP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rRDP.html">rRDP</a></td>
	<td class=" ">Michael Hahsler</td>
	<td class=" ">Interface to the RDP Classifier</td>
</tr><tr id="pkg_RRHO" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RRHO.html">RRHO</a></td>
	<td class=" ">Jonathan Rosenblatt</td>
	<td class=" ">Inference on agreement between ordered lists</td>
</tr><tr id="pkg_Rsamtools" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rsamtools.html">Rsamtools</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Binary alignment (BAM), FASTA, variant call (BCF), and tabix file import</td>
</tr><tr id="pkg_rsbml" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rsbml.html">rsbml</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">R support for SBML, using libsbml</td>
</tr><tr id="pkg_rSFFreader" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rSFFreader.html">rSFFreader</a></td>
	<td class=" ">Matt Settles</td>
	<td class=" ">rSFFreader reads in sff files generated by Roche 454 and Life Sciences Ion Torrent sequencers</td>
</tr><tr id="pkg_Rsubread" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rsubread.html">Rsubread</a></td>
	<td class=" ">Wei Shi</td>
	<td class=" ">Rsubread package: high-performance read alignment, quantification and mutation discovery</td>
</tr><tr id="pkg_RSVSim" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RSVSim.html">RSVSim</a></td>
	<td class=" ">Christoph Bartenhagen</td>
	<td class=" ">RSVSim: an R/Bioconductor package for the simulation of structural variations</td>
</tr><tr id="pkg_rTANDEM" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rTANDEM.html">rTANDEM</a></td>
	<td class=" ">Frederic Fournier</td>
	<td class=" ">Interfaces the tandem protein identification algorithm in R</td>
</tr><tr id="pkg_RTCA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RTCA.html">RTCA</a></td>
	<td class=" ">Jitao David Zhang</td>
	<td class=" ">Open-source toolkit to analyse data from xCELLigence System (RTCA)</td>
</tr><tr id="pkg_RTN" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RTN.html">RTN</a></td>
	<td class=" ">Mauro Castro</td>
	<td class=" ">Reconstruction of transcriptional networks and analysis of master regulators</td>
</tr><tr id="pkg_RTopper" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RTopper.html">RTopper</a></td>
	<td class=" ">Luigi Marchionni</td>
	<td class=" ">This package is designed to perform Gene Set Analysis across multiple genomic platforms</td>
</tr><tr id="pkg_rtracklayer" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rtracklayer.html">rtracklayer</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">R interface to genome browsers and their annotation tracks</td>
</tr><tr id="pkg_Rtreemix" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Rtreemix.html">Rtreemix</a></td>
	<td class=" ">Jasmina Bogojeska</td>
	<td class=" ">Rtreemix: Mutagenetic trees mixture models.</td>
</tr><tr id="pkg_rTRM" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rTRM.html">rTRM</a></td>
	<td class=" ">Diego Diez</td>
	<td class=" ">Identification of transcriptional regulatory modules from PPI networks</td>
</tr><tr id="pkg_rTRMui" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/rTRMui.html">rTRMui</a></td>
	<td class=" ">Diego Diez</td>
	<td class=" ">A shiny user interface for rTRM</td>
</tr><tr id="pkg_RUVcorr" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RUVcorr.html">RUVcorr</a></td>
	<td class=" ">Saskia Freytag</td>
	<td class=" ">Removal of unwanted variation for gene-gene correlations and related analysis</td>
</tr><tr id="pkg_RUVnormalize" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RUVnormalize.html">RUVnormalize</a></td>
	<td class=" ">Laurent Jacob</td>
	<td class=" ">RUV for normalization of expression array data</td>
</tr><tr id="pkg_RUVSeq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RUVSeq.html">RUVSeq</a></td>
	<td class=" ">Davide Risso</td>
	<td class=" ">Remove Unwanted Variation from RNA-Seq Data</td>
</tr><tr id="pkg_RWebServices" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/RWebServices.html">RWebServices</a></td>
	<td class=" ">Martin Morgan</td>
	<td class=" ">Expose R functions as web services through Java/Axis/Apache</td>
</tr><tr id="pkg_S4Vectors" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/S4Vectors.html">S4Vectors</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">S4 implementation of vectors and lists</td>
</tr><tr id="pkg_safe" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/safe.html">safe</a></td>
	<td class=" ">William T. Barry</td>
	<td class=" ">Significance Analysis of Function and Expression</td>
</tr><tr id="pkg_sagenhaft" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sagenhaft.html">sagenhaft</a></td>
	<td class=" ">Tim Beissbarth</td>
	<td class=" ">Collection of functions for reading and comparing SAGE libraries</td>
</tr><tr id="pkg_SAGx" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SAGx.html">SAGx</a></td>
	<td class=" ">Per Broberg,</td>
	<td class=" ">Statistical Analysis of the GeneChip</td>
</tr><tr id="pkg_SamSPECTRAL" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SamSPECTRAL.html">SamSPECTRAL</a></td>
	<td class=" ">Habil Zare</td>
	<td class=" ">Identifies cell population in flow cytometry data.</td>
</tr><tr id="pkg_sangerseqR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sangerseqR.html">sangerseqR</a></td>
	<td class=" ">Jonathon Hill</td>
	<td class=" ">Tools for Sanger Sequencing Data in R</td>
</tr><tr id="pkg_SANTA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SANTA.html">SANTA</a></td>
	<td class=" ">Alex J. Cornish</td>
	<td class=" ">Spatial Analysis of Network Associations</td>
</tr><tr id="pkg_sapFinder" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sapFinder.html">sapFinder</a></td>
	<td class=" ">Shaohang Xu, Bo Wen</td>
	<td class=" ">A package for variant peptides detection and visualization in shotgun proteomics.</td>
</tr><tr id="pkg_saps" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/saps.html">saps</a></td>
	<td class=" ">Daniel Schmolze</td>
	<td class=" ">Significance Analysis of Prognostic Signatures</td>
</tr><tr id="pkg_savR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/savR.html">savR</a></td>
	<td class=" ">R. Brent Calder</td>
	<td class=" ">Parse and analyze Illumina SAV files</td>
</tr><tr id="pkg_SBMLR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SBMLR.html">SBMLR</a></td>
	<td class=" ">Tomas Radivoyevitch</td>
	<td class=" ">SBML-R Interface and Analysis Tools</td>
</tr><tr id="pkg_SCAN.UPC" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SCAN.UPC.html">SCAN.UPC</a></td>
	<td class=" ">Stephen R. Piccolo</td>
	<td class=" ">Single-channel array normalization (SCAN) and Universal exPression Codes (UPC)</td>
</tr><tr id="pkg_ScISI" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ScISI.html">ScISI</a></td>
	<td class=" ">Tony Chiang</td>
	<td class=" ">In Silico Interactome</td>
</tr><tr id="pkg_scsR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/scsR.html">scsR</a></td>
	<td class=" ">Andrea Franceschini, Roger Meier, Christian von Mering</td>
	<td class=" ">SiRNA correction for seed mediated off-target effect</td>
</tr><tr id="pkg_segmentSeq" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/segmentSeq.html">segmentSeq</a></td>
	<td class=" ">Thomas J. Hardcastle</td>
	<td class=" ">Methods for identifying small RNA loci from high-throughput sequencing data</td>
</tr><tr id="pkg_SELEX" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SELEX.html">SELEX</a></td>
	<td class=" ">Harmen Bussemaker</td>
	<td class=" ">Functions for analyzing SELEX-seq data</td>
</tr><tr id="pkg_SemDist" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SemDist.html">SemDist</a></td>
	<td class=" ">Ian Gonzalez</td>
	<td class=" ">Information Accretion-based Function Predictor Evaluation</td>
</tr><tr id="pkg_seq2pathway" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/seq2pathway.html">seq2pathway</a></td>
	<td class=" ">Bin Wang</td>
	<td class=" ">a novel tool for functional gene-set (or termed as pathway) analysis of next-generation sequencing data</td>
</tr><tr id="pkg_SeqArray" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SeqArray.html">SeqArray</a></td>
	<td class=" ">Xiuwen Zheng</td>
	<td class=" ">Big Data Management of Genome-wide Sequencing Variants</td>
</tr><tr id="pkg_seqbias" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/seqbias.html">seqbias</a></td>
	<td class=" ">Daniel Jones</td>
	<td class=" ">Estimation of per-position bias in high-throughput sequencing data</td>
</tr><tr id="pkg_seqCNA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/seqCNA.html">seqCNA</a></td>
	<td class=" ">David Mosen-Ansorena</td>
	<td class=" ">Copy number analysis of high-throughput sequencing cancer data</td>
</tr><tr id="pkg_SeqGSEA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SeqGSEA.html">SeqGSEA</a></td>
	<td class=" ">Xi Wang</td>
	<td class=" ">Gene Set Enrichment Analysis (GSEA) of RNA-Seq Data: integrating differential expression and splicing</td>
</tr><tr id="pkg_seqLogo" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/seqLogo.html">seqLogo</a></td>
	<td class=" ">Oliver Bembom</td>
	<td class=" ">Sequence logos for DNA sequence alignments</td>
</tr><tr id="pkg_seqPattern" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/seqPattern.html">seqPattern</a></td>
	<td class=" ">Vanja Haberle</td>
	<td class=" ">Visualising oligonucleotide patterns and motif occurrences across a set of sorted sequences</td>
</tr><tr id="pkg_seqplots" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/seqplots.html">seqplots</a></td>
	<td class=" ">Przemyslaw Stempor</td>
	<td class=" ">An interactive tool for visualizing NGS signals and sequence motif densities along genomic features using average plots and heatmaps.</td>
</tr><tr id="pkg_seqTools" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/seqTools.html">seqTools</a></td>
	<td class=" ">Wolfgang Kaisers</td>
	<td class=" ">Analysis of nucleotide, sequence and quality content on fastq files.</td>
</tr><tr id="pkg_SeqVarTools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SeqVarTools.html">SeqVarTools</a></td>
	<td class=" ">Stephanie M. Gogarten, Xiuwen Zheng</td>
	<td class=" ">Tools for variant data</td>
</tr><tr id="pkg_SGSeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SGSeq.html">SGSeq</a></td>
	<td class=" ">Leonard Goldstein</td>
	<td class=" ">Splice event detection and quantification from RNA-seq data</td>
</tr><tr id="pkg_shinyMethyl" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/shinyMethyl.html">shinyMethyl</a></td>
	<td class=" ">Jean-Philippe Fortin</td>
	<td class=" ">Interactive visualization for Illumina's 450k methylation arrays</td>
</tr><tr id="pkg_shinyTANDEM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/shinyTANDEM.html">shinyTANDEM</a></td>
	<td class=" ">Frederic Fournier</td>
	<td class=" ">Provides a GUI for rTANDEM</td>
</tr><tr id="pkg_ShortRead" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ShortRead.html">ShortRead</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">FASTQ input and manipulation</td>
</tr><tr id="pkg_sigaR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sigaR.html">sigaR</a></td>
	<td class=" ">Wessel N. van Wieringen</td>
	<td class=" ">statistics for integrative genomics analyses in R</td>
</tr><tr id="pkg_SigCheck" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SigCheck.html">SigCheck</a></td>
	<td class=" ">Rory Stark</td>
	<td class=" ">Check a gene signature's prognostic performance against random signatures, known signatures, and permuted data/metadata</td>
</tr><tr id="pkg_SigFuge" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SigFuge.html">SigFuge</a></td>
	<td class=" ">Patrick Kimes</td>
	<td class=" ">SigFuge</td>
</tr><tr id="pkg_siggenes" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/siggenes.html">siggenes</a></td>
	<td class=" ">Holger Schwender</td>
	<td class=" ">Multiple testing using SAM and Efron's empirical Bayes approaches</td>
</tr><tr id="pkg_sigPathway" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sigPathway.html">sigPathway</a></td>
	<td class=" ">Weil Lai</td>
	<td class=" ">Pathway Analysis</td>
</tr><tr id="pkg_sigsquared" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sigsquared.html">sigsquared</a></td>
	<td class=" ">UnJin Lee</td>
	<td class=" ">Gene signature generation for functionally validated signaling pathways</td>
</tr><tr id="pkg_SIM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SIM.html">SIM</a></td>
	<td class=" ">Renee X. de Menezes</td>
	<td class=" ">Integrated Analysis on two human genomic datasets</td>
</tr><tr id="pkg_SIMAT" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SIMAT.html">SIMAT</a></td>
	<td class=" ">Mo R. Nezami Ranjbar</td>
	<td class=" ">GC-SIM-MS data processing and alaysis tool</td>
</tr><tr id="pkg_SimBindProfiles" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SimBindProfiles.html">SimBindProfiles</a></td>
	<td class=" ">Bettina Fischer</td>
	<td class=" ">Similar Binding Profiles</td>
</tr><tr id="pkg_similaRpeak" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/similaRpeak.html">similaRpeak</a></td>
	<td class=" ">Astrid Louise Deschenes</td>
	<td class=" ">similaRpeak: Metrics to estimate a level of similarity between two ChIP-Seq profiles</td>
</tr><tr id="pkg_simpleaffy" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/simpleaffy.html">simpleaffy</a></td>
	<td class=" ">Crispin Miller</td>
	<td class=" ">Very simple high level analysis of Affymetrix data</td>
</tr><tr id="pkg_simulatorZ" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/simulatorZ.html">simulatorZ</a></td>
	<td class=" ">Yuqing Zhang</td>
	<td class=" ">Simulator for Collections of Independent Genomic Data Sets</td>
</tr><tr id="pkg_sincell" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sincell.html">sincell</a></td>
	<td class=" ">Miguel JuliÃ¡, Antonio Rausell</td>
	<td class=" ">R package for the statistical assessment of cell state hierarchies from single-cell RNA-seq data</td>
</tr><tr id="pkg_sizepower" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sizepower.html">sizepower</a></td>
	<td class=" ">Weiliang Qiu</td>
	<td class=" ">Sample Size and Power Calculation in Micorarray Studies</td>
</tr><tr id="pkg_SJava" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SJava.html">SJava</a></td>
	<td class=" ">Martin Morgan</td>
	<td class=" ">The Omegahat interface for R and Java</td>
</tr><tr id="pkg_skewr" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/skewr.html">skewr</a></td>
	<td class=" ">Ryan Putney</td>
	<td class=" ">Visualize Intensities Produced by Illumina's Human Methylation 450k BeadChip</td>
</tr><tr id="pkg_SLGI" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SLGI.html">SLGI</a></td>
	<td class=" ">Nolwenn Le Meur</td>
	<td class=" ">Synthetic Lethal Genetic Interaction</td>
</tr><tr id="pkg_SLqPCR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SLqPCR.html">SLqPCR</a></td>
	<td class=" ">Matthias Kohl</td>
	<td class=" ">Functions for analysis of real-time quantitative PCR data at SIRS-Lab GmbH</td>
</tr><tr id="pkg_SMAP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SMAP.html">SMAP</a></td>
	<td class=" ">Robin Andersson</td>
	<td class=" ">A Segmental Maximum A Posteriori Approach to Array-CGH Copy Number Profiling</td>
</tr><tr id="pkg_SNAGEE" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SNAGEE.html">SNAGEE</a></td>
	<td class=" ">David Venet</td>
	<td class=" ">Signal-to-Noise applied to Gene Expression Experiments</td>
</tr><tr id="pkg_snapCGH" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/snapCGH.html">snapCGH</a></td>
	<td class=" ">John Marioni</td>
	<td class=" ">Segmentation, normalisation and processing of aCGH data.</td>
</tr><tr id="pkg_snm" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/snm.html">snm</a></td>
	<td class=" ">John D. Storey</td>
	<td class=" ">Supervised Normalization of Microarrays</td>
</tr><tr id="pkg_SNPchip" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SNPchip.html">SNPchip</a></td>
	<td class=" ">Robert Scharpf</td>
	<td class=" ">Visualizations for copy number alterations</td>
</tr><tr id="pkg_SNPRelate" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SNPRelate.html">SNPRelate</a></td>
	<td class=" ">Xiuwen Zheng</td>
	<td class=" ">Parallel Computing Toolset for Relatedness and Principal Component Analysis of SNP Data</td>
</tr><tr id="pkg_snpStats" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/snpStats.html">snpStats</a></td>
	<td class=" ">David Clayton</td>
	<td class=" ">SnpMatrix and XSnpMatrix classes and methods</td>
</tr><tr id="pkg_soGGi" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/soGGi.html">soGGi</a></td>
	<td class=" ">Tom Carroll</td>
	<td class=" ">Visualise ChIP-seq, MNase-seq and motif occurrence as aggregate plots Summarised Over Grouped Genomic Intervals</td>
</tr><tr id="pkg_SomatiCA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SomatiCA.html">SomatiCA</a></td>
	<td class=" ">Mengjie Chen</td>
	<td class=" ">SomatiCA: identifying, characterizing, and quantifying somatic copy number aberrations from cancer genome sequencing</td>
</tr><tr id="pkg_SomaticSignatures" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SomaticSignatures.html">SomaticSignatures</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Somatic Signatures</td>
</tr><tr id="pkg_SpacePAC" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SpacePAC.html">SpacePAC</a></td>
	<td class=" ">Gregory Ryslik</td>
	<td class=" ">Identification of Mutational Clusters in 3D Protein Space via Simulation.</td>
</tr><tr id="pkg_spade" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/spade.html">spade</a></td>
	<td class=" ">Zach Bjornson</td>
	<td class=" ">SPADE -- An analysis and visualization tool for Flow Cytometry</td>
</tr><tr id="pkg_specL" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/specL.html">specL</a></td>
	<td class=" ">Christian Panse</td>
	<td class=" ">specL - Prepare Peptide Spectrum Matches for Use in Targeted Proteomics</td>
</tr><tr id="pkg_SpeCond" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SpeCond.html">SpeCond</a></td>
	<td class=" ">Florence Cavalli</td>
	<td class=" ">Condition specific detection from expression data</td>
</tr><tr id="pkg_SPEM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SPEM.html">SPEM</a></td>
	<td class=" ">Xinyi YANG</td>
	<td class=" ">S-system parameter estimation method</td>
</tr><tr id="pkg_SPIA" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SPIA.html">SPIA</a></td>
	<td class=" ">Adi Laurentiu Tarca</td>
	<td class=" ">Signaling Pathway Impact Analysis (SPIA) using combined evidence of pathway over-representation and unusual signaling perturbations</td>
</tr><tr id="pkg_spikeLI" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/spikeLI.html">spikeLI</a></td>
	<td class=" ">Enrico Carlon</td>
	<td class=" ">Affymetrix Spike-in Langmuir Isotherm Data Analysis Tool</td>
</tr><tr id="pkg_spkTools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/spkTools.html">spkTools</a></td>
	<td class=" ">Matthew N McCall</td>
	<td class=" ">Methods for Spike-in Arrays</td>
</tr><tr id="pkg_splicegear" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/splicegear.html">splicegear</a></td>
	<td class=" ">Laurent Gautier</td>
	<td class=" ">splicegear</td>
</tr><tr id="pkg_spliceR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/spliceR.html">spliceR</a></td>
	<td class=" ">Johannes Waage, Kristoffer Vitting-Seerup</td>
	<td class=" ">Classification of alternative splicing and prediction of coding potential from RNA-seq data.</td>
</tr><tr id="pkg_spliceSites" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/spliceSites.html">spliceSites</a></td>
	<td class=" ">Wolfgang Kaisers</td>
	<td class=" ">Manages align gap positions from RNA-seq data</td>
</tr><tr id="pkg_SplicingGraphs" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SplicingGraphs.html">SplicingGraphs</a></td>
	<td class=" ">H. Pages</td>
	<td class=" ">Create, manipulate, visualize splicing graphs, and assign RNA-seq reads to them</td>
</tr><tr id="pkg_splots" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/splots.html">splots</a></td>
	<td class=" ">Wolfgang Huber</td>
	<td class=" ">Visualization of high-throughput assays in microtitre plate or slide format</td>
</tr><tr id="pkg_spotSegmentation" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/spotSegmentation.html">spotSegmentation</a></td>
	<td class=" ">Chris Fraley</td>
	<td class=" ">Microarray Spot Segmentation and Gridding for Blocks of Microarray Spots</td>
</tr><tr id="pkg_SQUADD" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SQUADD.html">SQUADD</a></td>
	<td class=" ">Martial Sankar</td>
	<td class=" ">Add-on of the SQUAD Software</td>
</tr><tr id="pkg_SRAdb" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SRAdb.html">SRAdb</a></td>
	<td class=" ">Jack Zhu</td>
	<td class=" ">A compilation of metadata from NCBI SRA and tools</td>
</tr><tr id="pkg_sRAP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sRAP.html">sRAP</a></td>
	<td class=" ">Charles Warden</td>
	<td class=" ">Simplified RNA-Seq Analysis Pipeline</td>
</tr><tr id="pkg_sscore" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sscore.html">sscore</a></td>
	<td class=" ">Richard Kennedy</td>
	<td class=" ">S-Score Algorithm for Affymetrix Oligonucleotide Microarrays</td>
</tr><tr id="pkg_sSeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sSeq.html">sSeq</a></td>
	<td class=" ">Danni Yu</td>
	<td class=" ">Shrinkage estimation of dispersion in Negative Binomial models for RNA-seq experiments with small sample size</td>
</tr><tr id="pkg_ssize" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ssize.html">ssize</a></td>
	<td class=" ">Gregory R. Warnes</td>
	<td class=" ">Estimate Microarray Sample Size</td>
</tr><tr id="pkg_SSPA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SSPA.html">SSPA</a></td>
	<td class=" ">Maarten van Iterson</td>
	<td class=" ">General Sample Size and Power Analysis for Microarray and Next-Generation Sequencing Data</td>
</tr><tr id="pkg_ssviz" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ssviz.html">ssviz</a></td>
	<td class=" ">Diana Low</td>
	<td class=" ">A small RNA-seq visualizer and analysis toolkit</td>
</tr><tr id="pkg_STAN" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/STAN.html">STAN</a></td>
	<td class=" ">Benedikt Zacher</td>
	<td class=" ">STrand-specific ANnotation of genomic data</td>
</tr><tr id="pkg_staRank" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/staRank.html">staRank</a></td>
	<td class=" ">Juliane Siebourg</td>
	<td class=" ">Stability Ranking</td>
</tr><tr id="pkg_Starr" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Starr.html">Starr</a></td>
	<td class=" ">Benedikt Zacher</td>
	<td class=" ">Simple tiling array analysis of Affymetrix ChIP-chip data</td>
</tr><tr id="pkg_STATegRa" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/STATegRa.html">STATegRa</a></td>
	<td class=" ">David Gomez-Cabrero, Patricia SebastiÃ¡n-LeÃ³n, Gordon Ball</td>
	<td class=" ">Classes and methods for multi-omics data integration</td>
</tr><tr id="pkg_stepNorm" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/stepNorm.html">stepNorm</a></td>
	<td class=" ">Yuanyuan Xiao</td>
	<td class=" ">Stepwise normalization functions for cDNA microarrays</td>
</tr><tr id="pkg_stepwiseCM" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/stepwiseCM.html">stepwiseCM</a></td>
	<td class=" ">Askar Obulkasim</td>
	<td class=" ">Stepwise Classification of Cancer Samples using High-dimensional Data Sets</td>
</tr><tr id="pkg_Streamer" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Streamer.html">Streamer</a></td>
	<td class=" ">Martin Morgan</td>
	<td class=" ">Enabling stream processing of large files</td>
</tr><tr id="pkg_STRINGdb" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/STRINGdb.html">STRINGdb</a></td>
	<td class=" ">Andrea Franceschini</td>
	<td class=" ">STRINGdb (Search Tool for the Retrieval of Interacting proteins database)</td>
</tr><tr id="pkg_supraHex" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/supraHex.html">supraHex</a></td>
	<td class=" ">Hai Fang</td>
	<td class=" ">A supra-hexagonal map for analysing tabular omics data</td>
</tr><tr id="pkg_survcomp" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/survcomp.html">survcomp</a></td>
	<td class=" ">Benjamin Haibe-Kains, Markus Schroeder, Catharina Olsen</td>
	<td class=" ">Performance Assessment and Comparison for Survival Analysis</td>
</tr><tr id="pkg_Sushi" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Sushi.html">Sushi</a></td>
	<td class=" ">Douglas H Phanstiel</td>
	<td class=" ">Tools for visualizing genomics data</td>
</tr><tr id="pkg_sva" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/sva.html">sva</a></td>
	<td class=" ">Jeffrey T. Leek, John D. Storey, W. Evan Johnson</td>
	<td class=" ">Surrogate Variable Analysis</td>
</tr><tr id="pkg_SVM2CRM" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SVM2CRM.html">SVM2CRM</a></td>
	<td class=" ">Guidantonio Malagoli Tagliazucchi</td>
	<td class=" ">SVM2CRM: support vector machine for cis-regulatory elements detections</td>
</tr><tr id="pkg_SwimR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/SwimR.html">SwimR</a></td>
	<td class=" ">Randy Blakely</td>
	<td class=" ">SwimR: A Suite of Analytical Tools for Quantification of C. elegans Swimming Behavior</td>
</tr><tr id="pkg_switchBox" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/switchBox.html">switchBox</a></td>
	<td class=" ">Bahman Afsari, Luigi Marchionni</td>
	<td class=" ">Utilities to train and validate classifiers based on pair switching using the K-Top-Scoring-Pair (KTSP) algorithm.</td>
</tr><tr id="pkg_synapter" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/synapter.html">synapter</a></td>
	<td class=" ">Laurent Gatto and Sebastian Gibb</td>
	<td class=" ">Label-free data analysis pipeline for optimal identification and quantitation</td>
</tr><tr id="pkg_systemPipeR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/systemPipeR.html">systemPipeR</a></td>
	<td class=" ">Thomas Girke</td>
	<td class=" ">systemPipeR: NGS workflow and report generation environment</td>
</tr><tr id="pkg_TargetScore" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TargetScore.html">TargetScore</a></td>
	<td class=" ">Yue Li</td>
	<td class=" ">TargetScore: Infer microRNA targets using microRNA-overexpression data and sequence information</td>
</tr><tr id="pkg_TargetSearch" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TargetSearch.html">TargetSearch</a></td>
	<td class=" ">Alvaro Cuadros-Inostroza</td>
	<td class=" ">A package for the analysis of GC-MS metabolite profiling data.</td>
</tr><tr id="pkg_TCC" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TCC.html">TCC</a></td>
	<td class=" ">Jianqiang Sun, Tomoaki Nishiyama</td>
	<td class=" ">TCC: Differential expression analysis for tag count data with robust normalization strategies</td>
</tr><tr id="pkg_TDARACNE" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TDARACNE.html">TDARACNE</a></td>
	<td class=" ">Zoppoli Pietro</td>
	<td class=" ">Network reverse engineering from time course data.</td>
</tr><tr id="pkg_TEQC" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TEQC.html">TEQC</a></td>
	<td class=" ">Manuela Hummel</td>
	<td class=" ">Quality control for target capture experiments</td>
</tr><tr id="pkg_ternarynet" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ternarynet.html">ternarynet</a></td>
	<td class=" ">Matthew N. McCall</td>
	<td class=" ">Ternary Network Estimation</td>
</tr><tr id="pkg_TFBSTools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TFBSTools.html">TFBSTools</a></td>
	<td class=" ">Ge Tan</td>
	<td class=" ">Software package for transcription factor binding site (TFBS) analysis</td>
</tr><tr id="pkg_tigre" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/tigre.html">tigre</a></td>
	<td class=" ">Antti Honkela</td>
	<td class=" ">Transcription factor Inference through Gaussian process Reconstruction of Expression</td>
</tr><tr id="pkg_tilingArray" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/tilingArray.html">tilingArray</a></td>
	<td class=" ">Zhenyu Xu</td>
	<td class=" ">Transcript mapping with high-density oligonucleotide tiling arrays</td>
</tr><tr id="pkg_timecourse" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/timecourse.html">timecourse</a></td>
	<td class=" ">Yu Chuan Tai</td>
	<td class=" ">Statistical Analysis for Developmental Microarray Time Course Data</td>
</tr><tr id="pkg_TIN" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TIN.html">TIN</a></td>
	<td class=" ">Bjarne Johannessen</td>
	<td class=" ">Transcriptome instability analysis</td>
</tr><tr id="pkg_TitanCNA" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TitanCNA.html">TitanCNA</a></td>
	<td class=" ">Gavin Ha, Sohrab P Shah</td>
	<td class=" ">Subclonal copy number and LOH prediction from whole genome sequencing of tumours</td>
</tr><tr id="pkg_tkWidgets" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/tkWidgets.html">tkWidgets</a></td>
	<td class=" ">J. Zhang</td>
	<td class=" ">R based tk widgets</td>
</tr><tr id="pkg_ToPASeq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/ToPASeq.html">ToPASeq</a></td>
	<td class=" ">Ivana Ihnatova</td>
	<td class=" ">Package for Topology-based Pathway Analysis of RNASeq data</td>
</tr><tr id="pkg_topGO" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/topGO.html">topGO</a></td>
	<td class=" ">Adrian Alexa</td>
	<td class=" ">topGO: Enrichment analysis for Gene Ontology</td>
</tr><tr id="pkg_TPP" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TPP.html">TPP</a></td>
	<td class=" ">Dorothee Childs</td>
	<td class=" ">Analyze thermal proteome profiling (TPP) experiments</td>
</tr><tr id="pkg_tracktables" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/tracktables.html">tracktables</a></td>
	<td class=" ">Tom Carroll</td>
	<td class=" ">Build IGV tracks and HTML reports</td>
</tr><tr id="pkg_trackViewer" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/trackViewer.html">trackViewer</a></td>
	<td class=" ">Jianhong Ou</td>
	<td class=" ">A bioconductor package with minimalist design for plotting elegant track layers</td>
</tr><tr id="pkg_tRanslatome" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/tRanslatome.html">tRanslatome</a></td>
	<td class=" ">Toma Tebaldi, Erik Dassi</td>
	<td class=" ">Comparison between multiple levels of gene expression</td>
</tr><tr id="pkg_TransView" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TransView.html">TransView</a></td>
	<td class=" ">Julius Muller</td>
	<td class=" ">Read density map construction and accession. Visualization of ChIPSeq and RNASeq data sets.</td>
</tr><tr id="pkg_triform" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/triform.html">triform</a></td>
	<td class=" ">Tony Handstad Developer</td>
	<td class=" ">Triform finds enriched regions (peaks) in transcription factor ChIP-sequencing data</td>
</tr><tr id="pkg_trigger" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/trigger.html">trigger</a></td>
	<td class=" ">John D. Storey</td>
	<td class=" ">Transcriptional Regulatory Inference from Genetics of Gene ExpRession</td>
</tr><tr id="pkg_trio" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/trio.html">trio</a></td>
	<td class=" ">Holger Schwender</td>
	<td class=" ">Testing of SNPs and SNP Interactions in Case-Parent Trio Studies</td>
</tr><tr id="pkg_triplex" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/triplex.html">triplex</a></td>
	<td class=" ">Jiri Hon</td>
	<td class=" ">Search and visualize intramolecular triplex-forming sequences in DNA</td>
</tr><tr id="pkg_TRONCO" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TRONCO.html">TRONCO</a></td>
	<td class=" ">Giulio Caravagna, Alex Graudenzi, Daniele Ramazzotti</td>
	<td class=" ">TRONCO, a package for TRanslational ONCOlogy</td>
</tr><tr id="pkg_TSCAN" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TSCAN.html">TSCAN</a></td>
	<td class=" ">Zhicheng Ji</td>
	<td class=" ">TSCAN: Tools for Single-Cell ANalysis</td>
</tr><tr id="pkg_tspair" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/tspair.html">tspair</a></td>
	<td class=" ">Jeffrey T. Leek</td>
	<td class=" ">Top Scoring Pairs for Microarray Classification</td>
</tr><tr id="pkg_TSSi" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TSSi.html">TSSi</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Transcription Start Site Identification</td>
</tr><tr id="pkg_TurboNorm" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TurboNorm.html">TurboNorm</a></td>
	<td class=" ">Maarten van Iterson</td>
	<td class=" ">A fast scatterplot smoother suitable for microarray normalization</td>
</tr><tr id="pkg_tweeDEseq" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/tweeDEseq.html">tweeDEseq</a></td>
	<td class=" ">Juan R Gonzalez</td>
	<td class=" ">RNA-seq data analysis using the Poisson-Tweedie family of distributions</td>
</tr><tr id="pkg_twilight" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/twilight.html">twilight</a></td>
	<td class=" ">Stefanie Scheid</td>
	<td class=" ">Estimation of local false discovery rate</td>
</tr><tr id="pkg_TypeInfo" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/TypeInfo.html">TypeInfo</a></td>
	<td class=" ">Duncan Temple Lang</td>
	<td class=" ">Optional Type Specification Prototype</td>
</tr><tr id="pkg_UNDO" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/UNDO.html">UNDO</a></td>
	<td class=" ">Niya Wang</td>
	<td class=" ">Unsupervised Deconvolution of Tumor-Stromal Mixed Expressions</td>
</tr><tr id="pkg_unifiedWMWqPCR" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/unifiedWMWqPCR.html">unifiedWMWqPCR</a></td>
	<td class=" ">Joris Meys</td>
	<td class=" ">Unified Wilcoxon-Mann Whitney Test for testing differential expression in qPCR data</td>
</tr><tr id="pkg_UniProt.ws" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/UniProt.ws.html">UniProt.ws</a></td>
	<td class=" ">Marc Carlson</td>
	<td class=" ">R Interface to UniProt Web Services</td>
</tr><tr id="pkg_VanillaICE" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/VanillaICE.html">VanillaICE</a></td>
	<td class=" ">Robert Scharpf</td>
	<td class=" ">A Hidden Markov Model for high throughput genotyping arrays</td>
</tr><tr id="pkg_VariantAnnotation" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html">VariantAnnotation</a></td>
	<td class=" ">Valerie Obenchain</td>
	<td class=" ">Annotation of Genetic Variants</td>
</tr><tr id="pkg_VariantFiltering" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/VariantFiltering.html">VariantFiltering</a></td>
	<td class=" ">Robert Castelo</td>
	<td class=" ">Filtering of coding and non-coding genetic variants</td>
</tr><tr id="pkg_VariantTools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/VariantTools.html">VariantTools</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">Tools for Working with Genetic Variants</td>
</tr><tr id="pkg_vbmp" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/vbmp.html">vbmp</a></td>
	<td class=" ">Nicola Lama</td>
	<td class=" ">Variational Bayesian Multinomial Probit Regression</td>
</tr><tr id="pkg_Vega" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/Vega.html">Vega</a></td>
	<td class=" ">Sandro Morganella</td>
	<td class=" ">An R package for copy number data segmentation</td>
</tr><tr id="pkg_VegaMC" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/VegaMC.html">VegaMC</a></td>
	<td class=" ">Sandro Morganella</td>
	<td class=" ">VegaMC: A Package Implementing a Variational Piecewise Smooth Model for Identification of Driver Chromosomal Imbalances in Cancer</td>
</tr><tr id="pkg_viper" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/viper.html">viper</a></td>
	<td class=" ">Mariano J Alvarez</td>
	<td class=" ">Virtual Inference of Protein-activity by Enriched Regulon analysis</td>
</tr><tr id="pkg_vsn" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/vsn.html">vsn</a></td>
	<td class=" ">Wolfgang Huber</td>
	<td class=" ">Variance stabilization and calibration for microarray data</td>
</tr><tr id="pkg_vtpnet" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/vtpnet.html">vtpnet</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">variant-transcription factor-phenotype networks</td>
</tr><tr id="pkg_wateRmelon" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/wateRmelon.html">wateRmelon</a></td>
	<td class=" ">Leo</td>
	<td class=" ">Illumina 450 methylation array normalization and metrics</td>
</tr><tr id="pkg_wavClusteR" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/wavClusteR.html">wavClusteR</a></td>
	<td class=" ">Federico Comoglio</td>
	<td class=" ">Sensitive and highly resolved identification of RNA-protein interaction sites in PAR-CLIP data</td>
</tr><tr id="pkg_waveTiling" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/waveTiling.html">waveTiling</a></td>
	<td class=" ">Kristof De Beuf</td>
	<td class=" ">Wavelet-Based Models for Tiling Array Transcriptome Analysis</td>
</tr><tr id="pkg_weaver" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/weaver.html">weaver</a></td>
	<td class=" ">Seth Falcon</td>
	<td class=" ">Tools and extensions for processing Sweave documents</td>
</tr><tr id="pkg_webbioc" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/webbioc.html">webbioc</a></td>
	<td class=" ">Colin A. Smith</td>
	<td class=" ">Bioconductor Web Interface</td>
</tr><tr id="pkg_widgetTools" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/widgetTools.html">widgetTools</a></td>
	<td class=" ">Jianhua Zhang</td>
	<td class=" ">Creates an interactive tcltk widget</td>
</tr><tr id="pkg_xcms" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/xcms.html">xcms</a></td>
	<td class=" ">Steffen Neumann</td>
	<td class=" ">LC/MS and GC/MS Data Analysis</td>
</tr><tr id="pkg_XDE" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/XDE.html">XDE</a></td>
	<td class=" ">Robert Scharpf</td>
	<td class=" ">XDE: a Bayesian hierarchical model for cross-study analysis of differential gene expression</td>
</tr><tr id="pkg_xmapbridge" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/xmapbridge.html">xmapbridge</a></td>
	<td class=" ">Chris Wirth</td>
	<td class=" ">Export plotting files to the xmapBridge for visualisation in X:Map</td>
</tr><tr id="pkg_xps" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/xps.html">xps</a></td>
	<td class=" ">Christian Stratowa</td>
	<td class=" ">Processing and Analysis of Affymetrix Oligonucleotide Arrays including Exon Arrays, Whole Genome Arrays and Plate Arrays</td>
</tr><tr id="pkg_XVector" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/XVector.html">XVector</a></td>
	<td class=" ">H. Pages</td>
	<td class=" ">Representation and manpulation of external sequences</td>
</tr><tr id="pkg_yaqcaffy" class="row_even">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/yaqcaffy.html">yaqcaffy</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">Affymetrix expression data quality control and reproducibility analysis</td>
</tr><tr id="pkg_zlibbioc" class="row_odd">
	<td class="  sorting_1"><a href="http://bioconductor.org/packages/release/bioc/html/zlibbioc.html">zlibbioc</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">An R packaged zlib-1.2.5</td>
</tr></tbody></table></div></div><div class="dataTables_info" id="biocViews_package_table_info">Showing 1 to 1,027 of 1,027 entries</div><div class="dataTables_paginate paging_two_button" id="biocViews_package_table_paginate"><a class="paginate_disabled_previous" tabindex="0" role="button" id="biocViews_package_table_previous" aria-controls="biocViews_package_table">Previous</a><a class="paginate_disabled_next" tabindex="0" role="button" id="biocViews_package_table_next" aria-controls="biocViews_package_table">Next</a></div></div>
</div>
        <div id="below_tree" class="biocViewsBelowTree"></div>
    </div>
</div>





        
      </div>
      <div id="SiteGlobalFooter" class="SiteGlobalFooter">

  <span class="global_footer_logo_bc noprint"><img src="/images/logo_global_footer_bioconductor.gif" height="51" width="154" alt=""></span>
  <span class="global_footer_logo_fhcrc noprint"><a href="http://www.fredhutch.org"><img src="/images/logo_global_footer_fhcrc.gif" height="51" width="154" alt="Fred Hutchinson Cancer Research Center"></a></span>

  <div class="attribution_copyright">
    <p>Contact us: <a href="https://support.bioconductor.org/">support.bioconductor.org</a></p>
    <p>Hosting provided by <a href="http://www.fredhutch.org">Fred Hutchinson Cancer Research Center</a></p>
    <p>Copyright © 2003 - 2015, Bioconductor</p>
  </div>

  <a name="site-map"></a> <!-- accessibility anchor -->
  <ul class="footer_nav_list noprint">
    <li class="footer_nav_list_element footer_nav_list_element_0">
      <b><a href="/index.html">Home</a></b>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_1">
      <ul>
        <li><b><a href="/install/index.html">Install</a></b></li>
        <li><a href="/install/index.html#install-R">Install R</a></li>
        <li><a href="/install/index.html#find-bioconductor-packages">Find Bioconductor Packages</a></li>
        <li><a href="/install/index.html#install-bioconductor-packages">Install Bioconductor Packages</a></li>
        <li><a href="/install/index.html#update-bioconductor-packages">Update Bioconductor Packages</a></li>
      </ul>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_2">
      <ul>
        <li><b><a href="/help/index.html">Help</a></b></li>     
        <li><a href="/help/workflows/">Workflows</a></li>
        <li><a href="/help/package-vignettes/">Package Vignettes</a></li>
        <li><a href="/help/faq/">FAQ</a></li>
        <li><a href="/help/support/">Support</a></li>
        <li><a href="http://cran.r-project.org/">Using R</a></li>
        <li><a href="/help/course-materials/">Courses</a></li>
        <li><a href="/help/publications/">Publications</a></li>
        <li><a href="/help/bioconductor-cloud-ami/">Cloud AMI</a></li>
        <li><a href="/help/community/">Community Resources</a></li>
        
      </ul>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_3">
      <ul>
        <li><b><a href="/developers/index.html">Developers</a></b></li>
    <li><a href="/developers/mentored-projects/">Mentored Projects</a></li>
		<li><a href="/developers/package-guidelines/">Package Guidelines</a></li>
		<li><a href="/developers/package-submission/">Package Submission</a></li>
		<li><a href="/developers/release-schedule/">Release Schedule</a></li>
		<li><a href="/developers/source-control/">Source Control</a></li>
      </ul>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_4">
      <ul>
        <li><b><a href="/about/index.html">About</a></b></li>     
        <li><a href="/about/advisory-board/">Advisory Board</a></li>
        <li><a href="/about/annual-reports/">Annual Reports</a></li>
        <li><a href="/about/core-team/">Core Team</a></li>
        <li><a href="/about/mirrors/">Mirrors</a></li>
        <li><a href="/about/related-projects/">Related Projects</a></li>
      </ul>
    </li>
  </ul>
  <br style="clear:both">    
  &nbsp;   

</div>
  

<div id="SiteMasthead" class="SiteMasthead">

  <a name="site-navigation"></a> <!-- accessibility anchor -->

  <span class="logo_vanity_bar noprint"></span>

  <a href="/">
	<img src="/images/logo_bioconductor.gif" border="0" class="masthead_logo" height="78" width="260" alt="Bioconductor - open source software for bioinformatics">
  </a>

  <div id="SiteMastheadRight" class="SiteMastheadRight">
    <div id="SiteMastheadRightBackground" class="SiteMastheadRightBackground">

      <a name="site-search"></a> <!-- accessibility anchor -->
      <form class="site_search" id="search_form" method="GET" action="/help/search/index.html"> 
        Search: <input id="q" name="q">
      </form>
      
      <ul class="masthead_nav noprint">
        <li class="masthead_nav_element" id="masthead_nav_element_1">
          <a href="/">Home</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_2">
          <a href="/install/">Install</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_3">
          <a href="/help/">Help</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_4">
          <a href="/developers/">Developers</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_5">
          <a href="/about/">About</a>
        </li>
      </ul>

    </div>
  </div>

</div>

    </div>
  

<div id="jstree-marker" style="display: none;">»</div><div id="jstree-marker-line" style="display: none;"></div><div id="vakata-contextmenu"></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all" id="ui-id-1" role="null" tabindex="0" style="display: none;"></ul></body></html>